From 49f4f5d0b30af666c11b4458f8309262ec7b0cfb Mon Sep 17 00:00:00 2001
From: Rafael Aquini <aquini@redhat.com>
Date: Fri, 30 Sep 2016 21:25:16 -0400
Subject: [redhat] kernel-3.10.0-512.el7

Signed-off-by: Rafael Aquini <aquini@redhat.com>

diff --git a/Makefile b/Makefile
index 44e1bd9..b1f079d 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 3
-RHEL_RELEASE = 511
+RHEL_RELEASE = 512
 
 #
 # DRM backport version
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 0ae11af..b3ff89b 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=511
+BUILD:=512
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index d178597..07b3266 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1544,6 +1544,17 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Fri Sep 30 2016 Rafael Aquini <aquini@redhat.com> [3.10.0-512.el7]
+- [fs] revert "ext4: pre-zero allocated blocks for DAX IO" (Eric Sandeen) [1380571]
+- [fs] nfsd: fix corruption in notifier registration ("J. Bruce Fields") [1378363]
+- [fs] xfs: log recovery tracepoints to track current lsn and buffer submission (Brian Foster) [1362730]
+- [fs] xfs: update metadata LSN in buffers during log recovery (Brian Foster) [1362730]
+- [fs] xfs: don't warn on buffers not being recovered due to LSN (Brian Foster) [1362730]
+- [fs] xfs: pass current lsn to log recovery buffer validation (Brian Foster) [1362730]
+- [fs] xfs: rework log recovery to submit buffers on LSN boundaries (Brian Foster) [1362730]
+- [x86] perf/uncore: Disable uncore on kdump kernel (Jiri Olsa) [1379569]
+- [netdrv] mlx4_core: Fix to clean devlink resources (Kamal Heib) [1379504]
+
 * Wed Sep 28 2016 Rafael Aquini <aquini@redhat.com> [3.10.0-511.el7]
 - [net] add recursion limit to GRO (Sabrina Dubroca) [1374191] {CVE-2016-7039}
 - [mm] cgroup: fix hugetlb_cgroup_read() (Jerome Marchand) [1378236]
-- 
1.7.1

