From e8e51cbbba2488d33b090bbd0c761e64972035d7 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 9 Dec 2014 14:22:14 -0500
Subject: [redhat] kernel-3.10.0-216.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 93f93b7..56b19a7 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 215
+RHEL_RELEASE = 216
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 23ae774..d95dd22 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=215
+BUILD:=216
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 1232439..5fa4b78 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1485,6 +1485,73 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Dec 09 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-216.el7]
+- [infiniband] iser: Adjust data_length to include protection information (Amir Vadai) [1107622]
+- [infiniband] iser: Bump version to 1.4.1 (Amir Vadai) [1107622]
+- [infiniband] iser: Allow bind only when connection state is UP (Amir Vadai) [1107622]
+- [infiniband] iser: Fix RX/TX CQ resource leak on error flow (Amir Vadai) [1107622]
+- [infiniband] iser: Clarify a duplicate counters check (Amir Vadai) [1107622]
+- [infiniband] iser: Replace connection waitqueue with completion object (Amir Vadai) [1107622]
+- [infiniband] iser: Protect iser state machine with a mutex (Amir Vadai) [1107622]
+- [infiniband] iser: Remove redundant return code in iser_free_ib_conn_res() (Amir Vadai) [1107622]
+- [infiniband] iser: Seperate iser_conn and iscsi_endpoint storage space (Amir Vadai) [1107622]
+- [infiniband] iser: Fix responder resources advertisement (Amir Vadai) [1107622]
+- [infiniband] iser: Add TIMEWAIT_EXIT event handling (Amir Vadai) [1107622]
+- [infiniband] iser: Support IPv6 address family (Amir Vadai) [1107622]
+- [infiniband] iser: Bump version to 1.4 (Amir Vadai) [1107622]
+- [infiniband] iser: Add missing newlines to logging messages (Amir Vadai) [1107622]
+- [infiniband] iser: Fix a possible race in iser connection states transition (Amir Vadai) [1107622]
+- [infiniband] iser: Simplify connection management (Amir Vadai) [1107622]
+- [infiniband] iser: Bump driver version to 1.3 (Amir Vadai) [1107622]
+- [infiniband] iser: Update Mellanox copyright note (Amir Vadai) [1107622]
+- [infiniband] iser: Print QP information once connection is established (Amir Vadai) [1107622]
+- [infiniband] iser: Remove struct iscsi_iser_conn (Amir Vadai) [1107622]
+- [infiniband] iser: Drain the tx cq once before looping on the rx cq (Amir Vadai) [1107622]
+- [infiniband] iser: Fix sector_t format warning (Amir Vadai) [1107622]
+- [infiniband] iser: Publish T10-PI support to SCSI midlayer (Amir Vadai) [1107622]
+- [infiniband] iser: Implement check_protection (Amir Vadai) [1107622]
+- [infiniband] iser: Support T10-PI operations (Amir Vadai) [1107622]
+- [infiniband] iser: Initialize T10-PI resources (Amir Vadai) [1107622]
+- [infiniband] iser: Introduce pi_enable, pi_guard module parameters (Amir Vadai) [1107622]
+- [infiniband] iser: Generalize fall_to_bounce_buf routine (Amir Vadai) [1107622]
+- [infiniband] iser: Generalize iser_unmap_task_data and finalize_rdma_unaligned_sg (Amir Vadai) [1107622]
+- [infiniband] iser: Replace fastreg descriptor valid bool with indicators container (Amir Vadai) [1107622]
+- [infiniband] iser: Keep IB device attributes under iser_device (Amir Vadai) [1107622]
+- [infiniband] iser: Move fast_reg_descriptor initialization to a function (Amir Vadai) [1107622]
+- [infiniband] iser: Push the decision what memory key to use into fast_reg_mr routine (Amir Vadai) [1107622]
+- [infiniband] iser: Avoid FRWR notation, use fastreg instead (Amir Vadai) [1107622]
+- [infiniband] iser: Suppress completions for fast registration work requests (Amir Vadai) [1107622]
+- [infiniband] iser: Fix use after free in iser_snd_completion() (Amir Vadai) [1107622]
+- [scsi] libiscsi: Add check_protection callback for transports (Amir Vadai) [1107622]
+- [mm] mem-hotplug: reset node present pages when hot-adding a new pgdat (Motohiro Kosaki) [1156396]
+- [mm] mem-hotplug: reset node managed pages when hot-adding a new pgdat (Motohiro Kosaki) [1156396]
+- [mm] make __free_pages_bootmem() only available at boot time (Motohiro Kosaki) [1156396]
+- [mm] use a dedicated lock to protect totalram_pages and zone->managed_pages (Motohiro Kosaki) [1156396]
+- [mm] accurately calculate zone->managed_pages for highmem zones (Motohiro Kosaki) [1156396]
+- [md] dm-cache: fix spurious cell_defer when dealing with partial block at end of device (Mike Snitzer) [1165050]
+- [md] dm-cache: dirty flag was mistakenly being cleared when promoting via overwrite (Mike Snitzer) [1165050]
+- [md] dm-cache: only use overwrite optimisation for promotion when in writeback mode (Mike Snitzer) [1165050]
+- [md] dm-cache: discard block size must be a multiple of cache block size (Mike Snitzer) [1165050]
+- [md] dm-cache: fix a harmless race when working out if a block is discarded (Mike Snitzer) [1165050]
+- [md] dm-cache: when reloading a discard bitset allow for a different discard block size (Mike Snitzer) [1165050]
+- [md] dm-cache: fix some issues with the new discard range support (Mike Snitzer) [1165050]
+- [md] dm-array: if resizing the array is a noop set the new root to the old one (Mike Snitzer) [1165050]
+- [md] dm-bufio: fix memleak when using a dm_buffer's inline bio (Mike Snitzer) [1165050]
+- [md] dm: use rcu_dereference_protected instead of rcu_dereference (Mike Snitzer) [1165246]
+- [md] dm-thin: suspend/resume active thin devices when reloading thin-pool (Mike Snitzer) [1165246]
+- [md] dm-thin: do not allow thin device activation while pool is suspended (Mike Snitzer) [1165246]
+- [md] dm-thin: fix a race in thin_dtr (Mike Snitzer) [1165246]
+- [md] dm-thin: remove stale 'trim' message in block comment above pool_message (Mike Snitzer) [1165246]
+- [md] dm: update wait_on_bit calls for RHEL (Mike Snitzer) [1165246]
+- [md] dm: enhance internal suspend and resume interface (Mike Snitzer) [1165246]
+- [md] dm: add presuspend_undo hook to target_type (Mike Snitzer) [1165246]
+- [md] dm: return earlier from dm_blk_ioctl if target doesn't implement .ioctl (Mike Snitzer) [1165246]
+- [md] dm: do not call dm_sync_table() when creating new devices (Mike Snitzer) [1165246]
+- [md] dm: sparse - Annotate field with __rcu for checking (Mike Snitzer) [1165246]
+- [md] dm: Use rcu_dereference() for accessing rcu pointer (Mike Snitzer) [1165246]
+- [md] dm: allow active and inactive tables to share dm_devs (Mike Snitzer) [1165246]
+- [md] dm-mpath: stop queueing IO when no valid paths exist (Mike Snitzer) [1165246]
+
 * Mon Dec 08 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-215.el7]
 - [net] vxlan: Do not reuse sockets for a different address family (Marcelo Leitner) [1146107]
 - [net] vti: Fix kernel panic due to tunnel not being removed on link deletion (Panu Matilainen) [1167725]
-- 
1.7.1