From 867e97a3acef204567901b642965f11bb5644952 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Mon, 10 Nov 2014 16:38:32 -0500
Subject: [redhat] kernel-3.10.0-201.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index ac1417f..2bd64d0 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 200
+RHEL_RELEASE = 201
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index ee234ce..35aa03a 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=200
+BUILD:=201
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 273bba8..6122255 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,39 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Mon Nov 10 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-201.el7]
+- [scsi] ipr: wait for aborted command responses (Gustavo Duarte) [1156530]
+- [ethernet] mlx4: Protect port type setting by mutex (Amir Vadai) [1095345]
+- [acpi] pm: Only set power states of devices that are power manageable (Amos Kong) [1142683]
+- [x86] setup: Mark Intel Haswell ULT as supported (Prarit Bhargava) [1159006]
+- [kernel] sched: Fix unreleased llc_shared_mask bit during CPU hotplug (Takahiro MUNEDA) [1116294]
+- [mm] do not walk all of system memory during show_mem (Johannes Weiner) [1125433]
+- [mm] remove noisy remainder of the scan_unevictable interface (Johannes Weiner) [1111215]
+- [pci] Rename sysfs 'enabled' file back to 'enable' (Myron Stowe) [1159655]
+- [kernel] sched/fair: Care divide error in update_task_scan_period() (Motohiro Kosaki) [1140979]
+- [powerpc] numa: ensure per-cpu NUMA mappings are correct on topology update (Gustavo Duarte) [1150097]
+- [powerpc] numa: use cached value of update->cpu in update_cpu_topology (Gustavo Duarte) [1150097]
+- [powerpc] numa: Add ability to disable and debug topology updates (Gustavo Duarte) [1150097]
+- [powerpc] numa: check error return from proc_create (Gustavo Duarte) [1150097]
+- [powerpc] some changes in numa_setup_cpu() (Gustavo Duarte) [1150097]
+- [powerpc] Only set numa node information for present cpus at boottime (Gustavo Duarte) [1150097]
+- [powerpc] Fix warning reported by verify_cpu_node_mapping() (Gustavo Duarte) [1150097]
+- [powerpc] reorder per-cpu NUMA information's initialization (Gustavo Duarte) [1150097]
+- [powerpc] pseries: Make CPU hotplug path endian safe (Steve Best) [1159579]
+- [powerpc] pseries: Fix endian issues in cpu hot-removal (Steve Best) [1159579]
+- [powerpc] pseries: Fix endian issues in onlining cpu threads (Steve Best) [1159579]
+- [x86] smpboot: Fix up typo in topology detection (Prarit Bhargava) [1156655]
+- [x86] smpboot: Add new topology for multi-NUMA-node CPUs (Prarit Bhargava) [1158269]
+- [kernel] sched: Rework sched_domain topology definition (Prarit Bhargava) [1158269]
+- [usb] hub: take hub->hdev reference when processing from eventlist (Don Zickus) [1151508]
+- [usb] ehci: unlink QHs even after the controller has stopped (Don Zickus) [1151491]
+- [tools] testing/selftests/powerpc: Correct DSCR during TM context switch (Gustavo Duarte) [1134511]
+- [tools] testing/selftests: Add infrastructure for powerpc selftests (Gustavo Duarte) [1134511]
+- [scsi] ibmvscsi: Abort init sequence during error recovery (Gustavo Duarte) [1105496]
+- [scsi] ibmvscsi: Add memory barriers for send / receive (Gustavo Duarte) [1105496]
+- [x86] fpu: __restore_xstate_sig()->math_state_restore() needs preempt_disable() (Oleg Nesterov) [1121784]
+- [x86] fpu: shift drop_init_fpu() from save_xstate_sig() to handle_signal() (Oleg Nesterov) [1121784]
+
 * Fri Nov 07 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-200.el7]
 - [fs] ext4: fix wrong assert in ext4_mb_normalize_request() (Lukas Czerner) [1146046]
 - [mm] Remove false WARN_ON from pagecache_isize_extended() (Lukas Czerner) [1156096]
-- 
1.7.1