From 5bdaaf3d059c97e60505599f54552cf6dddf30fe Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Mon, 1 Dec 2014 16:21:12 -0500
Subject: [redhat] kernel-3.10.0-212.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index cbdf1d0..ca7335b 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 211
+RHEL_RELEASE = 212
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 269087e..63bd2e7 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=211
+BUILD:=212
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index f4a439a..30a23a4 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,12 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Mon Dec 01 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-212.el7]
+- [fs] fsnotify: next_i is freed during fsnotify_unmount_inodes (Eric Sandeen) [1124997]
+- [fs] btrfs: fix regression of btrfs device replace (Eric Sandeen) [1162983]
+- [fs] ext4: don't count external journal blocks as overhead (Eric Sandeen) [1164366]
+- [fs] Fix oops when creating symlinks on smb3 (Sachin Prabhu) [1161429]
+
 * Wed Nov 26 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-211.el7]
 - [net] sctp: fix memory leak in auth key management (Daniel Borkmann) [1160928]
 - [net] sctp: fix NULL pointer dereference in af->from_addr_param on malformed packet (Daniel Borkmann) [1154002] {CVE-2014-7841}
-- 
1.7.1