From a85b05117b15328b2e5f7587ebfabf651f7eefdb Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 8 Jul 2014 15:45:48 -0400
Subject: [redhat] kernel-3.10.0-133.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index d954efa..6f8af40 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 132
+RHEL_RELEASE = 133
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index fd1b7f4..ef06192 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=132
+BUILD:=133
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index c2ff1f9..42e804f 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1471,6 +1471,41 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Jul 08 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-133.el7]
+- [s390] af_iucv: recvmsg problem for SOCK_STREAM sockets (Hendrik Brueckner) [1109703]
+- [s390] af_iucv: correct cleanup if listen backlog is full (Hendrik Brueckner) [1109033]
+- [kernel] auditsc: audit_krule mask accesses need bounds checking (Denys Vlasenko) [1102710] {CVE-2014-3917}
+- [fs] nfs: Don't mark the data cache as invalid if it has been flushed (Scott Mayhew) [1114054]
+- [fs] nfs: Clear NFS_INO_REVAL_PAGECACHE when we update the file size (Scott Mayhew) [1114054]
+- [fs] nfs: Fix cache_validity check in nfs_write_pageuptodate() (Scott Mayhew) [1114054]
+- [mm] avoid kABI breakage (Jerome Marchand) [1097419]
+- [mm] swap: change swap_list_head to plist, add swap_avail_head (Jerome Marchand) [1097419]
+- [lib] plist: add plist_requeue (Jerome Marchand) [1097419]
+- [lib] plist: add helper functions (Jerome Marchand) [1097419]
+- [mm] swap: change swap_info singly-linked list to list_head (Jerome Marchand) [1097419]
+- [block] mtip32xx: mtip_async_complete() bug fixes (Jeff Moyer) [1102281]
+- [block] mtip32xx: Unmap the DMA segments before completing the IO request (Jeff Moyer) [1102281]
+- [ethernet] tg3: Change nvram command timeout value to 50ms (Ivan Vecera) [1086397]
+- [ethernet] tg3: Override clock, link aware and link idle mode during NVRAM dump (Ivan Vecera) [1086397]
+- [edac] sb_edac: mark MCE messages as KERN_DEBUG (Aristeu Rozanski) [1056785]
+- [edac] sb_edac: use "event" instead of "exception" when MC wasnt signaled (Aristeu Rozanski) [1056785]
+- [Documentation] MAINTAINERS: Add DesignWare, i.MX6, Armada, R-Car PCI host maintainers (Myron Stowe) [1110890]
+- [pci] Disable Bus Master only on kexec reboot (Myron Stowe) [1110890]
+- [pci] Omit PCI ID macro strings to shorten quirk names (Myron Stowe) [1110890]
+- [pci] Move device_del() from pci_stop_dev() to pci_destroy_dev() (Myron Stowe) [1110890]
+- [pci] Avoid unnecessary CPU switch when calling driver .probe() method (Myron Stowe) [1110890]
+- [Documentation] MAINTAINERS: Add Jingoo Han as Samsung Exynos PCIe driver maintainer (Myron Stowe) [1110890]
+- [Documentation] MAINTAINERS: Add Tegra PCIe maintainer (Myron Stowe) [1110890]
+- [ethernet] ixgbe: set driver_max_VFs should be done before enabling SRIOV (Alex Williamson) [1090154]
+- [ethernet] ixgbe: define IXGBE_MAX_VFS_DRV_LIMIT macro and cleanup const 63 (Alex Williamson) [1090154]
+- [fs] nfs: advertise only supported callback netids (Jeff Layton) [1048477]
+- [fs] nfs: clean up, revert increase in READDIR RPC buffer max size (Jeff Layton) [1048477]
+- [net] sunrpc: Fix large reads on NFS/RDMA (Jeff Layton) [1048477]
+- [net] sunrpc: remove KERN_INFO from dprintk() call sites (Jeff Layton) [1048477]
+- [kernel] ftrace: Hardcode ftrace_module_init() call into load_module() (Takahiro MUNEDA) [1061553]
+- [kernel] trace: Make register/unregister_ftrace_command __init (Takahiro MUNEDA) [1061553]
+- [tools] turbostat: run on HSX (Prarit Bhargava) [1081657]
+
 * Thu Jul 03 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-132.el7]
 - [net] neigh: set nud_state to NUD_INCOMPLETE when probing router reachability (Hannes Frederic Sowa) [1110193]
 - [net] bridge: Set vlan_features to allow offloads on vlans (Florian Westphal) [1058544]
-- 
1.7.1