From 0bd545cfc856b91f2007f249112fc04ec36b223f Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Fri, 3 Oct 2014 09:58:39 -0400
Subject: [redhat] kernel-3.10.0-178.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 5222905..ee69c69 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 177
+RHEL_RELEASE = 178
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index f20bc66..18d8baf 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=177
+BUILD:=178
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 9413f04..3478650 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1483,6 +1483,45 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Fri Oct 03 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-178.el7]
+- [crypto] drbg: fix maximum value checks on 32 bit systems (Herbert Xu) [1138311]
+- [crypto] drbg: remove configuration of fixed values (Herbert Xu) [1138311]
+- [crypto] drbg: fix failure of generating multiple of 2**16 bytes (Herbert Xu) [1011562]
+- [crypto] drbg: drbg_exit() can be static (Herbert Xu) [1011562]
+- [crypto] drbg: HMAC-SHA1 DRBG has crypto strength of 128 bits (Herbert Xu) [1011562]
+- [crypto] drbg: Mix a time stamp into DRBG state (Herbert Xu) [1011562]
+- [crypto] drbg: Select correct DRBG core for stdrng (Herbert Xu) [1011562]
+- [crypto] drbg: Call CTR DRBG DF function only once (Herbert Xu) [1011562]
+- [crypto] drbg: Fix format string for debugging statements (Herbert Xu) [1011562]
+- [crypto] drbg: cleanup of preprocessor macros (Herbert Xu) [1011562]
+- [crypto] drbg: Use Kconfig to ensure at least one RNG option is set (Herbert Xu) [1011562]
+- [crypto] drbg: use of kernel linked list (Herbert Xu) [1011562]
+- [crypto] drbg: fix memory corruption for AES192 (Herbert Xu) [1011562]
+- [crypto] drbg: simplify ordering of linked list in drbg_ctr_df (Herbert Xu) [1011562]
+- [crypto] drbg: Add DRBG test code to testmgr (Herbert Xu) [1011562]
+- [crypto] drbg: DRBG testmgr test vectors (Herbert Xu) [1011562]
+- [crypto] drbg: compile the DRBG code (Herbert Xu) [1011562]
+- [crypto] drbg: DRBG kernel configuration options (Herbert Xu) [1011562]
+- [crypto] drbg: header file for DRBG (Herbert Xu) [1011562]
+- [crypto] drbg: SP800-90A Deterministic Random Bit Generator (Herbert Xu) [1011562]
+- [virt] virtio_scsi: avoid cancelling uninitialized work items (Paolo Bonzini) [1050834]
+- [net] macvtap: Fix race between device delete and open (Vlad Yasevich) [1075904]
+- [video] offb: Little endian fixes (David Gibson) [1144937]
+- [cpufreq] release policy->rwsem on error (Prarit Bhargava) [1140184]
+- [char] ipmi: Turn off all activity on an idle ipmi interface (Tony Camuso) [1111214]
+- [char] ipmi: Turn off default probing of interfaces (Tony Camuso) [1111214]
+- [char] ipmi: Reset the KCS timeout when starting error recovery (Tony Camuso) [1111214]
+- [char] ipmi: Fix a race restarting the timer (Tony Camuso) [1111214]
+- [char] ipmi: ipmi_bt_sm, fix infinite loop (Tony Camuso) [1111214]
+- [char] ipmi: delete non-required instances of include <linux/init.h> (Tony Camuso) [1111214]
+- [char] ipmi: Cleanup error return (Tony Camuso) [1111214]
+- [char] ipmi: fix timeout calculation when bmc is disconnected (Tony Camuso) [1111214]
+- [char] ipmi: use USEC_PER_SEC instead of 1000000 for more meaningful (Tony Camuso) [1111214]
+- [char] ipmi: remove deprecated IRQF_DISABLED (Tony Camuso) [1111214]
+- [char] ipmi: Initialize locals to avoid warning (Tony Camuso) [1111214]
+- [char] ipmi: info leak in compat_ipmi_ioctl() (Tony Camuso) [1111214]
+- [char] ipmi: Convert use of typedef ctl_table to struct ctl_table (Tony Camuso) [1111214]
+
 * Tue Sep 30 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-177.el7]
 - [x86] uv: Set n_lshift based on GAM_GR_CONFIG MMR for UV3 (George Beshers) [1098325]
 - [kernel] audit: correct AUDIT_GET_FEATURE return message type (Richard Guy Briggs) [1141791]
-- 
1.7.1