From 9c343d6e40da7f3996e5d76beeeb329402627349 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Wed, 24 Sep 2014 01:49:28 -0400
Subject: [redhat] infra: make sure kabi tarball exists for dist-git

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/Makefile b/redhat/Makefile
index 5d8add6..8639344 100644
--- a/redhat/Makefile
+++ b/redhat/Makefile
@@ -88,6 +88,10 @@ rh-stub-key:
 $(TARBALL):
  @scripts/create-tarball.sh $(GITID) $(TARBALL) linux-$(KVERSION)-$(PKGRELEASE)
 
+.PHONY: $(KABI_TARBALL)
+$(KABI_TARBALL):
+ @(cd kabi && tar cjvf $(SOURCES)/kernel-abi-whitelists.tar.bz2 kabi-rhel7* kabi-current)
+
 setup-source: rh-clean-sources
  @cp $(REDHAT)/$(SPECFILE).template $(SOURCES)/$(SPECFILE)
  @$(REDHAT)/genspec.sh $(SOURCES) $(SOURCES)/$(SPECFILE) $(PKGRELEASE) $(RPMVERSION) $(RELEASED_KERNEL) $(SPECRELEASE)
@@ -168,7 +172,7 @@ $(REDHAT)/rpm/SOURCES/kernel.spec:
  @echo "rh-sources"
  @$(MAKE) rh-sources
 
-rh-dist-git: $(REDHAT)/rpm/SOURCES/kernel.spec $(TARBALL)
+rh-dist-git: $(REDHAT)/rpm/SOURCES/kernel.spec $(TARBALL) $(KABI_TARBALL)
 ifeq ("$(RHDISTGIT_BRANCH)", "")
  $(error RHDISTGIT_BRANCH unset)
 endif
-- 
1.7.1