From e1c20ac621cf489f13f70ed16b669dbbc1f6fd75 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Wed, 17 Sep 2014 15:25:05 -0400
Subject: [redhat] kernel-3.10.0-163.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index fd39d84..de948da 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 162
+RHEL_RELEASE = 163
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index b3a444c..d49547a 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=162
+BUILD:=163
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 434818e..f2aad41 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1476,6 +1476,27 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Wed Sep 17 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-163.el7]
+- [fs] gfs2: Request demote when a "try" flock fails (Robert S Peterson) [1132154]
+- [fs] gfs2: Change maxlen variables to size_t (Robert S Peterson) [1127251]
+- [fs] gfs2: Fix return value in slot_get() (Abhijith Das) [1026406]
+- [fs] gfs2: replace kmalloc - __vmalloc / memset 0 (Abhijith Das) [1026406]
+- [fs] gfs2: Fix kbuild test robot reported warning (Abhijith Das) [1026406]
+- [fs] gfs2: Move quota bitmap operations under their own lock (Abhijith Das) [1026406]
+- [fs] gfs2: Clean up quota slot allocation (Abhijith Das) [1026406]
+- [fs] gfs2: Only run logd and quota when mounted read/write (Abhijith Das) [1026406]
+- [fs] gfs2: Use RCU/hlist_bl based hash for quotas (Abhijith Das) [1026406]
+- [fs] gfs2: Remove gfs2_quota_change_host structure (Abhijith Das) [1026406]
+- [fs] gfs2: Use generic list_lru for quota (Abhijith Das) [1026406]
+- [fs] list: add a new LRU list type (Abhijith Das) [1026406]
+- [fs] gfs2: Rename quota qd_lru_lock qd_lock (Abhijith Das) [1026406]
+- [fs] gfs2: Use reflink for quota data cache (Abhijith Das) [1026406]
+- [fs] gfs2: Protect quota sync generation (Abhijith Das) [1026406]
+- [fs] gfs2: Inline qd_trylock into gfs2_quota_unlock (Abhijith Das) [1026406]
+- [fs] gfs2: Make two similar quota code fragments into a function (Abhijith Das) [1026406]
+- [fs] gfs2: Remove obsolete quota tunable (Abhijith Das) [1026406]
+- [fs] gfs2: Move gfs2_icbit_munge into quota.c (Abhijith Das) [1026406]
+
 * Wed Sep 17 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-162.el7]
 - [powerpc] powernv: Fix permissions on sysparam sysfs entries (Don Zickus) [1127366]
 - [pci] rphahp: Fix endianess issues (Don Zickus) [1127366]
-- 
1.7.1