From dd4b4a7f87f4b2ed6f8a242e2bf81db8de514fbc Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Thu, 23 Oct 2014 11:54:43 -0400
Subject: [redhat] kernel-3.10.0-193.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index b732b3f..a1dd695 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 192
+RHEL_RELEASE = 193
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index e357877..1aaf0bf 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=192
+BUILD:=193
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 636ab0f..e215f60 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,46 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Thu Oct 23 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-193.el7]
+- [net] ipv4: fix a race in update_or_create_fnhe() (Hannes Frederic Sowa) [1155155]
+- [net] ipv4: irq safe sk_dst_[re]set() and ipv4_sk_update_pmtu() fix (Hannes Frederic Sowa) [1155155]
+- [net] ipv4: fix dst race in sk_dst_get() (Hannes Frederic Sowa) [1155155]
+- [net] ipv4: fix a race in ip4_datagram_release_cb() (Hannes Frederic Sowa) [1155155]
+- [net] sctp: fix remote memory pressure from excessive queueing (Daniel Borkmann) [1152755] {CVE-2014-3688}
+- [net] sctp: fix panic on duplicate ASCONF chunks (Daniel Borkmann) [1152755] {CVE-2014-3687}
+- [net] sctp: fix skb_over_panic when receiving malformed ASCONF chunks (Daniel Borkmann) [1152755] {CVE-2014-3673}
+- [net] sctp: handle association restarts when the socket is closed (Daniel Borkmann) [1152755]
+- [net] ipv4: fix nexthop attlen check in fib_nh_match (Jiri Pirko) [1153654]
+- [net] packet: handle too big packets for PACKET_V3 (Hannes Frederic Sowa) [1152248]
+- [net] sendmsg: fix NULL pointer dereference (Hannes Frederic Sowa) [1152251]
+- [net] ipv6: remove rt6i_genid (Hannes Frederic Sowa) [1150017]
+- [net] ipv6: remove prune parameter for fib6_clean_all (Hannes Frederic Sowa) [1150017]
+- [net] ipv6: bump genid when delete/add address (Hannes Frederic Sowa) [1150017]
+- [net] split rt_genid for ipv4 and ipv6 (Hannes Frederic Sowa) [1150017]
+- [net] ipv4: use separate genid for next hop exceptions (Hannes Frederic Sowa) [1150017]
+- [net] ipv4: properly refresh rtable entries on pmtu/redirect events (Hannes Frederic Sowa) [1150017]
+- [net] bridge: Don't include NDA_VLAN for FDB entries with vid 0 (Vlad Yasevich) [1139809]
+- [net] bridge: Prevent insertion of FDB entry with disallowed vlan (Vlad Yasevich) [1139809]
+- [net] bridge: Fix double free and memory leak around br_allowed_ingress (Vlad Yasevich) [1139809]
+- [net] bridge: Fix crash with vlan filtering and tcpdump (Vlad Yasevich) [1139809]
+- [net] bridge: Fix handling stacked vlan tags (Vlad Yasevich) [1139809]
+- [net] bridge: Fix inabillity to retrieve vlan tags when tx offload is disabled (Vlad Yasevich) [1139809]
+- [net] bridge: Use RCU_INIT_POINTER(x, NULL) in br_vlan.c (Vlad Yasevich) [1139809]
+- [net] bridge: Remove unnecessary vlan_put_tag in br_handle_vlan (Vlad Yasevich) [1139809]
+- [net] bridge: use list_for_each_entry_continue_reverse (Vlad Yasevich) [896669]
+- [net] bridge: Fix incorrect judgment of promisc (Vlad Yasevich) [896669]
+- [net] bridge: fix the unbalanced promiscuous count when add_if failed (Vlad Yasevich) [896669]
+- [net] bridge: fix spelling of promiscuous (Vlad Yasevich) [896669]
+- [net] bridge: fix build (Vlad Yasevich) [896669]
+- [net] bridge: Automatically manage port promiscuous mode (Vlad Yasevich) [896669]
+- [net] bridge: Add addresses from static fdbs to non-promisc ports (Vlad Yasevich) [896669]
+- [net] bridge: Introduce BR_PROMISC flag (Vlad Yasevich) [896669]
+- [net] bridge: Add functionality to sync static fdb entries to hw (Vlad Yasevich) [896669]
+- [net] bridge: Keep track of ports capable of automatic discovery (Vlad Yasevich) [896669]
+- [net] bridge: Turn flag change macro into a function (Vlad Yasevich) [896669]
+- [net] bridge: Add a flag to control unicast packet flood (Vlad Yasevich) [896669]
+- [net] bridge: Add flag to control mac learning (Vlad Yasevich) [896669]
+
 * Thu Oct 23 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-192.el7]
 - [infiniband] ocrdma: convert kernel va to pa for mmap in user (Doug Ledford) [1081067 1088577]
 - [infiniband] ocrdma: use right macro in query ah (Doug Ledford) [1081067 1088577]
-- 
1.7.1