From 0095a032b578f0884ae768c6e7adb4abc35a0031 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Mon, 20 Oct 2014 14:23:04 -0400
Subject: [redhat] kernel-3.10.0-190.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index aaa5917..d5a853d 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 189
+RHEL_RELEASE = 190
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 8e9798f..1b604b5 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=189
+BUILD:=190
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 1c654c5..c1b1261 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,40 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Mon Oct 20 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-190.el7]
+- [scsi] mpt3sas: Use pci_enable_msix_exact() instead of pci_enable_msix() (Tomas Henzl) [1088524]
+- [scsi] mpt2sas: Use pci_enable_msix_exact() instead of pci_enable_msix() (Tomas Henzl) [1088524]
+- [scsi] mpt2sas: fix undefined reference to `__udivdi3' compilation errors (Tomas Henzl) [1088524]
+- [scsi] mpt3sas, mpt2sas: fix scsi_add_host error handling problems in _scsih_probe (Tomas Henzl) [1088524]
+- [scsi] mpt3sas: Bump mpt3sas driver version to 04.100.00.00 (Tomas Henzl) [1088524]
+- [scsi] mpt3sas: Added Reply Descriptor Post Queue (RDPQ) Array support (Tomas Henzl) [1088524]
+- [scsi] mpt3sas: Added OEM branding Strings (Tomas Henzl) [1088524]
+- [scsi] mpt3sas: Copyright in driver sources is updated for year the 2014 (Tomas Henzl) [1088524]
+- [scsi] mpt3sas: MPI2.5 Rev H (2.5.3) specifications (Tomas Henzl) [1088524]
+- [scsi] mpt3sas: Bump mpt3sas driver version to 03.100.00.00 (Tomas Henzl) [1088524]
+- [scsi] mpt3sas: Clear PFA Status on SGPIO when PFA Drive is Removed or Replaced (Tomas Henzl) [1088524]
+- [scsi] mpt3sas: MPI2.5 Rev G (2.5.2) specifications (Tomas Henzl) [1088524]
+- [scsi] mpt3sas: delay scsi_add_host call to work with scsi-mq (Tomas Henzl) [1088524]
+- [scsi] mpt3sas: Rework the MSI-X grouping code (Tomas Henzl) [1088524]
+- [scsi] mpt3sas: combine fw_event_work and its event_data (Tomas Henzl) [1088524]
+- [scsi] mpt3sas: correct scsi_(target, device) hostdata allocation (Tomas Henzl) [1088524]
+- [scsi] mpt3sas: Remove phys on topology change (Tomas Henzl) [1088524]
+- [scsi] mpt2sas: Bump mpt2sas driver version to 18.100.00.00 (Tomas Henzl) [1088524]
+- [scsi] mpt2sas: Get IOC_FACTS information using handshake protocol only after HBA card gets into READY or Operational state (Tomas Henzl) [1088524]
+- [scsi] mpt2sas: Added Reply Descriptor Post Queue (RDPQ) Array support (Tomas Henzl) [1088524]
+- [scsi] mpt2sas: Avoid type casting for direct I/O commands (Tomas Henzl) [1088524]
+- [scsi] mpt2sas: Bump mpt2sas driver version to 17.100.00.00 (Tomas Henzl) [1088524]
+- [scsi] mpt2sas: Clear PFA Status on SGPIO when PFA Drive is Removed or Replaced (Tomas Henzl) [1088524]
+- [scsi] mpt2sas: Copyright in driver sources is updated for year the 2014 (Tomas Henzl) [1088524]
+- [scsi] mpt2sas: MPI2 Rev Y (2.00.17) and Rev Z (2.00.18) specifications (Tomas Henzl) [1088524]
+- [scsi] mpt2sas: Added driver module parameter max_msix_vectors (Tomas Henzl) [1088524]
+- [scsi] mpt2sas: delay scsi_add_host call to work with scsi-mq (Tomas Henzl) [1088524]
+- [scsi] mpt2sas: Rework the MSI-X grouping code (Tomas Henzl) [1088524]
+- [scsi] mpt2sas: annotate ioc->reply_post_host_index as __iomem (Tomas Henzl) [1088524]
+- [scsi] mpt2sas: combine fw_event_work and its event_data (Tomas Henzl) [1088524]
+- [scsi] mpt2sas: correct scsi_(target, device) hostdata allocation (Tomas Henzl) [1088524]
+- [scsi] mpt2sas: Don't disable device twice at suspend (Tomas Henzl) [1088524]
+
 * Fri Oct 17 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-189.el7]
 - [scsi] lpfc: driver release 10.2.8021.1 for RHEL7 (Rob Evers) [1149846]
 - [scsi] lpfc: 10.4.8000.0: Fixed crash from page fault caused by use after rport delete (Rob Evers) [1149846]
-- 
1.7.1