From 9f68b96d398d9710177ec6abb752bc4ee3f58b68 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 3 Jun 2014 11:37:43 -0400
Subject: [redhat] kernel-3.10.0-126.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index f7a5dda..6ee1270 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 125
+RHEL_RELEASE = 126
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 91ad05f..3c56b81 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=125
+BUILD:=126
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 919df80..e069396 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1471,6 +1471,19 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Jun 03 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-126.el7]
+- [fs] aio: plug memory disclosure and fix reqs_active accounting (Mateusz Guzik) [1094605] {CVE-2014-0206}
+- [kernel] mutexes: Give more informative mutex warning in the !lock->owner case (Larry Woodman) [1087655 1087919 1087922]
+- [kernel] mutex: replace CONFIG_HAVE_ARCH_MUTEX_CPU_RELAX with simple ifdef (Larry Woodman) [1087655 1087919 1087922]
+- [kernel] locking/mutexes: Introduce cancelable MCS lock for adaptive spinning (Larry Woodman) [1087655 1087919 1087922]
+- [kernel] locking/mutexes: Modify the way optimistic spinners are queued (Larry Woodman) [1087655 1087919 1087922]
+- [kernel] locking/mutexes: Return false if task need_resched() in mutex_can_spin_on_owner() (Larry Woodman) [1087655 1087919 1087922]
+- [kernel] Restructure the MCS lock defines and locking & Move mcs_spinlock.h into kernel/locking/ (Larry Woodman) [1087655 1087919 1087922]
+- [misc] arch: Introduce smp_load_acquire(), smp_store_release() (Larry Woodman) [1087655 1087919 1087922]
+- [kernel] locking/mutex: Fix debug_mutexes (Larry Woodman) [1087655 1087919 1087922]
+- [kernel] locking/mutex: Fix debug checks (Larry Woodman) [1087655 1087919 1087922]
+- [kernel] locking/mutexes: Unlock the mutex without the wait_lock (Larry Woodman) [1087655 1087919 1087922]
+
 * Tue Jun 03 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-125.el7]
 - [scsi] st: fix corruption of the st_modedef structures in st_set_options() (Maurizio Lombardi) [1084968]
 - [net] openvswitch: fix a possible deadlock and lockdep warning (Flavio Leitner) [1094867]
-- 
1.7.1