From f88e5a9c90d97c36f4b78bd87a29b2d5a8b4eac4 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 12 Aug 2014 11:14:19 -0400
Subject: [redhat] kernel-3.10.0-144.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 8b0f57f..860fb52 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 143
+RHEL_RELEASE = 144
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index bc5caca..513efce 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=143
+BUILD:=144
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 2699f1a..4cc5888 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1471,6 +1471,78 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Aug 12 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-144.el7]
+- [net] ipv6: Fix MLD Query message check (Jiri Pirko) [1114650]
+- [net] sctp: inherit auth_capable on INIT collisions (Daniel Borkmann) [1123763] {CVE-2014-5077}
+- [net] ipv6: ipv6_find_hdr restore prev functionality (Jesper Brouer) [1074516]
+- [net] ipvs: fix AF assignment in ip_vs_conn_new() (Jesper Brouer) [1074516]
+- [virt] kvm: always exit on EOIs for interrupts listed in the IOAPIC redir table (Alex Williamson) [1126997]
+- [iommu] amd: Fix interrupt remapping for aliased devices (Alex Williamson) [1090139]
+- [pci] Add Patsburg (X79) to Intel PCH root port ACS quirk (Alex Williamson) [1081428]
+- [kernel] time/alarmtimer: Fix bug where relative alarm timers were treated as absolute (Prarit Bhargava) [1124941]
+- [acpi] processor: Fix STARTING/DYING action in acpi_cpu_soft_notify() (Prarit Bhargava) [1099594]
+- [x86] mce_intel: Add raw_lock conversion again (Prarit Bhargava) [1127257]
+- [sound] alsa/control: Don't access controls outside of protected regions (Radomir Vrbovsky) [1117331] {CVE-2014-4653}
+- [kernel] tick: Clear broadcast pending bit when switching to oneshot (Alex Williamson) [1109996]
+- [x86] efi: Do not export efi runtime map in case old map (Dave Young) [1080109]
+- [x86] efi: Quirk out SGI UV (Dave Young) [1080109]
+- [x86] efi: Split efi_enter_virtual_mode (Dave Young) [1080109]
+- [x86] efi: Make efi virtual runtime map passing more robust (Dave Young) [1080109]
+- [x86] mm/pageattr: Export page unmapping interface (Dave Young) [1080109]
+- [x86] efi: Dump the EFI page table (Dave Young) [1080109]
+- [x86] mm/ptdump: Add the functionality to dump an arbitrary pagetable (Dave Young) [1080109]
+- [x86] efi: Style neatening (Dave Young) [1080109]
+- [x86] efi: parse_efi_setup() build fix (Dave Young) [1080109]
+- [x86] ksysfs.c build fix (Dave Young) [1080109]
+- [x86] efi: Delete superfluous global variables (Dave Young) [1080109]
+- [x86] setup: Reserve setup_data ranges late after parsing memmap cmdline (Dave Young) [1080109]
+- [x86] Export x86 boot_params to sysfs (Dave Young) [1080109]
+- [x86] boot: Add xloadflags bit for EFI runtime support on kexec (Dave Young) [1080109]
+- [firmware] efi: Pass necessary EFI data for kexec via setup_data (Dave Young) [1080109]
+- [firmware] efi: Export EFI runtime memory mapping to sysfs (Dave Young) [1080109]
+- [firmware] efi: Export more EFI table variables to sysfs (Dave Young) [1080109]
+- [firmware] efi: Cleanup efi_enter_virtual_mode() function (Dave Young) [1080109]
+- [firmware] efi: Fix off-by-one bug in EFI Boot Services reservation (Dave Young) [1080109]
+- [firmware] efi: Add a wrapper function efi_map_region_fixed() (Dave Young) [1080109]
+- [firmware] efi: Remove unused variables in __map_region() (Dave Young) [1080109]
+- [firmware] efi: Check krealloc return value (Dave Young) [1080109]
+- [firmware] efi: Runtime services virtual mapping (Dave Young) [1080109]
+- [x86] mm/pageattr: Map in an arbitrary pgd (Dave Young) [1080109]
+- [x86] mm/pageattr: Add last levels of error path (Dave Young) [1080109]
+- [x86] mm/pageattr: Add a PUD error unwinding path (Dave Young) [1080109]
+- [x86] mm/pageattr: Add a PTE pagetable populating function (Dave Young) [1080109]
+- [x86] mm/pageattr: Add a PMD pagetable populating function (Dave Young) [1080109]
+- [x86] mm/pageattr: Add a PUD pagetable populating function (Dave Young) [1080109]
+- [x86] mm/pageattr: Add a PGD pagetable populating function (Dave Young) [1080109]
+- [x86] mm/pageattr: Lookup address in an arbitrary PGD (Dave Young) [1080109]
+- [firmware] efi: Convert runtime services function ptrs (Dave Young) [1080109]
+- [firmware] efi: Simplify EFI_DEBUG (Dave Young) [1080109]
+- [firmware] efi: Fix config_table_type array termination (Dave Young) [1080109]
+- [firmware] efi: make efi_lookup_mapped_addr() a common function (Dave Young) [1080109]
+- [firmware] efi: provide a generic efi_config_init() (Dave Young) [1080109]
+- [iommu] vt-d: Use list_for_each_entry_safe() for dmar_domain->devices traversa (Myron Stowe) [1127467]
+- [iommu] vt-d: Use for_each_drhd_unit() instead of list_for_each_entry() (Myron Stowe) [1127467]
+- [iommu] Change iommu driver to call io_page_fault trace event (Myron Stowe) [1127467]
+- [iommu] Add iommu_error class event to iommu trace (Myron Stowe) [1127467]
+- [iommu] No need to pass '0x' when 'pa' is used (Myron Stowe) [1127467]
+- [iommu] Change iommu driver to call unmap trace event (Myron Stowe) [1127467]
+- [iommu] Change iommu driver to call map trace event (Myron Stowe) [1127467]
+- [iommu] Change iommu driver to call detach_device_to_domain trace event (Myron Stowe) [1127467]
+- [iommu] Change iommu driver to call attach_device_to_domain trace event (Myron Stowe) [1127467]
+- [iommu] Change iommu driver to call remove_device_to_group trace event (Myron Stowe) [1127467]
+- [iommu] Change iommu driver to call add_device_to_group trace event (Myron Stowe) [1127467]
+- [iommu] Add event tracing feature to iommu (Myron Stowe) [1127467]
+- [iommu] intel: correct ICS register offset (Myron Stowe) [1127467]
+- [Documentation] MAINTAINERS: add overall IOMMU section (Myron Stowe) [1127467]
+- [iommu] amd: Fix resource leak in iommu_init_device() (Myron Stowe) [1127467]
+- [iommu] amd: Clean up unnecessary MSI/MSI-X capability find (Myron Stowe) [1127467]
+- [powerpc] Add iommu domain pointer to device archdata (Myron Stowe) [1127467]
+- [iommu] vt-d: DMAR reporting table needs at least one DRHD (Myron Stowe) [1127467]
+- [iommu] vt-d: Downgrade the warning if enabling irq remapping fails (Myron Stowe) [1127467]
+- [net] l2tp: don't fall back on UDP [get|set]sockopt (Petr  Matousek) [1119466] {CVE-2014-4943}
+- [x86] ptrace: force IRET path after a ptrace_stop() (Oleg Nesterov) [1115935] {CVE-2014-4699}
+- [s390] ptrace: correct insufficient sanitization when setting psw mask (Hendrik Brueckner) [1113673] {CVE-2014-3534}
+
 * Wed Aug 06 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-143.el7]
 - [virt] kvm/async_pf: kill the unnecessary use_mm/unuse_mm async_pf_execute() (Paolo Bonzini) [1116936]
 - [virt] kvm/async_pf: change async_pf_execute() to use get_user_pages(tsk => NULL) (Paolo Bonzini) [1116936]
-- 
1.7.1