From af66a606eada046179c473e6082cae248f397990 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Wed, 17 Dec 2014 09:38:53 -0500
Subject: [redhat] kernel-3.10.0-218.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index f54685c..4228822 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 217
+RHEL_RELEASE = 218
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index a10539a..4fea47c 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=217
+BUILD:=218
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index bf973b2..0e2527c 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1493,6 +1493,47 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Wed Dec 17 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-218.el7]
+- [scsi] cxgb4i: Don't block unload/cxgb4 unload when remote closes TCP connection (Sai Vemuri) [1169941]
+- [kernel] kthread: partial revert of 81c98869faa5 ("kthread: ensure locality of task_struct allocations") (Gustavo Duarte) [953583]
+- [mm] slub: fall back to node_to_mem_node() node if allocating on memoryless node (Gustavo Duarte) [953583]
+- [mm] topology: add support for node_to_mem_node() to determine the fallback node (Gustavo Duarte) [953583]
+- [mm] slub: search partial list on numa_mem_id(), instead of numa_node_id() (Gustavo Duarte) [953583]
+- [kernel] kthread: ensure locality of task_struct allocations (Gustavo Duarte) [953583]
+- [md] dm-thin: fix missing out-of-data-space to write mode transition if blocks are released (Mike Snitzer) [1173181]
+- [md] dm-thin: fix inability to discard blocks when in out-of-data-space mode (Mike Snitzer) [1173181]
+- [wireless] iwlwifi/mvm: update values for Smart Fifo (Stanislaw Gruszka) [1155538]
+- [wireless] iwlwifi/dvm: fix flush support for old firmware (Stanislaw Gruszka) [1155538]
+- [wireless] ath5k: fix hardware queue index assignment (Stanislaw Gruszka) [1155538]
+- [wireless] ath9k: fix BE/BK queue order (Stanislaw Gruszka) [1155538]
+- [wireless] ath9k_hw: fix hardware queue allocation (Stanislaw Gruszka) [1155538]
+- [wireless] ath9k: Fix RTC_DERIVED_CLK usage (Stanislaw Gruszka) [1155538]
+- [wireless] rt2x00: do not align payload on modern H/W (Stanislaw Gruszka) [1155538]
+- [wireless] mac80211: Fix regression that triggers a kernel BUG with CCMP (Stanislaw Gruszka) [1155538]
+- [wireless] iwlwifi: fix RFkill while calibrating (Stanislaw Gruszka) [1155538]
+- [wireless] mac80211: fix use-after-free in defragmentation (Stanislaw Gruszka) [1155538]
+- [wireless] mac80211: properly flush delayed scan work on interface removal (Stanislaw Gruszka) [1155538]
+- [wireless] mac80211: schedule the actual switch of the station before CSA count 0 (Stanislaw Gruszka) [1155538]
+- [wireless] mac80211: use secondary channel offset IE also beacons during CSA (Stanislaw Gruszka) [1155538]
+- [wireless] rt2x00: add new rt2800usb device (Stanislaw Gruszka) [1155538]
+- [wireless] iwlwifi/dvm: drop non VO frames when flushing (Stanislaw Gruszka) [1155538]
+- [wireless] iwlwifi: configure the LTR (Stanislaw Gruszka) [1155538]
+- [wireless] mac80211: fix typo in starting baserate for rts_cts_rate_idx (Stanislaw Gruszka) [1155538]
+- [wireless] rt2x00: add new rt2800usb devices (Stanislaw Gruszka) [1155538]
+- [wireless] rt2x00: support Ralink 5362 (Stanislaw Gruszka) [1155538]
+- [wireless] rt2800: correct BBP1_TX_POWER_CTRL mask (Stanislaw Gruszka) [1155538]
+- [wireless] iwlwifi: Add missing PCI IDs for the 7260 series (Stanislaw Gruszka) [1155538]
+- [wireless] iwlwifi/mvm: disable BT Co-running by default (Stanislaw Gruszka) [1155538]
+- [wireless] nl80211: clear skb cb before passing to netlink (Stanislaw Gruszka) [1155538]
+- [wireless] ath9k/htc: fix random decryption failure (Stanislaw Gruszka) [1155538]
+- [wireless] brcmfmac: handle IF event for P2P_DEVICE interface (Stanislaw Gruszka) [1155538]
+- [wireless] rtlwifi/rtl8192cu: Add new ID (Stanislaw Gruszka) [1155538]
+- [wireless] iwlwifi/mvm: set MAC_FILTER_IN_BEACON correctly for STA/P2P client (Stanislaw Gruszka) [1155538]
+- [wireless] iwlwifi/mvm: treat EAPOLs like mgmt frames wrt rate (Stanislaw Gruszka) [1155538]
+- [wireless] iwlwifi: increase DEFAULT_MAX_TX_POWER (Stanislaw Gruszka) [1155538]
+- [wireless] iwlwifi/mvm: fix endianity issues with Smart Fifo commands (Stanislaw Gruszka) [1155538]
+- [wireless] carl9170: fix sending URBs with wrong type when using full-speed (Stanislaw Gruszka) [1155538]
+
 * Fri Dec 12 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-217.el7]
 - [net] ipv6: yet another new IPV6_MTU_DISCOVER option IPV6_PMTUDISC_OMIT (Hannes Frederic Sowa) [1170116]
 - [net] ipv4: yet another new IP_MTU_DISCOVER option IP_PMTUDISC_OMIT (Hannes Frederic Sowa) [1170116]
-- 
1.7.1