From 8188d063f66b5a9203a6691a39e6c5056766fd33 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Mon, 8 Dec 2014 09:49:49 -0500
Subject: [redhat] kernel-3.10.0-215.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index a8f3d26..93f93b7 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 214
+RHEL_RELEASE = 215
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index ad09123..23ae774 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=214
+BUILD:=215
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 0b74afa..2d34581 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,30 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Mon Dec 08 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-215.el7]
+- [net] vxlan: Do not reuse sockets for a different address family (Marcelo Leitner) [1146107]
+- [net] vti: Fix kernel panic due to tunnel not being removed on link deletion (Panu Matilainen) [1167725]
+- [net] sctp: test if association is dead in sctp_wake_up_waiters (Daniel Borkmann) [1166467]
+- [net] sctp: wake up all assocs if sndbuf policy is per socket (Daniel Borkmann) [1166467]
+- [net] ip: better estimate tunnel header cut for correct ufo handling (Alexander Duyck) [1159577]
+- [net] ipv6: gso: remove redundant locking (Alexander Duyck) [1159577]
+- [net] ipv6: Do not treat a GSO_TCPV4 request from UDP tunnel over IPv6 as invalid (Alexander Duyck) [1159577]
+- [net] ipv6: fib: fix fib dump restart (Panu Matilainen) [1163605]
+- [net] ipv6: drop unused fib6_clean_all_ro() function and rt6_proc_arg struct (Panu Matilainen) [1163605]
+- [net] ipv6: avoid high order memory allocations for /proc/net/ipv6_route (Panu Matilainen) [1163605]
+- [net] ipv4: Fix incorrect error code when adding an unreachable route (Panu Matilainen) [1165552]
+- [net] sctp: replace seq_printf with seq_puts (Daniel Borkmann) [1164214]
+- [net] sctp: add transport state in /proc/net/sctp/remaddr (Daniel Borkmann) [1164214]
+- [IB] isert: Adjust CQ size to HW limits (Andy Grover) [1166314]
+- [ib_isert] Add max_send_sge=2 minimum for control PDU responses (Andy Grover) [1166314]
+- [scsi] megaraid_sas: do not process IOCTLs and SCSI commands during driver removal (Tomas Henzl) [1162645]
+- [scsi] megaraid_sas: dndinaness related bug fixes (Tomas Henzl) [1162645]
+- [scsi] megaraid_sas: corrected return of wait_event from abort frame path (Tomas Henzl) [1162645]
+- [scsi] megaraid_sas: make HBA operational after LD_MAP_SYNC DCMD in OCR path (Tomas Henzl) [1162645]
+- [scsi] megaraid_sas: online Firmware upgrade support for Extended VD feature (Tomas Henzl) [1162645]
+- [scsi] megaraid_sas: update MAINTAINERS and copyright information for megaraid drivers (Tomas Henzl) [1162645]
+- [scsi] megaraid_sas: driver version upgrade and remove some meta data of driver (06.805.06.01-rc1) (Tomas Henzl) [1162645]
+
 * Mon Dec 08 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-214.el7]
 - [powerpc] Drop useless warning in eeh_init() (Gustavo Duarte) [1156651]
 - [powerpc] pseries: Decrease message level on EEH initialization (Gustavo Duarte) [1156651]
-- 
1.7.1