From 1bc9df1501b835cf54e5932a28da9a0cf5337af0 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Wed, 24 Sep 2014 01:37:59 -0400
Subject: [redhat] kernel-3.10.0-170.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 7cf13c8..0d73106 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 169
+RHEL_RELEASE = 170
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 34022b6..45f4e44 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=169
+BUILD:=170
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index ab6c6b9..56b7d9d 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1477,6 +1477,107 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Wed Sep 24 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-170.el7]
+- [powerpc] Enable IOMMU and VFIO for ppc64 (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s_32/pr - Access HTAB in big endian (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Rework SLB switching code (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s - Move little endian conflict to HV KVM (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Do dcbz32 patching with big endian instructions (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Fill pvinfo hcall instructions in big endian (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - PAPR - Access RTAS in big endian (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - PAPR - Access HTAB in big endian (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s_64/pr - Access shadow slb in big endian (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s_64/pr - Access HTAB in big endian (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s - ifdef on CONFIG_KVM_book3s_32_HANDLER for 32bit (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Disable AIL mode with OPAL (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Use SLB entry 0 (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: Graciously fail broken LE hypercalls (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s - Remove open coded make_dsisr in alignment handler (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s - Always use the saved DAR value (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: Make NX bit available with magic page (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: Disable NX for old magic page using guests (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Expose TM registers (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Expose EBB registers (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Expose TAR facility to guest (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Handle Facility interrupt and FSCR (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Emulate TIR register (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Ignore PMU SPRs (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Default to big endian guest (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Fix C/R bit setting (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Fix WARN_ON with debug options on (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Enable Little Endian PR guest (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Enable interrupts earlier (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Make svcpu -> vcpu store preempt savvy (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s - Ignore DABR register (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: Ignore PIR writes (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm: add PR mode KVM (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm: Move more code under CONFIG_HAVE_KVM_IRQFD (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: Enable IRQFD support for the XICS interrupt controller (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/s390: adapter interrupt sources (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm: Move irq notifier implementation into eventfd.c (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm: Move all accesses to kvm::irq_routing into irqchip.c (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/irqchip: Provide and use accessors for irq routing table (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm: Don't keep reference to irq routing table in irqfd struct (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: IRQ disabling cleanup (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: use fdget (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/hv - Put huge-page HPTEs in rmap chain for base address (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s - Move KVM_REG_PPC_WORT to an unused register number (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s - Add ONE_REG register names that were missed (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/hv - Add missing code for transaction reclaim on guest exit (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/hv - make _PAGE_NUMA take effect (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc/book3s: rename lookup_linux_pte() to lookup_linux_pte_and_update() (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/hv - Fix KVM hang with CONFIG_KVM_XICS=n (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/hv - Fix incorrect userspace exit on ioeventfd write (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/hv - Remove bogus duplicate code (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/hv - use xics_wake_cpu only when defined (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: NULL return of kvmppc_mmu_hpte_cache_next should be handled (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: Add devname:kvm aliases for modules (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/hv - Don't drop low-order page address bits (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: Get rid of KVM_HPAGE defines (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: Guard doorbell exception with CONFIG_PPC_DOORBELL (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: Make shared struct aka magic page guest endian (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/hv - Use base page size when comparing against slb value (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/hv - Fix machine check delivery to guest (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/hv - Add mixed page-size support for guest (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: fix couple of memory leaks in MPIC/XICS devices (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Don't clobber our exit handler id (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/hv - Fix physical address calculations (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Take SRCU read lock around RTAS kvm_read_guest() call (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/hv - Add H_SET_MODE hcall handling (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s - Allow only implemented hcalls to be enabled or disabled (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s - Controls for in-kernel sPAPR hypercall handling (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s - Move vcore definition to end of kvm_arch struct (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Export kvmppc_copy_to (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: Add CAP to indicate hcall fixes (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/hv - Prefer CMA region for hash page table allocation (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/hv - Add software abort codes for transactional memory (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: Unify kvmppc_get_last_inst and sc (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: Load/save FP/VMX/VSX state directly to/from vcpu struct (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm/ppc: book3s/pr - Fix compilation without CONFIG_ALTIVEC (David Gibson) [1123145 1123133 1123367]
+- [virt] kvm: Add per-vm capability enablement (David Gibson) [1123145 1123133 1123367]
+- [iommu] vt-d: Exclude devices using RMRRs from IOMMU API domains (Myron Stowe) [1097907]
+- [s390] uaccess: fix possible register corruption in strnlen_user_srst() (Hendrik Brueckner) [1099147]
+- [s390] uaccess: rework uaccess code - fix locking issues (Hendrik Brueckner) [1099147]
+- [s390] uaccess: introduce 'uaccesspt' kernel parameter (Hendrik Brueckner) [1099147]
+- [s390] uaccess: remove dead kernel parameter 'user_mode=' (Hendrik Brueckner) [1099147]
+- [s390] setup: get rid of MACHINE_HAS_MVCOS machine flag (Hendrik Brueckner) [1099147]
+- [s390] uaccess: consistent types (Hendrik Brueckner) [1099147]
+- [s390] uaccess: get rid of indirect function calls (Hendrik Brueckner) [1099147]
+- [s390] uaccess: normalize order of parameters of indirect uaccess function calls (Hendrik Brueckner) [1099147]
+- [s390] uaccess: remove dead extern declarations, make functions static (Hendrik Brueckner) [1099147]
+- [s390] uaccess: test if current->mm is set before walking page tables (Hendrik Brueckner) [1099147]
+- [s390] compat: fix PSW32_USER_BITS definition (Hendrik Brueckner) [1099147]
+- [s390] compat: make psw32_user_bits a constant value again (Hendrik Brueckner) [1099147]
+- [s390] uaccess: always run the kernel in home space (Hendrik Brueckner) [1099147]
+- [s390] kernel: remove page table walk for user space accesses (Hendrik Brueckner) [1099147]
+- [powerpc] start loop at section start of start in vmemmap_populated() (Gustavo Duarte) [1090174]
+- [powerpc] implement vmemmap_free() (Gustavo Duarte) [1090174]
+- [powerpc] implement vmemmap_remove_mapping() for BOOK3S (Gustavo Duarte) [1090174]
+- [powerpc] implement vmemmap_list_free() (Gustavo Duarte) [1090174]
+- [x86] Makefile: Add stack frame check (Prarit Bhargava) [1058753]
+- [virt] kvm/x86: fix TSC matching (Marcelo Tosatti) [1115764]
+- [drm] radeon: tweak ACCEL_WORKING2 query for hawaii (Jerome Glisse) [1140789]
+
 * Tue Sep 23 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-169.el7]
 - [lib] assoc_array: Fix termination condition in assoc array garbage collection (David Howells) [1139431] {CVE-2014-3631}
 - [acpi] x86/ras: Fix build warnings in <linux/aer.h> (Rui Wang) [1085141]
-- 
1.7.1