From 1d28e54b393d0a69883810ff6ffc29cdaad73d91 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 23 Sep 2014 18:15:07 -0400
Subject: [redhat] infra: properly export kabi bits to dist-git

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/.gitignore b/redhat/.gitignore
index ee4f316..15e6464 100644
--- a/redhat/.gitignore
+++ b/redhat/.gitignore
@@ -1,3 +1,10 @@
 *.bz2
 configs/kernel*.config
 configs/*-merged
+
+kabi/Module.kabi_ppc64
+kabi/Module.kabi_s390x
+kabi/Module.kabi_x86_64
+kabi/kabi-current
+kabi/kabi-rhel7?
+kabi/kabi-rhel7?/*
diff --git a/redhat/Makefile b/redhat/Makefile
index 5596d40..4e4ee91 100644
--- a/redhat/Makefile
+++ b/redhat/Makefile
@@ -172,7 +172,7 @@ rh-dist-git: $(REDHAT)/rpm/SOURCES/kernel.spec $(TARBALL)
 ifeq ("$(RHDISTGIT_BRANCH)", "")
  $(error RHDISTGIT_BRANCH unset)
 endif
- $(REDHAT)/scripts/rh-dist-git.sh "$(RHDISTGIT_BRANCH)" "$(RHDISTGIT_CACHE)" "$(RHDISTGIT_TMP)" "$(RHDISTGIT)" "$(TARBALL)"
+ $(REDHAT)/scripts/rh-dist-git.sh "$(RHDISTGIT_BRANCH)" "$(RHDISTGIT_CACHE)" "$(RHDISTGIT_TMP)" "$(RHDISTGIT)" "$(TARBALL)" "$(KABI_TARBALL)"
 
 rh-rtg: rh-release
  @$(MAKE) rh-release-tag
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index f32ceeb..e664533 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -34,3 +34,5 @@ PKGRELEASE:=$(PREBUILD)$(BUILD)$(DIST)$(BUILDID)
 
 TARFILE:=linux-$(KVERSION)-$(PKGRELEASE).tar.xz
 TARBALL:=$(REDHAT)/$(TARFILE)
+KABI_TARFILE:=kernel-abi-whitelists.tar.bz2
+KABI_TARBALL:=$(REDHAT)/rpm/SOURCES/$(KABI_TARFILE)
diff --git a/redhat/git/files b/redhat/git/files
index 5f1f16b..53000f2 100644
--- a/redhat/git/files
+++ b/redhat/git/files
@@ -16,9 +16,6 @@ sign-modules
 Module.kabi_ppc64
 Module.kabi_s390x
 Module.kabi_x86_64
-kabi_whitelist_ppc64
-kabi_whitelist_s390x
-kabi_whitelist_x86_64
 kernel-3.10.0-ppc64.config
 kernel-3.10.0-ppc64-debug.config
 kernel-3.10.0-ppc64le.config
diff --git a/redhat/scripts/rh-dist-git.sh b/redhat/scripts/rh-dist-git.sh
index 5a334fa..9c6f8cd 100755
--- a/redhat/scripts/rh-dist-git.sh
+++ b/redhat/scripts/rh-dist-git.sh
@@ -5,12 +5,15 @@
 # $2: local pristine clone of dist-git
 # $3: alternate tmp directory (if you have faster storage)
 # $4: alternate dist-git server
+# $5: kernel source tarball
+# $6: kabi whitelists tarball
 
 rhdistgit_branch=$1;
 rhdistgit_cache=$2;
 rhdistgit_tmp=$3;
 rhdistgit_server=$4;
 rhdistgit_tarball=$5;
+rhdistgit_kabi_tarball=$6;
 
 redhat=$(dirname $0)/..;
 topdir=$redhat/..;
@@ -39,11 +42,14 @@ echo "Copying updated files"
 # copy the required files (redhat/git/files)
 $redhat/scripts/copy_files.sh "$topdir" "$tmpdir"
 
-echo "Uploading new tarball"
-# upload tarball
+echo "Uploading new tarballs"
+# upload tarballs
 sed -i "/linux-3.*.el7.tar.xz/d" $tmpdir/kernel/sources;
+sed -i "/kernel-abi-whitelists.tar.bz2/d" $tmpdir/kernel/sources;
 sed -i "/linux-3.*.el7.tar.xz/d" $tmpdir/kernel/.gitignore;
-rhpkg upload $rhdistgit_tarball >/dev/null || die "uploading tarball";
+sed -i "/kernel-abi-whitelists.tar.bz2/d" $tmpdir/kernel/.gitignore;
+rhpkg upload $rhdistgit_tarball >/dev/null || die "uploading kernel tarball";
+rhpkg upload $rhdistgit_kabi_tarball >/dev/null || die "uploading kabi tarball";
 
 echo "Creating diff for review ($tmpdir/diff) and changelog"
 # diff the result (redhat/cvs/dontdiff). note: diff reuturns 1 if
-- 
1.7.1