From 8af20593e41d71cd7f9f7c213552995302cbe12f Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Fri, 12 Sep 2014 12:12:17 -0400
Subject: [redhat] kernel-3.10.0-158.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 8f5e70d..d864b8f 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 157
+RHEL_RELEASE = 158
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 89de34d..fe94b1a 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=157
+BUILD:=158
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 6168547..3c49e3f 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1474,6 +1474,29 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Fri Sep 12 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-158.el7]
+- [net] ipv4: implement igmp_qrv sysctl to tune igmp robustness variable (Hannes Frederic Sowa) [1110118]
+- [net] ipv6: add sysctl_mld_qrv to configure query robustness variable (Hannes Frederic Sowa) [1110118]
+- [net] rtnetlink: wait for unregistering devices in rtnl_link_unregister() (Florian Westphal) [1133741]
+- [net] netns: Delay default_device_exit_batch until no devices are unregistering (Florian Westphal) [1133741]
+- [net] netfilter: ctnetlink: fix refcnt leak in dying/unconfirmed list dumper (Florian Westphal) [1043012]
+- [net] netfilter: ctnetlink: fix dumping of dying/unconfirmed conntracks (Florian Westphal) [1043012]
+- [net] netfilter: nf_conntrack: initialize net.ct.generation (Florian Westphal) [1043012]
+- [net] netfilter: conntrack: Fix UP builds (Florian Westphal) [1043012]
+- [net] netfilter: conntrack: remove central spinlock nf_conntrack_lock (Florian Westphal) [1043012]
+- [net] netfilter: conntrack: seperate expect locking from nf_conntrack_lock (Florian Westphal) [1043012]
+- [net] netfilter: avoid race with exp->master ct (Florian Westphal) [1043012]
+- [net] netfilter: conntrack: spinlock per cpu to protect special lists (Florian Westphal) [1043012]
+- [net] netfilter: trivial code cleanup and doc changes (Florian Westphal) [1043012]
+- [net] sctp: fix ABI mismatch through sctp_assoc_to_state helper (Daniel Borkmann) [1135389]
+- [net] tcp: tsq: fix nonagle handling (Jiri Pirko) [1134402]
+- [netdrv] veth: extend features to support tunneling (Florian Westphal) [1098138]
+- [net] openvswitch: fix panic with multiple vlan headers (Jiri Benc) [1133109]
+- [net] netfilter: synproxy target: restrict to INPUT/FORWARD (Jesper Brouer) [1081339]
+- [net] fix rtnl notification in atomic context (Jiri Pirko) [1113925]
+- [net] always advertise rx_flags changes via netlink (Jiri Pirko) [1113925]
+- [net] update __dev_notify_flags() to send rtnl msg (Jiri Pirko) [1113925]
+
 * Fri Sep 12 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-157.el7]
 - [scsi] fusion: Remove use of DEF_SCSI_QCMD (Tomas Henzl) [1041675]
 - [scsi] fusion: Add free msg frames to the head, not tail of list (Tomas Henzl) [1041675]
-- 
1.7.1