From bffc7b652e5f6867fa3d70caab4aaa1ca1c03f15 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Fri, 25 Jul 2014 18:04:37 -0400
Subject: [redhat] kernel-3.10.0-139.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 9b557bc..4a18abe 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 138
+RHEL_RELEASE = 139
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 11a5cf9..4b1251a 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=138
+BUILD:=139
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 8f69289..9a33b52 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1471,6 +1471,79 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Fri Jul 25 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-139.el7]
+- [fs] GFS2: fix potential NULL pointer dereference (Robert S Peterson) [1095835]
+- [input] synaptics: fix resolution for manually provided min/max (Benjamin Tissoires) [1093449]
+- [input] synaptics: change min/max quirk table to pnp-id matching (Benjamin Tissoires) [1093449]
+- [input] synaptics: add a matches_pnp_id helper function (Benjamin Tissoires) [1093449]
+- [input] synaptics: T540p - unify with other LEN0034 models (Benjamin Tissoires) [1093449]
+- [input] synaptics: add min/max quirk for the ThinkPad W540 (Benjamin Tissoires) [1093449]
+- [input] synaptics: add min/max quirk for ThinkPad Edge E431 (Benjamin Tissoires) [1093449]
+- [input] synaptics: add min/max quirk for ThinkPad T431s, L440, L540, S1 Yoga and X1 (Benjamin Tissoires) [1093449]
+- [input] synaptics: report INPUT_PROP_TOPBUTTONPAD property (Benjamin Tissoires) [1093449]
+- [input] Add INPUT_PROP_TOPBUTTONPAD device property (Benjamin Tissoires) [1093449]
+- [input] i8042: add firmware_id support (Benjamin Tissoires) [1093449]
+- [input] serio: add firmware_id sysfs attribute (Benjamin Tissoires) [1093449]
+- [input] synaptics: add manual min/max quirk for ThinkPad X240 (Benjamin Tissoires) [1093449]
+- [input] synaptics: add manual min/max quirk (Benjamin Tissoires) [1093449]
+- [input] synaptics: fix incorrect placement of __initconst (Benjamin Tissoires) [1093449]
+- [s390] sclp_early: Return correct HSA block count also for zero (Hendrik Brueckner) [1107730]
+- [s390] sclp_early: Add function to detect sclp console capabilities (Hendrik Brueckner) [1107730]
+- [s390] sclp_early: Pass sccb pointer to every *_detect() function (Hendrik Brueckner) [1107730]
+- [s390] sclp_early: Replace early_read_info_sccb with sccb_early (Hendrik Brueckner) [1107730]
+- [s390] sclp_early: Get rid of sclp_early_read_info_sccb_valid (Hendrik Brueckner) [1107730]
+- [s390] sclp: replace uninitialized early_event_mask_sccb variable with sccb_early (Hendrik Brueckner) [1107730]
+- [s390] sclp: Consolidate early sclp init calls to sclp_early_detect() (Hendrik Brueckner) [1107730]
+- [s390] sclp: Move declarations for sclp_sdias into separate header file (Hendrik Brueckner) [1107730]
+- [s390] sclp: Move early code from sclp_cmd.c to sclp_early.c (Hendrik Brueckner) [1107730]
+- [s390] sclp: Determine HSA size dynamically for zfcpdump (Hendrik Brueckner) [1107730]
+- [s390] sclp: properly detect line mode console (Hendrik Brueckner) [1107730]
+- [s390] sclp: add parameter to specify number of buffer pages (Hendrik Brueckner) [1103744]
+- [md] dm-cache: set minimum_io_size to cache's data block size (Mike Snitzer) [1117872]
+- [md] dm-thin: set minimum_io_size to pool's data block size (Mike Snitzer) [1117872]
+- [md] dm-crypt: use per-bio data (Mike Snitzer) [1117872]
+- [block] use kmalloc alignment for bio slab (Mike Snitzer) [1117872]
+- [md] dm-crypt: fix cpu hotplug crash by removing per-cpu structure (Mike Snitzer) [1117872]
+- [md] dm-mpath: print more useful warnings in multipath_message() (Mike Snitzer) [1117872]
+- [md] dm-mpath: do not activate failed paths (Mike Snitzer) [1117872]
+- [md] dm-mpath: eliminate pg_ready() wrapper (Mike Snitzer) [1117872]
+- [md] dm-mpath: remove extra nesting in map function (Mike Snitzer) [1117872]
+- [md] dm-mpath: remove map_io() (Mike Snitzer) [1117872]
+- [md] dm-mpath: reduce memory pressure when requeuing (Mike Snitzer) [1117872]
+- [md] dm-mpath: really fix lockdep warning (Mike Snitzer) [1117872]
+- [md] dm-mpath: fix lock order inconsistency in multipath_ioctl (Mike Snitzer) [1117872]
+- [md] dm-mpath: remove process_queued_ios() (Mike Snitzer) [1117872]
+- [md] dm-mpath: fix IO hang due to logic bug in multipath_busy (Mike Snitzer) [1117872]
+- [md] dm-mpath: push back requests instead of queueing (Mike Snitzer) [1117872]
+- [md] dm-table: add dm_table_run_md_queue_async (Mike Snitzer) [1117872]
+- [md] dm-mpath: do not call pg_init when it is already running (Mike Snitzer) [1117872]
+- [md] dm-cache: fail migrations in the do_worker error path (Mike Snitzer) [1117872]
+- [md] dm-cache: simplify deferred set reference count increments (Mike Snitzer) [1117872]
+- [md] dm-cache-metadata: use dm-space-map-metadata.h defined size limits (Mike Snitzer) [1117872]
+- [md] dm-cache-metadata: do not allow the data block size to change (Mike Snitzer) [1117872]
+- [md] dm-thin-metadata: do not allow the data block size to change (Mike Snitzer) [1117872]
+- [md] dm-thin: relax external origin size constraints (Mike Snitzer) [1065474]
+- [md] dm-thin: switch to an atomic_t for tracking pending new block preparations (Mike Snitzer) [1065474]
+- [md] dm-thin: cleanup noflush_work to use a proper completion (Mike Snitzer) [1117872]
+- [md] dm-era: check for a non-NULL metadata object before closing it (Mike Snitzer) [1117872]
+- [md] dm-io: simplify dec_count and sync_io (Mike Snitzer) [1117872]
+- [md] dm-io: fix a race condition in the wake up code for sync_io (Mike Snitzer) [1117872]
+- [md] dm-crypt, dm-zero: update author name following legal name change (Mike Snitzer) [1117872]
+- [md] dm: allocate a special workqueue for deferred device removal (Mike Snitzer) [1117872]
+- [md] dm-snapshot: do not split read bios sent to snapshot-origin target (Mike Snitzer) [1117872]
+- [md] dm-snapshot: allocate a per-target structure for snapshot-origin target (Mike Snitzer) [1117872]
+- [md] dm: introduce dm_accept_partial_bio (Mike Snitzer) [1117872]
+- [md] dm: change sector_count member in clone_info from sector_t to unsigned (Mike Snitzer) [1117872]
+- [md] dm: remove symbol export for dm_set_device_limits (Mike Snitzer) [1117872]
+- [md] dm: disable WRITE SAME if it fails (Mike Snitzer) [1117872]
+- [md] dm: use RCU_INIT_POINTER instead of rcu_assign_pointer in __unbind (Mike Snitzer) [1117872]
+- [md] dm: stop using bi_private (Mike Snitzer) [1117872]
+- [md] dm: remove dm_get_mapinfo (Mike Snitzer) [1117872]
+- [md] dm-table: make dm_table_supports_discards static (Mike Snitzer) [1117872]
+- [md] dm: make dm_table_alloc_md_mempools static (Mike Snitzer) [1117872]
+- [crypto] fips: only panic on bad/missing crypto mod signatures (Jarod Wilson) [903562]
+- [x86] hugetlb: Add missing TLB page invalidation for hugetlb_cow() (Larry Woodman) [1097919]
+
 * Fri Jul 18 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-138.el7]
 - [mm] hugetlb: ensure hugepage access is denied if hugepages are not supported (David Gibson) [1081671]
 - [x86] Delete non-required instances of include <linux/init.h> (Prarit Bhargava) [1119074]
-- 
1.7.1