From aa239f622b4698331560696720f8b3632fd95269 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Mon, 29 Sep 2014 09:55:14 -0400
Subject: [redhat] kernel-3.10.0-173.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 168020f..2d5a405 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 172
+RHEL_RELEASE = 173
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 1dffb6e..03bc279 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=172
+BUILD:=173
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 89e735b..5c1e8aa 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1483,6 +1483,45 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Mon Sep 29 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-173.el7]
+- [crypto] pefile: Relax the check on the length of the PKCS#7 cert (Vivek Goyal) [852066]
+- [kernel] kexec, purgatory: add clean-up for purgatory directory (Vivek Goyal) [852066]
+- [x86] purgatory: use approprate -m64/-32 build flag for arch/x86/purgatory (Vivek Goyal) [852066]
+- [kernel] kexec: remove CONFIG_KEXEC dependency on crypto (Vivek Goyal) [852066]
+- [kernel] kexec: create a new config option CONFIG_KEXEC_FILE for new syscall (Vivek Goyal) [852066]
+- [kernel] resource: fix the case of null pointer access (Vivek Goyal) [852066]
+- [kernel] kexec: verify the signature of signed PE bzImage (Vivek Goyal) [852066]
+- [kernel] kexec: support kexec/kdump on EFI systems (Vivek Goyal) [852066]
+- [kernel] kexec: support for kexec on panic using new system call (Vivek Goyal) [852066]
+- [x86] kexec-bzimage64: support for loading bzImage using 64bit entry (Vivek Goyal) [852066]
+- [kernel] kexec: load and relocate purgatory at kernel load time (Vivek Goyal) [852066]
+- [x86] purgatory: core purgatory functionality (Vivek Goyal) [852066]
+- [x86] purgatory/sha256: provide implementation of sha256 in purgaotory context (Vivek Goyal) [852066]
+- [kernel] kexec: implementation of new syscall kexec_file_load (Vivek Goyal) [852066]
+- [kernel] kexec: new syscall kexec_file_load() declaration (Vivek Goyal) [852066]
+- [kernel] kexec: make kexec_segment user buffer pointer a union (Vivek Goyal) [852066]
+- [kernel] resource: provide new functions to walk through resources (Vivek Goyal) [852066]
+- [kernel] kexec: use common function for kimage_normal_alloc() and kimage_crash_alloc() (Vivek Goyal) [852066]
+- [kernel] kexec: move segment verification code in a separate function (Vivek Goyal) [852066]
+- [kernel] kexec: rename unusebale_pages to unusable_pages (Vivek Goyal) [852066]
+- [scripts] build bin2c based on config option CONFIG_BUILD_BIN2C (Vivek Goyal) [852066]
+- [scripts] bin2c: move bin2c in scripts/basic (Vivek Goyal) [852066]
+- [kernel] kexec: convert printk to pr_foo() (Vivek Goyal) [852066]
+- [kernel] kexec/compat: convert to COMPAT_SYSCALL_DEFINE with changing parameter types (Vivek Goyal) [852066]
+- [kernel] kexec: use vscnprintf() instead of vsnprintf() in vmcoreinfo_append_str() (Vivek Goyal) [852066]
+- [kernel] kexec: add sysctl to disable kexec_load (Vivek Goyal) [852066]
+- [kernel] kexec: Typo s/the/then/ (Vivek Goyal) [852066]
+- [kernel] kexec: remove unnecessary return (Vivek Goyal) [852066]
+- [asm] crash: Remove the original asm/crash.h files (Vivek Goyal) [852066]
+- [asm] crash: Rename crash.h to crash-driver.h (Vivek Goyal) [852066]
+- [x86] boot: Remove misc.h inclusion from compressed/string.c (Vivek Goyal) [852066]
+- [x86] boot: Do not include boot.h in string.c (Vivek Goyal) [852066]
+- [x86] boot: Move memset() definition in compressed/string.c (Vivek Goyal) [852066]
+- [x86] boot: Move memcmp() into string.h and string.c (Vivek Goyal) [852066]
+- [x86] boot: Move optimized memcpy() 32/64 bit versions to compressed/string.c (Vivek Goyal) [852066]
+- [x86] boot: Create a separate string.h file to provide standard string functions (Vivek Goyal) [852066]
+- [x86] boot: Undef memcmp before providing a new definition (Vivek Goyal) [852066]
+
 * Mon Sep 29 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-172.el7]
 - [fs] nfsd4: fix rd_dircount enforcement ("J. Bruce Fields") [1044798 1123721 1141932]
 - [fs] nfsd: add some comments to the nfsd4 object definitions ("J. Bruce Fields") [1044798 1123721 1141932]
-- 
1.7.1