From 6a6884ff39b4b008f02450f03ba9fb7892f66a7c Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 23 Sep 2014 17:48:04 -0400
Subject: [redhat] kernel-3.10.0-169.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 34b5793..7cf13c8 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 168
+RHEL_RELEASE = 169
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index e0889d9..f32ceeb 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=168
+BUILD:=169
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index af04b58..26006c2 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1474,6 +1474,34 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Sep 23 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-169.el7]
+- [lib] assoc_array: Fix termination condition in assoc array garbage collection (David Howells) [1139431] {CVE-2014-3631}
+- [acpi] x86/ras: Fix build warnings in <linux/aer.h> (Rui Wang) [1085141]
+- [acpi] ras/extlog: Adjust init flow (Rui Wang) [1085141]
+- [acpi] trace/emca: Add a knob to adjust where to save event log (Rui Wang) [1085141]
+- [ras] trace: Add eMCA trace event interface (Rui Wang) [1085141]
+- [ras] debugfs: Add debugfs interface for RAS subsystem (Rui Wang) [1085141]
+- [firmware] efi/cper: Adjust code flow of some functions (Rui Wang) [1085141]
+- [trace] aer: Move trace into unified interface (Rui Wang) [1085141]
+- [trace] ras: Add basic RAS trace event (Rui Wang) [1085141]
+- [trace] Add __field_struct macro for TRACE_EVENT() (Rui Wang) [1085141]
+- [trace] pci/aer: Fix severity usage in aer trace event (Rui Wang) [1085141]
+- [acpi] emca: Combine eMCA/EDAC event reporting priority (Rui Wang) [1076655]
+- [edac] sb_edac: Modify H/W event reporting policy (Rui Wang) [1076655]
+- [edac] Add an edac_report parameter to EDAC (Rui Wang) [1076655]
+- [acpi] Clean up inclusions of ACPI header files (Rui Wang) [1076655]
+- [acpi] x86: Export 'boot_cpu_physical_apicid' to modules (Rui Wang) [1076655]
+- [acpi] x86: Fix extended error log driver to depend on CONFIG_X86_LOCAL_APIC (Rui Wang) [1076655]
+- [firmware] dmi: Avoid unaligned memory access in save_mem_devices() (Rui Wang) [1076655]
+- [firmware] efi: Move cper.c from drivers/acpi/apei to drivers/firmware/efi (Rui Wang) [1076655]
+- [edac] ghes: Update ghes error record info (Rui Wang) [1076655]
+- [acpi] apei/cper: Cleanup CPER memory error output format (Rui Wang) [1076655]
+- [acpi] apei/cper: Enhance memory reporting capability (Rui Wang) [1076655]
+- [firmware] dmi: Parse memory device (type 17) in SMBIOS (Rui Wang) [1076655]
+- [acpi] x86: Extended error log driver for x86 platform (Rui Wang) [1076655]
+- [acpi] cper: Update cper info (Rui Wang) [1076655]
+- [acpi] apei/cper: Fix status check during error printing (Rui Wang) [1076655]
+
 * Tue Sep 23 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-168.el7]
 - [ethernet] bnx2x: Configure device endianity on driver load and reset endianity on removal (Michal Schmidt) [1089402]
 - [ethernet] bnx2x: Be more forgiving toward SW GRO (Michal Schmidt) [1089402]
-- 
1.7.1