From 8c1894c18c251264ef049a3643c8c37409c4b02a Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Wed, 10 Sep 2014 14:39:01 -0400
Subject: [redhat] kernel-3.10.0-154.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 3e336ce..f51f477 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 153
+RHEL_RELEASE = 154
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index ef5d631..faa8474 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=153
+BUILD:=154
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index d32bbec..9a29e52 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1472,6 +1472,76 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Wed Sep 10 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-154.el7]
+- [scsi] ipr: Add new CCIN definition for Grand Canyon support (Gustavo Duarte) [1088561]
+- [scsi] ipr: Format HCAM overlay ID 0x21 (Gustavo Duarte) [1088561]
+- [scsi] ipr: Handle early EEH (Gustavo Duarte) [1088561]
+- [scsi] ipr: Add new CCIN definition for new hardware support (Gustavo Duarte) [1088561]
+- [scsi] ipr: Remove extended delay bit on GSCSI reads/writes ops (Gustavo Duarte) [1088561]
+- [scsi] ipr: increase dump size in ipr driver (Gustavo Duarte) [1088561]
+- [kernel] trace: Remove function_trace_stop and HAVE_FUNCTION_TRACE_MCOUNT_TEST (Josh Poimboeuf) [1113829]
+- [s390] ftrace: remove check of obsolete variable function_trace_stop (Josh Poimboeuf) [1113829]
+- [x86] ftrace: Remove check of obsolete variable function_trace_stop (Josh Poimboeuf) [1113829]
+- [kernel] ftrace: Remove check for HAVE_FUNCTION_TRACE_MCOUNT_TEST (Josh Poimboeuf) [1113829]
+- [kernel] ftrace: Remove function_trace_stop check from list func (Josh Poimboeuf) [1113829]
+- [kernel] ftrace: Do no disable function tracing on enabling function tracing (Josh Poimboeuf) [1113829]
+- [kernel] ftrace: Remove ftrace_start/stop() (Josh Poimboeuf) [1113829]
+- [kernel] ftrace-graph: Remove usage of ftrace_stop() in ftrace_graph_stop() (Josh Poimboeuf) [1113829]
+- [powerpc] kernel/ftrace: Add call to ftrace_graph_is_dead() in function graph code (Josh Poimboeuf) [1113829]
+- [kernel] ftrace: Add call to ftrace_graph_is_dead() in function graph code (Josh Poimboeuf) [1113829]
+- [kernel] ftrace-graph: Remove dependency of ftrace_stop() from ftrace_graph_stop() (Josh Poimboeuf) [1113829]
+- [kernel] trace: Remove ftrace_stop/start() from reading the trace file (Josh Poimboeuf) [1113829]
+- [kernel] power: Remove ftrace_stop/start() from suspend and hibernate (Josh Poimboeuf) [1113829]
+- [x86] power/cpu: Annotate restore_processor_state() with notrace (Josh Poimboeuf) [1113829]
+- [kernel] ftrace: Move the mcount/fentry code out of entry_64.S (Josh Poimboeuf) [1113829]
+- [kernel] ftrace: Load ftrace_ops in parameter not the variable holding it (Josh Poimboeuf) [1113829]
+- [kernel] trace: Remove unused function ftrace_off_permanent() (Josh Poimboeuf) [1113829]
+- [kernel] ftrace: BUG when ftrace recovery fails (Josh Poimboeuf) [1113829]
+- [kernel] ftrace: Have ftrace_write() return -EPERM and clean up callers (Josh Poimboeuf) [1113829]
+- [kernel] ftrace: One more missing sync after fixup of function modification failure (Josh Poimboeuf) [1113829]
+- [kernel] ftrace: Run a sync after fixup on failure (Josh Poimboeuf) [1113829]
+- [kernel] ftrace: Use breakpoints for converting function graph caller (Josh Poimboeuf) [1113829]
+- [kernel] ftrace: skip over the breakpoint for ftrace caller (Josh Poimboeuf) [1113829]
+- [s390] kprobes: add support for pc-relative long displacement instructions (Hendrik Brueckner) [1123429]
+- [kernel] kprobes: allow to specify custom allocator for insn caches (Hendrik Brueckner) [1123429]
+- [kernel] kprobes: unify insn caches (Hendrik Brueckner) [1123429]
+- [kernel] uprobes: Rename arch_uprobe->def to ->defparam, minor comment updates (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Fix scratch register selection for rip-relative fixups (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Simplify rip-relative handling (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Simplify riprel_{pre,post}_xol() and make them similar (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Kill the "autask" arg of riprel_pre_xol() (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Rename *riprel* helpers to make the naming consistent (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Cleanup the usage of UPROBE_FIX_IP/UPROBE_FIX_CALL (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Kill adjust_ret_addr(), simplify UPROBE_FIX_CALL logic (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Introduce push_ret_address() (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Cleanup the usage of arch_uprobe->def.fixups, make it u8 (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Move default_xol_ops's data into arch_uprobe->def (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Move UPROBE_FIX_SETF logic from arch_uprobe_post_xol() to default_post_xol_op() (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Don't use arch_uprobe_abort_xol() in arch_uprobe_post_xol() (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Introduce uprobe_xol_ops->abort() and default_abort_op() (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Don't change the task's state if ->pre_xol() fails (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Fix is_64bit_mm() with CONFIG_X86_X32 (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Make good_insns_* depend on CONFIG_X86_* (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Shift "insn_complete" from branch_setup_xol_ops() to uprobe_init_insn() (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Add is_64bit_mm(), kill validate_insn_bits() (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Add uprobe_init_insn(), kill validate_insn_{32,64}bits() (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Refuse to attach uprobe to "word-sized" branch insns (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Emulate relative conditional "near" jmp's (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Emulate relative conditional "short" jmp's (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Emulate relative call's (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Emulate nop's using ops->emulate() (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Emulate unconditional relative jmp's (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Introduce sizeof_long(), cleanup adjust_ret_addr() and arch_uretprobe_hijack_return_addr() (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Teach arch_uprobe_post_xol() to restart if possible (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Send SIGILL if arch_uprobe_post_xol() fails (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Conditionalize the usage of handle_riprel_insn() (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Introduce uprobe_xol_ops and arch_uprobe->ops (Oleg Nesterov) [1073627]
+- [kernel] uprobes: move the UPROBE_FIX_{RIP,IP,CALL} code at the end of pre/post hooks (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Gather "riprel" functions together (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Kill the "ia32_compat" check in handle_riprel_insn(), remove "mm" arg (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Fold prepare_fixups() into arch_uprobe_analyze_insn() (Oleg Nesterov) [1073627]
+- [kernel] uprobes: Kill UPROBE_SKIP_SSTEP and can_skip_sstep() (Oleg Nesterov) [1073627]
+
 * Fri Sep 05 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-153.el7]
 - [ethernet] enic: Add ethtool support to show classifier filters added by the driver (Stefan Assmann) [1107934]
 - [ethernet] enic: remove #ifdef CONFIG_RFS_ACCEL around filter structures (Stefan Assmann) [1107934]
-- 
1.7.1