From 996f66a0d85e289aab64889cfef8182c60275142 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Fri, 18 Jul 2014 11:01:27 -0400
Subject: [redhat] kernel-3.10.0-138.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index cd3c498..9b557bc 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 137
+RHEL_RELEASE = 138
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 5b334db..11a5cf9 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=137
+BUILD:=138
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 888499f..8f69289 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1471,6 +1471,37 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Fri Jul 18 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-138.el7]
+- [mm] hugetlb: ensure hugepage access is denied if hugepages are not supported (David Gibson) [1081671]
+- [x86] Delete non-required instances of include <linux/init.h> (Prarit Bhargava) [1119074]
+- [mmc] sdhci-pci-o2micro: Add SeaBird SeaEagle SD3 support (David Milburn) [968020]
+- [mmc] sdhci-pci: add support of O2Micro/BayHubTech SD hosts (David Milburn) [968020]
+- [mmc] sdhci-pci: break out definitions to header file (David Milburn) [968020]
+- [ethernet] cnic: Don't take cnic_dev_lock in cnic_alloc_uio_rings() (Maurizio Lombardi) [1079384 1094613]
+- [ethernet] cnic: Don't take rcu_read_lock in cnic_rcv_netevent() (Maurizio Lombardi) [1079384 1094613]
+- [idle] intel_idle: fix IVT idle state table setting (Prarit Bhargava) [1119374]
+- [idle] intel_idle: fine-tune IVT residency targets (Prarit Bhargava) [1119374]
+- [idle] intel_idle: do C1E promotion disable quirk for hotplugged CPUs (Prarit Bhargava) [1119374]
+- [idle] intel_idle: remove superfluous dev->state_count initialization (Prarit Bhargava) [1119374]
+- [x86] acpi, idle: Restructure the mwait idle routines (Prarit Bhargava) [1119374]
+- [idle] intel_idle: use CPU_TASKS_FROZEN instead of a numeric constant (Prarit Bhargava) [1119374]
+- [idle] intel_idle: shrink states tables (Prarit Bhargava) [1119374]
+- [md] dm-thin: update discard_granularity to reflect the thin-pool blocksize (Mike Snitzer) [1099936]
+- [md] dm-bio-prison: implement per bucket locking in the dm_bio_prison hash table (Mike Snitzer) [1099936]
+- [md] dm-thin: return ENOSPC instead of EIO when error_if_no_space enabled (Mike Snitzer) [1099936]
+- [md] dm-cache: always split discards on cache block boundaries (Mike Snitzer) [1103836]
+- [md] dm-cache: fix writethrough mode quiescing in cache_map (Mike Snitzer) [1103836]
+- [md] dm-cache: fix a lock-inversion (Mike Snitzer) [1103836]
+- [md] dm-thin: add 'no_space_timeout' dm-thin-pool module param (Mike Snitzer) [1095639]
+- [md] dm-thin: add timeout to stop out-of-data-space mode holding IO forever (Mike Snitzer) [1095639]
+- [md] dm-thin: allow metadata commit if pool is in PM_OUT_OF_DATA_SPACE mode (Mike Snitzer) [1091852]
+- [md] dm-thin: use INIT_WORK_ONSTACK in noflush_work to avoid ODEBUG warning (Mike Snitzer) [1098643]
+- [md] dm-thin: fix rcu_read_lock being held in code that can sleep (Mike Snitzer) [1098643]
+- [md] dm-thin: irqsave must always be used with the pool->lock spinlock (Mike Snitzer) [1098643]
+- [md] dm-thin: sort the per thin deferred bios using an rb_tree (Mike Snitzer) [1098643]
+- [md] dm-thin: use per thin device deferred bio lists (Mike Snitzer) [1098643]
+- [md] dm-thin: simplify pool_is_congested (Mike Snitzer) [1098643]
+
 * Thu Jul 17 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-137.el7]
 - [powercap] Add to drivers Kconfig and Makefile (Steve Best) [1085575]
 - [powercap] add Documentation (Steve Best) [1085575]
-- 
1.7.1