From e3e634992a54a31d1838a27cb56d7479abc2c3e6 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Wed, 27 Aug 2014 16:50:10 -0400
Subject: [redhat] kernel-3.10.0-147.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 2ec827a..3b04e45 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 146
+RHEL_RELEASE = 147
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index ae7ed2d..752558f 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=146
+BUILD:=147
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 1434334..eaa0a47 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1472,6 +1472,49 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Wed Aug 27 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-147.el7]
+- [s390] fix restore of invalid floating-point-control (Hendrik Brueckner) [1121965]
+- [s390] crypto: fix aes_s390 crypto module unload problem (Hendrik Brueckner) [1103114]
+- [x86] avx-512: Enable AVX-512 States Context Switch (Rui Wang) [1085563]
+- [x86] avx-512: AVX-512 Feature Detection (Rui Wang) [1085563]
+- [ethernet] netxen: Remove DEFINE_PCI_DEVICE_TABLE macro use (Veaceslav Falico) [1089356]
+- [ethernet] netxen: slight optimization of addr compare (Veaceslav Falico) [1089356]
+- [ethernet] netxen: Update version to 4.0.82 (Veaceslav Falico) [1089356]
+- [ethernet] netxen: Miscellaneous conversions to ETH_ALEN (Veaceslav Falico) [1089356]
+- [ethernet] netxen: get rid of SET_ETHTOOL_OPS (Veaceslav Falico) [1089356]
+- [ethernet] netxen: Correct off-by-one errors in bounds checks (Veaceslav Falico) [1089356]
+- [ethernet] netxen: remove unnecessary pci_set_drvdata() (Veaceslav Falico) [1089356]
+- [ethernet] netxen: Print ULA information (Veaceslav Falico) [1089356]
+- [ethernet] netxen: lower NAPI weight (Veaceslav Falico) [1089356]
+- [ethernet] netxen: Remove extern from function prototypes (Veaceslav Falico) [1089356]
+- [x86] Mark Intel Broadwell processor as supported (Steve Best) [1089025]
+- [mm] Fix NULL pointer dereference in madvise(MADV_WILLNEED) support (Jan Stancek) [1072385]
+- [hid] logitech-dj: Fix USB 3.0 issue (Benjamin Tissoires) [1110523]
+- [lib] percpu_counter: fix bad percpu counter state during suspend (Eric Sandeen) [1129839]
+- [kernel] user_namespace: fix incorrect memory barriers (Mikulas Patocka) [1128950]
+- [kernel] capabilities: remove undefined caps from all processes (Paul Moore) [1123063]
+- [tty] hvc: don't free hvc_console_setup after init (Luiz Capitulino) [976490]
+- [mfd] lpc_ich: Add Device IDs for Intel Wildcat Point-LP PCH (Steve Best) [1086359]
+- [i2c] i801: Add Device IDs for Intel Wildcat Point-LP PCH (Steve Best) [1086359]
+- [x86] perf: Add model number for Avoton Silvermont (Steve Best) [1086821]
+- [x86] perf: Add Silvermont (22nm Atom) support (Steve Best) [1086821]
+- [x86] perf: use INTEL_UEVENT_EXTRA_REG to define MSR_OFFCORE_RSP_X (Steve Best) [1086821]
+- [virt] hyperv/storvsc: Correctly handle TEST_UNIT_READY failure (Fam Zheng) [1122317]
+- [virt] hyperv/storvsc: Set srb_flags in all cases (Fam Zheng) [1122317]
+- [virt] hyperv/storvsc: Implement a eh_timed_out handler (Fam Zheng) [1122317]
+- [virt] hyperv/storvsc: Fix a bug in handling VMBUS protocol version (Fam Zheng) [1122317]
+- [virt] hyperv/storvsc: Filter commands based on the storage protocol version (Fam Zheng) [1122317]
+- [virt] hyperv/storvsc: Set cmd_per_lun to reflect value supported by the Host (Fam Zheng) [1122317]
+- [virt] hyperv/storvsc: Change the limits to reflect the values on the host (Fam Zheng) [1122317]
+- [powerpc] perf: Add BHRB constraint and IFM MMCRA handling for EBB (Gustavo Duarte) [1121408]
+- [powerpc] perf: Reject EBB events which specify a sample_type (Gustavo Duarte) [1121408]
+- [powerpc] perf: Enable BHRB access for EBB events (Gustavo Duarte) [1121408]
+- [powerpc] perf: Avoid mutating event in power8_get_constraint() (Gustavo Duarte) [1121408]
+- [powerpc] perf: Clean up the EBB hash defines a little (Gustavo Duarte) [1121408]
+- [powerpc] mm: Check paca psize is up to date for huge mappings (Gustavo Duarte) [1107337]
+- [powerpc] perf: Add lost exception workaround (Gustavo Duarte) [1121407]
+- [powerpc] Add a cpu feature CPU_FTR_PMAO_BUG (Gustavo Duarte) [1121407]
+
 * Tue Aug 26 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-146.el7]
 - [edac] sb_edac: add support for Haswell based systems (Seth Jennings) [1127019]
 - [edac] add DDR4 and RDDR4 (Seth Jennings) [1127019]
-- 
1.7.1