From 1604ac7bd221c9d9cfeeac04656958ced29a2c8d Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Fri, 12 Sep 2014 10:54:54 -0400
Subject: [redhat] kernel-3.10.0-156.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index fd3bbfe..35ca6ba 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 155
+RHEL_RELEASE = 156
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 4577b81..7c61652 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=155
+BUILD:=156
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 641ccd9..27251d7 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1474,6 +1474,175 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Fri Sep 12 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-156.el7]
+- [tty] hvc_console: Fix wakeup of HVC thread on hvc_kick() (Steve Best) [1123045]
+- [tty] hvc_opal: Kick the HVC thread on OPAL console events (Steve Best) [1123045]
+- [powerpc] perf/hv-24x7: Catalog version number is be64, not be32 (Gustavo Duarte) [947159]
+- [powerpc] perf/hv-24x7: Remove [static 4096], sparse chokes on it (Gustavo Duarte) [947159]
+- [powerpc] perf/hv-24x7: Use (unsigned long) not (u32) values when calling plpar_hcall_norets() (Gustavo Duarte) [947159]
+- [powerpc] perf/hv-gpci: Make device attr static (Gustavo Duarte) [947159]
+- [powerpc] perf/hv_gpci: Probe failures use pr_debug(), and padding reduced (Gustavo Duarte) [947159]
+- [powerpc] perf/hv_24x7: Probe errors changed to pr_debug(), padding fixed (Gustavo Duarte) [947159]
+- [Documentation] powerpc/perf/hv_{gpci, 24x7}: Add documentation of device attributes (Gustavo Duarte) [947159]
+- [powerpc] perf: Add kconfig option for hypervisor provided counters (Gustavo Duarte) [947159]
+- [powerpc] perf: Add support for the hv 24x7 interface (Gustavo Duarte) [947159]
+- [powerpc] perf: Add support for the hv gpci (get performance counter info) interface (Gustavo Duarte) [947159]
+- [powerpc] perf: Add macros for defining event fields & formats (Gustavo Duarte) [947159]
+- [powerpc] perf: Add a shared interface to get gpci version and capabilities (Gustavo Duarte) [947159]
+- [powerpc] perf: Add 24x7 interface headers (Gustavo Duarte) [947159]
+- [powerpc] perf: Add hv_gpci interface header (Gustavo Duarte) [947159]
+- [powerpc] Add hvcalls for 24x7 and gpci (Get Performance Counter Info) (Gustavo Duarte) [947159]
+- [ethernet] sfc: Remove DEFINE_PCI_DEVICE_TABLE macro use (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Use __iowrite64_copy instead of a slightly different local function (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Add support for busy polling (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Add per-queue statistics in ethtool (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: add extra RX drop counters for nodesc_trunc and noskb_drop (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Add 40G link capability decoding (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Adding PCI ID for Solarflare 7000 series 40G network adapter (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: fix calling of free_irq with already free vector (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: get rid of SET_ETHTOOL_OPS (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: On MCDI timeout, issue an FLR (and mark MCDI to fail-fast) (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Call efx_set_channels() before efx->type->dimension_resources() (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Don't receive packets when the napi budget == 0 (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: check for NULL efx->ptp_data in efx_ptp_event (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Use pci_enable_msix_range() instead of pci_enable_msix() (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Add/remove blank lines to taste (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Fail self-test with -EBUSY, not -EIO, if the device is busy (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Cosmetic changes to self-test from the out-of-tree driver (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Update product naming (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Use canonical pointer type for MAC address in efx_set_mac_address() (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Rename 'use_options' variable in tso_start() to clearer 'use_opt_desc' (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Preserve rx_frm_trunc counters when resizing DMA rings (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Correct comment about number of TX queues used on EF10 (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Remove unused definitions of EF10 user-mode DMA descriptors (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Replace TSOH_OFFSET with the equivalent NET_IP_ALIGN (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Rewrite adjustment of PPS event in a clearer way (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Cache skb->data in local variable in efx_ptp_rx() (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Removed adhoc scheme to rate limit PTP event queue overflow message (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: fix build warning in ethernet/sfc/tx.c (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Use the correct maximum TX DMA ring size for SFC9100 (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Fix transposed ptp_{under, over}size_sync_windows statistics (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: Change efx_mcdi_reset_port to use ENTITY_RESET MC command (Nikolay Aleksandrov) [1110888]
+- [ethernet] sfc: calls skb_set_hash (Nikolay Aleksandrov) [1110888]
+- [iommu] vt-d: Use correct domain id to flush virtual machine domains (Myron Stowe) [1136539]
+- [iommu] Fix IOMMU sysfs stubs (Myron Stowe) [1136539]
+- [iommu] Fix compile error in iommu-sysfs.c (Myron Stowe) [1136539]
+- [pci] Add bridge DMA alias quirk for Intel 82801 bridge (Myron Stowe) [1136539]
+- [iommu] amd: Add sysfs support (Myron Stowe) [1136539]
+- [iommu] vt-d: Make use of IOMMU sysfs support (Myron Stowe) [1136539]
+- [iommu] Add sysfs support for IOMMUs (Myron Stowe) [1136539]
+- [iommu] core: Make iommu_group_get_for_dev() more robust (Myron Stowe) [1136537]
+- [iommu] Remove pci.h (Myron Stowe) [1136537]
+- [iommu] vt-d: Update to use PCI DMA aliases (Myron Stowe) [1136537]
+- [iommu] vt-d: Use iommu_group_get_for_dev() (Myron Stowe) [1136537]
+- [iommu] amd: Use iommu_group_get_for_dev() (Myron Stowe) [1136537]
+- [iommu] amd: Update to use PCI DMA aliases (Myron Stowe) [1136537]
+- [iommu] core: Create central IOMMU group lookup/creation interface (Myron Stowe) [1136537]
+- [iommu] vt-d: Suppress compiler warnings (Myron Stowe) [1136537]
+- [iommu] vt-d: Fix reference count in iommu_prepare_isa (Myron Stowe) [1136537]
+- [iommu] vt-d: fix bug in handling multiple RMRRs for the same PCI device (Myron Stowe) [1136537]
+- [pci] Add bridge DMA alias quirk for ITE bridge (Myron Stowe) [1136537]
+- [pci] Add bridge DMA alias quirk for ASMedia and Tundra bridges (Myron Stowe) [1136537]
+- [pci] Add support for PCIe-to-PCI bridge DMA alias quirks (Myron Stowe) [1136537]
+- [pci] Add function 1 DMA alias quirk for Marvell devices (Myron Stowe) [1136537]
+- [pci] Add function 0 DMA alias quirk for Ricoh devices (Myron Stowe) [1136537]
+- [pci] Add support for DMA alias quirks (Myron Stowe) [1136537]
+- [pci] Convert pci_dev_flags definitions to bit shifts (Myron Stowe) [1136537]
+- [pci] Add DMA alias iterator (Myron Stowe) [1136537]
+- [iommu] amd: fix enabling exclusion range for an exact device (Myron Stowe) [1129880 1087643]
+- [iommu] amd: Take mmap_sem when calling get_user_pages (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: fix bug in matching PCI devices with DRHD/RMRR descriptors (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Fix get_domain_for_dev() handling of upstream PCIe bridges (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: fix memory leakage caused by commit ea8ea46 (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Fix error handling in ANDD processing (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: returning free pointer in get_domain_for_dev() (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Only call dmar_acpi_dev_scope_init() if DRHD units present (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Check for NULL pointer in dmar_acpi_dev_scope_init() (Myron Stowe) [1129880 1087643]
+- [iommu] amd: Fix logic to determine and checking max PASID (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Include ACPI devices in iommu=pt (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Finally enable translation for non-PCI devices (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Remove to_pci_dev() in intel_map_page() (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Remove pdev from intel_iommu_attach_device() (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Remove pdev from iommu_no_mapping() (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Make domain_add_dev_info() take struct device (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Make domain_remove_one_dev_info() take struct device (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Rename 'hwdev' variables to 'dev' now that that's the norm (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Remove some pointless to_pci_dev() calls (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Make get_valid_domain_for_dev() take struct device (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Make iommu_should_identity_map() take struct device (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Handle RMRRs for non-PCI devices (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Make get_domain_for_dev() take struct device (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Make domain_context_mapp{ed, ing}() take struct device (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Make device_to_iommu() cope with non-PCI devices (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Make identity_mapping() take struct device not struct pci_dev (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Remove segment from struct device_domain_info() (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Store PCI segment number in struct intel_iommu (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Remove device_to_iommu() call from domain_remove_dev_info() (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Simplify iommu check in domain_remove_one_dev_info() (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Always store iommu in device_domain_info (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Use domain_remove_one_dev_info() in domain_add_dev_info() error path (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: use dmar_insert_dev_info() from dma_add_dev_info() (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Stop dmar_insert_dev_info() freeing domains on losing race (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Pass iommu to domain_context_mapping_one() and iommu_support_dev_iotlb() (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Use struct device in device_domain_info, not struct pci_dev (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Make dmar_insert_dev_info() take struct device instead of struct pci_dev (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Make iommu_dummy() take struct device instead of struct pci_dev (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Add ACPI devices into dmaru->devices[] array (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Change scope lists to struct device, bus, devfn (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Allocate space for ACPI devices (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Parse ANDD records (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Add ACPI namespace device reporting structures (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Be less pessimistic about domain coherency where possible (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Honour intel_iommu=sp_off for non-VMM domains (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Clean up and fix page table clear/free behaviour (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Clean up size handling for intel_iommu_unmap() (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Update IOMMU state when memory hotplug happens (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Unify the way to process DMAR device scope array (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Update DRHD/RMRR/ATSR device scope caches when PCI hotplug happe (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Use RCU to protect global resources in interrupt context (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Introduce a rwsem to protect global data structures (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Introduce macro for_each_dev_scope() to walk device scope entrie (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Fix error in detect ATS capability (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Check for NULL pointer when freeing IOMMU data structure (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Fix incorrect iommu_count for si_domain (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Reduce duplicated code to handle virtual machine domains (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Free resources if failed to create domain for PCIe endpoint (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Simplify function get_domain_for_dev() (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Move private structures and variables into intel-iommu.c (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Factor out dmar_alloc_dev_scope() for later reuse (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Avoid caching stale domain_device_info when hot-removing PCI dev (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Avoid caching stale domain_device_info and fix memory leak (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Avoid double free of g_iommus on error recovery path (Myron Stowe) [1129880 1087643]
+- [iommu] amd: Fix PASID format in INVALIDATE_IOTLB_PAGES command (Myron Stowe) [1129880 1087643]
+- [iommu] vt-d: Fix signedness bug in alloc_irte() (Myron Stowe) [1129808]
+- [iommu] vt-d: free all resources if failed to initialize DMARs (Myron Stowe) [1129808]
+- [iommu] vt-d: clean sparse warnings (Myron Stowe) [1129808]
+- [iommu] vt-d: fix wrong return value of dmar_table_init() (Myron Stowe) [1129808]
+- [iommu] vt-d: release invalidation queue when destroying IOMMU unit (Myron Stowe) [1129808]
+- [iommu] vt-d: fix access after free issue in function free_dmar_iommu() (Myron Stowe) [1129808]
+- [iommu] vt-d: keep shared resources when failed to initialize iommu devices (Myron Stowe) [1129808]
+- [iommu] vt-d: fix invalid memory access when freeing DMAR irq (Myron Stowe) [1129808]
+- [iommu] vt-d: simplify code with existing macros (Myron Stowe) [1129808]
+- [iommu] vt-d: use defined macro instead of hardcoding (Myron Stowe) [1129808]
+- [iommu] vt-d: mark internal functions as static (Myron Stowe) [1129808]
+- [iommu] vt-d: clean up unused code (Myron Stowe) [1129808]
+- [iommu] vt-d: check suitable flag in function detect_intel_iommu() (Myron Stowe) [1129808]
+- [iommu] vt-d: print correct domain id of static identity domain (Myron Stowe) [1129808]
+- [iommu] vt-d: refine support of 64bit guest address (Myron Stowe) [1129808]
+- [iommu] vt-d: fix resource leakage on error recovery path in iommu_init_domain (Myron Stowe) [1129808]
+- [iommu] vt-d: fix a race window in allocating domain ID for virtual machines (Myron Stowe) [1129808]
+- [iommu] vt-d: fix PCI device reference leakage on error recovery path (Myron Stowe) [1129808]
+- [iommu] vt-d: use dedicated bitmap to track remapping entry allocation status (Myron Stowe) [1129808]
+- [iommu] Rename domain_has_cap to iommu_domain_has_cap (Myron Stowe) [1129808]
+- [iommu] vt-d: Use dev_is_pci() to check whether it is pci device (Myron Stowe) [1129808]
+- [iommu] amd: Use dev_is_pci() to check whether it is pci device (Myron Stowe) [1129808]
+- [iommu] add missing include (Myron Stowe) [1129808]
+- [iommu] vt-d: Mark function eoi_ioapic_pin_remapped() as static in irq_remappi (Myron Stowe) [1129808]
+- [iommu] vt-d: Mark functions as static in intel_irq_remapping.c (Myron Stowe) [1129808]
+- [iommu] vt-d: Mark functions as static in dmar.c (Myron Stowe) [1129808]
+- [iommu] add IOMMU_EXEC flag for safely allowing XN mappings (Myron Stowe) [1129808]
+- [iommu] acpica: Update DMAR table definitions (Myron Stowe) [1129808]
+
 * Wed Sep 10 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-155.el7]
 - [tools] perf/bench: Fix NULL pointer dereference in "perf bench all" (Jiri Olsa) [1133083]
 - [tools] perf/bench/numa: Make no args mean 'run all tests' (Jiri Olsa) [1133083]
-- 
1.7.1