From bbe8bdc5dc5c3a00158baca6b2e9be5340cdc318 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Mon, 13 Oct 2014 09:32:12 -0400
Subject: [redhat] kernel-3.10.0-188.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 97260f8..d7ba0e8 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 187
+RHEL_RELEASE = 188
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index cceac71..85d841f 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=187
+BUILD:=188
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index fde8832..7307828 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,58 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Mon Oct 13 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-188.el7]
+- [kernel] workqueue: Add quiescent state between work items (Jiri Pirko) [1149250]
+- [crypto] qat: Enable interrupts from all 32 bundles (Nikolay Aleksandrov) [1091130]
+- [char] ipmi: Clear drvdata when interface is removed (Tony Camuso) [1136589]
+- [md] raid5: disable 'DISCARD' by default due to safety concerns (Jes Sorensen) [1149931]
+- [md] raid1: fix_read_error should act on all non-faulty devices (Jes Sorensen) [1149931]
+- [virt] hyperv: NULL dereference on error (Jason Wang) [1146357]
+- [virt] hyperv: Increase the buffer length for netvsc_channel_cb() (Jason Wang) [1146357]
+- [virt] hyperv: Adjust the size of sendbuf region to support ws2008r2 (Jason Wang) [1146357]
+- [virt] hyperv/net: Increase the size of the sendbuf region (Jason Wang) [1146357]
+- [virt] hyperv: Fix error return code in netvsc_init_buf() (Jason Wang) [1146357]
+- [virt] hyperv: Add netpoll support (Jason Wang) [1146357]
+- [virt] hyperv/net: remove unnecessary null test before kfree (Jason Wang) [1146357]
+- [virt] hyperv: Add handler for RNDIS_STATUS_NETWORK_CHANGE event (Jason Wang) [1146357]
+- [virt] hyperv: fix apparent cut-n-paste error in send path teardown (Jason Wang) [1146357]
+- [virt] hyperv: Add hash value into RNDIS Per-packet info (Jason Wang) [1146357]
+- [virt] hyperv/net: Add support for netvsc build without CONFIG_SYSFS flag (Jason Wang) [1146357]
+- [virt] hyperv: Properly handle checksum offload (Jason Wang) [1146357]
+- [virt] hyperv: Enable sendbuf mechanism on the send path (Jason Wang) [1146357]
+- [virt] hyperv: Simplify the send_completion variables (Jason Wang) [1146357]
+- [virt] hyperv: Remove recv_pkt_list and lock (Jason Wang) [1146357]
+- [virt] hyperv: Add support for virtual Receive Side Scaling (vRSS) (Jason Wang) [1146357]
+- [virt] hyperv/net: Address UDP checksum issues (Jason Wang) [1146357]
+- [virt] hyperv/net: Negotiate suitable ndis version for offload support (Jason Wang) [1146357]
+- [virt] hyperv/net: Allocate memory for all possible per-pecket information (Jason Wang) [1146357]
+- [virt] hyperv/net: Enable large send offload (Jason Wang) [1146357]
+- [virt] hyperv/net: Enable send side checksum offload (Jason Wang) [1146357]
+- [virt] hyperv/net: Enable receive side IP checksum offload (Jason Wang) [1146357]
+- [virt] hyperv/net: Enable offloads on the host (Jason Wang) [1146357]
+- [virt] hyperv/net: Cleanup the send path (Jason Wang) [1146357]
+- [virt] hyperv/net: Enable scatter gather I/O (Jason Wang) [1146357]
+- [virt] hyperv: Move state setting for link query (Jason Wang) [1146357]
+- [virt] hyperv: Add latest NetVSP versions to auto negotiation (Jason Wang) [1146357]
+- [virt] hyperv/net: Cleanup the netvsc receive callback functio (Jason Wang) [1146357]
+- [virt] hyperv/net: Cleanup the receive path (Jason Wang) [1146357]
+- [virt] hyperv/net: Get rid of the rndis_filter_packet structure (Jason Wang) [1146357]
+- [virt] hyperv: Fix the carrier status setting (Jason Wang) [1146357]
+- [virt] hyperv/util: Fix a bug in the KVP code (Jason Wang) [1102584]
+- [virt] hyperv/vmbus: Fix a bug in the channel callback dispatch code (Jason Wang) [1146357]
+- [virt] hyperv: remove meaningless pr_err() in vmbus_recvpacket_raw() (Jason Wang) [1146357]
+- [virt] tools/hyperv: Handle the case when the target file exists correctly (Jason Wang) [1146358]
+- [virt] hyperv/hv_fcopy: fix a race condition for SMP guest (Jason Wang) [1146358]
+- [virt] hyperv: fcopy_open() can be static (Jason Wang) [1146358]
+- [virt] hyperv: Implement the file copy service (Jason Wang) [1146358]
+- [virt] hyperv: Add hyperv.h to uapi headers (Jason Wang) [1146358]
+- [virt] hyperv/vmbus: Fix a bug in vmbus_open() (Jason Wang) [1146358]
+- [virt] hyperv/vmbus: Properly protect calls to smp_processor_id() (Jason Wang) [1146358]
+- [virt] hyperv: Remove unnecessary comparison of unsigned against 0 (Jason Wang) [1146358]
+- [virt] hyperv/vmbus: Implement per-CPU mapping of relid to channel (Jason Wang) [1146358]
+- [virt] hyperv: Eliminate the channel spinlock in the callback path (Jason Wang) [1146358]
+- [virt] hyperv/vmbus: Support per-channel driver state (Jason Wang) [1146358]
+
 * Fri Oct 10 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-187.el7]
 - [ethernet] qlcnic: Remove duplicate object file from Makefile (Chad Dupuis) [1089352]
 - [ethernet] qlcnic: Initialize dcbnl_ops before register_netdev (Chad Dupuis) [1089352]
-- 
1.7.1