From 0c5cf87f46ceb43958b03e29372d9a9648d4b59a Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Fri, 7 Nov 2014 13:03:48 -0500
Subject: [redhat] kernel-3.10.0-200.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index c0bef78..ac1417f 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 199
+RHEL_RELEASE = 200
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 3356734..ee234ce 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=199
+BUILD:=200
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 1e07fa5..273bba8 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,96 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Fri Nov 07 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-200.el7]
+- [fs] ext4: fix wrong assert in ext4_mb_normalize_request() (Lukas Czerner) [1146046]
+- [mm] Remove false WARN_ON from pagecache_isize_extended() (Lukas Czerner) [1156096]
+- [fs] ext4: check s_chksum_driver when looking for bg csum presence (Lukas Czerner) [1156096]
+- [fs] ext4: move error report out of atomic context in ext4_init_block_bitmap() (Lukas Czerner) [1156096]
+- [fs] ext4: Replace open coded mdata csum feature to helper function (Lukas Czerner) [1156096]
+- [fs] ext4: fix reservation overflow in ext4_da_write_begin (Lukas Czerner) [1156096]
+- [fs] ext4: add ext4_iget_normal() which is to be used for dir tree lookups (Lukas Czerner) [1156096]
+- [fs] ext4: don't orphan or truncate the boot loader inode (Lukas Czerner) [1156096]
+- [fs] ext4: grab missed write_count for EXT4_IOC_SWAP_BOOT (Lukas Czerner) [1156096]
+- [fs] ext4: get rid of code duplication (Lukas Czerner) [1156096]
+- [fs] ext4: fix over-defensive complaint after journal abort (Lukas Czerner) [1156096]
+- [fs] ext4: fix return value of ext4_do_update_inode (Lukas Czerner) [1156096]
+- [fs] ext4: fix mmap data corruption when blocksize < pagesize (Lukas Czerner) [1156096]
+- [fs] vfs: fix data corruption when blocksize < pagesize for mmaped data (Lukas Czerner) [1156096]
+- [fs] ext4: don't check quota format when there are no quota files (Lukas Czerner) [1156096]
+- [fs] jbd2: avoid pointless scanning of checkpoint lists (Lukas Czerner) [1156096]
+- [fs] ext4: explicitly inform user about orphan list cleanup (Lukas Czerner) [1156096]
+- [fs] jbd2: jbd2_log_wait_for_space improve error detetcion (Lukas Czerner) [1156096]
+- [fs] jbd2: free bh when descriptor block checksum fails (Lukas Czerner) [1156096]
+- [fs] ext4: check EA value offset when loading (Lukas Czerner) [1156096]
+- [fs] ext4: don't keep using page if inline conversion fails (Lukas Czerner) [1156096]
+- [fs] ext4: validate external journal superblock checksum (Lukas Czerner) [1156096]
+- [fs] jbd2: fix journal checksum feature flag handling (Lukas Czerner) [1156096]
+- [fs] ext4: provide separate operations for sysfs feature files (Lukas Czerner) [1156096]
+- [fs] ext4: add sysfs entry showing whether the fs contains errors (Lukas Czerner) [1156096]
+- [fs] ext4: renumber EXT4_EX_* flags to avoid flag aliasing problems (Lukas Czerner) [1156096]
+- [fs] ext4: fix comments about get_blocks (Lukas Czerner) [1156096]
+- [fs] ext4: fix accidental flag aliasing in ext4_map_blocks flags (Lukas Czerner) [1156096]
+- [fs] ext4: fix ZERO_RANGE bug hidden by flag aliasing (Lukas Czerner) [1156096]
+- [fs] ext4: use ext4_update_i_disksize instead of opencoded ones (Lukas Czerner) [1156096]
+- [fs] ext4: remove a duplicate call in ext4_init_new_dir() (Lukas Czerner) [1156096]
+- [fs] ext4: add missing BUFFER_TRACE before ext4_journal_get_write_access (Lukas Czerner) [1156096]
+- [fs] ext4: check inline directory before converting (Lukas Czerner) [1156096]
+- [fs] ext4: fix incorrect locking in move_extent_per_page (Lukas Czerner) [1156096]
+- [fs] ext4: use correct depth value (Lukas Czerner) [1156096]
+- [fs] ext4: add i_data_sem sanity check (Lukas Czerner) [1156096]
+- [fs] ext4: fix wrong size computation in ext4_mb_normalize_request() (Lukas Czerner) [1156096]
+- [fs] ext4: make ext4_has_inline_data() as a inline function (Lukas Czerner) [1156096]
+- [fs] ext4: remove readpage() check in ext4_mmap_file() (Lukas Czerner) [1156096]
+- [fs] ext4: remove metadata reservation checks (Lukas Czerner) [1156096]
+- [fs] ext4: rearrange initialization to fix EXT4FS_DEBUG (Lukas Czerner) [1156096]
+- [fs] ext4: fix potential null pointer dereference in ext4_free_inode (Lukas Czerner) [1156096]
+- [fs] ext4: decrement free clusters/inodes counters when block group declared bad (Lukas Czerner) [1156096]
+- [fs] ext4: handle symlink properly with inline_data (Lukas Czerner) [1156096]
+- [fs] ext4: reduce contention on s_orphan_lock (Lukas Czerner) [1156096]
+- [fs] ext4: use sbi in ext4_orphan_[add|del]() (Lukas Czerner) [1156096]
+- [fs] ext4: remove unnecessary double parentheses (Lukas Czerner) [1156096]
+- [fs] ext4: do not destroy ext4_groupinfo_caches if ext4_mb_init() fails (Lukas Czerner) [1156096]
+- [fs] ext4: make local functions static (Lukas Czerner) [1156096]
+- [fs] ext4: fix block bitmap validation when bigalloc, ^flex_bg (Lukas Czerner) [1156096]
+- [fs] ext4: fix block bitmap initialization under sparse_super2 (Lukas Czerner) [1156096]
+- [fs] ext4: find the group descriptors on a 1k-block bigalloc, meta_bg filesystem (Lukas Czerner) [1156096]
+- [fs] ext4: avoid unneeded lookup when xattr name is invalid (Lukas Czerner) [1156096]
+- [fs] ext4: remove obsoleted check (Lukas Czerner) [1156096]
+- [fs] ext4: add a new spinlock i_raw_lock to protect the ext4's raw inode (Lukas Czerner) [1156096]
+- [fs] ext4: revert Disable punch hole on non-extent mapped files (Lukas Czerner) [1150178]
+- [fs] ext4: fix transaction issues for ext4_fallocate and ext_zero_range (Lukas Czerner) [1150171]
+- [fs] ext4: move i_size, i_disksize update routines to helper function (Lukas Czerner) [1150171]
+- [fs] ext4: fix incorect journal credits reservation in ext4_zero_range (Lukas Czerner) [1150171]
+- [fs] ext4: fix COLLAPSE RANGE test for bigalloc file systems (Lukas Czerner) [1150171]
+- [fs] ext4: fix punch hole on files with indirect mapping (Lukas Czerner) [1150171]
+- [fs] ext4: Fix block zeroing when punching holes in indirect block files (Lukas Czerner) [1150171]
+- [fs] ext4: fix ZERO_RANGE test failure in data journalling (Lukas Czerner) [1150171]
+- [fs] ext4: use EXT_MAX_BLOCKS in ext4_es_can_be_merged() (Lukas Czerner) [1150171]
+- [fs] ext4: rename uninitialized extents to unwritten (Lukas Czerner) [1150171]
+- [fs] ext4: disable COLLAPSE_RANGE for bigalloc (Lukas Czerner) [1150171]
+- [fs] ext4: fix COLLAPSE_RANGE failure with 1KB block size (Lukas Czerner) [1150171]
+- [fs] ext4: use EINVAL if not a regular file in ext4_collapse_range() (Lukas Czerner) [1150171]
+- [fs] ext4: enforce we are operating on a regular file in ext4_zero_range() (Lukas Czerner) [1150171]
+- [fs] ext4: fix extent merging in ext4_ext_shift_path_extents() (Lukas Czerner) [1150171]
+- [fs] ext4: discard preallocations after removing space (Lukas Czerner) [1150171]
+- [fs] ext4: no need to truncate pagecache twice in collapse range (Lukas Czerner) [1150171]
+- [fs] ext4: fix removing status extents in ext4_collapse_range() (Lukas Czerner) [1150171]
+- [fs] ext4: use filemap_write_and_wait_range() correctly in collapse range (Lukas Czerner) [1150171]
+- [fs] ext4: use truncate_pagecache() in collapse range (Lukas Czerner) [1150171]
+- [fs] ext4: always check ext4_ext_find_extent result (Lukas Czerner) [1150171]
+- [fs] ext4: COLLAPSE_RANGE only works on extent-based files (Lukas Czerner) [1150171]
+- [fs] ext4: fix byte order problems introduced by the COLLAPSE_RANGE patches (Lukas Czerner) [1150171]
+- [fs] ext4: disallow all fallocate operation on active swapfile (Lukas Czerner) [1150171]
+- [fs] ext4: move falloc collapse range check into the filesystem methods (Lukas Czerner) [1150171]
+- [fs] ext4: fix COLLAPSE_RANGE test failure in data journalling mode (Lukas Czerner) [1150171]
+- [fs] ext4: remove unneeded test of ret variable (Lukas Czerner) [1150171]
+- [fs] ext4: Introduce FALLOC_FL_ZERO_RANGE flag for fallocate (Lukas Czerner) [1150171]
+- [fs] ext4: Introduce FALLOC_FL_ZERO_RANGE flag for fallocate (Lukas Czerner) [1150171]
+- [fs] ext4: refactor ext4_fallocate code (Lukas Czerner) [1150171]
+- [fs] ext4: Update inode i_size after the preallocation (Lukas Czerner) [1150171]
+- [fs] ext4: Add new flag(FALLOC_FL_COLLAPSE_RANGE) for fallocate (Lukas Czerner) [1150171]
+- [fs] ext4: Add support FALLOC_FL_COLLAPSE_RANGE for fallocate (Lukas Czerner) [1150171]
+
 * Wed Nov 05 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-199.el7]
 - [fs] quota: remove dqptr_sem (Lukas Czerner) [1053111]
 - [fs] quota: simplify remove_inode_dquot_ref() (Lukas Czerner) [1053111]
-- 
1.7.1