From 6665f0bce2ab2505d186db87efbed2072316578d Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 27 Jan 2015 11:34:56 -0500
Subject: [redhat] kernel-3.10.0-227.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index e3c1c41..56fed70 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 226
+RHEL_RELEASE = 227
 RHEL_DRM_VERSION = 3
 RHEL_DRM_PATCHLEVEL = 16
 RHEL_DRM_SUBLEVEL = 7
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 5d5f19e..b544795 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=226
+BUILD:=227
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 87eea71..7465523 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1492,6 +1492,16 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Jan 27 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-227.el7]
+- [fs] ext4: fix overwrite race condition (Jacob Tanenbaum) [1152607] {CVE-2014-8086}
+- [media] ttusb-dec: buffer overflow in ioctl (Alexander Gordeev) [1167116] {CVE-2014-8884}
+- [drm] i915: demote opregion excessive timeout WARN_ONCE to DRM_INFO_ONCE (Rob Clark) [1145627]
+- [drm] i915: further quiet i915 (Rob Clark) [1163074]
+- [scsi] megaraid_sas: endianness related bug fixes and code optimization (Tomas Henzl) [1179748]
+- [s390] crypto: kernel oops at insmod of the z90crypt device driver (Hendrik Brueckner) [1172136]
+- [drm] mgag200: Add command line option to specify preferred depth (Dave Airlie) [1044555]
+- [drm] mgag200: Consolidate depth/bpp handling (Dave Airlie) [1044555]
+
 * Sun Jan 25 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-226.el7]
 - [md] dm-cache: fix problematic dual use of a single migration count variable (Mike Snitzer) [1182665]
 - [md] dm-cache: share cache-metadata object across inactive and active DM tables (Mike Snitzer) [1182665]
-- 
1.7.1