From 9c8a1064975bcf2b6fa66cc0aeb5eb8ce27d5722 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Thu, 4 Sep 2014 12:04:32 -0400
Subject: [redhat] kernel-3.10.0-152.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index a259a68..2e68fb3 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 151
+RHEL_RELEASE = 152
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index bc52f9c..821af38 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=151
+BUILD:=152
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 6e2506a..a578dfd 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1472,6 +1472,67 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Thu Sep 04 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-152.el7]
+- [scsi] scsi_lib: remove the description string in scsi_io_completion() (Maurizio Lombardi) [1045998]
+- [scsi] sd: fix a bug in deriving the FLUSH_TIMEOUT from the basic I/O timeout (Ewan Milne) [1123906]
+- [scsi] scsi_debug: simple short transfer injection (Ewan Milne) [1132100]
+- [scsi] scsi_debug: add ability to enable clustering (Ewan Milne) [1132100]
+- [scsi] scsi_debug: protect device access with atomic_rw lock (Ewan Milne) [1132100]
+- [scsi] scsi_debug: prepare to enable clustering (Ewan Milne) [1132100]
+- [scsi] scsi_debug: fix resp_xdwriteread() return value when running out of memory (Ewan Milne) [1132100]
+- [scsi] scsi_debug: fix duplicate dif_errors increment (Ewan Milne) [1132100]
+- [scsi] scsi_debug: make pseudo_primary static (Ewan Milne) [1132100]
+- [scsi] scsi_debug: fix false positive logical block reference tag check fail (Ewan Milne) [1132100]
+- [scsi] scsi_debug: simplify creation and destruction of driver attribute files (Ewan Milne) [1132100]
+- [scsi] scsi_debug: fix sparse warnings related to data integrity field (Ewan Milne) [1132100]
+- [scsi] scsi_debug: fix invalid value check for guard module parameter (Ewan Milne) [1132100]
+- [scsi] scsi_debug: avoid partial copying PI from prot_sglist to dif_storep (Ewan Milne) [1132100]
+- [scsi] scsi_debug: factor out copying PI from dif_storep to prot_sglist (Ewan Milne) [1132100]
+- [scsi] scsi_debug: fix buffer overrun when DIF/DIX is enabled and virtual_gb > 0 (Ewan Milne) [1132100]
+- [scsi] scsi_debug: fix logical block provisioning support when unmap_alignment != 0 (Ewan Milne) [1132100]
+- [scsi] scsi_debug: fix do_device_access() with wrap around range (Ewan Milne) [1132100]
+- [scsi] lib/scatterlist: introduce sg_pcopy_from_buffer() and sg_pcopy_to_buffer() (Ewan Milne) [1132100]
+- [scsi] lib/scatterlist: factor out sg_miter_get_next_page() from sg_miter_next() (Ewan Milne) [1132100]
+- [scsi] scsi_debug: reduce duplication between prot_verify_read and prot_verify_write (Ewan Milne) [1132100]
+- [scsi] scsi_debug: simplify offset calculation for dif_storep (Ewan Milne) [1132100]
+- [scsi] scsi_debug: invalidate protection info for unmapped region (Ewan Milne) [1132100]
+- [scsi] scsi_debug: fix NULL pointer dereference with parameters dif=0 dix=1 (Ewan Milne) [1132100]
+- [scsi] scsi_debug: fix incorrectly nested kmap_atomic() (Ewan Milne) [1132100]
+- [scsi] scsi_debug: fix invalid address passed to kunmap_atomic() (Ewan Milne) [1132100]
+- [scsi] use dev_printk variants where possible (Ewan Milne) [998695]
+- [scsi] use dev_printk() variants for ioctl (Ewan Milne) [998695]
+- [scsi] Implement st_printk() (Ewan Milne) [998695]
+- [scsi] Implement ch_printk() (Ewan Milne) [998695]
+- [scsi] Implement sg_printk() (Ewan Milne) [998695]
+- [scsi] Implement sr_printk() (Ewan Milne) [998695]
+- [scsi] ses: Use vpd information from scsi_device (Ewan Milne) [1132092]
+- [scsi] Add EVPD page 0x83 and 0x80 to sysfs (Ewan Milne) [1132092]
+- [scsi] Return VPD page length in scsi_vpd_inquiry() (Ewan Milne) [1132092]
+- [scsi] Workaround for disks that report bad optimal transfer length (Ewan Milne) [1132092]
+- [scsi] sg: O_EXCL and other lock handling (Ewan Milne) [1132092]
+- [scsi] sg: add SG_FLAG_Q_AT_TAIL flag (Ewan Milne) [1132092]
+- [scsi] sg: relax 16 byte cdb restriction (Ewan Milne) [1132092]
+- [scsi] scsi constants: command, sense key + additional sense strings (Ewan Milne) [1132092]
+- [scsi] scsi_error: disable eh_deadline if no host_reset_handler is set (Ewan Milne) [1132092]
+- [scsi] cleanup switch in scsi_adjust_queue_depth (Ewan Milne) [1132092]
+- [scsi] set correct completion code in scsi_send_eh_cmnd() (Ewan Milne) [1132092]
+- [scsi] Spelling hsot -> host (Ewan Milne) [1132092]
+- [scsi] scsi_sysfs: Implement 'is_visible' callback (Ewan Milne) [1132092]
+- [scsi] export device_busy for sdev (Ewan Milne) [1132092]
+- [scsi] fc: ensure scan_work isn't active when freeing fc_rport (Ewan Milne) [1132092]
+- [scsi] scsi_transport_fc: Add 32Gbps speed definition (Ewan Milne) [1132092]
+- [scsi] sd: bad return code of init_sd (Ewan Milne) [1132092]
+- [scsi] sd: notify block layer when using temporary change to cache_type (Ewan Milne) [1132092]
+- [scsi] sd: medium access timeout counter fails to reset (Ewan Milne) [1101729]
+- [scsi] sd: Quiesce mode sense error messages (Ewan Milne) [1132092]
+- [scsi] sd: Do not call do_div() with a 64-bit divisor (Ewan Milne) [1132092]
+- [scsi] sd: Reduce buffer size for vpd request (Ewan Milne) [1132092]
+- [scsi] sd: avoid deadlocks when running under multipath (Ewan Milne) [1132092]
+- [scsi] st: fix enlarge_buffer (Ewan Milne) [1132092]
+- [scsi] use the scsi data buffer length to extract transfer size (Ewan Milne) [1132092]
+- [scsi] scsi_cmnd: Introduce scsi_transfer_length helper (Ewan Milne) [1132092]
+- [scsi] Add CDB definition for COMPARE_AND_WRITE (Ewan Milne) [1132092]
+
 * Thu Sep 04 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-151.el7]
 - [md] raid6: avoid data corruption during recovery of double-degraded RAID6 (Jes Sorensen) [1130905]
 - [block] scsi_ioctl: verify return pointer from blk_get_request (Jeff Moyer) [1104324]
-- 
1.7.1