From ce09ac14ed56f5268996162216fe6109e2a69bff Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 23 Sep 2014 03:05:32 -0400
Subject: [redhat] kernel-3.10.0-168.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 2c89a6b..34b5793 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 167
+RHEL_RELEASE = 168
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index b3c6a7e..e0889d9 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=167
+BUILD:=168
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index d15b769..b6e8765 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1476,6 +1476,122 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Sep 23 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-168.el7]
+- [ethernet] bnx2x: Configure device endianity on driver load and reset endianity on removal (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Be more forgiving toward SW GRO (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: VF clean statistics (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Fix stop-on-error (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: ethtool -d might cause timeout in log (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Use is_kdump_kernel() to detect kdump kernel (Michal Schmidt) [1059554 1089402]
+- [ethernet] bnx2x: FW assertion changes (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Make BP_VF more robust (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Prevent pci_disable_sriov with assigned VFs (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Prevent IOV if no entries in CAM (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Fix static checker warning regarding `txdata_ptr' (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Safe bnx2x_panic_dump() (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Update driver version to 1.710.51 (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Code cleanup (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Fix sparse warnings (Michal Schmidt) [1119861 1060671 1089402]
+- [ethernet] bnx2x: Fix timesync endianity (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Add timestamping and PTP hardware clock support (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: fix tunneled GSO over IPv6 (Michal Schmidt) [1119861 1060671 1089402]
+- [ethernet] bnx2x: prevent incorrect byte-swap in BE (Michal Schmidt) [1119861 1060671 1089402]
+- [ethernet] bnx2x: Utilize FW 7.10.51 (Michal Schmidt) [1119861 1060671 1089402]
+- [ethernet] bnx2x: Remove redundant #ifdef (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Remove DEFINE_PCI_DEVICE_TABLE macro use (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: fix set_setting for some PHYs (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: fix possible panic under memory stress (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Fix the MSI flags (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Fail probe of VFs using an old incompatible driver (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: enlarge minimal alignemnt of data offset (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: VF can report link speed (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: remove null test before kfree (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Enlarge the dorq threshold for VFs (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Check for UNDI in uncommon branch (Michal Schmidt) [1094099 1089402]
+- [ethernet] bnx2x: Fix 1G-baseT link (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Fix link for KR with swapped polarity lane (Michal Schmidt) [1123867 1089402]
+- [ethernet] bnx2x: Remove useless return variables (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: update MAINTAINERS for bnx2x and e-mail addresses (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: get rid of SET_ETHTOOL_OPS (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: fix build when BNX2X_SRIOV is not enabled (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Convert return 0 to return rc (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Fix UNDI driver unload (Michal Schmidt) [1094099 1089402]
+- [ethernet] bnx2x: Memory leak during VF removal (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: include irq.h for irqreturn_t definitions (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Fix possible memory leak on iov error flow (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Fix compilation when CONFIG_BNX2X_SRIOV is not set (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Don't allow VFs to become promiscuous (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Don't show port statistics for VFs (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Fix failure to configure VF multicast filters (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Fix vlan credit issues for VFs (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Remove the sriov VFOP mechanism (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Create workqueue for IOV related tasks (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Support mng. request for driver version (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Don't receive packets when the napi budget == 0 (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: save RAM in kdump kernel by disabling TPA (Michal Schmidt) [1059554 1089402]
+- [ethernet] bnx2x: save RAM in kdump kernel by using a single queue (Michal Schmidt) [1059554 1089402]
+- [ethernet] bnx2x: clamp num_queues to prevent passing a negative value (Michal Schmidt) [1059554 1089402]
+- [ethernet] bnx2x: Remove hidden flow control goto from BNX2X_ALLOC macros (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Use pci_enable_msix_range() instead of pci_enable_msix() (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: (semantic) revise scheduling of sp_rtnl (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Fix bnx2x_panic_dump for VFs (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Revise IOV vlan/mac validation (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Add support in PF driver for RSC (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Semantic Validate vlan/mac changes (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Remove unnecessary internal mem config (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Remove unused iov code (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: Debug - change verbosity of some prints (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: fix L2-GRE TCP issues (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: fix sparse warning (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: prevent WARN during driver unload (Michal Schmidt) [1089402]
+- [ethernet] bnx2x: slight optimization of addr compare (Michal Schmidt) [1089402]
+- [kernel] crash_dump: Make is_kdump_kernel() accessible from modules (Michal Schmidt) [1089402 1059554]
+- [ethernet] qlge: Fix TSO for non-accelerated vlan traffic (Vlad Yasevich) [1135619]
+- [ethernet] mvneta: Fix TSO and checksum for non-acceleration vlan traffic (Vlad Yasevich) [1135619]
+- [ethernet] i40evf: Fix TSO and hw checksums for non-accelerated vlan packets (Vlad Yasevich) [1135619]
+- [ethernet] i40e: Fix TSO and hw checksums for non-accelerated vlan packets (Vlad Yasevich) [1135619]
+- [ethernet] ehea: Fix TSO and hw checksums with non-accelerated vlan packets (Vlad Yasevich) [1135619]
+- [ethernet] e1000: Fix TSO for non-accelerated vlan traffic (Vlad Yasevich) [1135619]
+- [ethernet] e1000e: Fix TSO with non-accelerated vlans (Vlad Yasevich) [1135619]
+- [net] macvlan: Initialize vlan_features to turn on offload support (Vlad Yasevich) [1135619]
+- [net] macvtap: Fix checksum errors for non-gso packets in bridge mode (Vlad Yasevich) [1135619]
+- [net] Correctly set segment mac_len in skb_segment() (Vlad Yasevich) [1135619]
+- [crypto] pkcs7: Fix the parser cleanup to drain parsed out X.509 certs (Vivek Goyal) [1129779]
+- [crypto] pkcs7: Provide a single place to do signed info block freeing (Vivek Goyal) [1129779]
+- [security] keys: Reinstate EPERM for a key type name beginning with a '.' (Vivek Goyal) [1129779]
+- [crypto] pkcs7: Add a missing static (Vivek Goyal) [1129779]
+- [crypto] x509: Need to export x509_request_asymmetric_key() (Vivek Goyal) [1129779]
+- [crypto] pkcs7: X.509 certificate issuer and subject are mandatory fields in the ASN.1 (Vivek Goyal) [1129779]
+- [crypto] pkcs7: Use x509_request_asymmetric_key() (Vivek Goyal) [1129779]
+- [crypto] x509: x509_request_asymmetric_keys() doesn't need string length arguments (Vivek Goyal) [1129779]
+- [crypto] pkcs7: fix sparse non static symbol warning (Vivek Goyal) [1129779]
+- [crypto] pkcs7: Missing inclusion of linux/err.h (Vivek Goyal) [1129779]
+- [security] ima: define '.ima' as a builtin 'trusted' keyring (Vivek Goyal) [1129779]
+- [crypto] keys: validate certificate trust only with builtin keys (Vivek Goyal) [1129779]
+- [crypto] keys: validate certificate trust only with selected key (Vivek Goyal) [1129779]
+- [crypto] keys: make partial key id matching as a dedicated function (Vivek Goyal) [1129779]
+- [crypto] keys: verify a certificate is signed by a 'trusted' key (Vivek Goyal) [1129779]
+- [security] keys: special dot prefixed keyring name bug fix (Vivek Goyal) [1129779]
+- [crypto] pefile: Validate PKCS#7 trust chain (Vivek Goyal) [1129779]
+- [crypto] pefile: Digest the PE binary and compare to the PKCS#7 data (Vivek Goyal) [1129779]
+- [crypto] pefile: Handle pesign using the wrong OID (Vivek Goyal) [1129779]
+- [crypto] pefile: Parse the "Microsoft individual code signing" data blob (Vivek Goyal) [1129779]
+- [crypto] pefile: Parse the presumed PKCS#7 content of the certificate blob (Vivek Goyal) [1129779]
+- [crypto] pefile: Strip the wrapper off of the cert data block (Vivek Goyal) [1129779]
+- [crypto] pefile: Parse a PE binary to find a key and a signature contained therein (Vivek Goyal) [1129779]
+- [misc] Provide PE binary definitions (Vivek Goyal) [1129779]
+- [crypto] keys/x509: Fix a spelling mistake (Vivek Goyal) [1129779]
+- [crypto] pkcs7: Provide a key type for testing PKCS#7 (Vivek Goyal) [1129779]
+- [crypto] pkcs7: Find intersection between PKCS#7 message and known, trusted keys (Vivek Goyal) [1129779]
+- [crypto] pkcs7: Verify internal certificate chain (Vivek Goyal) [1129779]
+- [crypto] pkcs7: Find the right key in the PKCS#7 key list and verify the signature (Vivek Goyal) [1129779]
+- [crypto] pkcs7: Digest the data in a signed-data message (Vivek Goyal) [1129779]
+- [crypto] pkcs7: Implement a parser for RFC 2315 (Vivek Goyal) [1129779]
+- [crypto] x509: Export certificate parse and free functions (Vivek Goyal) [1129779]
+- [crypto] x509: Add bits needed for PKCS#7 (Vivek Goyal) [1129779]
+- [crypto] keys: change asymmetric keys to use common hash definitions (Vivek Goyal) [1129779]
+- [crypto] provide single place for hash algo information (Vivek Goyal) [1129779]
+
 * Sun Sep 21 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-167.el7]
 - [fs] ceph: remove extra definitions of U32_MAX (Sage Weil) [1122174]
 - [fs] vfs: call d_op->d_prune() before unhashing dentry (Sage Weil) [1122174]
-- 
1.7.1