From f609c0e57f16a35e056b8411ff0f06f7dad67e7f Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 1 Jul 2014 11:26:52 -0400
Subject: [redhat] kernel-3.10.0-130.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index e7bc1ad..bfcce6c 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 129
+RHEL_RELEASE = 130
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index dc824fe..0a5c5ef 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=129
+BUILD:=130
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index e4ce2cf..10df409 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1471,6 +1471,55 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Jul 01 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-130.el7]
+- [scsi] qla2xxx: Update version number to 8.06.00.08.07.0-k3 (Chad Dupuis) [1090378]
+- [scsi] qla2xxx: Reduce the time we wait for a command to complete during SCSI error handling (Chad Dupuis) [1090378]
+- [scsi] qla2xxx: Clear loop_id for ports that are marked lost during fabric scanning (Chad Dupuis) [1090378]
+- [scsi] qla2xxx: Avoid escalating the SCSI error handler if the command is not found in firmware (Chad Dupuis) [1090378]
+- [scsi] qla2xxx: Don't check for firmware hung during the reset context for ISP82XX (Chad Dupuis) [1090378]
+- [scsi] qla2xxx: Issue abort command for outstanding commands during cleanup when only firmware is alive (Chad Dupuis) [1090378]
+- [scsi] set DID_TIME_OUT correctly (Ewan Milne) [1103881]
+- [scsi] fix invalid setting of host byte (Ewan Milne) [1103881]
+- [scsi] More USB deadlock fixes (Ewan Milne) [1103881]
+- [scsi] Fix USB deadlock caused by SCSI error handling (Ewan Milne) [1103881]
+- [scsi] Fix command result state propagation (Ewan Milne) [1103881]
+- [scsi] Fix spurious request sense in error handling (Ewan Milne) [1103881]
+- [scsi] Add timeout to avoid infinite command retry (Ewan Milne) [1061871]
+- [scsi] lpfc: Update version for 10.2.8021.0 driver release (Rob Evers) [1088574]
+- [scsi] lpfc 10.2.8001.0: Fix ExpressLane priority setup (Rob Evers) [1088574]
+- [scsi] lpfc 10.2.8001.0: Removed obsolete PCI IDs from the driver (Rob Evers) [1088574]
+- [scsi] lpfc 10.2.8001.0: Fix for initializing RRQ bitmap (Rob Evers) [1088574]
+- [scsi] lpfc 10.2.8001.0: Fix for cleaning up stale ring flag and sp_queue_event entries (Rob Evers) [1088574]
+- [scsi] lpfc: Add iotag memory barrier (Rob Evers) [1088574]
+- [scsi] lpfc 10.2.8000.0: Update Copyright on changed files from 8.3.45 patches (Rob Evers) [1088574]
+- [scsi] lpfc 10.2.8000.0: Fixed locking for scsi task management commands (Rob Evers) [1088574]
+- [scsi] lpfc 10.2.8000.0: Convert runtime references to old xlane cfg param to fof cfg param (Rob Evers) [1088574]
+- [scsi] lpfc 10.2.8000.0: Fix FW dump using sysfs (Rob Evers) [1088574]
+- [scsi] lpfc 10.2.8000.0: Fix SLI4 s abort loop to process all FCP rings and under ring_lock (Rob Evers) [1088574]
+- [scsi] lpfc 10.2.8000.0: Fixed kernel panic in lpfc_abort_handler (Rob Evers) [1088574]
+- [scsi] lpfc 10.2.8000.0: Fix locking for postbufq when freeing (Rob Evers) [1088574]
+- [scsi] lpfc: remove self-assignments (Rob Evers) [1088574]
+- [scsi] lpfc: Using plain integer as NULL pointer (Rob Evers) [1088574]
+- [scsi] lpfc 10.2.8000.0: Fix locking for lpfc_hba_down_post (Rob Evers) [1088574]
+- [scsi] lpfc 10.2.8000.0: Fix dynamic transitions of FirstBurst from on to off (Rob Evers) [1088574]
+- [scsi] lpfc 8.3.45: Fixed crash during driver unload (Rob Evers) [1088574]
+- [scsi] lpfc 8.3.45: Fixed driver error messages after firmware download (Rob Evers) [1088574]
+- [scsi] lpfc 8.3.45: Fixed missing initialization for task management IOCBs (Rob Evers) [1088574]
+- [scsi] lpfc 8.3.45: Fix sysfs buffer overrun in read of lpfc_fcp_cpu_map for 128 CPUs (Rob Evers) [1088574]
+- [scsi] lpfc 8.3.45: Incorporate changes to use reason in change_queue_depth function (Rob Evers) [1088574]
+- [scsi] lpfc 8.3.45: Incorporated support of a low-latency io path (Rob Evers) [1088574]
+- [scsi] lpfc 8.3.45: Added dport mailbox pass through support (Rob Evers) [1088574]
+- [scsi] be2iscsi: Bump the driver version (Rob Evers) [1088576]
+- [scsi] be2iscsi: fix bad if expression (Rob Evers) [1088576]
+- [scsi] be2iscsi: fix memory leak in error path (Rob Evers) [1088576]
+- [scsi] be2iscsi: Jump to 'free_memory' is apparently missing (Rob Evers) [1088576]
+- [scsi] be2iscsi: Fix destroy MCC-CQ before MCC-EQ is destroyed (Rob Evers) [1088576]
+- [scsi] be2iscsi: Fix memory corruption in MBX path (Rob Evers) [1088576]
+- [scsi] be2iscsi: Fix TCP parameters while connection offloading (Rob Evers) [1088576]
+- [scsi] be2iscsi: Fix interrupt Coalescing mechanism (Rob Evers) [1088576]
+- [scsi] be2iscsi: Fix exposing Host in sysfs after adapter initialization is complete (Rob Evers) [1088576]
+- [scsi] be2iscsi: Fix retreving MCCQ_WRB in non-embedded Mbox path (Rob Evers) [1088576]
+
 * Tue Jul 01 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-129.el7]
 - [drm] nouveau/bios: fix a bit shift error introduced by recent commit (Ulrich Obergfell) [1089936]
 - [ethernet] sfc: PIO:Restrict to 64bit arch and use 64-bit writes (Nikolay Aleksandrov) [1089024]
-- 
1.7.1