From a9be953c0a3dfc8992ab7e1e7027e5b9d79d3bde Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 21 Oct 2014 16:57:05 -0400
Subject: [redhat] kernel-3.10.0-191.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index d5a853d..ca9592c 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 190
+RHEL_RELEASE = 191
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 1b604b5..4bbaa23 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=190
+BUILD:=191
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index c1b1261..1cc6ae4 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,14 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Oct 21 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-191.el7]
+- [fs] fuse: ignore entry-timeout LOOKUP_REVAL (Carlos Maiolino) [1124880]
+- [fs] fuse: Timeout comparison fix (Carlos Maiolino) [1128802]
+- [fs] lockd: Try to reconnect if statd has moved (Benjamin Coddington) [1120850]
+- [fs] sunrpc: Don't wake tasks during connection abort (Benjamin Coddington) [1120850]
+- [fs] nfsd4: fix corruption of NFSv4 read data ("J. Bruce Fields") [1148055]
+- [fs] gfs2: revert "GFS2: d_splice_alias() can't return error" (Robert S Peterson) [1141899]
+
 * Mon Oct 20 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-190.el7]
 - [scsi] mpt3sas: Use pci_enable_msix_exact() instead of pci_enable_msix() (Tomas Henzl) [1088524]
 - [scsi] mpt2sas: Use pci_enable_msix_exact() instead of pci_enable_msix() (Tomas Henzl) [1088524]
-- 
1.7.1