From 7726d6a473fbdd1df56d75a92e9f4a322508cbe4 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 20 May 2014 13:54:23 -0400
Subject: [redhat] kernel-3.10.0-124.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index cb3ea13..420aedf 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 123
+RHEL_RELEASE = 124
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 9fb2317..cecb2ce 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=123
+BUILD:=124
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 11a127d..7eff2d3 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1465,6 +1465,15 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue May 20 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-124.el7]
+- [tty] n_tty: Fix n_tty_write crash when echoing in raw mode (Aristeu Rozanski) [1094242] {CVE-2014-0196}
+- [fs] gfs2: Fix uninitialized VFS inode in gfs2_create_inode (Abhijith Das) [1087995]
+- [kernel] cputime: Fix jiffies based cputime assumption on steal accounting (Frederic Weisbecker) [1047732]
+- [kernel] cputime: Bring cputime -> nsecs conversion (Frederic Weisbecker) [1047732]
+- [kernel] cputime: Default implementation of nsecs -> cputime conversion (Frederic Weisbecker) [1047732]
+- [x86] irq, pic: Probe for legacy PIC and set legacy_pic appropriately (Vivek Goyal) [1037957]
+- [virt] hyperv/vmbus: Negotiate version 3.0 when running on ws2012r2 hosts (Vivek Goyal) [1037957]
+
 * Mon May 05 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-123.el7]
 - [mm] rmap: try_to_unmap_cluster() should lock_page() before mlocking (Larry Woodman) [1078349] {CVE-2014-3122}
 - [mm] huge_memory: Fix cpuset cgroups so all pages for a task remain on correct node (Larry Woodman) [1076613]
-- 
1.7.1