From f4f0d9b2329571ee01d35da82da6b322cdd98480 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Fri, 6 Jun 2014 12:36:32 -0400
Subject: [redhat] kernel-3.10.0-127.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 6ee1270..6cb37bd 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 126
+RHEL_RELEASE = 127
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 3c56b81..07068eb 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=126
+BUILD:=127
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index e069396..1349fe7 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1471,6 +1471,18 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Fri Jun 06 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-127.el7]
+- [net] mac80211: fix crash due to AP powersave TX vs. wakeup race (Jacob Tanenbaum) [1083534] {CVE-2014-2706}
+- [ethernet] qlcnic: Fix ethtool statistics length calculation (Michal Schmidt) [1099634]
+- [kernel] futex: Make lookup_pi_state more robust (Larry Woodman) [1104520] {CVE-2014-3153}
+- [kernel] futex: Always cleanup owner tid in unlock_pi (Larry Woodman) [1104520] {CVE-2014-3153}
+- [kernel] futex: Validate atomic acquisition in futex_lock_pi_atomic() (Larry Woodman) [1104520] {CVE-2014-3153}
+- [kernel] futex: prevent requeue pi on same futex (Larry Woodman) [1104520] {CVE-2014-3153}
+- [net] gro: restore frag0 optimization (and fix crash) (Michal Schmidt) [1069741]
+- [net] ipv4: current group_info should be put after using (Jiri Benc) [1087416] {CVE-2014-2851}
+- [net] core, nfqueue, openvswitch: Orphan frags in skb_zerocopy and handle errors (Jiri Pirko) [1079014] {CVE-2014-2568}
+- [fs] aio: fix plug memory disclosure and fix reqs_active accounting backport (Jeff Moyer) [1094605] {CVE-2014-0206}
+
 * Tue Jun 03 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-126.el7]
 - [fs] aio: plug memory disclosure and fix reqs_active accounting (Mateusz Guzik) [1094605] {CVE-2014-0206}
 - [kernel] mutexes: Give more informative mutex warning in the !lock->owner case (Larry Woodman) [1087655 1087919 1087922]
-- 
1.7.1