From 545da86d7af8d8668757f769698142b58f425818 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Wed, 2 Jul 2014 13:45:07 -0400
Subject: [redhat] kernel-3.10.0-131.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index bfcce6c..b39b90d 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 130
+RHEL_RELEASE = 131
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 0a5c5ef..716c1a1 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=130
+BUILD:=131
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 10df409..a0f489d 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1471,6 +1471,17 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Wed Jul 02 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-131.el7]
+- [scsi] pm80xx: Spinlock fix (Rich Bono) [1044636]
+- [scsi] aacraid: Update unsupported card list (Rich Bono) [1080712]
+- [scsi] vmw_pvscsi: Some improvements in pvscsi driver (Ewan Milne) [1075090]
+- [scsi] vmw_pvscsi: Add support for I/O requests coalescing (Ewan Milne) [1075090]
+- [scsi] vmw_pvscsi: Fix pvscsi_abort() function (Ewan Milne) [1075090]
+- [net] sctp: Fix sk_ack_backlog wrap-around problem (Daniel Borkmann) [1112726] {CVE-2014-4667}
+- [net] ipv4/tunnels: fix an oops when using ipip/sit with IPsec (Jiri Pirko) [1108857]
+- [net] filter: let bpf_tell_extensions return SKF_AD_MAX (Jiri Benc) [1079524]
+- [net] filter: introduce SO_BPF_EXTENSIONS (Jiri Benc) [1079524]
+
 * Tue Jul 01 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-130.el7]
 - [scsi] qla2xxx: Update version number to 8.06.00.08.07.0-k3 (Chad Dupuis) [1090378]
 - [scsi] qla2xxx: Reduce the time we wait for a command to complete during SCSI error handling (Chad Dupuis) [1090378]
-- 
1.7.1