From d756991c1ff4acd52b98f4520979d349ed728ad9 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Fri, 3 Oct 2014 15:34:58 -0400
Subject: [redhat] kernel-3.10.0-180.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 5418312..9b60d40 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 179
+RHEL_RELEASE = 180
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index b82f0cc..d2aa2f5 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=179
+BUILD:=180
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index f54342f..9241d3a 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1483,6 +1483,130 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Fri Oct 03 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-180.el7]
+- [net] ethtool: Check that reserved fields of struct ethtool_rxfh are 0 (Ivan Vecera) [1138823]
+- [net] ethtool: Replace ethtool_ops::{get, set}_rxfh_indir() with {get, set}_rxfh() (Ivan Vecera) [1138823]
+- [net] ethtool: constify array pointer parameters to ethtool_ops::set_rxfh (Ivan Vecera) [1138823]
+- [net] ethtool: Disallow ETHTOOL_SRSSH with both indir table and hash key unchanged (Ivan Vecera) [1138823]
+- [net] ethtool: Expand documentation of ethtool_ops::{get, set}_rxfh() (Ivan Vecera) [1138823]
+- [net] ethtool: Improve explanation of the two arrays following struct ethtool_rxfh (Ivan Vecera) [1138823]
+- [net] ethtool: Name the 'no change' value for setting RSS hash key but not indir table (Ivan Vecera) [1138823]
+- [net] ethtool: Return immediately on error in ethtool_copy_validate_indir() (Ivan Vecera) [1138823]
+- [net] ethtool: exit the loop when invalid index occurs (Ivan Vecera) [1138823]
+- [net] ethtool: Support for configurable RSS hash key (Ivan Vecera) [1138823]
+- [lib] reciprocal_divide: update/correction of the algorithm (Jiri Benc) [1140665]
+- [net] introduce reciprocal_scale helper and convert users (Jiri Benc) [1140665]
+- [net] random32: add prandom_u32_max and convert open coded users (Jiri Benc) [1140665]
+- [net] bpf: do not use reciprocal divide (Jiri Benc) [1140665]
+- [net] genetlink: remove superfluous assignment (Jiri Benc) [1140656]
+- [scsi] pmcraid: use proper genetlink multicast API (Jiri Benc) [1140656]
+- [net] genetlink: Fix uninitialized variable in genl_validate_assign_mc_groups() (Jiri Benc) [1140656]
+- [net] genetlink: fix genlmsg_multicast() bug (Jiri Benc) [1140656]
+- [net] genetlink: make multicast groups const, prevent abuse (Jiri Benc) [1140656]
+- [net] genetlink: pass family to functions using groups (Jiri Benc) [1140656]
+- [net] genetlink: add and use genl_set_err() (Jiri Benc) [1140656]
+- [net] genetlink: remove family pointer from genl_multicast_group (Jiri Benc) [1140656]
+- [net] genetlink: remove genl_unregister_mc_group() (Jiri Benc) [1140656]
+- [fs] quota: use proper genetlink multicast APIs (Jiri Benc) [1140656]
+- [net] drop_monitor: use proper genetlink multicast APIs (Jiri Benc) [1140656]
+- [net] genetlink: only pass array to genl_register_family_with_ops() (Jiri Benc) [1140656]
+- [net] genetlink: rename shadowed variable (Jiri Benc) [1140656]
+- [net] genetlink: unify registration functions (Jiri Benc) [1140656]
+- [net] genetlink: make genl_ops flags a u8 and move to end (Jiri Benc) [1140656]
+- [net] genetlink: make all genl_ops users const (Jiri Benc) [1140656]
+- [net] genetlink: allow making ops const (Jiri Benc) [1140656]
+- [net] genetlink: register family ops as array (Jiri Benc) [1140656]
+- [net] genetlink: remove genl_register_ops/genl_unregister_ops (Jiri Benc) [1140656]
+- [net] ieee802154: use genl_register_family_with_ops() (Jiri Benc) [1140656]
+- [kernel] taskstats: use genl_register_family_with_ops() (Jiri Benc) [1140656]
+- [net] genetlink: Remove extern from function prototypes (Jiri Benc) [1140656]
+- [net] genetlink: fix usage of NLM_F_EXCL or NLM_F_REPLACE (Jiri Benc) [1140656]
+- [net] udp: Fix inverted NAPI_GRO_CB(skb)->flush test (Florian Westphal) [1131999]
+- [net] gre: gro: Fix a bug that breaks the forwarding path (Florian Westphal) [1131999]
+- [net] gro: reset skb->truesize in napi_reuse_skb() (Florian Westphal) [1131999]
+- [net] netdev_features: work around NETIF_F kabi breakage (Florian Westphal) [1131999]
+- [net] l2tp: Enable checksum unnecessary conversions for l2tp/UDP sockets (Florian Westphal) [1131999]
+- [net] vxlan: Enable checksum unnecessary conversions for vxlan/UDP sockets (Florian Westphal) [1131999]
+- [net] gre: Add support for checksum unnecessary conversions (Florian Westphal) [1131999]
+- [net] udp: Add support for doing checksum unnecessary conversion (Florian Westphal) [1131999]
+- [net] Infrastructure for checksum unnecessary conversions (Florian Westphal) [1131999]
+- [net] Support for csum_bad in skbuff (Florian Westphal) [1131999]
+- [net] sctp: Change sctp to implement csum_levels (Florian Westphal) [1131999]
+- [net] Allow GRO to use and set levels of checksum unnecessary (Florian Westphal) [1131999]
+- [net] Clarification of CHECKSUM_UNNECESSARY (Florian Westphal) [1131999]
+- [net] skbuff: improve comment on checksumming (Florian Westphal) [1131999]
+- [net] skbuff: Use ALIGN macro instead of open coding it (Florian Westphal) [1131999]
+- [net] Allocate a new 16 bits for flags in skbuff (Florian Westphal) [1131999]
+- [net] gre: When GRE csum is present count as encap layer wrt csum (Florian Westphal) [1131999]
+- [net] udp: additional GRO support (Florian Westphal) [1131999]
+- [net] tcp: Call skb_gro_checksum_validate (Florian Westphal) [1131999]
+- [net] ipv6: gro: fix CHECKSUM_COMPLETE support (Florian Westphal) [1131999]
+- [net] gre: call skb_gro_checksum_simple_validate (Florian Westphal) [1131999]
+- [net] add gro_compute_pseudo functions (Florian Westphal) [1131999]
+- [net] skb_gro_checksum_* functions (Florian Westphal) [1131999]
+- [net] fix setting csum_start in skb_segment() (Florian Westphal) [1131999]
+- [net] vxlan: Checksum fixes (Florian Westphal) [1131999]
+- [net] add skb_pop_rcv_encapsulation (Florian Westphal) [1131999]
+- [net] udp: call __skb_checksum_complete when doing full checksum (Florian Westphal) [1131999]
+- [net] Fix save software checksum complete (Florian Westphal) [1131999]
+- [net] Fix GSO constants to match NETIF flags (Florian Westphal) [1131999]
+- [net] Add skb_gro_postpull_rcsum to udp and vxlan (Florian Westphal) [1131999]
+- [net] Save software checksum complete (Florian Westphal) [1131999]
+- [net] Preserve CHECKSUM_COMPLETE at validation (Florian Westphal) [1131999]
+- [net] vxlan: Add support for UDP checksums (v4 sending, v6 zero csums) (Florian Westphal) [1131999]
+- [net] gre: Call gso_make_checksum (Florian Westphal) [1131999]
+- [net] gre: simplify GRE header length calculation in gre_gso_segment() (Florian Westphal) [1131999]
+- [net] Add GSO support for UDP tunnels with checksum (Florian Westphal) [1131999]
+- [net] tcp: Call gso_make_checksum (Florian Westphal) [1131999]
+- [net] Support for multiple checksums with gso (Florian Westphal) [1131999]
+- [net] l2tp: call udp{6}_set_csum (Florian Westphal) [1131999]
+- [net] udp: Generic functions to set checksum (Florian Westphal) [1131999]
+- [net] l2tp: Add support for zero IPv6 checksums (Florian Westphal) [1131999]
+- [net] udp: Make enabling of zero UDP6 csums more restrictive (Florian Westphal) [1131999]
+- [net] Split sk_no_check into sk_no_check_{rx, tx} (Florian Westphal) [1131999]
+- [net] Eliminate no_check from protosw (Florian Westphal) [1131999]
+- [net] sunrpc: Remove sk_no_check setting (Florian Westphal) [1131999]
+- [net] l2tp: Remove UDP checksum verification (Florian Westphal) [1131999]
+- [net] udp: Verify UDP checksum before handoff to encap (Florian Westphal) [1131999]
+- [net] icmp6: Call skb_checksum_validate (Florian Westphal) [1131999]
+- [net] icmp: Call skb_checksum_simple_validate (Florian Westphal) [1131999]
+- [net] igmp: Call skb_checksum_simple_validate (Florian Westphal) [1131999]
+- [net] gre6: Call skb_checksum_simple_validate (Florian Westphal) [1131999]
+- [net] gre: Call skb_checksum_simple_validate (Florian Westphal) [1131999]
+- [net] ipv6: Need to sock_put on csum error (Florian Westphal) [1131999]
+- [net] ipv6: Implmement RFC 6936 (zero RX csums for UDP/IPv6) (Florian Westphal) [1131999]
+- [net] udp: Only allow busy read/poll on connected sockets (Florian Westphal) [1078978 1131999]
+- [net] ipv6: log src and dst along with "udp checksum is 0" (Florian Westphal) [1131999]
+- [net] ipv6: Call skb_checksum_init in IPv6 (Florian Westphal) [1131999]
+- [net] ipv4: Call skb_checksum_init in IPv4 (Florian Westphal) [1131999]
+- [net] Generalize checksum_init functions (Florian Westphal) [1131999]
+- [net] Change x86_64 add32_with_carry to allow memory operand (Florian Westphal) [1131999]
+- [x86_64] csum_add for x86_64 (Florian Westphal) [1131999]
+- [net] Allow csum_add to be provided in arch (Florian Westphal) [1131999]
+- [net] ipv4: add a sock pointer to dst->output() path (Jiri Pirko) [1081956]
+- [net] iptunnels: remove net arg from iptunnel_xmit() (Jiri Pirko) [1081956]
+- [net] vti: Use the tunnel mark for lookup in the error handlers (Jiri Pirko) [1091561]
+- [net] xfrm4: Remove duplicate semicolon (Jiri Pirko) [1091561]
+- [net] xfrm4: Properly handle unsupported protocols (Jiri Pirko) [1091561]
+- [net] vti4: Don't count header length twice (Jiri Pirko) [1091561]
+- [net] vti4: don't allow to add the same tunnel twice (Jiri Pirko) [1091561]
+- [net] vti4: Enable namespace changing (Jiri Pirko) [1091561]
+- [net] vti4: Check the tunnel endpoints of the xfrm state and the vti interface (Jiri Pirko) [1091561]
+- [net] vti4: Support inter address family tunneling (Jiri Pirko) [1091561]
+- [net] vti4: Use the on xfrm_lookup returned dst_entry directly (Jiri Pirko) [1091561]
+- [net] vti4: Update the ipv4 side to use it's own receive hook (Jiri Pirko) [1091561]
+- [net] skb: allow skb_scrub_packet() to be used by tunnels (Jiri Pirko) [1091561]
+- [net] ipv4: xfrm: Introduce xfrm_tunnel_notifier for xfrm tunnel mode callback (Jiri Pirko) [1091561]
+- [net] ip_tunnel: Remove double unregister of the fallback device (Jiri Pirko) [1091561]
+- [net] ipip: add x-netns support (Jiri Pirko) [1091561]
+- [net] vti4: switch to new ip tunnel code (Jiri Pirko) [1091561]
+- [net] ip_tunnel: Make vti work with i_key set (Jiri Pirko) [1091561]
+- [net] xfrm: Add xfrm_tunnel_skb_cb to the skb common buffer (Jiri Pirko) [1091561]
+- [net] ipcomp4: Use the IPsec protocol multiplexer API (Jiri Pirko) [1091561]
+- [net] ah4: Use the IPsec protocol multiplexer API (Jiri Pirko) [1091561]
+- [net] esp4: Use the IPsec protocol multiplexer API (Jiri Pirko) [1091561]
+- [net] xfrm4: Add IPsec protocol multiplexer (Jiri Pirko) [1091561]
+
 * Fri Oct 03 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-179.el7]
 - [drm] i915: don't crash if unable to setup stolen (Rob Clark) [1128939]
 - [fs] nfs: Don't reset pg_moreio in __nfs_pageio_add_request (Steve Dickson) [1120728]
-- 
1.7.1