From f9fad0fc6ceaa50df98526decf0053419fb722a2 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 7 Oct 2014 10:01:23 -0400
Subject: [redhat] kernel-3.10.0-182.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 62067ae..8cb36e9 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 181
+RHEL_RELEASE = 182
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index afe2df4..8a9a15d 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=181
+BUILD:=182
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 4d79304..47ed68c 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,62 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Oct 07 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-182.el7]
+- [ethernet] qlge: Remove DEFINE_PCI_DEVICE_TABLE macro use (Chad Dupuis) [1089357]
+- [ethernet] qlge: get rid of SET_ETHTOOL_OPS (Chad Dupuis) [1089357]
+- [ethernet] qlge: Convert /n to \n (Chad Dupuis) [1089357]
+- [ethernet] qlge: remove open-coded skb_cow_head (Chad Dupuis) [1089357]
+- [ethernet] qlge: Do not propaged vlan tag offloads to vlans (Chad Dupuis) [1089357]
+- [ethernet] qlge: Get rid of an redundant assignment (Chad Dupuis) [1089357]
+- [ethernet] qlge: delete non-required instances of include <linux/init.h> (Chad Dupuis) [1089357]
+- [ethernet] qlge: Fix vlan netdev features (Chad Dupuis) [1089357]
+- [ethernet] qlge: make local function static (Chad Dupuis) [1089357]
+- [ethernet] qlge: Update version to 1.00.00.34 (Chad Dupuis) [1089357]
+- [ethernet] qlge: Allow enable/disable rx/tx vlan acceleration independently (Chad Dupuis) [1089357]
+- [ethernet] qlge: Fix ethtool statistics (Chad Dupuis) [1089357]
+- [ethernet] qlge: remove unnecessary pci_set_drvdata() (Chad Dupuis) [1089357]
+- [ethernet] qlge: Update version to 1.00.00.33 (Chad Dupuis) [1089357]
+- [ethernet] qlge: Enhance nested VLAN (Q-in-Q) handling (Chad Dupuis) [1089357]
+- [ethernet] qlge: call ql_core_dump() only if dump memory was allocated (Chad Dupuis) [1089357]
+- [kernel] audit: allow user processes to log from another PID namespace (Richard Guy Briggs) [1010455]
+- [kernel] audit: anchor all pid references in the initial pid namespace (Richard Guy Briggs) [1010455]
+- [kernel] audit: convert PPIDs to the inital PID namespace (Richard Guy Briggs) [1010455]
+- [kernel] sched: declare pid_alive as inline (Richard Guy Briggs) [1010455]
+- [kernel] pid: get pid_t ppid of task in init_pid_ns (Richard Guy Briggs) [1010455]
+- [net] netlabel: shorter names for the NetLabel catmap funcs/structs (Paul Moore) [1119662]
+- [net] netlabel: fix the catmap walking functions (Paul Moore) [1119662]
+- [net] netlabel: fix the horribly broken catmap functions (Paul Moore) [1119662]
+- [net] netlabel: fix a problem when setting bits below the previously lowest bit (Paul Moore) [1119662]
+- [kernel] sched/time: fix lock inversion in thread_group_cputime (Rik van Riel) [1120307]
+- [kernel] seqlock: add irqsave variant of read_seqbegin_or_lock (Rik van Riel) [1120307]
+- [kernel] sched: make lockless sys_times kABI-friendly (Rik van Riel) [1120307]
+- [kernel] sched/cputime: atomically increment stime & utime (Rik van Riel) [1120307]
+- [kernel] time, signal: protect resource use statistics with seqlock (Rik van Riel) [1120307]
+- [kernel] exit: always reap resource stats in __exit_signal (Rik van Riel) [1120307]
+- [kernel] seqcount: Provide raw_read_seqcount() (Rik van Riel) [1120307]
+- [kernel] take read_seqbegin_or_lock() and friends to seqlock.h (Rik van Riel) [1120307]
+- [kernel] sched: change thread_group_cputime() to use for_each_thread() (Rik van Riel) [1120307]
+- [usb] usb3503: Fix a warning for using an uninitialized variable (Don Zickus) [1148954]
+- [ethernet] tg3: prevent ifup/ifdown during PCI error recovery (Ivan Vecera) [1088637]
+- [ethernet] tg3: fix return value in tg3_get_stats64 (Ivan Vecera) [1088637]
+- [ethernet] tg3: Remove DEFINE_PCI_DEVICE_TABLE macro use (Ivan Vecera) [1088637]
+- [ethernet] tg3: Modify tg3_tso_bug() to handle multiple TX rings (Ivan Vecera) [1088637]
+- [ethernet] tg3: Clear NETIF_F_TSO6 flag before doing software GSO (Ivan Vecera) [1088637]
+- [ethernet] tg3: Replace ethtool_ops::{get, set}_rxfh_indir() with {get, set}_rxfh() (Ivan Vecera) [1088637]
+- [ethernet] tg3: Update copyright and version to 3.137 (Ivan Vecera) [1088637]
+- [ethernet] tg3: Prevent page allocation failure during TSO workaround (Ivan Vecera) [1088637]
+- [ethernet] tg3: Don't modify ip header fields when doing GSO (Ivan Vecera) [1088637]
+- [ethernet] tg3: update rx_jumbo_pending ring param only when jumbo frames are enabled (Ivan Vecera) [1088637]
+- [ethernet] tg3: remove open-coded skb_cow_head (Ivan Vecera) [1088637]
+- [ethernet] tg3: remove empty MDIO bus reset function (Ivan Vecera) [1088637]
+- [ethernet] tg3: Do not include vlan acceleration features in vlan_features (Ivan Vecera) [1088637]
+- [ethernet] tg3: Call dev_kfree_skby_any instead of dev_kfree_skb (Ivan Vecera) [1088637]
+- [ethernet] tg3: Use pci_enable_msix_range() instead of pci_enable_msix() (Ivan Vecera) [1088637]
+- [ethernet] tg3: Fix deadlock in tg3_change_mtu() (Ivan Vecera) [1088637]
+- [ethernet] tg3: cleanup an error path in tg3_phy_reset_5703_4_5() (Ivan Vecera) [1088637]
+- [ethernet] tg3: delete non-required instances of include <linux/init.h> (Ivan Vecera) [1088637]
+- [ethernet] tg3: Implement the SIOCGHWTSTAMP ioctl (Ivan Vecera) [1088637]
+
 * Mon Oct 06 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-181.el7]
 - [tools] perf/powerpc: Adjust callchain based on DWARF debug info (Jiri Olsa) [1113736]
 - [tools] perf: Allow to specify lib compile variable for spec usage (Jiri Olsa) [879133]
-- 
1.7.1