From 12546b8649c307cea7d7502f7a0b79f43ac9e8b2 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Mon, 8 Dec 2014 09:10:33 -0500
Subject: [redhat] kernel-3.10.0-214.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 0a78c62..a8f3d26 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 213
+RHEL_RELEASE = 214
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 76dc2ae..ad09123 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=213
+BUILD:=214
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 578250e..0b74afa 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,72 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Mon Dec 08 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-214.el7]
+- [powerpc] Drop useless warning in eeh_init() (Gustavo Duarte) [1156651]
+- [powerpc] pseries: Decrease message level on EEH initialization (Gustavo Duarte) [1156651]
+- [net] ceph: fixup includes in pagelist.h (Ilya Dryomov) [1165232]
+- [net] ceph: change from BUG to WARN for __remove_osd() asserts (Ilya Dryomov) [1165232]
+- [net] ceph: clear r_req_lru_item in __unregister_linger_request() (Ilya Dryomov) [1165232]
+- [net] ceph: unlink from o_linger_requests when clearing r_osd (Ilya Dryomov) [1165232]
+- [net] ceph: do not crash on large auth tickets (Ilya Dryomov) [1165232]
+- [fs] ceph: fix flush tid comparision (Ilya Dryomov) [1165232]
+- [net] ceph: eliminate unnecessary allocation in process_one_ticket() (Ilya Dryomov) [1165232]
+- [block] rbd: Fix error recovery in rbd_obj_read_sync() (Ilya Dryomov) [1165232]
+- [net] ceph: use memalloc flags for net IO (Ilya Dryomov) [1165232]
+- [block] rbd: use a single workqueue for all devices (Ilya Dryomov) [1165232]
+- [fs] ceph: fix divide-by-zero in __validate_layout() (Ilya Dryomov) [1165232]
+- [block] rbd: rbd workqueues need a resque worker (Ilya Dryomov) [1165232]
+- [net] ceph: ceph-msgr workqueue needs a resque worker (Ilya Dryomov) [1165232]
+- [fs] ceph: fix bool assignments (Ilya Dryomov) [1165232]
+- [net] ceph: separate multiple ops with commas in debugfs output (Ilya Dryomov) [1165232]
+- [net] ceph: sync osd op definitions in rados.h (Ilya Dryomov) [1165232]
+- [net] ceph: remove redundant declaration (Ilya Dryomov) [1165232]
+- [fs] ceph: additional debugfs output (Ilya Dryomov) [1165232]
+- [fs] ceph: export ceph_session_state_name function (Ilya Dryomov) [1165232]
+- [fs] ceph: use pagelist to present MDS request data (Ilya Dryomov) [1165232]
+- [net] ceph: reference counting pagelist (Ilya Dryomov) [1165232]
+- [fs] ceph: fix llistxattr on symlink (Ilya Dryomov) [1165232]
+- [fs] ceph: send client metadata to MDS (Ilya Dryomov) [1165232]
+- [fs] ceph: remove redundant code for max file size verification (Ilya Dryomov) [1165232]
+- [fs] ceph: move ceph_find_inode() outside the s_mutex (Ilya Dryomov) [1165232]
+- [fs] ceph: request xattrs if xattr_version is zero (Ilya Dryomov) [1165232]
+- [block] rbd: set the remaining discard properties to enable support (Ilya Dryomov) [1165232]
+- [block] rbd: use helpers to handle discard for layered images correctly (Ilya Dryomov) [1165232]
+- [block] rbd: extract a method for adding object operations (Ilya Dryomov) [1165232]
+- [block] rbd: make discard trigger copy-on-write (Ilya Dryomov) [1165232]
+- [block] rbd: tolerate -ENOENT for discard operations (Ilya Dryomov) [1165232]
+- [block] rbd: fix snapshot context reference count for discards (Ilya Dryomov) [1165232]
+- [block] rbd: read image size for discard check safely (Ilya Dryomov) [1165232]
+- [block] rbd: initial discard bits (Ilya Dryomov) [1165232]
+- [block] rbd: extend the operation type (Ilya Dryomov) [1165232]
+- [block] rbd: skip the copyup when an entire object writing (Ilya Dryomov) [1165232]
+- [block] rbd: add img_obj_request_simple() helper (Ilya Dryomov) [1165232]
+- [block] rbd: access snapshot context and mapping size safely (Ilya Dryomov) [1165232]
+- [block] rbd: do not return -ERANGE on auth failures (Ilya Dryomov) [1165232]
+- [net] ceph: don't try checking queue_work() return value (Ilya Dryomov) [1165232]
+- [fs] ceph: make sure request isn't in any waiting list when kicking request (Ilya Dryomov) [1165232]
+- [fs] ceph: protect kick_requests() with mdsc->mutex (Ilya Dryomov) [1165232]
+- [net] ceph: Convert pr_warning to pr_warn (Ilya Dryomov) [1165232]
+- [fs] ceph: trim unused inodes before reconnecting to recovering MDS (Ilya Dryomov) [1165232]
+- [net] ceph: fix a use after free issue in osdmap_set_max_osd (Ilya Dryomov) [1165232]
+- [net] ceph: select CRYPTO_CBC in addition to CRYPTO_AES (Ilya Dryomov) [1165232]
+- [net] ceph: resend lingering requests with a new tid (Ilya Dryomov) [1165232]
+- [net] ceph: abstract out ceph_osd_request enqueue logic (Ilya Dryomov) [1165232]
+- [block] rbd: fix error return code in rbd_dev_device_setup() (Ilya Dryomov) [1165232]
+- [block] rbd: avoid format-security warning inside alloc_workqueue() (Ilya Dryomov) [1165232]
+- [kernel] printk/register_console: prevent adding the same console twice (Artem Savkov) [1169766]
+- [mm] hugetlb: add cond_resched_lock() in, return_unused_surplus_pages() (Motohiro Kosaki) [1142698]
+- [mm] hugetlb: fix softlockup when a large number of, hugepages are freed (Motohiro Kosaki) [1142698]
+- [kernel] sched: Use new KABI macros (Don Zickus) [1164383]
+- [net] Use new KABI macros (Don Zickus) [1164383]
+- [scsi] Use new KABI macros (Don Zickus) [1164383]
+- [kernel] Use new KABI macros (Don Zickus) [1164383]
+- [block] Use new KABI macros (Don Zickus) [1164383]
+- [block] include: Use new KABI macros (Don Zickus) [1164383]
+- [misc] Use new KABI macros (Don Zickus) [1164383]
+- [x86] Use new KABI macros (Don Zickus) [1164383]
+- [powerpc] Use new KABI macros (Don Zickus) [1164383]
+
 * Wed Dec 03 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-213.el7]
 - [scsi] ipr: don't log error messages when applications issues illegal requests (Gustavo Duarte) [1163019]
 - [net] macvlan: Allow setting multicast filter on all macvlan types (Vlad Yasevich) [848197]
-- 
1.7.1