From c8c14d0e31d884c144f8a30a93c79de00de9f2c2 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 2 Sep 2014 13:42:59 -0400
Subject: [redhat] kernel-3.10.0-150.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index a813f0d..96e4d83 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 149
+RHEL_RELEASE = 150
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 86bc4df..27398fe 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=149
+BUILD:=150
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 66fa0ac..dfbff70 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1472,6 +1472,41 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Sep 02 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-150.el7]
+- [net] team: set IFF_TEAM_PORT priv_flag after rx_handler is registered (Jiri Pirko) [1132943]
+- [net] ipv6: fix calculation of option len in ip6_append_data (Hannes Frederic Sowa) [1127218]
+- [net] ipv6: pmtudisc setting not respected with UFO/CORK (Hannes Frederic Sowa) [1127218]
+- [net] ipv4: fix DO and PROBE pmtu mode regarding local fragmentation with UFO/CORK (Hannes Frederic Sowa) [1127218]
+- [net] clear local_df when passing skb between namespaces (Jiri Pirko) [1128223]
+- [net] rtnetlink: fix VF info size (Jiri Benc) [1128267]
+- [net] netfilter: connlimit: move lock array out of struct connlimit_data (Florian Westphal) [1052282]
+- [net] netfilter: connlimit: use rbtree for per-host conntrack obj storage (Florian Westphal) [1052282]
+- [net] netfilter: connlimit: make same_source_net signed (Florian Westphal) [1052282]
+- [net] netfilter: connlimit: use keyed locks (Florian Westphal) [1052282]
+- [net] netfilter: connlimit: use kmem_cache for conn objects (Florian Westphal) [1052282]
+- [net] netfilter: connlimit: move insertion of new element out of count function (Florian Westphal) [1052282]
+- [net] netfilter: connlimit: improve packet-to-closed-connection logic (Florian Westphal) [1052282]
+- [net] netfilter: connlimit: factor hlist search into new function (Florian Westphal) [1052282]
+- [net] ipv6: increase ip6_rt_max_size to 16384 (Hannes Frederic Sowa) [1119364]
+- [net] ipv6: don't count addrconf generated routes against gc limit (Hannes Frederic Sowa) [1119364]
+- [net] tcp: add tcp_syncookies mode to allow unconditionally generation of syncookies (Florian Westphal) [1084439]
+- [net] tcp: syncookies: do not use getnstimeofday() (Florian Westphal) [1084439]
+- [net] tcp: syncookies: reduce mss table to four values (Florian Westphal) [1084439]
+- [net] tcp: syncookies: reduce cookie lifetime to 128 seconds (Florian Westphal) [1084439]
+- [net] sctp: only warn in proc_sctp_do_alpha_beta if write (Daniel Borkmann) [1110290]
+- [net] sctp: check proc_dointvec result in proc_sctp_do_auth (Daniel Borkmann) [1110290]
+- [net] sctp: propagate sysctl errors from proc_do* properly (Daniel Borkmann) [1110290]
+- [net] sctp: fix permissions for rto_alpha and rto_beta knobs (Daniel Borkmann) [1110290]
+- [net] sctp: cache auth_enable per endpoint (Daniel Borkmann) [1110290]
+- [net] sctp: fix a missed .data initialization (Daniel Borkmann) [1110290]
+- [net] sctp: fix up a spacing (Daniel Borkmann) [1110290]
+- [net] sctp: add check rto_min and rto_max in sysctl (Daniel Borkmann) [1110290]
+- [net] ip: push gso skb forwarding handling down the stack (Florian Westphal) [1109214]
+- [net] ipv6: send pkttoobig immediately if orig frag size > mtu (Florian Westphal) [1109214]
+- [net] ipv4: ip_forward: fix inverted local_df test (Florian Westphal) [1109214]
+- [net] netfilter: ipv4: defrag: set local_df flag on defragmented skb (Florian Westphal) [1109214]
+- [net] don't account for udp header size when computing seglen (Florian Westphal) [1109214]
+
 * Tue Sep 02 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-149.el7]
 - [fs] ext4: fix type declaration of ext4_validate_block_bitmap (Lukas Czerner) [1091055]
 - [fs] ext4: error out if verifying the block bitmap fails (Lukas Czerner) [1091055]
-- 
1.7.1