From b55762aa39e194b471b47af34f1017a966b2c8a0 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Sat, 22 Nov 2014 13:24:58 -0500
Subject: [redhat] kernel-3.10.0-208.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index be90f7e..a695fbf 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 207
+RHEL_RELEASE = 208
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 21593e4..f262f9a 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=207
+BUILD:=208
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 081d8ae..7f28ebe 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,71 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Sat Nov 22 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-208.el7]
+- [scsi] cxgb4i: send abort_rpl correctly (Sai Vemuri) [1163467]
+- [Documentation] cxgbi: add maintainer for cxgb3i/cxgb4i (Sai Vemuri) [1163467]
+- [ethernet] cxgb4vf: FL Starvation Threshold needs to be larger than the SGE's Egress Congestion Threshold (Sai Vemuri) [1163467]
+- [ethernet] cxgb4: For T5 use Packing and Padding Boundaries for SGE DMA transfers (Sai Vemuri) [1163467]
+- [ethernet] cxgb4vf: Move fl_starv_thres into adapter->sge data structure (Sai Vemuri) [1163467]
+- [ethernet] cxgb4vf: Replace repetitive pci device ID's with right ones (Sai Vemuri) [1163467]
+- [infinband] cxgb4: Make c4iw_wr_log_size_order static (Sai Vemuri) [1163467]
+- [infinband] cxgb4: Add missing neigh_release in find_route (Sai Vemuri) [1163467]
+- [infinband] cxgb4: Fix ntuple calculation for ipv6 and remove duplicate line (Sai Vemuri) [1163467]
+- [ethernet] cxgb4: Fix FW flash logic using ethtool (Sai Vemuri) [1163467]
+- [infiniband] cxgb4: Take IPv6 into account for best_mtu and set_emss (Sai Vemuri) [1163467]
+- [ethernet] cxgb4: Wait for device to get ready before reading any register (Sai Vemuri) [1163467]
+- [ethernet] cxgb4vf: Add 40G support for cxgb4vf driver (Sai Vemuri) [1163467]
+- [ethernet] cxgb4: Updated the LSO transfer length in CPL_TX_PKT_LSO for T5 (Sai Vemuri) [1163467]
+- [ethernet] cxgb4: Add support for adaptive rx (Sai Vemuri) [1163467]
+- [ethernet] cxgb4: Change default Interrupt Holdoff Packet Count Threshold (Sai Vemuri) [1163467]
+- [ethernet] cxgb4: Add Devicde ID for two more adapter (Sai Vemuri) [1163467]
+- [ethernet] cxgb4vf: Remove superfluous "idx" parameter of CH_DEVICE() macro (Sai Vemuri) [1163467]
+- [ethernet] cxgb4: Use BAR2 Going To Sleep (GTS) for T5 and later (Sai Vemuri) [1163467]
+- [scsi] cxgbi: support ipv6 address host_param (Sai Vemuri) [1153834]
+- [scsi] cxgb4i: Fix -Wmaybe-uninitialized warning (Sai Vemuri) [1153834]
+- [scsi] cxgb4i: Remove duplicate call to dst_neigh_lookup() (Sai Vemuri) [1153834]
+- [scsi] cxgb4i: Fix -Wunused-function warning (Sai Vemuri) [1153834]
+- [ethernet] cxgb4: Fix build failure in cxgb4 when ipv6 is disabled/not in-built (Sai Vemuri) [1153834]
+- [scsi] cxgb4i: Remove duplicated CLIP handling code (Sai Vemuri) [1153834]
+- [ethernet] be2net: fix alignment on line wrap (Ivan Vecera) [1165755]
+- [ethernet] be2net: remove multiple assignments on a single line (Ivan Vecera) [1165755]
+- [ethernet] be2net: remove space after typecasts (Ivan Vecera) [1165755]
+- [ethernet] be2net: remove unnecessary blank lines after an open brace (Ivan Vecera) [1165755]
+- [ethernet] be2net: insert a blank line after function/struct//enum definitions (Ivan Vecera) [1165755]
+- [ethernet] be2net: remove multiple blank lines (Ivan Vecera) [1165755]
+- [ethernet] be2net: add blank line after declarations (Ivan Vecera) [1165755]
+- [ethernet] be2net: remove return statements for void functions (Ivan Vecera) [1165755]
+- [ethernet] be2net: add speed reporting for 20G-KR interface (Ivan Vecera) [1165755]
+- [ethernet] be2net: add speed reporting for 40G/KR interface (Ivan Vecera) [1165755]
+- [ethernet] be2net: fix sparse warnings in be_cmd_req_port_type{} (Ivan Vecera) [1165755]
+- [ethernet] be2net: fix a sparse warning in be_cmd_modify_eqd() (Ivan Vecera) [1165755]
+- [ethernet] be2net: enable PCIe error reporting on VFs too (Ivan Vecera) [1165755]
+- [ethernet] be2net: send a max of 8 EQs to be_cmd_modify_eqd() on Lancer (Ivan Vecera) [1165755]
+- [ethernet] be2net: fix port-type reporting in get_settings (Ivan Vecera) [1165755]
+- [ethernet] be2net: add ethtool "-m" option support (Ivan Vecera) [1165755]
+- [ethernet] be2net: fix RX fragment posting for jumbo frames (Ivan Vecera) [1165755]
+- [ethernet] be2net: replace strcpy with strlcpy (Ivan Vecera) [1165755]
+- [ethernet] be2net: fix some log messages (Ivan Vecera) [1165755]
+- [ethernet] bna: fix skb->truesize underestimation (Ivan Vecera) [1165759]
+- [ethernet] bna: allow transmit tagged frames (Ivan Vecera) [1165759]
+- [ethernet] bna: use container_of to resolve bufdesc_ex from bufdesc (Ivan Vecera) [1165759]
+- [ethernet] r8169: add support for RTL8168EP (Ivan Vecera) [1165764]
+- [ethernet] r8169: add support for Byte Queue Limits (Ivan Vecera) [1165764]
+- [ethernet] r8169: call "rtl8168_driver_start" "rtl8168_driver_stop" only when hardware dash function is enabled (Ivan Vecera) [1165764]
+- [ethernet] r8169: modify the behavior of function "rtl8168_oob_notify" (Ivan Vecera) [1165764]
+- [ethernet] r8169: change the name of function "r8168dp_check_dash" to "r8168_check_dash" (Ivan Vecera) [1165764]
+- [ethernet] r8169: change the name of function"rtl_w1w0_eri" (Ivan Vecera) [1165764]
+- [ethernet] r8169: for function "rtl_w1w0_phy" change its name and behavior (Ivan Vecera) [1165764]
+- [ethernet] r8169: add more chips to support magic packet v2 (Ivan Vecera) [1165764]
+- [ethernet] r8169: add support more chips to get mac address from backup mac address register (Ivan Vecera) [1165764]
+- [ethernet] r8169: add disable/enable RTL8411B pll function (Ivan Vecera) [1165764]
+- [ethernet] r8169: add disable/enable RTL8168G pll function (Ivan Vecera) [1165764]
+- [ethernet] r8169: change uppercase number to lowercase number (Ivan Vecera) [1165764]
+- [ethernet] r8169: fix an if condition (Ivan Vecera) [1165764]
+- [ethernet] r8169: adjust __rtl8169_set_features (Ivan Vecera) [1165764]
+- [ethernet] r8169: fix setting rx vlan (Ivan Vecera) [1165764]
+- [ethernet] r8169: fix the default setting of rx vlan (Ivan Vecera) [1165764]
+
 * Sat Nov 22 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-207.el7]
 - [powerpc] use device_online/offline() instead of cpu_up/down() (Gustavo Duarte) [1157737]
 - [ethernet] i40e: disable FCoE (Stefan Assmann) [1165175]
-- 
1.7.1