From 1f75a7b16647631dc17043313b261e0b7458c10b Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Thu, 8 Jan 2015 23:42:32 -0500
Subject: [redhat] kernel-3.10.0-222.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index f6deaec..76b0a5d 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 221
+RHEL_RELEASE = 222
 RHEL_DRM_VERSION = 3
 RHEL_DRM_PATCHLEVEL = 16
 RHEL_DRM_SUBLEVEL = 7
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index aa5ebfe..e23879b 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=221
+BUILD:=222
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 7786cb2..c448817 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1493,6 +1493,33 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Thu Jan 08 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-222.el7]
+- [scsi] qla2xxx: Update version number to 8.07.00.08.07.1-k2 (Chad Dupuis) [1085239]
+- [scsi] qla2xxx: Move mailbox failure messages to a default debug level (Chad Dupuis) [1085239]
+- [security] commoncap: don't alloc the credential unless needed in cap_task_prctl (Paul Moore) [1056347]
+- [iommu] vt-d: Fix dmar_domain leak in iommu_attach_device (Myron Stowe) [1109829]
+- [iommu] vt-d: Only remove domain when device is removed (Myron Stowe) [1109829]
+- [base] core: Add BUS_NOTIFY_REMOVED_DEVICE event (Myron Stowe) [1109829]
+- [powerpc] kdump: Ignore failure in enabling big endian exception during crash (Steve Best) [1170362]
+- [infiniband] srpt: convert printk's to pr_* functions (Doug Ledford) [1174910]
+- [infiniband] srpt: Handle GID change events (Doug Ledford) [1174910]
+- [input] alps: fix v4 button press recognition (Benjamin Tissoires) [1107819]
+- [input] alps: v7 - document the v7 touchpad packet protocol (Benjamin Tissoires) [1107819]
+- [input] alps: v7 - fix finger counting for > 2 fingers on clickpads (Benjamin Tissoires) [1107819]
+- [input] alps: v7 - sometimes a single touch is reported in mt[1] (Benjamin Tissoires) [1107819]
+- [input] alps: v7 - ignore new packets (Benjamin Tissoires) [1107819]
+- [powerpc] perf/hv-24x7: Use kmem_cache_free() instead of kfree (Gustavo Duarte) [1171795]
+- [powerpc] perf/hv-24x7: Use per-cpu page buffer (Gustavo Duarte) [1171795]
+- [powerpc] perf/hv-24x7: use kmem_cache instead of aligned stack allocations (Gustavo Duarte) [1171795]
+- [powerpc] perf/hv-24x7: Use kmem_cache_free (Gustavo Duarte) [1171795]
+- [powerpc] Fill in si_addr_lsb siginfo field (Gustavo Duarte) [1173267]
+- [powerpc] Add VM_FAULT_HWPOISON handling to powerpc page fault handler (Gustavo Duarte) [1173267]
+- [fs] dlm: fix missing endian conversion of rcom_status flags (Andrew Price) [1175900]
+- [scsi] add Intel Multi-Flex to scsi scan blacklist (Hannes Frederic Sowa) [1175862]
+- [scsi] do not issue SCSI RSOC command to Promise Vtrak E610f (Hannes Frederic Sowa) [1175862]
+- [scsi] scsi_lib: rate-limit the error message from failing commands (Tomas Henzl) [1175785]
+- [scsi] iscsi_ibft: Fix finding Broadcom specific ibft sign (Chris Leech) [1095169]
+
 * Tue Jan 06 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-221.el7]
 - [ethernet] enic: fix rx skb checksum (Stefan Assmann) [1154182]
 - [x86] uv: make kdump default action for 'power nmi' (George Beshers) [1175560]
-- 
1.7.1