From 321481b78321dc43428a88e1239b6fd75ad0f924 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 13 Jan 2015 09:21:15 -0500
Subject: [redhat] kernel-3.10.0-223.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 76b0a5d..eebab8f 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 222
+RHEL_RELEASE = 223
 RHEL_DRM_VERSION = 3
 RHEL_DRM_PATCHLEVEL = 16
 RHEL_DRM_SUBLEVEL = 7
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index e23879b..c70bf89 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=222
+BUILD:=223
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index c448817..a6f3659 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1493,6 +1493,35 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Jan 13 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-223.el7]
+- [md] dm-thin: fix crash by initializing thin device's refcount and completion earlier (Mike Snitzer) [1175282]
+- [scsi] storvsc: Fix a bug in storvsc limits (Vitaly Kuznetsov) [1174162]
+- [iser-target] Ignore non TEXT + LOGOUT opcodes for discovery (Andy Grover) [1058736]
+- [iser-target] Add support for ISCSI_OP_TEXT opcode + payload handling (Andy Grover) [1058736]
+- [iser-target] Rename sense_buf_dma/len to pdu_dma/len (Andy Grover) [1058736]
+- [iscsi-target] Add IFC_SENDTARGETS_SINGLE support (Andy Grover) [1058736]
+- [iscsi-target] Move sendtargets parsing into iscsit_process_text_cmd (Andy Grover) [1058736]
+- [iscsi-target] Allow ->MaxXmitDataSegmentLength assignment for iser discovery (Andy Grover) [1058736]
+- [iscsi-target] Refactor ISCSI_OP_TEXT_RSP TX handling (Andy Grover) [1058736]
+- [iscsi-target] Refactor ISCSI_OP_TEXT RX handling (Andy Grover) [1058736]
+- [iscsi] isert-target: Refactor ISCSI_OP_NOOP RX handling (Andy Grover) [1058736]
+- [net] description of dma_cookie cause make xmldocs warning (Jiri Benc) [1173444]
+- [net] tcp: make tcp_cleanup_rbuf private (Jiri Benc) [1173444]
+- [net] net_dma: revert 'copied_early' (Jiri Benc) [1173444]
+- [net] net_dma: mark broken (Jiri Benc) [1173444]
+- [net] unix: allow set_peek_off to fail (Jiri Benc) [1123777]
+- [net] ppp: ppp-ioctl.h: pull in ppp_defs.h (Jiri Benc) [1159802]
+- [net] bridge: Add filtering support for default_pvid (Vlad Yasevich) [1164653]
+- [net] bridge: Simplify pvid checks (Vlad Yasevich) [1164653]
+- [net] bridge: Add a default_pvid sysfs attribute (Vlad Yasevich) [1164653]
+- [net] bridge: Prepare for 802.1ad vlan filtering support (Vlad Yasevich) [1164653]
+- [net] bridge: Fix the way to check if a local fdb entry can be deleted (Vlad Yasevich) [1164653]
+- [net] bridge: Fix the way to insert new local fdb entries in br_fdb_changeaddr (Vlad Yasevich) [1164653]
+- [net] Remove extern from function prototypes (Vlad Yasevich) [1164653]
+- [ethernet] mlx5: Add more supported devices (Amir Vadai) [1169277]
+- [infiniband] mlx4: Fix wrong usage of IPv4 protocol for multicast attach/detach (Amir Vadai) [1151331]
+- [ethernet] mlx4: mlx4_en_set_settings() always fails when autoneg is set (Amir Vadai) [1170129]
+
 * Thu Jan 08 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-222.el7]
 - [scsi] qla2xxx: Update version number to 8.07.00.08.07.1-k2 (Chad Dupuis) [1085239]
 - [scsi] qla2xxx: Move mailbox failure messages to a default debug level (Chad Dupuis) [1085239]
-- 
1.7.1