From 2a9a33821cf9de178da669572bc645852187cc3f Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Sun, 25 Jan 2015 23:16:07 -0500
Subject: [redhat] kernel-3.10.0-226.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index a2a1d7c..e3c1c41 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 225
+RHEL_RELEASE = 226
 RHEL_DRM_VERSION = 3
 RHEL_DRM_PATCHLEVEL = 16
 RHEL_DRM_SUBLEVEL = 7
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 7182826..5d5f19e 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=225
+BUILD:=226
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 4bf49be..87eea71 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1492,6 +1492,16 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Sun Jan 25 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-226.el7]
+- [md] dm-cache: fix problematic dual use of a single migration count variable (Mike Snitzer) [1182665]
+- [md] dm-cache: share cache-metadata object across inactive and active DM tables (Mike Snitzer) [1182665]
+- [net] tun/macvtap: use consume_skb() instead of kfree_skb() when needed (Jiri Pirko) [1182805]
+- [virt] kvm/vmx: invalid host cr4 handling across vm entries (Jacob Tanenbaum) [1153329] {CVE-2014-3690}
+- [virt] virtio-scsi: Fix the race condition in virtscsi_handle_event (Fam Zheng) [1152140]
+- [virt] kvm: workaround SuSE's 2.6.16 pvclock vs masterclock issue (Marcelo Tosatti) [1177718]
+- [fs] bdi: avoid oops on device removal (Fam Zheng) [1087179]
+- [mm] backing_dev: fix hung task on sync (Fam Zheng) [1087179]
+
 * Thu Jan 22 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-225.el7]
 - [net] team: avoid possible underflow of count_pending value for notify_peers and mcast_rejoin (Jiri Pirko) [1176697]
 - [fs] seq_file: don't include mm.h in genksyms calculation (Ian Kent) [1183280]
-- 
1.7.1