From 603fddc722e4900bbd57a9ad2e861fd270ce56fa Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Wed, 12 Nov 2014 15:30:59 -0500
Subject: [redhat] kernel-3.10.0-203.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index c3f60ba..a06aea1 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 202
+RHEL_RELEASE = 203
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index c3b2af3..07f485b 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=202
+BUILD:=203
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index ed76953..aaad278 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,18 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Wed Nov 12 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-203.el7]
+- [fs] GFS2: If we use up our block reservation, request more next time (Robert S Peterson) [1142238]
+- [fs] GFS2: Only increase rs_sizehint (Robert S Peterson) [1142238]
+- [fs] GFS2: Set of distributed preferences for rgrps (Robert S Peterson) [1142238]
+- [fs] autofs: fix symlinks aren't checked for expiry (Ian Kent) [1116182]
+- [fs] GFS2: fix regression in dir_double_exhash (Robert S Peterson) [1160229]
+- [fs] gfs2_atomic_open(): skip lookups on hashed dentry (Robert S Peterson) [1158150]
+- [fs] splice: perform generic write checks (Eric Sandeen) [1155907]
+- [fs] fs: seq_file: fallback to vmalloc allocation (Ian Kent) [1095623]
+- [fs] fs: /proc/stat: convert to single_open_size() (Ian Kent) [1095623]
+- [fs] fs: seq_file: always clear m->count when we free m->buf (Ian Kent) [1095623]
+
 * Wed Nov 12 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-202.el7]
 - [ethernet] mlx4: Use PTYS register to set ethtool settings (Speed) (Amir Vadai) [1060221]
 - [ethernet] mlx4: Use PTYS register to query ethtool settings (Amir Vadai) [1060221]
-- 
1.7.1