From 792b4b11029e69e2ea06d41c79613cf556914fc5 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Wed, 16 Jul 2014 10:58:26 -0400
Subject: [redhat] kernel-3.10.0-135.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index c1bf7cf..dd448fb 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 134
+RHEL_RELEASE = 135
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index aef15a7..394d3ab 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=134
+BUILD:=135
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index e2616ba..4d20afa 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1471,6 +1471,138 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Wed Jul 16 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-135.el7]
+- [pci] hotplug: Avoid warning when _ADR not present (Myron Stowe) [1114228]
+- [pci] hotplug/acpi: Drop unused acpiphp_debug declaration (Myron Stowe) [1114228]
+- [acpi] scan: Set flags.match_driver in acpi_bus_scan_fixed() (Myron Stowe) [1114228]
+- [acpi] pci_root: Clear driver_data before failing enumeration (Myron Stowe) [1114228]
+- [acpi] hotplug: Fix PCI host bridge hot removal (Myron Stowe) [1114228]
+- [acpi] hotplug: Fix acpi_bus_get_device() return value check (Myron Stowe) [1114228]
+- [acpi] hotplug: Consolidate deferred execution of ACPI hotplug routines (Myron Stowe) [1114228]
+- [acpi] hotplug: Do not execute "insert in progress" _OST (Myron Stowe) [1114228]
+- [acpi] hotplug: Carry out PCI root eject directly (Myron Stowe) [1114228]
+- [acpi] hotplug: Merge device hot-removal routines (Myron Stowe) [1114228]
+- [acpi] hotplug: Make acpi_bus_hot_remove_device() internal (Myron Stowe) [1114228]
+- [acpi] hotplug: Simplify device ejection routines (Myron Stowe) [1114228]
+- [acpi] hotplug: Fix handle_root_bridge_removal() (Myron Stowe) [1114228]
+- [acpi] hotplug: Refuse to hot-remove all objects with disabled hotplug (Myron Stowe) [1114228]
+- [acpi] scan: Start matching drivers after trying scan handlers (Myron Stowe) [1114228]
+- [acpi] Remove acpi_pci_slot_init() headers from internal.h (Myron Stowe) [1114228]
+- [base] Release device_hotplug_lock when store_mem_state returns EINVAL (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Drop WARN_ON() from acpiphp_enumerate_slots() (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Fix error code path in acpiphp_enumerate_slots() (Myron Stowe) [1114228]
+- [acpi] memhotplug: Use defined marco METHOD_NAME__STA (Myron Stowe) [1114228]
+- [acpi] hotplug: Use kobject_init_and_add() instead of _init() and _add() (Myron Stowe) [1114228]
+- [acpi] hotplug: Don't set kobject parent pointer explicitly (Myron Stowe) [1114228]
+- [acpi] hotplug: Set kobject name via kobject_add(), not kobject_set_name() (Myron Stowe) [1114228]
+- [base] hotplug: Remove cpu_hotplug_driver_lock() (Myron Stowe) [1114228]
+- [x86] hotplug: Disable ARCH_CPU_PROBE_RELEASE on x86 (Myron Stowe) [1114228]
+- [x86] hotplug: Add hotplug lock to missing places (Myron Stowe) [1114228]
+- [x86] hotplug: Fix online state in cpu0 debug interface (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Avoid parent bus rescans on spurious device checks (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Use _OST to notify firmware about notify status (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Avoid doing too much for spurious notifies (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Don't trim devices before scanning the namespace (Myron Stowe) [1114228]
+- [kernel] power/hibernate/memory_hotplug: Rework mutual exclusion (Myron Stowe) [1114228]
+- [kernel] power/hibernate: Create memory bitmaps after freezing user space (Myron Stowe) [1114228]
+- [acpi] scan: Change ordering of locks for device hotplug (Myron Stowe) [1114228]
+- [acpi] hotplug: Remove containers synchronously (Myron Stowe) [1114228]
+- [base] core/acpi: Avoid device hot remove locking issues (Myron Stowe) [1114228]
+- [base] introduce help macro to_memory_block (Myron Stowe) [1114228]
+- [base] use standard device online/offline for state change (Myron Stowe) [1114228]
+- [base] refactor add_memory_section() to add_memory_block() (Myron Stowe) [1114228]
+- [base] remove improper get/put in add_memory_section() (Myron Stowe) [1114228]
+- [base] reduce add_memory_section() for boot-time only (Myron Stowe) [1114228]
+- [base] unshare add_memory_section() from hotplug (Myron Stowe) [1114228]
+- [base] use device get/put functions (Myron Stowe) [1114228]
+- [base] remove unneeded variable (Myron Stowe) [1114228]
+- [base] move mutex lock out of add_memory_section() (Myron Stowe) [1114228]
+- [acpi] sleep: Hold acpi_scan_lock over system PM transitions (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Fix NULL pointer dereference in cleanup_bridge() (Myron Stowe) [1114228]
+- [base] cpu: Check if NUMA node is valid before bringing CPU up (Myron Stowe) [1114228]
+- [pci] acpi: Use dev_dbg() instead of dev_info() in acpi_pci_set_power_state() (Myron Stowe) [1114228]
+- [acpi] sleep: Introduce acpi_os_prepare_extended_sleep() for extended sleep (Myron Stowe) [1114228]
+- [base] core: replace strict_strto*() with kstrto*() (Myron Stowe) [1114228]
+- [acpi] dock: fix error return code in dock_add() (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Get rid of check_sub_bridges() (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Clean up bridge_mutex usage (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Redefine enable_device() and disable_device() (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Sanitize acpiphp_get_(latch) (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Get rid of unused constants in acpiphp.h (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Check for new devices on enabled slots (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Allow slots without new devices to be rescanned (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Do not check SLOT_ENABLED in enable_device() (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Do not exectute _PS0 and _PS3 directly (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Do not queue up event handling work items in vain (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Consolidate slot disabling and ejecting (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Drop redundant checks from check_hotplug_bridge() (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Rework namespace scanning and trimming routines (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Store parent in functions and bus in slots (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Drop handle field from struct acpiphp_bridge (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Drop handle field from struct acpiphp_func (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Embed function struct into struct acpiphp_context (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Drop flags field from struct acpiphp_bridge (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Drop sun field from struct acpiphp_slot (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Register all devices under the given bridge (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Refactor slot allocation code in register_slot() (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Drop func field from struct acpiphp_bridge (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Merge hotplug event handling functions (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Pass hotplug context objects to event handlers (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Rework acpiphp_handle_to_bridge() (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Unified notify handler for hotplug events (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Hotplug context objects for bridges and functions (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Always return success after adding a function (Myron Stowe) [1114228]
+- [acpi] hotplug/pci: Consolidate acpiphp_enumerate_slots() (Myron Stowe) [1114228]
+- [acpi] pci: Make bus registration and unregistration symmetric (Myron Stowe) [1114228]
+- [x86] pci: prevent re-allocation of already existing bridge and ROM resource (Myron Stowe) [1114228]
+- [acpi] dock: Drop unnecessary local variable from dock_add() (Myron Stowe) [1114228]
+- [acpi] dock/pci: Drop ACPI dock notifier chain (Myron Stowe) [1114228]
+- [acpi] dock: Do not check CONFIG_ACPI_DOCK_MODULE (Myron Stowe) [1114228]
+- [acpi] dock: Do not leak memory on falilures to add a dock station (Myron Stowe) [1114228]
+- [acpi] Drop ACPI bus notifier call chain (Myron Stowe) [1114228]
+- [acpi] dock: Rework the handling of notifications (Myron Stowe) [1114228]
+- [acpi] dock: Simplify dock_init_hotplug() and dock_release_hotplug() (Myron Stowe) [1114228]
+- [acpi] dock: Walk list in reverse order during removal of devices (Myron Stowe) [1114228]
+- [acpi] dock: Rework and simplify find_dock_devices() (Myron Stowe) [1114228]
+- [acpi] dock: Drop the hp_lock mutex from struct dock_station (Myron Stowe) [1114228]
+- [acpi] simplify acpiphp driver with new helper functions (Myron Stowe) [1114228]
+- [acpi] simplify dock driver with new helper functions (Myron Stowe) [1114228]
+- [acpi] Export acpi_(bay) (Myron Stowe) [1114228]
+- [acpi] introduce two helper functions for _EJ0 and _LCK (Myron Stowe) [1114228]
+- [acpi] introduce helper function acpi_execute_simple_method() (Myron Stowe) [1114228]
+- [acpi] introduce helper function acpi_has_method() (Myron Stowe) [1114228]
+- [acpi] dock: simplify dock_create_acpi_device() (Myron Stowe) [1114228]
+- [acpi] dock: mark initialization functions with __init (Myron Stowe) [1114228]
+- [acpi] dock: drop redundant spin lock in dock station object (Myron Stowe) [1114228]
+- [acpi] dock: avoid initializing acpi_dock_notifier_list multiple times (Myron Stowe) [1114228]
+- [base] Use attribute groups to create sysfs memory files (Myron Stowe) [1114228]
+- [mm] hugetlb: add support for gigantic page allocation at runtime (Luiz Capitulino) [996763]
+- [mm] hugetlb: move helpers up in the file (Luiz Capitulino) [996763]
+- [mm] hugetlb: update_and_free_page(), don't clear PG_reserved bit (Luiz Capitulino) [996763]
+- [mm] hugetlb: add hstate_is_gigantic() (Luiz Capitulino) [996763]
+- [mm] hugetlb: fix and clean-up node iteration code to alloc or free (Luiz Capitulino) [996763]
+- [mm] dma-contiguous: clean source code and prepare for device tree (Luiz Capitulino) [996763]
+- [mm] cma: Move dma contiguous changes into a seperate config (Luiz Capitulino) [996763]
+- [thermal] intel_powerclamp: Fix cstate counter detection (Steve Best) [1103376]
+- [thermal] Tidy up error handling in powerclamp_init (Steve Best) [1103376]
+- [thermal] intel_powerclamp: Add newer CPU models (Steve Best) [1103376]
+- [security] selinux: Increase ebitmap_node size for 64-bit configuration (Paul Moore) [922752]
+- [security] selinux: Reduce overhead of mls_level_isvalid() function call (Paul Moore) [922752]
+- [powerpc] mm: fix ".__node_distance" undefined (David Milburn) [1081734]
+- [block] nvme: Retry failed commands with non-fatal error (David Milburn) [1081734]
+- [block] nvme: Fix divide-by-zero in nvme_trans_io_get_num_cmds (David Milburn) [1081734]
+- [block] nvme: Add getgeo to block ops (David Milburn) [1081734]
+- [block] nvme: Start-stop nvme_thread during device add-remove (David Milburn) [1081734]
+- [block] nvme: Make I/O timeout a module parameter (David Milburn) [1081734]
+- [block] nvme: CPU hot plug notification (David Milburn) [1081734]
+- [block] nvme: per-cpu io queues (David Milburn) [1081734]
+- [block] nvme: Replace DEFINE_PCI_DEVICE_TABLE (David Milburn) [1081734]
+- [block] nvme: IOCTL path RCU protect queue access (David Milburn) [1081734]
+- [block] nvme: RCU protected access to io queues (David Milburn) [1081734]
+- [block] nvme: Initialize device reference count earlier (David Milburn) [1081734]
+- [scsi] fnic: fix broken FIP discovery by initializing multicast address (Chris Leech) [1100078]
+- [scsi] libfcoe: Make fcoe_sysfs optional / fix fnic NULL exception (Chris Leech) [1100078]
+
 * Mon Jul 14 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-134.el7]
 - [fs] nfs: populate ->net in mount data when remounting (Steve Dickson) [1113248]
 - [fs] nfs: fix lockup caused by pnfs_generic_pg_test (Steve Dickson) [1113248]
-- 
1.7.1