From 146e35b6c776e75f8aca1244b28afa8993463bfe Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Mon, 19 Jan 2015 15:48:08 -0500
Subject: [redhat] kernel-3.10.0-224.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index eebab8f..cab016e 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 223
+RHEL_RELEASE = 224
 RHEL_DRM_VERSION = 3
 RHEL_DRM_PATCHLEVEL = 16
 RHEL_DRM_SUBLEVEL = 7
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index c70bf89..10cfce6 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=223
+BUILD:=224
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index a6f3659..0350b56 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1493,6 +1493,30 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Mon Jan 19 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-224.el7]
+- [fs] xfs: catch invalid negative blknos in _xfs_buf_find() (Eric Sandeen) [1164128]
+- [fs] proc: make proc_fd_permission() thread-friendly (Carlos Maiolino) [1171242]
+- [fs] rpc: fix xdr_truncate_encode to handle buffer ending on page boundary ("J. Bruce Fields") [1176641]
+- [fs] nfs: nfs4_fl_prepare_ds, fix bugs when the connect attempt fails (Steve Dickson) [1113248]
+- [fs] gfs2: fix bad inode i_goal values during block allocation (Abhijith Das) [1144209]
+- [fs] nfsd: allow turning off nfsv3 readdir_plus (Steve Dickson) [1178949]
+- [fs] nfsd4: fix xdr4 count of server in fs_location4 (Benjamin Coddington) [1164055]
+- [fs] nfsd4: fix xdr4 inclusion of escaped char (Benjamin Coddington) [1164055]
+- [fs] xfs: replace global xfslogd wq with per-mount wq (Brian Foster) [1155929]
+- [fs] xfs: mark all internal workqueues as freezable (Brian Foster) [1155929]
+- [fs] overlayfs: Add call to mark_tech_preview (BZ 1180613) (David Howells) [1180613]
+- [fs] aio: fix uncorrent dirty pages accouting when truncating AIO ring buffer (Jeff Moyer) [1159346]
+- [infiniband] ocrdma: fix hardcoded max cqe and max send wr (Doug Ledford) [1158148]
+- [crypto] aesni-intel: Add support for 192 & 256 bit keys to AESNI RFC4106 (Jarod Wilson) [1176266]
+- [block] blk-mq: Fix a use-after-free (Fam Zheng) [1152159]
+- [crypto] drbg: panic on continuous self test error (Jarod Wilson) [1179496]
+- [ethernet] mlx4: Cache line CQE/EQE stride fixes (Doug Ledford) [1088499 1173483]
+- [ethernet] mlx4: Add mlx4_en_get_cqe helper (Doug Ledford) [1088499 1173483]
+- [ethernet] mlx4: Cache line EQE size support (Doug Ledford) [1088499 1173483]
+- [infiniband] ocrdma: Fix ocrdma_query_qp() to report q_key value for UD QPs (Doug Ledford) [1167256]
+- [infiniband] ocrdma: Always resolve destination mac from GRH for UD QPs (Doug Ledford) [1167256]
+- [net] gre: fix the inner mac header in nbma tunnel xmit path (Alexander Duyck) [1168608]
+
 * Tue Jan 13 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-223.el7]
 - [md] dm-thin: fix crash by initializing thin device's refcount and completion earlier (Mike Snitzer) [1175282]
 - [scsi] storvsc: Fix a bug in storvsc limits (Vitaly Kuznetsov) [1174162]
-- 
1.7.1