From 2104b26f4a4c3957695044f556d2d892ed598b5a Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 6 Jan 2015 14:35:53 -0500
Subject: [redhat] kernel-3.10.0-221.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index cc6e775..f6deaec 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 220
+RHEL_RELEASE = 221
 RHEL_DRM_VERSION = 3
 RHEL_DRM_PATCHLEVEL = 16
 RHEL_DRM_SUBLEVEL = 7
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index bd6104d..aa5ebfe 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=220
+BUILD:=221
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 3580c78..7786cb2 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1493,6 +1493,112 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Jan 06 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-221.el7]
+- [ethernet] enic: fix rx skb checksum (Stefan Assmann) [1154182]
+- [x86] uv: make kdump default action for 'power nmi' (George Beshers) [1175560]
+- [virt] powerpc/kvm: book3s_hv - Fix KSM memory corruption (David Gibson) [1170394]
+- [powercap] rapl: add support for CPU model 0x3f (Rui Wang) [1177579]
+- [kernel] audit: don't attempt to lookup PIDs when changing PID filtering audit rules (Paul Moore) [1172624]
+- [ethernet] ixgbe: avoid possible read_reg panic caused by late method binding (John Greene) [1145772]
+- [ethernet] ixgbe: bump version number (John Greene) [1145772]
+- [ethernet] ixgbe: Add X550 support function pointers (John Greene) [1145772]
+- [ethernet] ixgbe: Add new support for X550 MAC's (John Greene) [1145772]
+- [ethernet] ixgbe: Add x550 SW/FW semaphore support (John Greene) [1145772]
+- [ethernet] ixgbe: add methods for combined read and write operations (John Greene) [1145772]
+- [ethernet] ixgbe: cleanup checksum to allow error results (John Greene) [1145772]
+- [ethernet] ixgbe: Add timeout parameter to ixgbe_host_interface_command (John Greene) [1145772]
+- [ethernet] ixgbe: Fix spurious release of semaphore in EEPROM access (John Greene) [1145772]
+- [drm] i915: remove the IRQs enabled WARN from intel_disable_gt_powersave (Rob Clark) [1173317]
+- [drm] i915: tame the chattermouth (Rob Clark) [1173317]
+- [drm] ttm: Avoid memory allocation from shrinker functions (Rob Clark) [1173317]
+- [drm] ttm: Fix possible stack overflow by recursive shrinker calls (Rob Clark) [1173317]
+- [drm] ttm: Use mutex_trylock() to avoid deadlock inside shrinker functions (Rob Clark) [1173317]
+- [drm] video/fb: Propagate error code from failing to unregister conflicting fb (Rob Clark) [1173317]
+- [drm] i915: save/restore GMBUS freq across suspend/resume on gen4 (Rob Clark) [1173317]
+- [drm] i915: resume MST after reading back hw state (Rob Clark) [1173317]
+- [drm] dp-mst: Remove branches before dropping the reference (Rob Clark) [1173317]
+- [drm] fb_helper: move deferred fb checking into restore mode (Rob Clark) [1173317]
+- [drm] dp: retry AUX transactions 32 times (v1.1) (Rob Clark) [1173317]
+- [drm] i915: Ignore long hpds on eDP ports (Rob Clark) [1173317]
+- [drm] i915/dp: only use training pattern 3 on platforms that support it (Rob Clark) [1173317]
+- [drm] radeon: sync all BOs involved in a CS (Rob Clark) [1173317]
+- [drm] radeon: kernel panic in drm_calc_vbltimestamp_from_scanoutpos with 3.18.0-rc6 (Rob Clark) [1173317]
+- [drm] i915: Unlock panel even when LVDS is disabled (Rob Clark) [1173317]
+- [drm] i915: More cautious with pch fifo underruns (Rob Clark) [1173317]
+- [drm] i915: Ignore SURFLIVE and flip counter when the GPU gets reset (Rob Clark) [1173317]
+- [drm] i915: Kick fbdev before vgacon (Rob Clark) [1173317]
+- [drm] i915: Handle failure to kick out a conflicting fb driver (Rob Clark) [1173317]
+- [drm] i915: drop WaSetupGtModeTdRowDispatch:snb (Rob Clark) [1173317]
+- [drm] radeon: add locking around atombios scratch space usage (Rob Clark) [1173317]
+- [drm] radeon: add missing crtc unlock when setting up the MC (Rob Clark) [1173317]
+- [drm] i915: Disable caches for Global GTT (Rob Clark) [1173317]
+- [drm] i915: Do not leak pages when freeing userptr objects (Rob Clark) [1173317]
+- [drm] ast: Fix HW cursor image (Rob Clark) [1173317]
+- [drm] radeon: Use drm_malloc_ab instead of kmalloc_array (Rob Clark) [1173317]
+- [drm] radeon/dpm: disable ulv support on SI (Rob Clark) [1173317]
+- [drm] i915: Do a dummy DPCD read before the actual read (Rob Clark) [1173317]
+- [drm] nouveau/bios: memset dcb struct to zero before parsing (Rob Clark) [1173317]
+- [drm] nv50/disp: fix dpms regression on certain boards (Rob Clark) [1173317]
+- [drm] nouveau/ltc: fix cbc issues on certain boards (Rob Clark) [1173317]
+- [drm] nouveau/ltc: fix tag base address getting truncated if above 4GiB (Rob Clark) [1173317]
+- [drm] nvc0-/fb/ram: fix use of non-existant ram if partitions aren't uniform (Rob Clark) [1173317]
+- [drm] nouveau/bar: behave better if ioremap failed (Rob Clark) [1173317]
+- [drm] nouveau: make sure display hardware is reinitialised on runtime resume (Rob Clark) [1173317]
+- [drm] nouveau: punt fbcon resume out to a workqueue (Rob Clark) [1173317]
+- [drm] nouveau/kms: restore acceleration before fb_set_suspend() resumes (Rob Clark) [1173317]
+- [drm] nouveau/kms: take more care when pulling down accelerated fbcon (Rob Clark) [1173317]
+- [drm] i915: Flush the PTEs after updating them before suspend (Rob Clark) [1153301]
+- [drm] radeon/cik: use a separate counter for CP init timeout (Rob Clark) [1173317]
+- [drm] radeon/dpm: fix resume on mullins (Rob Clark) [1173317]
+- [drm] radeon: don't reset dma on r6xx-evergreen init (Rob Clark) [1173317]
+- [drm] radeon: don't reset sdma on CIK init (Rob Clark) [1173317]
+- [drm] radeon: don't reset dma on NI/SI init (Rob Clark) [1173317]
+- [drm] radeon: add connector quirk for fujitsu board (Rob Clark) [1173317]
+- [drm] radeon/dpm: set the thermal type properly for special configs (Rob Clark) [1173317]
+- [drm] radeon: fix semaphore value init (Rob Clark) [1173317]
+- [drm] radeon: handle broken disabled rb mask gracefully (6xx/7xx) (Rob Clark) [1173317]
+- [drm] radeon: fix active_cu mask on SI and CIK after re-init (v3) (Rob Clark) [1173317]
+- [drm] radeon: fix active cu count for SI and CIK (Rob Clark) [1173317]
+- [drm] radeon: fix pm handling in radeon_gpu_reset (Rob Clark) [1173317]
+- [drm] radeon: properly document reloc priority mask (Rob Clark) [1173317]
+- [drm] radeon/dpm: select the appropriate vce power state for KV/KB/ML (Rob Clark) [1173317]
+- [drm] radeon: Add missing lines to ci_set_thermal_temperature_range (Rob Clark) [1173317]
+- [drm] radeon: Add ability to get and change dpm state when radeon PX card is turned off (Rob Clark) [1173317]
+- [drm] vmwgfx: Fix a potential infinite spin waiting for fifo idle (Rob Clark) [1173317]
+- [drm] ast: AST2000 cannot be detected correctly (Rob Clark) [1173317]
+- [drm] ast: open key before detect chips (Rob Clark) [1173317]
+- [drm] i915: Don't leak command parser tables on suspend/resume (Rob Clark) [1153301]
+- [drm] i915/hdmi: fix hdmi audio state readout (Rob Clark) [1153301]
+- [drm] i915: Wait for vblank before enabling the TV encoder (Rob Clark) [1153301]
+- [drm] i915: Fix EIO/wedged handling in gem fault handler (Rob Clark) [1153301]
+- [drm] i915: Fix lock dropping in intel_tv_detect() (Rob Clark) [1153301]
+- [drm] i915: Remove bogus __init annotation from DMI callbacks (Rob Clark) [1153301]
+- [drm] i915: Ignore VBT backlight presence check on Acer C720 (4005U) (Rob Clark) [1153301]
+- [drm] i915: fix plane/cursor handling when runtime suspended (Rob Clark) [1153301]
+- [drm] i915: don't try to retrain a DP link on an inactive CRTC (Rob Clark) [1153301]
+- [drm] i915: Fix locking for intel_enable_pipe_a() (Rob Clark) [1153301]
+- [drm] i915: Skip load detect when intel_crtc->new_enable==true (Rob Clark) [1153301]
+- [drm] i915: Disable RCS flips on Ivybridge (Rob Clark) [1153301]
+- [drm] i915: read HEAD register back in init_ring_common() to enforce ordering (Rob Clark) [1153301]
+- [drm] i915: Fix crash when failing to parse MIPI VBT (Rob Clark) [1153301]
+- [drm] radeon: tweak ACCEL_WORKING2 query for hawaii (Rob Clark) [1173317]
+- [drm] radeon/atom: add new voltage fetch function for hawaii (Rob Clark) [1173317]
+- [drm] radeon: set VM base addr using the PFP (Rob Clark) [1173317]
+- [drm] radeon: load the lm63 driver for an lm64 thermal chip (Rob Clark) [1173317]
+- [drm] radeon: re-enable dpm by default on BTC (Rob Clark) [1173317]
+- [drm] radeon: re-enable dpm by default on cayman (Rob Clark) [1173317]
+- [drm] radeon/dpm: handle voltage info fetching on hawaii (Rob Clark) [1173317]
+- [drm] ttm: Choose a pool to shrink correctly in ttm_dma_pool_shrink_scan() (Rob Clark) [1173317]
+- [drm] ttm: Fix possible division by 0 in ttm_dma_pool_shrink_scan() (Rob Clark) [1173317]
+- [drm] ttm: fix handling of TTM_PL_FLAG_TOPDOWN (Rob Clark) [1173317]
+- [drm] nouveau: Bump version from 1.1.1 to 1.1.2 (Rob Clark) [1173317]
+- [drm] nouveau: Dis/Enable vblank irqs during suspend/resume (Rob Clark) [1173317]
+- [drm] radeon: add additional SI pci ids (Rob Clark) [1173317]
+- [drm] radeon: add new bonaire pci ids (Rob Clark) [1173317]
+- [drm] radeon: add new KV pci id (Rob Clark) [1173317]
+- [powerpc] add little endian flag to syscall_get_arch() (Richard Guy Briggs) [1169461]
+- [powerpc] simplify syscall_get_arch() (Richard Guy Briggs) [1169461]
+
 * Tue Dec 23 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-220.el7]
 - [scsi] libcxgbi: fix freeing skb prematurely (Sai Vemuri) [1174982]
 - [scsi] cxgb4i: use set_wr_txq() to set tx queues (Sai Vemuri) [1174982]
-- 
1.7.1