From 7bb84322083b1a9669e23e7662e0c1014812ecb6 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Thu, 3 Jul 2014 16:46:02 -0400
Subject: [redhat] kernel-3.10.0-132.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index b39b90d..d954efa 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 131
+RHEL_RELEASE = 132
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 716c1a1..fd1b7f4 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=131
+BUILD:=132
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index a0f489d..c2ff1f9 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1471,6 +1471,27 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Thu Jul 03 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-132.el7]
+- [net] neigh: set nud_state to NUD_INCOMPLETE when probing router reachability (Hannes Frederic Sowa) [1110193]
+- [net] bridge: Set vlan_features to allow offloads on vlans (Florian Westphal) [1058544]
+- [net] team: fix mtu setting (Jiri Pirko) [1103083]
+- [net] ptp: validate the requested frequency adjustment (Jiri Benc) [1092969]
+- [net] netfilter: ctnetlink: don't add null bindings if no nat requested (Florian Westphal) [1089929]
+- [net] netfilter: ctnetlink: force null nat binding on insert (Florian Westphal) [1089929]
+- [net] netfilter: nf_conntrack: don't release a conntrack with non-zero refcnt (Florian Westphal) [1089929]
+- [net] netfilter: nf_conntrack: fix RCU race in nf_conntrack_find_get (Florian Westphal) [1089929]
+- [net] rtnetlink: Only supply IFLA_VF_PORTS information when RTEXT_FILTER_VF is set (Jiri Pirko) [1092400]
+- [net] rtnetlink: Warn when interface's information won't fit in our packet (Jiri Pirko) [1092400]
+- [net] fib_frontend: fix possible NULL pointer dereference (Jiri Benc) [1091142]
+- [net] team: forbid incorrect fall-through in notifier (Jiri Pirko) [1090495]
+- [net] gre: don't allow to add the same tunnel twice (Florian Westphal) [1086498]
+- [net] sctp: confirm route during forward progress (Daniel Borkmann) [1089629]
+- [net] nf_conntrack: decrement global counter after object release (Jiri Pirko) [1089191]
+- [net] add busy_poll device feature (Jiri Pirko) [1080093]
+- [net] ipv6: reallocate addrconf router for ipv6 address when lo device up (Jiri Benc) [1077053]
+- [net] macvlan: Add support for 'always_on' offload features (Florian Westphal) [928644]
+- [net] macvlan: fix netdev feature propagation from lower device (Florian Westphal) [928644]
+
 * Wed Jul 02 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-131.el7]
 - [scsi] pm80xx: Spinlock fix (Rich Bono) [1044636]
 - [scsi] aacraid: Update unsupported card list (Rich Bono) [1080712]
-- 
1.7.1