From 9617dc9881da37fbc1c115be262f053a756d01ce Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Fri, 24 Oct 2014 10:17:13 -0400
Subject: [redhat] kernel-3.10.0-194.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index a1dd695..51baf67 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 193
+RHEL_RELEASE = 194
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 1aaf0bf..fe25cd2 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=193
+BUILD:=194
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index e215f60..1eab48f 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,69 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Fri Oct 24 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-194.el7]
+- [fs] xfs: flush the range before zero range conversion (Eric Sandeen) [1145837]
+- [fs] xfs: only set extent size hint when asked (Eric Sandeen) [1145837]
+- [fs] xfs: project id inheritance is a directory only flag (Eric Sandeen) [1145837]
+- [fs] xfs: use ranged writeback and invalidation for direct IO (Eric Sandeen) [1145837]
+- [fs] xfs: don't zero partial page cache pages during O_DIRECT writes (Eric Sandeen) [1145837]
+- [fs] xfs: don't zero partial page cache pages during O_DIRECT reads (Eric Sandeen) [1130294 1145837]
+- [fs] xfs: don't dirty buffers beyond EOF (Eric Sandeen) [1145837]
+- [fs] xfs: fix coccinelle warnings (Eric Sandeen) [1145837]
+- [fs] xfs: flush both inodes in xfs_swap_extents (Eric Sandeen) [1145837]
+- [fs] xfs: fix swapext ilock deadlock (Eric Sandeen) [1145837]
+- [fs] xfs: dquot recovery needs verifiers (Eric Sandeen) [1145837]
+- [fs] xfs: quotacheck leaves dquot buffers without verifiers (Eric Sandeen) [1145837]
+- [fs] xfs: ensure verifiers are attached to recovered buffers (Eric Sandeen) [1145837]
+- [fs] xfs: catch buffers written without verifiers attached (Eric Sandeen) [1145837]
+- [fs] xfs: avoid false quotacheck after unclean shutdown (Eric Sandeen) [1145837]
+- [fs] xfs: fix rounding error of fiemap length parameter (Eric Sandeen) [1145837]
+- [fs] xfs: fix uflags detection at xfs_fs_rm_xquota (Eric Sandeen) [1145837]
+- [fs] xfs: fix cil push sequence after log recovery (Eric Sandeen) [1145837]
+- [fs] xfs: squash prealloc while over quota free space as well (Eric Sandeen) [1145837]
+- [fs] xfs: run an eofblocks scan on ENOSPC/EDQUOT (Eric Sandeen) [1145837]
+- [fs] xfs: support a union-based filter for eofblocks scans (Eric Sandeen) [1145837]
+- [fs] xfs: add scan owner field to xfs_eofblocks (Eric Sandeen) [1145837]
+- [fs] xfs: fix a couple error sequence jumps in xfs_mountfs() (Eric Sandeen) [1145837]
+- [fs] xfs: null unused quota inodes when quota is on (Eric Sandeen) [1145837]
+- [fs] xfs: refine the allocation stack switch (Eric Sandeen) [1145837]
+- [fs] xfs: Fix rounding in xfs_alloc_fix_len() (Eric Sandeen) [1145837]
+- [fs] xfs: tone down writepage/releasepage WARN_ONs (Eric Sandeen) [1145837]
+- [fs] xfs: remove redundant checks from xfs_da_read_buf (Eric Sandeen) [1145837]
+- [fs] xfs: keep sb_bad_features2 the same a sb_features2 (Eric Sandeen) [1145837]
+- [fs] xfs: negate xfs_icsb_init_counters error value (Eric Sandeen) [1145837]
+- [fs] xfs: negate mount workqueue init error value (Eric Sandeen) [1145837]
+- [fs] xfs: fix wrong errno from xfs_initxattrs (Eric Sandeen) [1145837]
+- [fs] xfs: xfs_commit_metadata returns wrong errno (Eric Sandeen) [1145837]
+- [fs] xfs: fix incorrect error sign in xfs_file_aio_read (Eric Sandeen) [1145837]
+- [fs] xfs: xfs_dir_fsync() returns positive errno (Eric Sandeen) [1145837]
+- [fs] xfs: fix directory readahead offset off-by-one (Eric Sandeen) [1145837]
+- [fs] xfs: remote attribute overwrite causes transaction overrun (Eric Sandeen) [1145837]
+- [fs] xfs: Fix wrong error codes being returned (Eric Sandeen) [1145837]
+- [fs] xfs: bulletfproof xfs_qm_scall_trunc_qfiles() (Eric Sandeen) [1145837]
+- [fs] xfs: fix Q_XQUOTARM ioctl (Eric Sandeen) [1145837]
+- [fs] xfs: fully support v5 format filesystems (Eric Sandeen) [1145837]
+- [fs] xfs: wrong error sign conversion during failed DIO writes (Eric Sandeen) [1145837]
+- [fs] xfs: unmount does not wait for shutdown during unmount (Eric Sandeen) [1145837]
+- [fs] xfs: don't map ranges that span EOF for direct IO (Eric Sandeen) [1145837]
+- [fs] xfs: zeroing space needs to punch delalloc blocks (Eric Sandeen) [1145837]
+- [fs] xfs: kill buffers over failed write ranges properly (Eric Sandeen) [1145837]
+- [fs] xfs: extra semi-colon breaks a condition (Eric Sandeen) [1145837]
+- [fs] xfs: xfs_check_page_type buffer checks need help (Eric Sandeen) [1145837]
+- [wireless] Backport mwl8k driver from linux 3.16 (Stanislaw Gruszka) [1136324]
+- [wireless] Backport mwifiex driver from linux 3.16 (Stanislaw Gruszka) [1136324]
+- [wireless] Backport brcm80211 drivers from linux 3.16 (Stanislaw Gruszka) [1136324]
+- [wireless] Backport rtlwifi drivers from linux 3.16 (Stanislaw Gruszka) [1136324]
+- [wireless] Backport rt2x00 driver from linux 3.16 (Stanislaw Gruszka) [1136324]
+- [wireless] Backport ath drivers from linux 3.16 (Stanislaw Gruszka) [1136324]
+- [wireless] Backport iwlegacy drivers from linux 3.16 (Stanislaw Gruszka) [1136324]
+- [wireless] Backport iwlwifi driver from linux 3.16 (Stanislaw Gruszka) [1085079 1107878 1136324]
+- [wireless] Backport mac80211 from linux 3.16 (Stanislaw Gruszka) [1136324]
+- [wireless] Backport wireless core from linux 3.16 (Stanislaw Gruszka) [1136324]
+- [kernel] if_ether: add IEEE 802.21 Ethertype (Stanislaw Gruszka) [1136324]
+- [kernel] PTR_RET is now PTR_ERR_OR_ZERO (Stanislaw Gruszka) [1136324]
+- [net] etherdevice: Add ether_addr_equal_unaligned (Stanislaw Gruszka) [1136324]
+
 * Thu Oct 23 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-193.el7]
 - [net] ipv4: fix a race in update_or_create_fnhe() (Hannes Frederic Sowa) [1155155]
 - [net] ipv4: irq safe sk_dst_[re]set() and ipv4_sk_update_pmtu() fix (Hannes Frederic Sowa) [1155155]
-- 
1.7.1