From 43a6e7ed0d7fd520a15cf4fd67a927a9811d7e74 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Thu, 23 Oct 2014 08:15:01 -0400
Subject: [redhat] kernel-3.10.0-192.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index ca9592c..b732b3f 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 191
+RHEL_RELEASE = 192
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 4bbaa23..e357877 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=191
+BUILD:=192
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 1cc6ae4..636ab0f 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,88 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Thu Oct 23 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-192.el7]
+- [infiniband] ocrdma: convert kernel va to pa for mmap in user (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: use right macro in query ah (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: resolve L2 address when creating user AH (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: get vlan tag from ib_qp_attrs (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: add default gid at index 0 (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: do not skip setting deffered_arm (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Report actual value of max_fast_reg_page_list_len (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: report asic-id in query device (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Update sli data structure for endianness (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Obtain SL from device structure (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Update the ocrdma module version string (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Fix a sparse warning (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Initialize the GID table while registering the device (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Increase the size of STAG array in dev structure to 16K (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Add missing adapter mailbox opcodes (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Return proper value for max_mr_size (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Do proper cleanup even if FW is in error state (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Allow only SEND opcode in case of UD QPs (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Avoid reporting wrong completions in case of error CQEs (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Delete AH table if ocrdma_init_hw fails after AH table creation (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Remove hardcoding of the max DPP QPs supported (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Handle shutdown event from be2net driver (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Add hca_type and fixing fw_version string in device atrributes (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Query and initalize the PFC SL (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Avoid posting DPP requests for RDMA READ (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Convert to use simple_open() (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Unregister inet notifier when unloading ocrdma (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Fix warnings about pointer <-> integer casts (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Code clean-up (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Display FW version (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Query controller information (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Support non-embedded mailbox commands (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Handle CQ overrun error (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Display proper value for max_mw (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Use non-zero tag in SRQ posting (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Memory leak fix in ocrdma_dereg_mr() (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Increment abi version count (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Update version string (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: ABI versioning between ocrdma and be2net (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Allow DPP QP creation (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Read ASIC_ID register to select asic_gen (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: SQ and RQ doorbell offset clean up (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: EQ full catastrophe avoidance (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Fix load time panic during GID table init (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Fix traffic class shift (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Add dependency on INET (Doug Ledford) [1081067 1088577]
+- [infiniband] ocrdma: Move ocrdma_inetaddr_event outside of "#if CONFIG_IPV6" (Doug Ledford) [1081067 1088577]
+- [mm] thp: do not allow thp faults to avoid cpuset restrictions (Frank Ramsay) [1149786]
+- [mm] thp: add VM_INIT_DEF_MASK and PRCTL_THP_DISABLE (Frank Ramsay) [1149786]
+- [fs] exec: kill the unnecessary mm->def_flags setting in load_elf_binary() (Frank Ramsay) [1149786]
+- [block] disable entropy contributions for nonrot devices (Mike Snitzer) [852425]
+- [crypto] qat: Enforce valid numa configuration (Nikolay Aleksandrov) [1152642]
+- [crypto] qat: Prevent dma mapping zero length assoc data (Nikolay Aleksandrov) [1152642]
+- [pci] Add ACS quirk for Intel 10G NICs (Alex Williamson) [1141399]
+- [pci] Add ACS quirk for Solarflare SFC9120 & SFC9140 (Alex Williamson) [1131552]
+- [infiniband] ipoib: validate struct ipoib_cb size (Doug Ledford) [1107621 1140601]
+- [infiniband] ipoib: No longer use flush as a parameter (Doug Ledford) [1107621]
+- [infiniband] ipoib: Make ipoib_mcast_stop_thread flush the workqueue (Doug Ledford) [1107621]
+- [infiniband] ipoib: Use dedicated workqueues per interface (Doug Ledford) [1107621]
+- [infiniband] ipoib: change init sequence ordering (Doug Ledford) [1107621]
+- [infiniband] ipoib: fix mcast_dev_flush/mcast_restart_task race (Doug Ledford) [1107621]
+- [infiniband] ipoib: fix MCAST_FLAG_BUSY usage (Doug Ledford) [1107621]
+- [infiniband] ipoib: Make the carrier_on_task race aware (Doug Ledford) [1107621]
+- [infiniband] ipoib: Consolidate rtnl_lock tasks in workqueue (Doug Ledford) [1107621]
+- [infiniband] ipoib: Remove unnecessary test for NULL before debugfs_remove() (Doug Ledford) [1107621]
+- [infiniband] ipoib: Avoid multicast join attempts with invalid P_key (Doug Ledford) [1107621]
+- [infiniband] ipoib: Avoid flushing the workqueue from worker context (Doug Ledford) [1107621]
+- [infiniband] ipoib: Use P_Key change event instead of P_Key polling mechanism (Doug Ledford) [1107621]
+- [infiniband] ipoib: Report operstate consistently when brought up without a link (Doug Ledford) [1107621]
+- [net] svcrdma: remove rdma_create_qp() failure recovery logic (Doug Ledford) [1107621]
+- [net] svcrdma: send_write() must not overflow the device's max sge (Doug Ledford) [1107621]
+- [net] svcrdma: Fence LOCAL_INV work requests (Doug Ledford) [1107621]
+- [net] svcrdma: refactor marshalling logic (Doug Ledford) [1107621]
+- [net] sunrpc: Fix regression in NFSRDMA server (Doug Ledford) [1107621]
+- [net] svcrdma: fix printk when memory allocation fails (Doug Ledford) [1107621]
+- [kernel] uapi: Include socket.h in rdma_user_cm.h (Doug Ledford) [1107621]
+- [infiniband] Return error for unsupported QP creation flags (Doug Ledford) [1107621]
+- [infiniband] Allow build of hw/ and ulp/ subdirectories independently (Doug Ledford) [1107621]
+- [infiniband] ehca: Returns an error on ib_copy_to_udata() failure (Doug Ledford) [1107621]
+- [net] sched: shrink struct qdisc_skb_cb to 28 bytes (Doug Ledford) [1107621 1140601]
+
 * Tue Oct 21 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-191.el7]
 - [fs] fuse: ignore entry-timeout LOOKUP_REVAL (Carlos Maiolino) [1124880]
 - [fs] fuse: Timeout comparison fix (Carlos Maiolino) [1128802]
-- 
1.7.1