From 6806ea8ff4eeab0911fa853380b334c87a41d650 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Mon, 15 Sep 2014 16:13:40 -0400
Subject: [redhat] kernel-3.10.0-159.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index d864b8f..aa8e56b 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 158
+RHEL_RELEASE = 159
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index fe94b1a..9cfece3 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=158
+BUILD:=159
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 3c49e3f..73505dc 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1474,6 +1474,142 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Mon Sep 15 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-159.el7]
+- [ethernet] bna: Support TSO and partial checksum with non-accelerated vlans (Ivan Vecera) [1136912]
+- [ethernet] bna: Remove DEFINE_PCI_DEVICE_TABLE macro use (Ivan Vecera) [1136912]
+- [ethernet] bna: fix performance regression (Ivan Vecera) [1136912]
+- [ethernet] bna: fill the magic in bnad_get_eeprom() instead of validating (Ivan Vecera) [1136912]
+- [ethernet] bna: remove unnecessary break after return (Ivan Vecera) [1136912]
+- [ethernet] bna: use SPEED_UNKNOWN and DUPLEX_UNKNOWN when appropriate (Ivan Vecera) [1136912]
+- [ethernet] bna: get rid of SET_ETHTOOL_OPS (Ivan Vecera) [1136912]
+- [ethernet] bna: remove open-coded skb_cow_head (Ivan Vecera) [1136912]
+- [ethernet] bna: Call dev_kfree_skb_any instead of dev_kfree_skb (Ivan Vecera) [1136912]
+- [ethernet] bna: Convert uses of __constant_<foo> to <foo> (Ivan Vecera) [1136912]
+- [ethernet] bna: Replace large udelay() with mdelay() (Ivan Vecera) [1136912]
+- [ethernet] bna: Use pci_enable_msix_range() instead of pci_enable_msix() (Ivan Vecera) [1136912]
+- [ethernet] bna: bnad code cleanup (Ivan Vecera) [1136912]
+- [ethernet] e1000: fix possible reset_task running after adapter down (John Greene) [1091129]
+- [ethernet] e1000: prevent oops when adapter is being closed and reset simultaneously (John Greene) [1091129]
+- [ethernet] r8169: add missing MODULE_FIRMWARE (Ivan Vecera) [1139247]
+- [ethernet] r8169: add support for RTL8168H and RTL8107E (Ivan Vecera) [1139247]
+- [ethernet] r8169: Remove DEFINE_PCI_DEVICE_TABLE macro use (Ivan Vecera) [1139247]
+- [ethernet] r8169: Enable RX_MULTI_EN for RTL_GIGA_MAC_VER_40 (Ivan Vecera) [1139247]
+- [ethernet] r8169: support IPv6 (Ivan Vecera) [1139247]
+- [ethernet] r8169: use Giant Send (Ivan Vecera) [1139247]
+- [ethernet] r8169: split rtl8169_tso_csum (Ivan Vecera) [1139247]
+- [ethernet] r8169: disable L23 (Ivan Vecera) [1139247]
+- [ethernet] r8169: get rid of SET_ETHTOOL_OPS (Ivan Vecera) [1139247]
+- [ethernet] r8169: Call dev_kfree_skby_any instead of dev_kfree_skb (Ivan Vecera) [1139247]
+- [ethernet] r8169: fix the incorrect tx descriptor version (Ivan Vecera) [1139247]
+- [ethernet] r8169: initialize rtl8169_stats seqlock (Ivan Vecera) [1139247]
+- [ethernet] r8169: delete non-required instances of include <linux/init.h> (Ivan Vecera) [1139247]
+- [ethernet] bnx2: Remove DEFINE_PCI_DEVICE_TABLE macro use (Michal Schmidt) [1092495]
+- [ethernet] bnx2: Rebranding bnx2 driver (Michal Schmidt) [1092495]
+- [ethernet] bnx2: use SPEED_UNKNOWN and DUPLEX_UNKNOWN when appropriate (Michal Schmidt) [1092495]
+- [ethernet] bnx2: Don't build unused suspend/resume functions not enabled (Michal Schmidt) [1092495]
+- [ethernet] bnx2: Don't receive packets when the napi budget == 0 (Michal Schmidt) [1092495]
+- [ethernet] bnx2: Call dev_kfree_skby_any instead of dev_kfree_skb (Michal Schmidt) [1092495]
+- [ethernet] bnx2: Use pci_enable_msix_range() instead of pci_enable_msix() (Michal Schmidt) [1092495]
+- [ethernet] bnx2: delete non-required instances of include <linux/init.h> (Michal Schmidt) [1092495]
+- [ethernet] ixgbevf: Remove unused get_supported_physical_layer pointer (John Greene) [1091124]
+- [ethernet] ixgbevf: use SPEED_UNKNOWN and DUPLEX_UNKNOWN when appropriate (John Greene) [1091124]
+- [ethernet] ixgbevf: get rid of SET_ETHTOOL_OPS (John Greene) [1091124]
+- [ethernet] ixgbevf: remove 82599 from the module description (John Greene) [1091124]
+- [ethernet] ixgbevf: remove open-coded skb_cow_head (John Greene) [1091124]
+- [ethernet] ixgbevf: Add bit to mark work queue initialization (John Greene) [1091124]
+- [ethernet] ixgbevf: Fix rcu warnings induced by LER (John Greene) [1091124]
+- [ethernet] ixgbevf: Change ixgbe_read_reg to ixgbevf_read_reg (John Greene) [1091124]
+- [ethernet] ixgbevf: Additional adapter removal checks (John Greene) [1091124]
+- [ethernet] ixgbevf: Check for adapter removal on register writes (John Greene) [1091124]
+- [ethernet] ixgbevf: Check register reads for adapter removal (John Greene) [1091124]
+- [ethernet] ixgbevf: Make the ethtool register test use accessors (John Greene) [1091124]
+- [ethernet] ixgbevf: Use static inlines instead of macros (John Greene) [1091124]
+- [ethernet] ixgbevf: Convert uses of __constant_<foo> to <foo> (John Greene) [1091124]
+- [ethernet] ixgbevf: Protect ixgbevf_down with __IXGBEVF_DOWN bit (John Greene) [1091124]
+- [ethernet] ixgbevf: Indicate removal state explicitly (John Greene) [1091124]
+- [ethernet] ixgbevf: delete unneeded call to pci_set_power_state (John Greene) [1091124]
+- [ethernet] ixgbevf: fix skb->pkt_type checks (John Greene) [1091124]
+- [ethernet] ixgbevf: add check for CHECKSUM_PARTIAL when doing TSO (John Greene) [1091124]
+- [ethernet] ixgbevf: fix handling of tx checksumming (John Greene) [1091124]
+- [ethernet] ixgbevf: Use pci_enable_msix_range() instead of pci_enable_msix() (John Greene) [1091124]
+- [ethernet] ixgbevf: merge ixgbevf_tx_map and ixgbevf_tx_queue into a single function (John Greene) [1091124]
+- [ethernet] ixgbevf: redo dma mapping using the tx buffer info (John Greene) [1091124]
+- [ethernet] ixgbevf: make the first tx_buffer a repository for most of the skb info (John Greene) [1091124]
+- [ethernet] ixgbevf: add tx counters (John Greene) [1091124]
+- [ethernet] ixgbevf: remove counters for Tx/Rx checksum offload (John Greene) [1091124]
+- [ethernet] ixgbevf: move ring specific stats into ring specific structure (John Greene) [1091124]
+- [ethernet] ixgbevf: make use of the dev pointer in the ixgbevf_ring struct (John Greene) [1091124]
+- [ethernet] ixgbevf: bump version (John Greene) [1091124]
+- [ethernet] ixgbevf: create function for all of ring init (John Greene) [1091124]
+- [ethernet] ixgbevf: Convert ring storage form pointer to an array to array of pointers (John Greene) [1091124]
+- [ethernet] ixgbevf: use pci drvdata correctly in ixgbevf_suspend() (John Greene) [1091124]
+- [ethernet] ixgbevf: set the disable state when ixgbevf_qv_disable is called (John Greene) [1091124]
+- [ethernet] ixgbevf: add DCB configuration into queue setup (John Greene) [1091124]
+- [ethernet] ixgbe: Focus config of head, tail ntc, and ntu all into a single function (John Greene) [1091124]
+- [ethernet] ixgbe: cleanup IXGBE_DESC_UNUSED (John Greene) [1091124]
+- [ethernet] ixgbevf: remove redundant workaround (John Greene) [1091124]
+- [ethernet] ixgbevf: Add zero_base handler to network statistics (John Greene) [1091124]
+- [ethernet] ixgbevf: add BP_EXTENDED_STATS for CONFIG_NET_RX_BUSY_POLL (John Greene) [1091124]
+- [ethernet] ixgbevf: implement CONFIG_NET_RX_BUSY_POLL (John Greene) [1091124]
+- [ethernet] ixgbevf: have clean_rx_irq return total_rx_packets cleaned (John Greene) [1091124]
+- [ethernet] ixgbevf: add ixgbevf_rx_skb (John Greene) [1091124]
+- [ethernet] ixgbevf: bump driver version (John Greene) [1091124]
+- [ethernet] ixgbevf: Remove extern from function prototypes (John Greene) [1091124]
+- [ethernet] ixgbevf: Adds function to set PSRTYPE register (John Greene) [1091124]
+- [ethernet] ixgbevf: Miscellaneous conversions to ETH_ALEN (John Greene) [1091124]
+- [ethernet] ixgbevf: add wait for Rx queue disable (John Greene) [1091123]
+- [ethernet] ixgbevf: fix 32-bit DMA mask handling (John Greene) [1091123]
+- [ethernet] ixgbe: fix message terminations (John Greene) [1091123]
+- [ethernet] ixgbe: clean up Rx time stamping code (John Greene) [1091123]
+- [ethernet] ixgbe: remove open-coded skb_cow_head (John Greene) [1091123]
+- [ethernet] ixgbe: Add bit to mark service task initialization (John Greene) [1091123]
+- [ethernet] ixgbe: Fix rcu warnings induced by LER (John Greene) [1091123]
+- [ethernet] ixgbe: Indicate removal state explicitly (John Greene) [1091123]
+- [ethernet] ixgbe: fix ixgbe_check_reset_blocked() declaration (John Greene) [1091123]
+- [ethernet] ixgbe: fix race conditions on queuing skb for HW time stamp (John Greene) [1091123]
+- [ethernet] ixgbe: never generate both software and hardware timestamps (John Greene) [1091123]
+- [ethernet] ixgbe: remove redundant if clause from PTP work (John Greene) [1091123]
+- [ethernet] ixgbe: Break recursion in case of removal (John Greene) [1091123]
+- [ethernet] ixgbe: Stop cacheing if the MNG FW enabled (John Greene) [1091123]
+- [ethernet] ixgbe: clean up ixgbe_atr_compute_perfect_hash_82599 (John Greene) [1091123]
+- [ethernet] ixgbe: use ixgbe_read_pci_cfg_word (John Greene) [1091123]
+- [ethernet] ixgbe: remove unused media type (John Greene) [1091123]
+- [ethernet] ixgbe: fix ixgbe_setup_mac_link_82599 autoc variables (John Greene) [1091123]
+- [ethernet] ixgbe: fix ixgbe_stop_mac_link_on_d3_82599 to check mng correctly (John Greene) [1091123]
+- [ethernet] ixgbe: check Core Clock Disable bit (John Greene) [1091123]
+- [ethernet] ixgbe: fix errors related to protected AUTOC calls (John Greene) [1091123]
+- [ethernet] ixgbe: Convert uses of __constant_<foo> to <foo> (John Greene) [1091123]
+- [ethernet] ixgbe: Don't receive packets when the napi budget == 0 (John Greene) [1091123]
+- [ethernet] ixgbe: add ixgbe_write_pci_cfg_word with ixgbe_removed check (John Greene) [1091123]
+- [ethernet] ixgbe: fix some multiline hw_dbg prints (John Greene) [1091123]
+- [ethernet] ixgbe: fixup header for ixgbe_set_rxpba_82598 (John Greene) [1091123]
+- [ethernet] ixgbe: add Linux NICS mailing list to contact info (John Greene) [1091123]
+- [ethernet] ixgbe: move setting rx_pb_size into get_invariants (John Greene) [1091123]
+- [ethernet] ixgbe: Fix format string in ixgbe_fcoe.c (John Greene) [1091123]
+- [ethernet] ixgbe: ixgbe calls skb_set_hash (John Greene) [1091123]
+- [ethernet] ixgbe: implement SIOCGHWTSTAMP ioctl (John Greene) [1091123]
+- [ethernet] ixgbe: Check config reads for removal (John Greene) [1091123]
+- [ethernet] ixgbe: Fix up some ethtool results when adapter is removed (John Greene) [1091123]
+- [ethernet] ixgbe: Restore hw_addr in LER recovery paths (John Greene) [1091123]
+- [ethernet] ixgbe: Additional adapter removal checks (John Greene) [1091123]
+- [ethernet] ixgbe: Check for adapter removal on register writes (John Greene) [1091123]
+- [ethernet] ixgbe: Check register reads for adapter removal (John Greene) [1091123]
+- [ethernet] ixgbe: Use static inlines instead of macros (John Greene) [1091123]
+- [ethernet] ixgbe: Add check for FW veto bit (John Greene) [1091123]
+- [ethernet] ixgbe: fix bit toggled for 82599 reset fix (John Greene) [1091123]
+- [ethernet] ixgbe: collect all 82599 AUTOC code in one function (John Greene) [1091123]
+- [ethernet] ixgbe: Remove extern from function prototypes (John Greene) [1091123]
+- [ethernet] ixgbe: fix to use correct timeout interval for memory read completion (John Greene) [1091123]
+- [ethernet] ixgbe: Add WoL support for a new device (John Greene) [1091123]
+- [ethernet] ixgbe: don't use magic size number to assign ptp_caps.name (John Greene) [1091123]
+- [ethernet] ixgbe: modify behavior on receiving a HW ECC error (John Greene) [1091123]
+- [ethernet] ixgbe: Use pci_enable_msix_range() instead of pci_enable_msix() (John Greene) [1091123]
+- [ethernet] ixgbe: bump version number (John Greene) [1091123]
+- [ethernet] ixgbe: add braces around else condition in ixgbe_qv_lock_* calls (John Greene) [1091123]
+- [ethernet] ixgbe: fix qv_lock_napi call in ixgbe_napi_disable_all (John Greene) [1091123]
+- [ethernet] ixgbe: Reduce memory consumption with larger page sizes (John Greene) [1091123]
+- [ethernet] ixgbe: Cleanup the use of tabs and spaces (John Greene) [1091124 1091123]
+
 * Fri Sep 12 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-158.el7]
 - [net] ipv4: implement igmp_qrv sysctl to tune igmp robustness variable (Hannes Frederic Sowa) [1110118]
 - [net] ipv6: add sysctl_mld_qrv to configure query robustness variable (Hannes Frederic Sowa) [1110118]
-- 
1.7.1