From 20968c81af8dc52435d0a8fcc4a796806b9c0327 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Thu, 20 Nov 2014 08:01:53 -0500
Subject: [redhat] kernel-3.10.0-206.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index b1a28ed..31644cc 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 205
+RHEL_RELEASE = 206
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index db03475..40ad649 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=205
+BUILD:=206
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index d641a94..d5229c8 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,33 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Thu Nov 20 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-206.el7]
+- [x86] quirks: Print the Intel graphics stolen memory range (Rob Clark) [1154053]
+- [x86] quirks: Add Intel graphics stolen memory quirk for gen2 platforms (Rob Clark) [1154053]
+- [x86] quirks: Add vfunc for Intel graphics stolen memory base address (Rob Clark) [1154053]
+- [x86] quirks: use gen6 stolen detection for VLV (Rob Clark) [1154053]
+- [x86] quirks: support GMS and GGMS changes on i915/bdw (Rob Clark) [1154053]
+- [x86] quirks: add early quirk for reserving Intel graphics stolen memory v5 (Rob Clark) [1154053]
+- [net] vmxnet3: fix netpoll race condition (Neil Horman) [1158001]
+- [virt] virtio_balloon: update_balloon_size() - update correct field (Luiz Capitulino) [1163567]
+- [firmware] memmap: don't create memmap sysfs of same firmware_map_entry (Takahiro MUNEDA) [1160173]
+- [mm] memory-hotplug: clear pgdat which is allocated by bootmem in try_offline_node() (Larry Woodman) [1156393]
+- [kernel] add panic_on_warn (Prarit Bhargava) [1163852]
+- [virt] hyperv: Fix the total_data_buflen in send path (Jason Wang) [1156305]
+- [virt] hyperv: Add handling of IP header with option field in netvsc_set_hash() (Jason Wang) [1156305]
+- [virt] hyperv: Fix a bug in netvsc_start_xmit() (Jason Wang) [1156305]
+- [virt] hyperv: Fix a bug in netvsc_send() (Jason Wang) [1156305]
+- [powerpc] kexec: adjust crashkernel reservation for 2GB-4GB systems (Gustavo Duarte) [1074924]
+- [virt] kvm/ioapic: conditionally delay irq delivery duringeoi broadcast (John Snow) [921526]
+- [fs] file_table: get rid of s_files and files_lock (Gustavo Duarte) [1112805]
+- [fs] super: uninline destroy_super(), consolidate alloc_super() (Gustavo Duarte) [1112805]
+- [ethernet] mlx4: Advertize encapsulation offloads features only when VXLAN tunnel is set (Florian Westphal) [1097478]
+- [ethernet] mlx4: Avoid leaking steering rules on flow creation error flow (Florian Westphal) [1097478]
+- [ethernet] mlx4: Don't attempt to TX offload the outer UDP checksum for VXLAN (Florian Westphal) [1097478]
+- [scsi] bnx2fc: fix tgt spinlock locking (Maurizio Lombardi) [1165169]
+- [scsi] TUR path is down after adapter gets reset with multipath (Ewan Milne) [1153738]
+- [scsi] call device handler for failed TUR command (Ewan Milne) [1153738]
+
 * Sat Nov 15 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-205.el7]
 - [mm] shmem: fix splicing from a hole while it's punched (Denys Vlasenko) [1118245] {CVE-2014-4171}
 - [mm] shmem: fix faulting into a hole, not taking i_mutex (Denys Vlasenko) [1118245] {CVE-2014-4171}
-- 
1.7.1