From c2974ba500f3ab64b4ea3f0814439bcce1f9500f Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Thu, 9 Oct 2014 17:39:37 -0400
Subject: [redhat] kernel-3.10.0-186.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index e453171..7104b0a 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 185
+RHEL_RELEASE = 186
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 6eb2b00..415408d 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=185
+BUILD:=186
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 4747c43..441cbaa 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,36 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Thu Oct 09 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-186.el7]
+- [kernel] sched/balancing: Reduce the rate of needless idle load balancing (Larry Woodman) [1120318]
+- [kernel] sched/fair: Implement fast idling of CPUs when the system is partially loaded (Larry Woodman) [1120318]
+- [kernel] sched: Skip double execution of pick_next_task_fair() (Larry Woodman) [1120318]
+- [virt] kvm: nested vmx - disable perf cpuid reporting (Marcelo Tosatti) [1083969]
+- [mm] slub: Handle NULL parameter in kmem_cache_flags (Ilya Dryomov) [1143028]
+- [crypto] algif: avoid excessive use of socket buffer in skcipher (Ondrej Kozina) [1136075]
+- [kernel] time/clocksource: Move cycle_last validation to core code (Prarit Bhargava) [1148398]
+- [kernel] time/clocksource: Make delta calculation a function (Prarit Bhargava) [1148398]
+- [mm] vmscan: clear kswapd's special reclaim powers before exiting (Motohiro Kosaki) [810042]
+- [x86] mm/numa: fix array index overflow when synchronizing nid to memblock.reserved (Motohiro Kosaki) [810042]
+- [x86] mm/numa: initialize numa_kernel_nodes in numa_clear_kernel_node_hotplug() (Motohiro Kosaki) [810042]
+- [mm] page_alloc: make movable_node have higher priority (Motohiro Kosaki) [810042]
+- [x86] memblock, mem_hotplug: make memblock skip hotpluggable regions if needed (Motohiro Kosaki) [810042]
+- [x86] mm: mark all nodes the kernel resides un-hotpluggable (Motohiro Kosaki) [810042]
+- [x86] mm: mark hotpluggable memory in memblock (Motohiro Kosaki) [810042]
+- [mm] memblock: make memblock_set_node() support different memblock_type (Motohiro Kosaki) [810042]
+- [mm] memblock/mem_hotplug: introduce MEMBLOCK_HOTPLUG flag to mark hotpluggable regions (Motohiro Kosaki) [810042]
+- [mm] memblock: numa - introduce flags field into memblock (Motohiro Kosaki) [810042]
+- [mm] memblock: debug - correct displaying of upper memory boundary (Motohiro Kosaki) [810042]
+- [x86] mm: get pg_data_t's memory from other node (Motohiro Kosaki) [810042]
+- [mm] mem-hotplug: introduce movable_node boot option (Motohiro Kosaki) [810042]
+- [x86] kernel/setup: do reserve_crashkernel() after SRAT is parsed (Motohiro Kosaki) [810042]
+- [x86] mem-hotplug: support initialize page tables in bottom-up (Motohiro Kosaki) [810042]
+- [x86] mm: factor out of top-down direct mapping setup (Motohiro Kosaki) [810042]
+- [mm] memblock: introduce bottom-up allocation mode (Motohiro Kosaki) [810042]
+- [mm] memblock: factor out of top-down allocation (Motohiro Kosaki) [810042]
+- [x86] mm: Add 'step_size' comments to init_mem_mapping() (Motohiro Kosaki) [810042]
+- [x86] srat: Print Hot-Pluggable Field in SRAT (Motohiro Kosaki) [810042]
+
 * Thu Oct 09 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-185.el7]
 - [fs] udf: Avoid infinite loop when processing indirect ICBs (Jacob Tanenbaum) [1142322] {CVE-2014-6410}
 - [fs] eliminate BUG() call when there's an unexpected lock on file close (Frank Sorenson) [1148130]
-- 
1.7.1