From 6e92a4be759ae49facb4a30409f4f5afb9adaa63 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 16 Sep 2014 12:54:28 -0400
Subject: [redhat] kernel-3.10.0-161.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 6cb11c3..78f4f42 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 160
+RHEL_RELEASE = 161
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index f8f1c36..0cfe837 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=160
+BUILD:=161
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 0d7192b..346ac02 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1474,6 +1474,146 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Sep 16 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-161.el7]
+- [target] iscsi: Explicily clear login response PDU in exception path (Andy Grover) [1129387]
+- [target] iscsi: Avoid rejecting incorrect ITT for Data-Out (Andy Grover) [1129387]
+- [target] Fix left-over se_lun->lun_sep pointer OOPs (Andy Grover) [1129387]
+- [infiniband] ulp/isert: Wait for proper cleanup before unloading (Andy Grover) [1129387]
+- [infiniband] ulp/isert: Improve cm events handling (Andy Grover) [1129387]
+- [target] Explicitly clear ramdisk_mcp backend pages (Andy Grover) [1129387]
+- [target] Report correct response length for some commands (Andy Grover) [1129387]
+- [target] iscsi: Fix ABORT_TASK + connection reset iscsi_queue_req memory leak (Andy Grover) [1129387]
+- [target] Use complete_all for se_cmd->t_transport_stop_comp (Andy Grover) [1129387]
+- [target] lio: Set CMD_T_ACTIVE bit for Task Management Requests (Andy Grover) [1129387]
+- [target] iscsi, iser: Fix hangs in connection teardown (Andy Grover) [1129387]
+- [infiniband] iser: Bail from accept_np if np_thread is trying to close (Andy Grover) [1129387]
+- [target] iscsi: Reject mutual authentication with reflected CHAP_C (Andy Grover) [1129387]
+- [target] Fix alua_access_state attribute OOPs for un-configured devices (Andy Grover) [1129387]
+- [infiniband] iser: Add missing target_put_sess_cmd for ImmedateData failure (Andy Grover) [1129387]
+- [target] Don't allow setting WC emulation if device doesn't support (Andy Grover) [1129387]
+- [infiniband] iser: Add missing se_cmd put for WRITE_PENDING in tx_comp_err (Andy Grover) [1129387]
+- [target] iscsi: Fix ERL=2 ASYNC_EVENT connection pointer bug (Andy Grover) [1129387]
+- [infiniband] ib_srpt: Use correct ib_sg_dma primitives (Andy Grover) [1129387]
+- [infiniband] iser: Fix post_send_buf_count for RDMA READ/WRITE (Andy Grover) [1129387]
+- [target] iscsi, iser: Fix isert_conn->state hung shutdown issues (Andy Grover) [1129387]
+- [target] iscsi, iser:  Use list_del_init for ->i_conn_node (Andy Grover) [1129387]
+- [target] iscsi: Fix iscsit_get_tpg_from_np tpg_state bug (Andy Grover) [1129387]
+- [target] Update hw_max_sectors based on current block_size (Andy Grover) [1129387]
+- [target] iscsi: Fix-up all zero data-length CDBs with R/W_BIT set (Andy Grover) [1129387]
+- [target] Allow READ_CAPACITY opcode in ALUA Standby access state (Andy Grover) [1129387]
+- [target] iscsi: Fix wrong buffer / buffer overrun in iscsi_change_param_value() (Andy Grover) [1129387]
+- [target] iscsi: Fix multi network portal shutdown regression (Andy Grover) [1129387]
+- [target] iscsi, iser: Avoid accepting transport connections during stop stage (Andy Grover) [1129387]
+- [infiniband] ulp/isert: Fix iscsit_accept_np and rdma_cm racy flow (Andy Grover) [1129387]
+- [infiniband] ulp/isert: Fix wrong connection requests list addition (Andy Grover) [1129387]
+- [scsi] hpsa: do not require board "not ready" status after hard reset (Tomas Henzl) [1069185]
+- [scsi] hpsa: make tech preview info more verbose (Tomas Henzl) [1083556]
+- [scsi] hpsa: fix bad -ENOMEM return value in hpsa_big_passthru_ioctl (Tomas Henzl) [1069185]
+- [scsi] hpsa: remove online devices from offline device list (Tomas Henzl) [1069185]
+- [scsi] hpsa: fix non-x86 builds (Tomas Henzl) [1069185]
+- [scsi] hpsa: do not unconditionally copy sense data (Tomas Henzl) [1069185]
+- [scsi] hpsa: fix 6-byte READ/WRITE with 0 length data xfer (Tomas Henzl) [1069185]
+- [scsi] hpsa: make hpsa_init_one return -ENOMEM if allocation of h->lockup_detected fails (Tomas Henzl) [1069185]
+- [scsi] hpsa: fix handling of hpsa_volume_offline return value (Tomas Henzl) [1069185]
+- [scsi] hpsa: return -ENOMEM not -1 on kzalloc failure in hpsa_get_device_id (Tomas Henzl) [1069185]
+- [scsi] hpsa: remove messages about volume status VPD inquiry page not supported (Tomas Henzl) [1069185]
+- [scsi] hpsa: report check condition even if no sense data present for ioaccel2 mode (Tomas Henzl) [1069185]
+- [scsi] hpsa: remove bad unlikely annotation from device list updating code (Tomas Henzl) [1069185]
+- [scsi] hpsa: fix event filtering to prevent excessive rescans with old firmware (Tomas Henzl) [1069185]
+- [scsi] hpsa: kill annoying messages about SSD Smart Path retries (Tomas Henzl) [1069185]
+- [scsi] hpsa: define extended_report_lun_entry data structure (Tomas Henzl) [1069185]
+- [scsi] hpsa: Rearrange start_io to avoid one unlock/lock sequence in main io path (Tomas Henzl) [1069185]
+- [scsi] hpsa: avoid unnecessary readl on every command submission (Tomas Henzl) [1069185]
+- [scsi] hpsa: use per-cpu variable for lockup_detected (Tomas Henzl) [1069185]
+- [scsi] hpsa: remove unnecessary pci_set_drvdata() (Tomas Henzl) [1069185]
+- [scsi] hpsa: set irq affinity hints to route MSI-X vectors across CPUs (Tomas Henzl) [1069185]
+- [scsi] hpsa: allocate reply queues individually (Tomas Henzl) [1069185]
+- [scsi] hpsa: choose number of reply queues more intelligently (Tomas Henzl) [1069185]
+- [scsi] hpsa: remove dev_dbg() calls from hot paths (Tomas Henzl) [1069185]
+- [scsi] hpsa: use gcc aligned attribute instead of manually padding structs (Tomas Henzl) [1069185]
+- [scsi] hpsa: change doorbell reset delay to ten seconds (Tomas Henzl) [1069185]
+- [scsi] hpsa: allow passthru ioctls to work with bidirectional commands (Tomas Henzl) [1069185]
+- [scsi] hpsa: remove unused fields from struct ctlr_info (Tomas Henzl) [1069185]
+- [scsi] hpsa: fix bad comparison of signed with unsigned in hpsa_update_scsi_devices (Tomas Henzl) [1069185]
+- [scsi] hpsa: do not ignore failure of sense controller parameters command (Tomas Henzl) [1069185]
+- [scsi] hpsa: fix memory leak in hpsa_hba_mode_enabled (Tomas Henzl) [1069185]
+- [scsi] hpsa: Checking for a NULL return from a kzalloc call (Tomas Henzl) [1069185]
+- [scsi] hpsa: fix NULL dereference in hpsa_put_ctlr_into_performant_mode() (Tomas Henzl) [1069185]
+- [scsi] hpsa: update driver version to 3.4.4-1 (Tomas Henzl) [1069185]
+- [scsi] hpsa: fix bad endif placement in RAID 5 mapper code (Tomas Henzl) [1069185]
+- [scsi] hpsa: Do not zero fields of ioaccel2 command structure twice (Tomas Henzl) [1069185]
+- [scsi] hpsa: Add hba mode to the hpsa driver (Tomas Henzl) [1069185]
+- [scsi] hpsa: remove unused struct request from CommandList (Tomas Henzl) [1069185]
+- [scsi] hpsa: increase the probability of a reported success after a device reset (Tomas Henzl) [1069185]
+- [scsi] hpsa: bring format-in-progress drives online when ready (Tomas Henzl) [1069185]
+- [scsi] hpsa: remove unused kthread.h header (Tomas Henzl) [1069185]
+- [scsi] hpsa: Add support for a few HP Storage controllers (Tomas Henzl) [1069185]
+- [scsi] hpsa: add HP/3PAR vendor id to pci_ids.h (Tomas Henzl) [1069185]
+- [scsi] hpsa: add sysfs debug switch for raid map debugging messages (Tomas Henzl) [1069185]
+- [scsi] hpsa: improve error messages for driver initiated commands (Tomas Henzl) [1069185]
+- [scsi] hpsa: only do device rescan for certain events (Tomas Henzl) [1069185]
+- [scsi] hpsa: when switching out of accel mode await only accel command completions (Tomas Henzl) [1069185]
+- [scsi] hpsa: add controller base data-at-rest encryption compatibility ioaccel2 (Tomas Henzl) [1069185]
+- [scsi] hpsa: update source file copyrights (Tomas Henzl) [1069185]
+- [scsi] hpsa: retry certain ioaccel error cases on the RAID path (Tomas Henzl) [1069185]
+- [scsi] hpsa: do not inquire for unsupported ioaccel status vpd page (Tomas Henzl) [1069185]
+- [scsi] hpsa: allow VPD page zero to be queried (Tomas Henzl) [1069185]
+- [scsi] hpsa: rescan devices on ioaccel2 error (Tomas Henzl) [1069185]
+- [scsi] hpsa: allow user to disable accelerated i/o path (Tomas Henzl) [1069185]
+- [scsi] hpsa: complete the ioaccel raidmap code (Tomas Henzl) [1069185]
+- [scsi] hpsa: make device update copy the raid map also (Tomas Henzl) [1069185]
+- [scsi] hpsa: add task management for ioaccel mode 2 (Tomas Henzl) [1069185]
+- [scsi] hpsa: teach hpsa_device_reset to do either target or lun reset (Tomas Henzl) [1069185]
+- [scsi] hpsa: get ioaccel mode 2 i/o working (Tomas Henzl) [1069185]
+- [scsi] hpsa: initialize controller to perform io accelerator mode 2 (Tomas Henzl) [1069185]
+- [scsi] hpsa: get physical device handles for io accel mode 2 as well as mode 1 (Tomas Henzl) [1069185]
+- [scsi] hpsa: do ioaccel mode 2 resource allocations (Tomas Henzl) [1069185]
+- [scsi] hpsa: Acknowledge controller events in ioaccell mode 2 as well as mode 1 (Tomas Henzl) [1069185]
+- [scsi] hpsa: add ioaccel mode 2 structure definitions (Tomas Henzl) [1069185]
+- [scsi] hpsa: complain if physical or logical aborts are not supported (Tomas Henzl) [1069185]
+- [scsi] hpsa: add hp_ssd_smart_path_enabled sysfs attribute (Tomas Henzl) [1069185]
+- [scsi] hpsa: do not rescan controllers known to be locked up (Tomas Henzl) [1069185]
+- [scsi] hpsa: poll controller to detect device change event (Tomas Henzl) [1069185]
+- [scsi] hpsa: update raid offload status on device rescan (Tomas Henzl) [1069185]
+- [scsi] hpsa: add ioaccell mode 1 RAID offload support (Tomas Henzl) [1069185]
+- [scsi] hpsa: fix task management for mode-1 ioaccell path (Tomas Henzl) [1069185]
+- [scsi] hpsa: only allow REQ_TYPE_FS to use fast path (Tomas Henzl) [1069185]
+- [scsi] hpsa: add support for 'fastpath' i/o (Tomas Henzl) [1069185]
+- [scsi] hpsa: mark last scatter gather element as the last (Tomas Henzl) [1069185]
+- [scsi] hpsa: use extended report luns command for HP SSD SmartPath (Tomas Henzl) [1069185]
+- [scsi] hpsa: fixup MSI-X registration (Tomas Henzl) [1069185]
+- [scsi] hpsa: prevent stalled i/o (Tomas Henzl) [1069185]
+- [scsi] hpsa: cap CCISS_PASSTHRU at 20 concurrent commands (Tomas Henzl) [1069185]
+- [scsi] hpsa: add MSA 2040 to list of external target devices (Tomas Henzl) [1069185]
+- [scsi] hpsa: fix memory leak in CCISS_BIG_PASSTHRU ioctl (Tomas Henzl) [1069185]
+- [scsi] hpsa: remove unneeded include of seq_file.h (Tomas Henzl) [1069185]
+- [scsi] hpsa: add 5 second delay after doorbell reset (Tomas Henzl) [1069185]
+- [scsi] hpsa: do not attempt to flush the cache on locked up controllers (Tomas Henzl) [1069185]
+- [scsi] bnx2i: Make boot_nic entry visible in the sysfs session objects (Maurizio Lombardi) [1139139]
+- [scsi] bnx2fc: fix incorrect DMA memory mapping in bnx2fc_unmap_sg_list() (Maurizio Lombardi) [1102526]
+- [ethernet] cnic: Replace rcu_dereference() with rcu_access_pointer() (Maurizio Lombardi) [1089401]
+- [ethernet] cnic: Rebranding cnic driver (Maurizio Lombardi) [1089401]
+- [ethernet] cnic: Fix missing ISCSI_KEVENT_IF_DOWN message (Maurizio Lombardi) [1089401]
+- [ethernet] cnic: Update version to 2.5.20 and copyright year (Maurizio Lombardi) [1089401]
+- [ethernet] cnic: Use proper ulp_ops for per device operations (Maurizio Lombardi) [1089401]
+- [ethernet] broadcom: Remove extern from function prototypes (Maurizio Lombardi) [1089401]
+- [scsi] bnx2i: Update driver version to 2.7.10.1 (Maurizio Lombardi) [1089400]
+- [scsi] bnx2i: Rebranding bnx2i driver (Maurizio Lombardi) [1089400]
+- [scsi] bnx2i, be2iscsi: fix custom stats length (Maurizio Lombardi) [1089400]
+- [ethernet] cnic, bnx2i, bnx2fc: Fix inconsistent use of page size (Maurizio Lombardi) [1089400 1089399 1089401]
+- [scsi] bnx2fc: Rebranding bnx2fc driver (Maurizio Lombardi) [1089399]
+- [scsi] bnx2fc: do not scan uninitialized lists in case of error (Maurizio Lombardi) [1089399]
+- [scsi] bnx2fc: fix memory leak in bnx2fc_allocate_hash_table() (Maurizio Lombardi) [1089399]
+- [scsi] bnx2fc: fix memory leak and potential NULL pointer dereference (Maurizio Lombardi) [1089399]
+- [scsi] bnx2fc: remove unused variable hash_table_size (Maurizio Lombardi) [1089399]
+- [scsi] bnx2fc: Updated version to 2.4.2 (Maurizio Lombardi) [1089399]
+- [scsi] bnx2fc: Fixed the handling for the SCSI retry delay (Maurizio Lombardi) [1089399]
+- [scsi] be2iscsi: Bump driver version (Rob Evers) [1130072]
+- [scsi] be2iscsi: Fix processing CQE before connection resources are freed (Rob Evers) [1130072]
+- [scsi] be2iscsi: Fix updating the boot enteries in sysfs (Rob Evers) [1130072]
+- [scsi] be2iscsi: Fix the copyright year (Rob Evers) [1130072]
+- [scsi] be2iscsi: Fix the sparse warning introduced in previous submission (Rob Evers) [1130072]
+
 * Mon Sep 15 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-160.el7]
 - [x86] module: work around kabi module breakage when 16K stacks are enabled (Kyle McMartin) [1108378]
 - [kernel] redhat: bump RHEL_MINOR to 1 (Kyle McMartin) [1108378]
-- 
1.7.1