From fb4859e3f4905a324d67d23695230cab12d4e7c6 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Mon, 15 Sep 2014 23:34:24 -0400
Subject: [redhat] kernel-3.10.0-160.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 1ae9eeb..6cb11c3 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 159
+RHEL_RELEASE = 160
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 9cfece3..f8f1c36 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=159
+BUILD:=160
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 73505dc..0d7192b 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1474,6 +1474,14 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Mon Sep 15 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-160.el7]
+- [x86] module: work around kabi module breakage when 16K stacks are enabled (Kyle McMartin) [1108378]
+- [kernel] redhat: bump RHEL_MINOR to 1 (Kyle McMartin) [1108378]
+- [kernel] modules: export check_module_rhelversion (Kyle McMartin) [1108378]
+- [kernel] modules: Add module_ext struct (Prarit Bhargava) [1110315]
+- [kernel] modules: add rhelversion MODULE_INFO tag (Kyle McMartin) [1110315]
+- [x86] expand 64-bit kernel stack to 16K (Johannes Weiner) [1108378]
+
 * Mon Sep 15 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-159.el7]
 - [ethernet] bna: Support TSO and partial checksum with non-accelerated vlans (Ivan Vecera) [1136912]
 - [ethernet] bna: Remove DEFINE_PCI_DEVICE_TABLE macro use (Ivan Vecera) [1136912]
-- 
1.7.1