From b029c5996e355898b82e8b71b98bd9f5ab06f8a4 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 23 Dec 2014 09:59:03 -0500
Subject: [redhat] kernel-3.10.0-220.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index fcb14e0..5db3915 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 219
+RHEL_RELEASE = 220
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 0c61af9..bd6104d 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=219
+BUILD:=220
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 0326363..3580c78 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1493,6 +1493,73 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Dec 23 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-220.el7]
+- [scsi] libcxgbi: fix freeing skb prematurely (Sai Vemuri) [1174982]
+- [scsi] cxgb4i: use set_wr_txq() to set tx queues (Sai Vemuri) [1174982]
+- [scsi] cxgb4i: handle non-pdu-aligned rx data (Sai Vemuri) [1174982]
+- [scsi] cxgb4i: additional types of negative advice (Sai Vemuri) [1174982]
+- [scsi] cxgb4i: set the max. pdu length in firmware (Sai Vemuri) [1174982]
+- [scsi] cxgb4i: fix credit check for tx_data_wr (Sai Vemuri) [1174982]
+- [scsi] cxgb4i: fix tx immediate data credit check (Sai Vemuri) [1174982]
+- [net] ipv6: update Destination Cache entries when gateway turn into host (Jiri Pirko) [1114781]
+- [net] ipsec: Don't update the pmtu on ICMPV6_DEST_UNREACH (Herbert Xu) [1158771]
+- [s390] zfcp: remove access control tables interface (port leftovers) (Hendrik Brueckner) [1173553]
+- [x86] perf: Use extended offcore mask on Haswell (Don Zickus) [1170795]
+- [fs] ovl: ovl_dir_fsync() cleanup (David Howells) [985875]
+- [fs] ovl: pass dentry into ovl_dir_read_merged() (David Howells) [985875]
+- [fs] ovl: use lockless_dereference() for upperdentry (David Howells) [985875]
+- [fs] ovl: allow filenames with comma (David Howells) [985875]
+- [fs] ovl: fix race in private xattr checks (David Howells) [985875]
+- [fs] ovl: fix remove/copy-up race (David Howells) [985875]
+- [fs] ovl: rename filesystem type to "overlay" (David Howells) [985875]
+- [fs] Don't warn if both ->rename() and ->rename2() iops are defined (David Howells) [985875]
+- [fs] overlayfs: Fix the kABI for overlayfs (David Howells) [985875]
+- [fs] overlayfs: don't poison cursor (David Howells) [985875]
+- [fs] overlayfs: initialize ->is_cursor (David Howells) [985875]
+- [fs] overlayfs: fix lockdep misannotation (David Howells) [985875]
+- [fs] overlayfs: fix check for cursor (David Howells) [985875]
+- [fs] overlayfs: barriers for opening upper-layer directory (David Howells) [985875]
+- [kernel] rcu: Provide counterpart to rcu_dereference() for non-RCU situations (David Howells) [985875]
+- [fs] overlayfs: embed middle into overlay_readdir_data (David Howells) [985875]
+- [fs] overlayfs: embed root into overlay_readdir_data (David Howells) [985875]
+- [fs] overlayfs: make ovl_cache_entry->name an array instead of pointer (David Howells) [985875]
+- [fs] overlayfs: don't hold ->i_mutex over opening the real directory (David Howells) [985875]
+- [fs] overlayfs: limit filesystem stacking depth (David Howells) [985875]
+- [fs] overlayfs: overlay filesystem documentation (David Howells) [985875]
+- [fs] overlayfs: implement show_options (David Howells) [985875]
+- [fs] overlayfs: add statfs support (David Howells) [985875]
+- [fs] overlayfs: filesystem (David Howells) [985875]
+- [mm] shmem: support RENAME_WHITEOUT (David Howells) [985875]
+- [fs] ext4: support RENAME_WHITEOUT (David Howells) [985875]
+- [fs] vfs: add RENAME_WHITEOUT (David Howells) [985875]
+- [fs] vfs: add whiteout support (David Howells) [985875]
+- [fs] vfs: export check_sticky() (David Howells) [985875]
+- [fs] vfs: introduce clone_private_mount() (David Howells) [985875]
+- [fs] vfs: export __inode_permission() to modules (David Howells) [985875]
+- [fs] vfs: export do_splice_direct() to modules (David Howells) [985875]
+- [fs] vfs: add i_op->dentry_open() (David Howells) [985875]
+- [fs] namei: trivial fix to vfs_rename_dir comment (David Howells) [985875]
+- [fs] cifs: support RENAME_NOREPLACE (David Howells) [985875]
+- [fs] hostfs: support rename flags (David Howells) [985875]
+- [mm] shmem: support RENAME_EXCHANGE (David Howells) [985875]
+- [mm] shmem: support RENAME_NOREPLACE (David Howells) [985875]
+- [fs] btrfs: add RENAME_NOREPLACE (David Howells) [985875]
+- [fs] bad_inode: add ->rename2() (David Howells) [985875]
+- [fs] call rename2 if exists (David Howells) [985875]
+- [fs] fuse: restructure ->rename2() (David Howells) [985875]
+- [fs] fuse: add renameat2 support (David Howells) [985875]
+- [fs] dcache: fix races between __d_instantiate() and checks of dentry flags (David Howells) [985875]
+- [fs] ext4: add cross rename support (David Howells) [985875]
+- [fs] vfs: add cross-rename (David Howells) [985875]
+- [fs] vfs: lock_two_nondirectories - allow directory args (David Howells) [985875]
+- [security] add flags to rename hooks (David Howells) [985875]
+- [fs] vfs: add RENAME_NOREPLACE flag (David Howells) [985875]
+- [fs] vfs: add renameat2 syscall (David Howells) [985875]
+- [fs] namei: use common code for dir and non-dir (David Howells) [985875]
+- [fs] namei: move d_move() up (David Howells) [985875]
+- [fs] vfs: add d_is_dir() (David Howells) [985875]
+- [fs] vfs: Put a small type field into struct dentry::d_flags (David Howells) [985875]
+
 * Thu Dec 18 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-219.el7]
 - [mm] vmstat: on-demand vmstat workers V8 (Larry Woodman) [1157802]
 - [mm] vmstat: use this_cpu() to avoid irqon/off sequence in refresh_cpu_vm_stats (Larry Woodman) [1157802]
-- 
1.7.1