From c8368252393acc3e0f1cd248a4a3d615115afbcb Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Thu, 17 Jul 2014 15:59:09 -0400
Subject: [redhat] kernel-3.10.0-137.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 2f19bc9..cd3c498 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 136
+RHEL_RELEASE = 137
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 8307d71..5b334db 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=136
+BUILD:=137
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 4af4a10..888499f 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1471,6 +1471,94 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Thu Jul 17 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-137.el7]
+- [powercap] Add to drivers Kconfig and Makefile (Steve Best) [1085575]
+- [powercap] add Documentation (Steve Best) [1085575]
+- [powercap] Fix build error with option -Werror=format-security (Steve Best) [1085575]
+- [powercap] Introduce Intel RAPL power capping driver (Steve Best) [1085575]
+- [powercap] Add class driver (Steve Best) [1085575]
+- [x86] msr: add 64bit _on_cpu access functions (Steve Best) [1085575]
+- [misc] bitops: Introduce BIT_ULL (Steve Best) [1085575]
+- [virt] xen/spinlock: Don't use __initdate for xen_pv_spin (Vitaly Kuznetsov) [1006977]
+- [virt] xen/spinlock: Don't setup xen spinlock IPI kicker if disabled (Vitaly Kuznetsov) [1006977]
+- [virt] xen/smp: Update pv_lock_ops functions before alternative code starts under PVHVM (Vitaly Kuznetsov) [1006977]
+- [virt] xen/spinlock: We don't need the old structure anymore (Vitaly Kuznetsov) [1006977]
+- [virt] xen/spinlock: Fix locking path engaging too soon under PVHVM (Vitaly Kuznetsov) [1006977]
+- [ata] ahci: Do not receive interrupts sent by dummy ports (David Milburn) [1067838]
+- [ata] ahci: do not request irq for dummy port (David Milburn) [1067838]
+- [ata] ahci: fix Null pointer dereference in achi_host_active() (David Milburn) [1067838]
+- [ata] ahci: Make distinct names for ports in /proc/interrupts (David Milburn) [1067838]
+- [s390] 3270: fix crash with multiple reset device requests (Hendrik Brueckner) [1088320]
+- [ata] ahci: accommodate tag ordered controller (David Milburn) [1083746]
+- [scsi] libsas: introduce scmd_dbg() to quiet false positive "timeout" messages (David Milburn) [1074987]
+- [ethernet] be2net: enable interrupts in EEH resume (Ivan Vecera) [1076682]
+- [ethernet] bnx2: Fix shutdown sequence (Neil Horman) [1074656]
+- [ethernet] be2net: Fix invocation of be_close() after be_clear() (Ivan Vecera) [1066644]
+- [virt] hv/balloon: Ensure pressure reports are posted regularly (Luiz Capitulino) [1102888]
+- [virt] kvm/x86: expose invariant tsc cpuid bit (Marcelo Tosatti) [1095990]
+- [virt] kvm/x86: disable master clock if TSC is reset during suspend (Marcelo Tosatti) [1098604]
+- [virt] xen/pvhvm: Initialize xen panic handler for PVHVM guests (Vitaly Kuznetsov) [839966]
+- [fs] cifs: Wait for writebacks to complete before attempting write (Sachin Prabhu) [1040329]
+- [fs] xfs: xfs_readsb needs to check for magic numbers (Eric Sandeen) [1102940]
+- [net] svcrdma: fix offset calculation for non-page aligned sge entries (Jeff Layton) [1088954]
+- [fs] lockd: ensure we tear down any live sockets when socket creation fails during lockd_up (Jeff Layton) [1085774]
+- [net] svcrpc: fix gss-proxy NULL dereference in some error cases (Jeff Layton) [1085616]
+- [mm] use paravirt friendly ops for NUMA hinting ptes (Vitaly Kuznetsov) [1101274]
+- [mm] swap: discard while swapping only if SWAP_FLAG_DISCARD_PAGES (Rafael Aquini) [1053189]
+- [x86] hwpoison: remove unused global variable in do_machine_check() (Prarit Bhargava) [1119306]
+- [x86] mce: Panic when a core has reached a timeout (Prarit Bhargava) [1119306]
+- [x86] mce: Improve mcheck_init_device() error handling (Prarit Bhargava) [1119306]
+- [x86] mce: Fix CMCI preemption bugs (Prarit Bhargava) [1119306]
+- [x86] cmci: Add proper detection of end of CMCI storms (Prarit Bhargava) [1119306]
+- [x86] acpi, apei, ghes: Do not report only correctable errors with SCI (Prarit Bhargava) [1119306]
+- [x86] mce: Call put_device on device_register failure (Prarit Bhargava) [1119306]
+- [acpi] apei, cper: Add UEFI 2.4 support for memory error (Prarit Bhargava) [1119306]
+- [x86] mce: Fix mce regression from recent cleanup (Prarit Bhargava) [1119306]
+- [x86] mce: Update MCE severity condition check (Prarit Bhargava) [1119306]
+- [x86] mce: Add comments to clarify usage of the various bitfields in the MCA subsystem (Prarit Bhargava) [1119306]
+- [x86] mcheck, therm_throt: Process package thresholds (Prarit Bhargava) [1119306]
+- [x86] mce: Fix "braodcast" typo (Prarit Bhargava) [1119306]
+- [mm] vmstat: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [mm] vmstat: set N_CPU to node_states during boot (Prarit Bhargava) [1119078]
+- [kernel] trace, ring-buffer: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [kernel] profile: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [mm] zswap: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [hwmon] via-cputemp: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [hwmon] coretemp: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [oprofile] nmi-timer: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [idle] intel-idle: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [base] topology: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [scsi] fcoe: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [cpufreq] acpi-cpufreq: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [scsi] bnx2i: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [scsi] bnx2fc: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [virt] x86, kvm: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [x86] amd, uncore: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [x86] pci, amd-bus: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [x86] hpet: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [x86] intel, cacheinfo: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [x86] therm_throt: Remove unused therm_cpu_lock (Prarit Bhargava) [1119078]
+- [x86] oprofile, nmi: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [x86] amd, ibs: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [x86] msr: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [powerpc] sysfs: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [x86] mce: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [x86] therm_throt: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [x86] cpuid: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [x86] vsyscall: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [x86] intel, uncore: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [s390] smp: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [s390] cacheinfo: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [kernel] cpu_hotplug, perf: Fix CPU hotplug callback registration (Prarit Bhargava) [1119078]
+- [kernel] cpu/hotplug: Add lockdep annotations to get/put_online_cpus() (Prarit Bhargava) [1119078]
+- [kernel] cpu/hotplug: Provide lockless versions of callback registration functions (Prarit Bhargava) [1119078]
+- [Documentation] cpu-hotplug: Specify race-free way to register CPU hotplug callbacks (Prarit Bhargava) [1119078]
+- [pci] Fail new_id for vendor/device values already built into driver (Bandan Das) [1055867]
+- [init] main: add initcall_blacklist kernel parameter (Prarit Bhargava) [1080995]
+- [kernel] pid_namespace: pidns_get() should check task_active_pid_ns() != NULL (Jan Stancek) [1106368]
+- [i2c] i2c-ismt: support I2C_SMBUS_I2C_BLOCK_DATA transaction type (Neil Horman) [1094516]
+- [misc] mei: fix lockdep warning when registering mei watchdog (Prarit Bhargava) [971855]
+
 * Wed Jul 16 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-136.el7]
 - [block] mtip32xx: minor performance enhancements (Mike Snitzer) [1105703]
 - [block] mtip32xx: let blk_mq_tag_to_rq() take blk_mq_tags as the main parameter (Mike Snitzer) [1105703]
-- 
1.7.1