From ef2acb1c00742fd96bcffc6f77f861a357e37373 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Sat, 30 Aug 2014 02:11:00 -0400
Subject: [redhat] kernel-3.10.0-148.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 3b04e45..eb379f5 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 147
+RHEL_RELEASE = 148
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 752558f..3d08ad5 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=147
+BUILD:=148
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index eaa0a47..7d5ea11 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1472,6 +1472,111 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Sat Aug 30 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-148.el7]
+- [ethernet] igbvf: Remove DEFINE_PCI_DEVICE_TABLE macro use (Stefan Assmann) [1091122]
+- [ethernet] igbvf: use SPEED_UNKNOWN and DUPLEX_UNKNOWN when appropriate (Stefan Assmann) [1091122]
+- [ethernet] igbvf: remove usless return statements (Stefan Assmann) [1091122]
+- [ethernet] igbvf: get rid of SET_ETHTOOL_OPS (Stefan Assmann) [1091122]
+- [ethernet] igbvf: remove open-coded skb_cow_head (Stefan Assmann) [1091122]
+- [ethernet] igbvf: Convert uses of __constant_<foo> to <foo> (Stefan Assmann) [1091122]
+- [ethernet] igbvf: Use pci_enable_msix_range() instead of pci_enable_msix() (Stefan Assmann) [1091122]
+- [ethernet] igbvf: slight optimization of addr compare (Stefan Assmann) [1091122]
+- [ethernet] igbvf: add missing iounmap() on error in igbvf_probe() (Stefan Assmann) [1091122]
+- [ethernet] igbvf: integer wrapping bug setting the mtu (Stefan Assmann) [1091122]
+- [ethernet] igbvf: Miscellaneous conversions to ETH_ALEN (Stefan Assmann) [1091122]
+- [ethernet] igbvf: Remove extern from function prototypes (Stefan Assmann) [1091122]
+- [ethernet] igbvf: fix 32-bit DMA mask handling (Stefan Assmann) [1091122]
+- [ethernet] igb: bump igb version to 5.2.13 (Stefan Assmann) [1091121]
+- [ethernet] igb: Add message when malformed packets detected by hw (Stefan Assmann) [1091121]
+- [ethernet] igb: remove unnecessary break after return (Stefan Assmann) [1091121]
+- [ethernet] igb: remove unnecessary break after goto (Stefan Assmann) [1091121]
+- [ethernet] igb: do a reset on SR-IOV re-init if device is down (Stefan Assmann) [1091121]
+- [ethernet] igb: workaround for i210 errata 25, Slow System Clock (Stefan Assmann) [1091121]
+- [ethernet] igb: bring link up when PHY is powered up (Stefan Assmann) [1091121]
+- [ethernet] igb: separate hardware setting from the set_ts_config ioctl (Stefan Assmann) [1091121]
+- [ethernet] igb: Replace 1/0 return values with true/false (Stefan Assmann) [1091121]
+- [ethernet] igb: unhide invariant returns (Stefan Assmann) [1091121]
+- [ethernet] igb: use ethtool_cmd_speed_set helper to set ethtool speed value (Stefan Assmann) [1091121]
+- [ethernet] igb: use SPEED_UNKNOWN and DUPLEX_UNKNOWN when appropriate (Stefan Assmann) [1091121]
+- [ethernet] igb: add defaults for i210 TX/RX PBSIZE (Stefan Assmann) [1091121]
+- [ethernet] igb: use mac loopback for i354 backplane (Stefan Assmann) [1091121]
+- [ethernet] igb: rename igb_ptp_enable to igb_ptp_feature_enable (Stefan Assmann) [1091121]
+- [ethernet] igb: remove redundant PHY power down register write (Stefan Assmann) [1091121]
+- [ethernet] igb: remove usless return statements (Stefan Assmann) [1091121]
+- [ethernet] igb: remove return statements for void functions (Stefan Assmann) [1091121]
+- [ethernet] igb: get rid of SET_ETHTOOL_OPS (Stefan Assmann) [1091121]
+- [ethernet] igb: Change memcpy to struct assignment (Stefan Assmann) [1091121]
+- [ethernet] igb: Cleanups to remove unneeded extern declaration (Stefan Assmann) [1091121]
+- [ethernet] igb: Cleanups to replace deprecated DEFINE_PCI_DEVICE_TABLE (Stefan Assmann) [1091121]
+- [ethernet] igb: Cleanups to fix static initialization (Stefan Assmann) [1091121]
+- [ethernet] igb: Cleanups to fix msleep warnings (Stefan Assmann) [1091121]
+- [ethernet] igb: Cleanups to fix line length warnings (Stefan Assmann) [1091121]
+- [ethernet] igb: Cleanups to remove return parentheses (Stefan Assmann) [1091121]
+- [ethernet] igb: Cleanups to fix missing break in switch statements (Stefan Assmann) [1091121]
+- [ethernet] igb: Cleanups to fix assignment in if error (Stefan Assmann) [1091121]
+- [ethernet] igb: Cleanups to change comment style on license headers (Stefan Assmann) [1091121]
+- [ethernet] igb: Cleanups to fix for trailing statement (Stefan Assmann) [1091121]
+- [ethernet] igb: Cleanups to fix pointer location error (Stefan Assmann) [1091121]
+- [ethernet] igb: Cleanups to fix incorrect indentation (Stefan Assmann) [1091121]
+- [ethernet] igb: Cleanups to fix braces location warnings (Stefan Assmann) [1091121]
+- [ethernet] igb: Cleanups for messaging (Stefan Assmann) [1091121]
+- [ethernet] igb: fix message terminations (Stefan Assmann) [1091121]
+- [ethernet] igb: fix stats for i210 rx_fifo_errors (Stefan Assmann) [1091121]
+- [ethernet] igb: fix last_rx_timestamp usage (Stefan Assmann) [1091121]
+- [ethernet] igb: remove open-coded skb_cow_head (Stefan Assmann) [1091121]
+- [ethernet] igb: Convert iounmap to pci_iounmap (Stefan Assmann) [1091121]
+- [ethernet] igb: fix race conditions on queuing skb for HW time stamp (Stefan Assmann) [1091121]
+- [ethernet] igb: never generate both software and hardware timestamps (Stefan Assmann) [1091121]
+- [ethernet] igb: Unset IGB_FLAG_HAS_MSIX-flag when falling back to msi-only (Stefan Assmann) [1091121]
+- [ethernet] igb: Fix Null-pointer dereference in igb_reset_q_vector (Stefan Assmann) [1091121]
+- [ethernet] igb: specify phc_index of 82575 for get_ts_info (Stefan Assmann) [1091121]
+- [ethernet] igb: Fix memory leak in igb_get_module_eeprom() (Stefan Assmann) [1091121]
+- [ethernet] igb: add register rd/wr for surprise removal (Stefan Assmann) [1091121]
+- [ethernet] igb: implement SIOCGHWTSTAMP ioctl (Stefan Assmann) [1091121]
+- [ethernet] igb: Convert uses of __constant_<foo> to <foo> (Stefan Assmann) [1091121]
+- [ethernet] igb: enable VLAN stripping for VMs with i350 (Stefan Assmann) [1091121]
+- [ethernet] igb: Add register defines needed for time sync functions (Stefan Assmann) [1091121]
+- [ethernet] igb: remove references to long gone command line parameters (Stefan Assmann) [1091121]
+- [ethernet] igb: Don't receive packets when the napi budget == 0 (Stefan Assmann) [1091121]
+- [ethernet] igb: Fix code comment (Stefan Assmann) [1091121]
+- [ethernet] igb: Fix for devices using ethtool for EEE settings (Stefan Assmann) [1091121]
+- [ethernet] igb: fix calls to skb_set_hash (Stefan Assmann) [1091121]
+- [ethernet] igb: fix warning if !CONFIG_IGB_HWMON (Stefan Assmann) [1091121]
+- [ethernet] igb: fix array size calculation (Stefan Assmann) [1091121]
+- [ethernet] igb: Update license text to remove FSF address and update copyright (Stefan Assmann) [1091121]
+- [ethernet] igb: make local functions static and remove dead code (Stefan Assmann) [1091121]
+- [ethernet] igb: Use pci_enable_msix_range() instead of pci_enable_msix() (Stefan Assmann) [1091121]
+- [ethernet] igb: Change to use statically allocated array for MSIx entries (Stefan Assmann) [1091121]
+- [ethernet] igb: Fix queue allocation method to accommodate changing during runtime (Stefan Assmann) [1091121]
+- [ethernet] igb: Fix for issue where values could be too high for udelay function (Stefan Assmann) [1091121]
+- [ethernet] igb: Start temperature sensor attribute index with 1 (Stefan Assmann) [1091121]
+- [ethernet] igb: Add new feature Media Auto Sense for 82580 devices only (Stefan Assmann) [1091121]
+- [ethernet] igb: Support ports mapped in 64-bit PCI space (Stefan Assmann) [1091121]
+- [ethernet] igb: Add media switching feature for i354 PHY's (Stefan Assmann) [1091121]
+- [ethernet] igb: Fixed Wake On LAN support (Stefan Assmann) [1091121]
+- [ethernet] igb: Update link modes display in ethtool (Stefan Assmann) [1091121]
+- [ethernet] igb: Explicitly initialize u64_stats_sync structures for lockdep (Stefan Assmann) [1091121]
+- [ethernet] igb: Don't let ethtool try to write to iNVM in i210/i211 (Stefan Assmann) [1091121]
+- [ethernet] igb: Fix master/slave mode for all m88 i354 PHY's (Stefan Assmann) [1091121]
+- [ethernet] igb: Miscellaneous conversions to ETH_ALEN (Stefan Assmann) [1091121]
+- [ethernet] igb: Avoid uninitialized advertised variable in eee_set_cur (Stefan Assmann) [1091121]
+- [ethernet] igb: Add ethtool support to configure number of channels (Stefan Assmann) [1091121]
+- [ethernet] igb: Add ethtool offline tests for i354 (Stefan Assmann) [1091121]
+- [ethernet] igb: Fix ethtool loopback test for 82580 copper (Stefan Assmann) [1091121]
+- [ethernet] igb: Remove extern from function prototypes (Stefan Assmann) [1091121]
+- [ethernet] igb: fix 32-bit DMA mask handling (Stefan Assmann) [1091121]
+- [ethernet] igb: Read flow control for i350 from correct EEPROM section (Stefan Assmann) [1091121]
+- [ethernet] igb: Add additional get_phy_id call for i354 devices (Stefan Assmann) [1091121]
+- [kernel] workqueue: zero cpumask of wq_numa_possible_cpumask on init (Motohiro Kosaki) [1117184]
+- [firmware] memmap: don't allocate firmware_map_entry of same memory range (Motohiro Kosaki) [1117186]
+- [firmware] memmap: pass the correct argument to firmware_map_find_entry_bootmem() (Motohiro Kosaki) [1117186]
+- [cpufreq] acpi-cpufreq: skip loading acpi_cpufreq after intel_pstate (Motohiro Kosaki) [1123250]
+- [x86] mem-hotplug: modify PGD entry when removing memory (Motohiro Kosaki) [1116286]
+- [x86] mem-hotplug: pass sync_global_pgds() a correct argument in remove_pagetable() (Motohiro Kosaki) [1116286]
+- [virt] kvm: increase x86 VCPU limit to 240 (Radim Krcmar) [1061403]
+- [kernel] sched/fair: Rework sched_fair time accounting (Rik van Riel) [1123731]
+- [kernel] math64: Add mul_u64_u32_shr() (Rik van Riel) [1123731]
+
 * Wed Aug 27 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-147.el7]
 - [s390] fix restore of invalid floating-point-control (Hendrik Brueckner) [1121965]
 - [s390] crypto: fix aes_s390 crypto module unload problem (Hendrik Brueckner) [1103114]
-- 
1.7.1