From 09b799702df81cf73190bb9c35444c83b0a87f0f Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 30 Sep 2014 16:40:51 -0400
Subject: [redhat] kernel-3.10.0-177.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index e3d24ab..5222905 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 176
+RHEL_RELEASE = 177
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 5f4cbbc..f20bc66 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=176
+BUILD:=177
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index babf4c7..9413f04 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1483,6 +1483,36 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Sep 30 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-177.el7]
+- [x86] uv: Set n_lshift based on GAM_GR_CONFIG MMR for UV3 (George Beshers) [1098325]
+- [kernel] audit: correct AUDIT_GET_FEATURE return message type (Richard Guy Briggs) [1141791]
+- [mm] fix the theoretical compound_lock() vs prep_new_page() race (Andrea Arcangeli) [1135506]
+- [mm] hugetlb: call MMU notifiers when copying a hugetlb page range (Andrea Arcangeli) [1135506]
+- [mm] page_alloc: convert hot/cold parameter and immediate callers to bool (Andrea Arcangeli) [1135506]
+- [mm] introdule compound_head_by_tail() (Andrea Arcangeli) [1135506]
+- [mm] swap: split put_compound_page() (Andrea Arcangeli) [1135506]
+- [mm] slub: do not VM_BUG_ON_PAGE() for temporary on-stack pages (Andrea Arcangeli) [1135506]
+- [mm] page_alloc: change mm debug routines back to EXPORT_SYMBOL (Andrea Arcangeli) [1135506]
+- [mm] dump page when hitting a VM_BUG_ON using VM_BUG_ON_PAGE (Andrea Arcangeli) [1135506]
+- [mm] thp: __get_page_tail_foll() can use get_huge_page_tail() (Andrea Arcangeli) [1135506]
+- [mm] print more details for bad_page() (Andrea Arcangeli) [1135506]
+- [mm] thp: turn compound_head() into BUG_ON(!PageTail) in get_huge_page_tail() (Andrea Arcangeli) [1135506]
+- [mm] hugetlbfs: Add some VM_BUG_ON()s to catch non-hugetlbfs pages (Andrea Arcangeli) [1135506]
+- [mm] swap: introduce put_[un]refcounted_compound_page helpers for splitting put_compound_page() (Andrea Arcangeli) [1135506]
+- [mm] close PageTail race (Andrea Arcangeli) [1135506]
+- [block] aoe: adjust ref of head for compound page tails (Andrea Arcangeli) [1135506]
+- [mm] swap: reorganize put_compound_page() (Andrea Arcangeli) [1135506]
+- [mm] hugetlbfs: use __compound_tail_refcounted in __get_page_tail too (Andrea Arcangeli) [1135506]
+- [mm] tail page refcounting optimization for slab and hugetlbfs (Andrea Arcangeli) [1135506]
+- [mm] hugetlbfs: move the put/get_page slab and hugetlbfs optimization in a faster path (Andrea Arcangeli) [1135506]
+- [mm] hugetlbfs: fix hugetlbfs optimization (Andrea Arcangeli) [1135506]
+- [mm] make lru_add_drain_all() selective (Andrea Arcangeli) [1135506]
+- [mm] fix aio performance regression for database caused by THP (Andrea Arcangeli) [1135506]
+- [powerpc] Add smp_mb()s to arch_spin_unlock_wait() (Gustavo Duarte) [1136528]
+- [powerpc] Add smp_mb() to arch_spin_is_locked() (Gustavo Duarte) [1136528]
+- [acpi] scan: not cache _SUN value in struct acpi_device_pnp (Prarit Bhargava) [1140567]
+- [acpi] scan: ACPI device object sysfs attribute for _STA evaluation (Prarit Bhargava) [1140567]
+
 * Tue Sep 30 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-176.el7]
 - [alsa] hda: Set up initial pins for Acer Aspire V5 (Jaroslav Kysela) [1112200]
 - [alsa] ice1712: Replacing hex with #defines (Jaroslav Kysela) [1112200]
-- 
1.7.1