From c54178fbd994222e00cd00070754f95440d4ef61 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Mon, 3 Nov 2014 16:02:30 -0500
Subject: [redhat] kernel-3.10.0-198.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 1efc6de..bc34942 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 197
+RHEL_RELEASE = 198
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index c30bde3..eec5b2b 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=197
+BUILD:=198
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index ac3106b..485034a 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,63 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Mon Nov 03 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-198.el7]
+- [block] nvme: Clear QUEUE_FLAG_STACKABLE (David Milburn) [1155716]
+- [md] dm-raid: ensure superblock's size matches device's logical block size (Mike Snitzer) [1112804]
+- [md] dm-thin: refactor requeue_io to eliminate spinlock bouncing (Mike Snitzer) [1156164]
+- [md] dm-thin: optimize retry_bios_on_resume (Mike Snitzer) [1156164]
+- [md] dm-thin: sort the deferred cells (Mike Snitzer) [1156164]
+- [md] dm-thin: direct dispatch when breaking sharing (Mike Snitzer) [1156164]
+- [md] dm-thin: remap the bios in a cell immediately (Mike Snitzer) [1156164]
+- [md] dm-thin: defer whole cells rather than individual bios (Mike Snitzer) [1156164]
+- [md] dm-thin: factor out remap_and_issue_overwrite (Mike Snitzer) [1156164]
+- [md] dm-thin: performance improvement to discard processing (Mike Snitzer) [1156164]
+- [md] dm-thin: grab a virtual cell before looking up the mapping (Mike Snitzer) [1156164]
+- [md] dm-thin: implement thin_merge (Mike Snitzer) [1156164]
+- [md] dm: improve documentation and code clarity in dm_merge_bvec (Mike Snitzer) [1156164]
+- [md] dm-thin: adjust max_sectors_kb based on thinp blocksize (Mike Snitzer) [1156164]
+- [block] fix alignment_offset math that assumes io_min is a power-of-2 (Mike Snitzer) [1156164]
+- [md] dm-thin: throttle incoming IO (Mike Snitzer) [1156161]
+- [md] dm-thin: prefetch missing metadata pages (Mike Snitzer) [1156161]
+- [md] dm-transaction-manager: add support for prefetching blocks of metadata (Mike Snitzer) [1156161]
+- [md] dm-thin-metadata: change dm_thin_find_block to allow blocking, but not issuing, IO (Mike Snitzer) [1156161]
+- [md] dm-bio-prison: switch to using a red black tree (Mike Snitzer) [1156161]
+- [md] dm-bufio: evict buffers that are past the max age but retain some buffers (Mike Snitzer) [1156161]
+- [md] dm-bufio: switch from a huge hash table to an rbtree (Mike Snitzer) [1156161]
+- [md] dm-bufio: change __GFP_IO to __GFP_FS in shrinker callbacks (Mike Snitzer) [1156161]
+- [md] dm-bufio: when done scanning return from __scan immediately (Mike Snitzer) [1156161]
+- [md] dm-bufio: update last_accessed when relinking a buffer (Mike Snitzer) [1156161]
+- [md] dm-bufio: fully initialize shrinker (Mike Snitzer) [1156161]
+- [misc] genwqe: Remove unnecessary include (Gustavo Duarte) [1088491]
+- [misc] genwqe: Increase driver version number (Gustavo Duarte) [1088491]
+- [misc] genwqe: Improve hardware error recovery (Gustavo Duarte) [1088491]
+- [misc] genwqe: Add support for EEH error recovery (Gustavo Duarte) [1088491]
+- [misc] genwqe: Add sysfs interface for bitstream reload (Gustavo Duarte) [1088491]
+- [misc] genwqe: fix uninitialized return value in genwqe_free_sync_sgl() (Gustavo Duarte) [1088491]
+- [misc] genwqe: Fix format string mismatch in card_debugfs.c (Gustavo Duarte) [1088491]
+- [misc] genwqe: Increase driver version number (Gustavo Duarte) [1088491]
+- [misc] genwqe: Fix multithreading problems (Gustavo Duarte) [1088491]
+- [misc] genwqe: Ensure rc is not returning an uninitialized value (Gustavo Duarte) [1088491]
+- [misc] genwqe: Add wmb before DDCB is started (Gustavo Duarte) [1088491]
+- [misc] genwqe: Enable access to VPD flash area (Gustavo Duarte) [1088491]
+- [misc] genwqe: Fix potential memory leak when pinning memory (Gustavo Duarte) [1088491]
+- [misc] genwqe: delete non-required instances of include <linux/init.h> (Gustavo Duarte) [1088491]
+- [misc] genwqe: fix return value check in genwqe_device_create() (Gustavo Duarte) [1088491]
+- [misc] genwqe: Fix warnings for sparc (Gustavo Duarte) [1088491]
+- [misc] genwqe: Fix compile problems for Alpha (Gustavo Duarte) [1088491]
+- [misc] genwqe: Rework return code for flash-update ioctl (Gustavo Duarte) [1088491]
+- [misc] genwqe: Accidently casting to u32 where u64 is required (Gustavo Duarte) [1088491]
+- [misc] genwqe: Replace dynamic_hex_dump with print_hex_dump_debug (Gustavo Duarte) [1088491]
+- [misc] genwqe: Fix endian issues detected by sparse (Gustavo Duarte) [1088491]
+- [misc] genwqe: Enable driver (Gustavo Duarte) [1088491]
+- [misc] genwqe: Sysfs interfaces (Gustavo Duarte) [1088491]
+- [misc] genwqe: Debugfs interfaces (Gustavo Duarte) [1088491]
+- [misc] genwqe: Utility functions (Gustavo Duarte) [1088491]
+- [misc] genwqe: Character device and DDCB queue (Gustavo Duarte) [1088491]
+- [misc] genwqe: GenWQE PCI support, health monitoring and recovery (Gustavo Duarte) [1088491]
+- [virt] kvm/vmx: handle invvpid vm exit gracefully (Petr  Matousek) [1144829] {CVE-2014-3646}
+- [virt] kvm: fix PIT timer race condition (Petr  Matousek) [1144880] {CVE-2014-3611}
+
 * Thu Oct 30 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-197.el7]
 - [scsi] qla2xxx: Update version number to 8.07.00.08.07.1-k1 (Chad Dupuis) [1067179]
 - [scsi] qla2xxx: Disable PCI device in shutdown handler (Chad Dupuis) [1067179]
-- 
1.7.1