From 4c503eb3917e2d33a78df45c4ca726fc73b7970d Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Mon, 24 Nov 2014 16:11:59 -0500
Subject: [redhat] kernel-3.10.0-209.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index a695fbf..51e8358 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 208
+RHEL_RELEASE = 209
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index f262f9a..8c52ed5 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=208
+BUILD:=209
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 7f28ebe..f2a0bd6 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,41 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Mon Nov 24 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-209.el7]
+- [fs] xfs: write failure beyond EOF truncates too much data (Brian Foster) [1032968]
+- [fs] xfs: xfs_vm_write_end truncates too much on failure (Brian Foster) [1032968]
+- [fs] xfs: use ->invalidatepage() length argument (Brian Foster) [1032968]
+- [fs] xfs: change invalidatepage prototype to accept length (Brian Foster) [1032968]
+- [fs] xfs: restore buffer_head unwritten bit on ioend cancel (Brian Foster) [1162953]
+- [fs] xfs: allow inode allocations in post-growfs disk space (Eric Sandeen) [1115201]
+- [scsi] pm8001: Update nvmd response data to request buffer (Rich Bono) [1110943]
+- [scsi] pm8001: fix pm8001_store_update_fw (Rich Bono) [1110943]
+- [scsi] pm8001: Fix erratic calculation in update_flash (Rich Bono) [1110943]
+- [scsi] pm8001: Fix invalid return when request_irq() failed (Rich Bono) [1110943]
+- [scsi] pm8001: fix a memory leak in nvmd_resp (Rich Bono) [1110943]
+- [scsi] pm8001: fix update_flash (Rich Bono) [1110943]
+- [scsi] pm8001: fix a memory leak in flash_update (Rich Bono) [1110943]
+- [scsi] pm8001: Cleaning up uninitialized variables (Rich Bono) [1110943]
+- [scsi] pm8001: Fix to remove null pointer checks that could never happen (Rich Bono) [1110943]
+- [scsi] pm8001: more fixes to honor return value (Rich Bono) [1110943]
+- [scsi] pm8001: add a new spinlock to protect the CCB (Rich Bono) [1110943]
+- [scsi] pm8001: honor return value (Rich Bono) [1110943]
+- [scsi] pm8001: clean bitmap management functions (Rich Bono) [1110943]
+- [scsi] pm8001: Fix hibernation issue (Rich Bono) [1110943]
+- [scsi] pm8001: Fix potential null pointer dereference and memory leak (Rich Bono) [1110943]
+- [scsi] pm80xx: Fix missing NULL pointer checks and memory leaks (Rich Bono) [1110943]
+- [scsi] drivers/scsi/pm8001/pm8001_ctl.c: avoid world-writable sysfs files (Rich Bono) [1110943]
+- [scsi] pm80xx: fix problem of pm8001_work_fn reseting incorrect phy device (Rich Bono) [1110943]
+- [scsi] pm80xx: Fix missing NULL pointer checks and memory leaks (Rich Bono) [1110943]
+- [scsi] pm80xx: Enable BAR shift to avoid BIOS conflict with MPI space for ATTO pm8001 based HBAs (Rich Bono) [1110943]
+- [scsi] pm80xx: Read saved WWN from NVMD for ATTO pm8001 based HBAs (Rich Bono) [1110943]
+- [scsi] pm80xx: Fixed return value issue (Rich Bono) [1110943]
+- [md] dm-thin: fix pool_io_hints to avoid looking at max_hw_sectors (Mike Snitzer) [1156164]
+- [kernel] audit: keep inode pinned (Paul Moore) [1162261]
+- [block] nvme: cleanup nvme_split_flush_data() (David Milburn) [1161766]
+- [scsi] ibmvfc: fix little endian issues (Steve Best) [1159781]
+- [scsi] ibmvfc: Fix for offlining devices during error recovery (Steve Best) [1159781]
+
 * Sat Nov 22 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-208.el7]
 - [scsi] cxgb4i: send abort_rpl correctly (Sai Vemuri) [1163467]
 - [Documentation] cxgbi: add maintainer for cxgb3i/cxgb4i (Sai Vemuri) [1163467]
-- 
1.7.1