From 1685ed85bc1ddf9246810cee2c2b8ac54c3c6a95 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Fri, 19 Sep 2014 17:24:07 -0400
Subject: [redhat] kernel-3.10.0-165.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index f736c7b..e925f1a 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 164
+RHEL_RELEASE = 165
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index a2c7a4c..533f4ad 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=164
+BUILD:=165
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 7055073..3d0205c 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1476,6 +1476,72 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Fri Sep 19 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-165.el7]
+- [virt] virtio_scsi: fix various bad behavior on aborted requests (Paolo Bonzini) [1098506]
+- [x86] perf/intel: ignore CondChgd bit to avoid false NMI handling (Don Zickus) [1110264]
+- [lib] cmdline: add size unit t/p/e to memparse (Eric Sandeen) [1058608]
+- [virt] virtio_net: fix packets dropping issue (Marcel Apfelbaum) [1131733]
+- [pci] hotplug: Prevent NULL dereference during pciehp probe (Myron Stowe) [1133107]
+- [security] selinux: fix a problem with IPv6 traffic denials in selinux_ip_postroute() (Paul Moore) [1102641]
+- [platform] toshiba: re-enable acpi hotkeys after suspend to disk (Benjamin Tissoires) [1107814]
+- [ethernet] cxgb4: use remove handler as shutdown handler (Steve Best) [1123146]
+- [idle] intel_idle: Broadwell support (Steve Best) [1086453]
+- [s390] dasd: fix unresponsive device during format (Hendrik Brueckner) [1132370]
+- [s390] dasd: validate request size before building CCW/TCW request (Hendrik Brueckner) [1132386]
+- [s390] dasd: fix error recovery for alias devices during format (Hendrik Brueckner) [1132372]
+- [s390] pci: introduce lazy IOTLB flushing for DMA unmap (Hendrik Brueckner) [1132382]
+- [s390] dasd: use aliases for formatted devices during format (Hendrik Brueckner) [1132369]
+- [s390] dasd: fix list_del corruption during format (Hendrik Brueckner) [1132371]
+- [s390] con3215: fix hanging console issue (Hendrik Brueckner) [1132345]
+- [mm] hugetlb: improve page-fault scalability (Larry Woodman) [1087955]
+- [s390] spinlock: fix system hang with spin_retry <= 0 (Hendrik Brueckner) [1096329]
+- [s390] qeth: Accurate ethtool output (Hendrik Brueckner) [1088581]
+- [s390] smp: limit number of cpus in possible cpu mask (Hendrik Brueckner) [1103711]
+- [scsi] isci: fix needless ata reset escalation (David Milburn) [1074984]
+- [idle] intel_idle: close avn_cstates array with correct marker (Steve Best) [1048050]
+- [idle] intel_idle: Fixed C6 state on Avoton/Rangeley processors (Steve Best) [1048050]
+- [idle] intel_idle: Support Intel Atom Processor C2000 Product Family (Steve Best) [1048050]
+- [ethernet] i40e: Bump i40e/i40evf versions (Stefan Assmann) [1140994]
+- [ethernet] i40e: fix panic due to too-early Tx queue enable (Stefan Assmann) [1140994]
+- [ethernet] i40e: Fix an issue when PF reset fails (Stefan Assmann) [1140994]
+- [ethernet] i40e: make warning less verbose (Stefan Assmann) [1140994]
+- [ethernet] i40e: Tell OS link is going down when calling set_phy_config (Stefan Assmann) [1140994]
+- [ethernet] i40e: Remove unnecessary assignment (Stefan Assmann) [1140994]
+- [ethernet] i40e: Change wording to be more consistent (Stefan Assmann) [1140994]
+- [ethernet] i40e: Allow user to change link settings if link is down (Stefan Assmann) [1140994]
+- [ethernet] i40e: Add dual speed module support (Stefan Assmann) [1140994]
+- [mm] memory-failure: support use of a dedicated thread to handle SIGBUS(BUS_MCEERR_AO) (Rui Wang) [1108357]
+- [mm] memory-failure: don't let collect_procs() skip over processes for MF_ACTION_REQUIRED (Rui Wang) [1108357]
+- [mm] memory-failure: send right signal code to correct thread (Rui Wang) [1108357]
+- [crypto] qat: Use pci_enable_msix_exact() instead of pci_enable_msix() (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: Fix return value check in adf_chr_drv_create() (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: Fixed SKU1 dev issue (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: Use hweight for bit counting (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: Updated print outputs (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: change ae_num to ae_id (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: change slice->regions to slice->region (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: use min_t macro (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: remove unnecessary parentheses (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: remove unneeded header (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: checkpatch blank lines (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: remove unnecessary return codes (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: remove an unneeded cast (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: Fix error path crash when no firmware is present (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: Fixed new checkpatch warnings (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: Updated Firmware Info Metadata (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: Fix random config build warnings (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: Fix build problem with O= (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: Update to makefiles (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: Intel(R) QAT DH895xcc accelerator (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: Intel(R) QAT accelengine part of fw loader (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: Intel(R) QAT ucode part of fw loader (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: Intel(R) QAT crypto interface (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: Intel(R) QAT FW interface (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: Intel(R) QAT transport code (Nikolay Aleksandrov) [1091130]
+- [crypto] qat: Intel(R) QAT driver framework (Nikolay Aleksandrov) [1091130]
+- [crypto] Sort drivers/crypto/Makefile (Nikolay Aleksandrov) [1091130]
+- [crypto] authenc: Export key parsing helper function (Nikolay Aleksandrov) [1091130]
+
 * Thu Sep 18 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-164.el7]
 - [drm] mst: rework payload table allocation to conform better (Dave Airlie) [1140440]
 - [drm] dp: Staticize a couple of DP utility functions (Dave Airlie) [1140440]
-- 
1.7.1