From 3b78d2e63c8bc6737243f12e74fe133be865c3d4 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Thu, 29 Jan 2015 18:15:53 -0500
Subject: [redhat] kernel-3.10.0-229.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 4c2c975..274dd90 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 228
+RHEL_RELEASE = 229
 RHEL_DRM_VERSION = 3
 RHEL_DRM_PATCHLEVEL = 16
 RHEL_DRM_SUBLEVEL = 7
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 5384695..c09687a 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=228
+BUILD:=229
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 1d0ab58..3a1f286 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1492,6 +1492,9 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Thu Jan 29 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-229.el7]
+- [net] rtnetlink: allow to register ops without ops->setup set (Jiri Benc) [1186492]
+
 * Thu Jan 29 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-228.el7]
 - [fs] NFSv4.1: Fix an Oops in nfs41_walk_client_list (Steve Dickson) [1185784]
 - [misc] redhat: don't suppress Revert patches from changelog (Jarod Wilson) [1187353]
-- 
1.7.1