From 6f86a087edfe8e6a66a2577d233259268d1ddd25 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Wed, 12 Nov 2014 15:14:27 -0500
Subject: [redhat] kernel-3.10.0-202.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 2bd64d0..c3f60ba 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 201
+RHEL_RELEASE = 202
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 35aa03a..c3b2af3 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=201
+BUILD:=202
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 6122255..ed76953 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,39 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Wed Nov 12 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-202.el7]
+- [ethernet] mlx4: Use PTYS register to set ethtool settings (Speed) (Amir Vadai) [1060221]
+- [ethernet] mlx4: Use PTYS register to query ethtool settings (Amir Vadai) [1060221]
+- [ethernet] mlx4: use SPEED_UNKNOWN and DUPLEX_UNKNOWN when appropriate (Amir Vadai) [1060221]
+- [ethernet] mlx4: Add 100M, 20G, 56G speeds ethtool reporting support (Amir Vadai) [1060221]
+- [ethernet] mlx4: Add ethernet backplane autoneg device capability (Amir Vadai) [1060221]
+- [ethernet] mlx4: Introduce ACCESS_REG CMD and eth_prot_ctrl dev cap (Amir Vadai) [1060221]
+- [ethernet] mlx4: Cable info, get_module_info/eeprom ethtool support (Amir Vadai) [1060221]
+- [ethernet] mlx4: Introduce mlx4_get_module_info for cable module info reading (Amir Vadai) [1060221]
+- [ethernet] mlx4: Enable CQE/EQE stride support (Amir Vadai) [1060221]
+- [virt] kvm/vmx: defer load of APIC access page address during reset (Paolo Bonzini) [1140974]
+- [virt] kvm: do not handle APIC access page if in-kernel irqchip is not in use (Paolo Bonzini) [1140974]
+- [virt] kvm: Unpin and remove kvm_arch->apic_access_page (Paolo Bonzini) [1140974]
+- [virt] kvm/vmx: Implement set_apic_access_page_addr (Paolo Bonzini) [1140974]
+- [virt] kvm: Add request bit to reload APIC access page address (Paolo Bonzini) [1140974]
+- [virt] kvm: Add arch specific mmu notifier for page invalidation (Paolo Bonzini) [1140974]
+- [virt] kvm: Rename make_all_cpus_request() to kvm_make_all_cpus_request() and make it non-static (Paolo Bonzini) [1140974]
+- [virt] kvm: Remove ept_identity_pagetable from struct kvm_arch (Paolo Bonzini) [1140974]
+- [virt] kvm: Use APIC_DEFAULT_PHYS_BASE macro as the apic access page address (Paolo Bonzini) [1140974]
+- [drm] vmwgfx: respect 'nomodeset' (Rob Clark) [1101381]
+- [s390] qeth: don't query for info if hardware not ready (Hendrik Brueckner) [1147573]
+- [block] Fix dev_t minor allocation lifetime (Jeff Moyer) [1139898]
+- [md] dm-crypt: fix access beyond the end of allocated space (Mike Snitzer) [1135066]
+- [fs] isofs: unbound recursion when processing relocated directories (Jacob Tanenbaum) [1142271] {CVE-2014-5471 CVE-2014-5472}
+- [ethernet] be2net: use v1 of SET_FLOW_CONTROL command (Ivan Vecera) [1087128]
+- [acpi] return 1 after successfully install cmos_rtc space handler (Amos Kong) [1159465]
+- [x86] hyperv: Bypass the timer_irq_works() check (Jason Wang) [1058105]
+- [mm] hugetlb: initialize PG_reserved for tail pages of gigantic compound pages (Luiz Capitulino) [1158506]
+- [kernel] cpuset: PF_SPREAD_PAGE and PF_SPREAD_SLAB should be atomic flags (Aaron Tomlin) [1160360]
+- [infiniband] qib: Correct reference counting in debugfs qp_stats (Rui Wang) [1150001]
+- [x86] uv: Check for alloc_cpumask_var() failures properly in uv_nmi_setup() (George Beshers) [1155754]
+- [powerpc] fadump: Fix endianess issues in firmware assisted dump handling (Steve Best) [1159773]
+
 * Mon Nov 10 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-201.el7]
 - [scsi] ipr: wait for aborted command responses (Gustavo Duarte) [1156530]
 - [ethernet] mlx4: Protect port type setting by mutex (Amir Vadai) [1095345]
-- 
1.7.1