From 99c1b841335bf776a1d75c406d4aca5666395e79 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Fri, 3 Oct 2014 12:16:17 -0400
Subject: [redhat] kernel-3.10.0-179.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index ee69c69..5418312 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 178
+RHEL_RELEASE = 179
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 18d8baf..b82f0cc 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=178
+BUILD:=179
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 1bb4e4c..f54342f 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1483,6 +1483,16 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Fri Oct 03 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-179.el7]
+- [drm] i915: don't crash if unable to setup stolen (Rob Clark) [1128939]
+- [fs] nfs: Don't reset pg_moreio in __nfs_pageio_add_request (Steve Dickson) [1120728]
+- [fs] nfs: Remove 2 unused variables (Steve Dickson) [1120728]
+- [fs] nfs: handle multiple reqs in nfs_wb_page_cancel (Steve Dickson) [1120728]
+- [fs] nfs: nfs_page should take a ref on the head req (Steve Dickson) [1120728]
+- [fs] nfsv4: test SECINFO RPC_AUTH_GSS pseudoflavors for support (Steve Dickson) [1120728]
+- [fs] nfs: Return -EPERM if no supported or matching SECINFO flavor (Steve Dickson) [1120728]
+- [fs] nfs: check the return of nfs4_negotiate_security in nfs4_submount (Steve Dickson) [1120728]
+
 * Fri Oct 03 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-178.el7]
 - [crypto] drbg: fix maximum value checks on 32 bit systems (Herbert Xu) [1138311]
 - [crypto] drbg: remove configuration of fixed values (Herbert Xu) [1138311]
-- 
1.7.1