From 26fa8112d7325eec6bf6937c2574638498a89381 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Fri, 12 Dec 2014 13:46:56 -0500
Subject: [redhat] kernel-3.10.0-217.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 56b19a7..f54685c 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 216
+RHEL_RELEASE = 217
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index d95dd22..a10539a 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=216
+BUILD:=217
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 0abf68b..bf973b2 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1493,6 +1493,38 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Fri Dec 12 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-217.el7]
+- [net] ipv6: yet another new IPV6_MTU_DISCOVER option IPV6_PMTUDISC_OMIT (Hannes Frederic Sowa) [1170116]
+- [net] ipv4: yet another new IP_MTU_DISCOVER option IP_PMTUDISC_OMIT (Hannes Frederic Sowa) [1170116]
+- [net] ipv4: use ip_skb_dst_mtu to determine mtu in ip_fragment (Hannes Frederic Sowa) [1170116]
+- [net] ipv4: introduce ip_dst_mtu_maybe_forward and protect forwarding path against pmtu spoofing (Hannes Frederic Sowa) [1170116]
+- [net] ipv6: move ip6_sk_accept_pmtu from generic pmtu update path to ipv6 one (Hannes Frederic Sowa) [1170116]
+- [net] ipv6: support IPV6_PMTU_INTERFACE on sockets (Hannes Frederic Sowa) [1170116]
+- [net] udp: do not report ICMP redirects to user space (Hannes Frederic Sowa) [1170116]
+- [net] ipv4: new ip_no_pmtu_disc mode to always discard incoming frag needed msgs (Hannes Frederic Sowa) [1170116]
+- [net] inet: make no_pmtu_disc per namespace and kill ipv4_config (Hannes Frederic Sowa) [1170116]
+- [net] ipv4: improve documentation of ip_no_pmtu_disc (Hannes Frederic Sowa) [1170116]
+- [net] ipv4: introduce new IP_MTU_DISCOVER mode IP_PMTUDISC_INTERFACE (Hannes Frederic Sowa) [1170116]
+- [net] xfrm: revert ipv4 mtu determination to dst_mtu (Hannes Frederic Sowa) [1170116]
+- [net] xfrm: introduce helper for safe determination of mtu (Hannes Frederic Sowa) [1170116]
+- [net] netfilter: conntrack: disable generic tracking for known protocols (Daniel Borkmann) [1170520]
+- [net] gre: Fix use-after-free panic in ipgre_rcv() (Panu Matilainen) [1117543]
+- [net] netfilter: nf_conntrack_h323: lookup route from proper net namespace (Florian Westphal) [1163847]
+- [net] netfilter: xt_tcpmss: lookup route from proper net namespace (Florian Westphal) [1163847]
+- [net] netfilter: xt_tcpmss: Get mtu only if clamp-mss-to-pmtu is specified (Florian Westphal) [1163847]
+- [wireless] cfg80211: don't WARN about two consecutive Country IE hint (Stanislaw Gruszka) [1164282]
+- [fs] aio: fix race between aio event completion and reaping (Jeff Moyer) [1131312]
+- [fs] proc/task_mmu: fix missing check during hugepage migration (Jacob Tanenbaum) [1105040] {CVE-2014-3940}
+- [kernel] trace: insufficient syscall number validation in perf and ftrace subsystems (Jacob Tanenbaum) [1161570] {CVE-2014-7825 CVE-2014-7826}
+- [ethernet] i40e: get pf_id from HW rather than PCI function (Stefan Assmann) [1078740]
+- [ethernet] i40e: increase ARQ size (Stefan Assmann) [1078740]
+- [x86] uv: Update the UV3 TLB shootdown logic (Frank Ramsay) [1170253]
+- [tools] peeksiginfo: add PAGE_SIZE definition (Steve Best) [1172250]
+- [base] bus: Fix unbalanced device reference in drivers_probe (Alex Williamson) [1158862]
+- [char] tpm: Fix NULL return in tpm_ibmvtpm_get_desired_dma (Gustavo Duarte) [1154818]
+- [powerpc] kvm: book3s_hv - Reserve cma region only in hypervisor mode (Gustavo Duarte) [1147740]
+- [x86] traps: stop using IST for #SS (Petr  Matousek) [1172813] {CVE-2014-9322}
+
 * Tue Dec 09 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-216.el7]
 - [infiniband] iser: Adjust data_length to include protection information (Amir Vadai) [1107622]
 - [infiniband] iser: Bump version to 1.4.1 (Amir Vadai) [1107622]
-- 
1.7.1