From 97171d975d6730ebb59266604254d7c066eadf5c Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Thu, 29 Jan 2015 16:57:29 -0500
Subject: [redhat] kernel-3.10.0-228.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 56fed70..4c2c975 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 227
+RHEL_RELEASE = 228
 RHEL_DRM_VERSION = 3
 RHEL_DRM_PATCHLEVEL = 16
 RHEL_DRM_SUBLEVEL = 7
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 2241f6b..5384695 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=227
+BUILD:=228
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index cc539d7..1d0ab58 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1492,6 +1492,29 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Thu Jan 29 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-228.el7]
+- [fs] NFSv4.1: Fix an Oops in nfs41_walk_client_list (Steve Dickson) [1185784]
+- [misc] redhat: don't suppress Revert patches from changelog (Jarod Wilson) [1187353]
+- [infiniband] Revert: ipoib: Consolidate rtnl_lock tasks in workqueue (Doug Ledford) [1179740]
+- [infiniband] Revert: ipoib: Make the carrier_on_task race aware (Doug Ledford) [1179740]
+- [infiniband] Revert: ipoib: fix MCAST_FLAG_BUSY usage (Doug Ledford) [1179740]
+- [infiniband] Revert: ipoib: fix mcast_dev_flush/mcast_restart_task race (Doug Ledford) [1179740]
+- [infiniband] Revert: ipoib: change init sequence ordering (Doug Ledford) [1179740]
+- [infiniband] Revert: ipoib: Use dedicated workqueues per interface (Doug Ledford) [1179740]
+- [infiniband] Revert: ipoib: Make ipoib_mcast_stop_thread flush the workqueue (Doug Ledford) [1179740]
+- [infiniband] Revert: ipoib: No longer use flush as a parameter (Doug Ledford) [1179740]
+- [fs] fix deadlock in cifs_ioctl_clone() (Sachin Prabhu) [1183980]
+- [md] dm-cache: fix missing ERR_PTR returns and handling (Mike Snitzer) [1182665]
+- [fs] cifs: fix regression in cifs_create_mf_symlink() (Sachin Prabhu) [1186324]
+- [net] ipv4: try to cache dst_entries which would cause a redirect (Hannes Frederic Sowa) [1181819]
+- [fs] coredump: add new P variable in core_pattern (Jiri Olsa) [1186360]
+- [drm] fix fb-helper vs MST dangling connector ptrs (Rob Clark) [1184968]
+- [net] bridge: Program port vlan filters only if filtering is enabled in bridge (Vlad Yasevich) [1183958]
+- [fs] cifs: Complete oplock break jobs before closing file handle (Sachin Prabhu) [1177215]
+- [fs] LOCKD: Fix a race when initialising nlmsvc_timeout (Benjamin Coddington) [1144982]
+- [scsi] hpsa: add in P840ar controller model name (Joseph Szczypek) [1185467]
+- [scsi] hpsa: add in gen9 controller model names (Joseph Szczypek) [1185467]
+
 * Tue Jan 27 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-227.el7]
 - [fs] ext4: fix overwrite race condition (Jacob Tanenbaum) [1152607] {CVE-2014-8086}
 - [media] ttusb-dec: buffer overflow in ioctl (Alexander Gordeev) [1167116] {CVE-2014-8884}
-- 
1.7.1