From caca95e40d208d5ceb15efd19a520843f28589ed Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Sun, 21 Sep 2014 22:40:36 -0400
Subject: [redhat] kernel-3.10.0-166.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index e925f1a..bd6547f 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 165
+RHEL_RELEASE = 166
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 533f4ad..3033f0d 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=165
+BUILD:=166
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 3d0205c..4c0646a 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1476,6 +1476,36 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Sun Sep 21 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-166.el7]
+- [scsi] megaraid_sas: Driver version update (06.805.06.00-rc1) (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: MFI MPT linked list corruption fix (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: N-drive primary raid level 1 load balancing (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Add module parameter to disable IRQ-CPU affinity hint (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Round down max sge supported by controller to power of two (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Host lock less mode to enabled asynchronous IO submission (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Extended VD support (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Firmware crash dump feature support (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Update threshold based reply post host index register (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Use writeq for 64bit pci write to avoid spinlock overhead (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Do not scan non syspd drives (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Version and Changelog update (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Fix reset_mutex leak (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Remove unused variables in megasas_instance (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Add missing initial call to megasas_get_ld_vf_affiliation() (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Fix LD/VF affiliation parsing (Tomas Henzl) [1088523]
+- [scsi] megaraid: Fail resume if MSI-X re-initialization failed (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Version and Changelog update (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Add Dell PowerEdge VRTX SR-IOV VF support (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Return leaked MPT frames to MPT frame pool (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Fix megasas_ioc_init_fusion (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: Load correct raid context timeout (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: check return value for megasas_get_pd_list() (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas_fusion: Return correct error value in megasas_get_ld_map_info() (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas_fusion: correctly pass queue info pointer (Tomas Henzl) [1088523]
+- [scsi] megaraid: Use resource_size_t for PCI resources, not long (Tomas Henzl) [1088523]
+- [scsi] megaraid: minor cut and paste error fixed (Tomas Henzl) [1088523]
+- [scsi] megaraid_sas: fix a bug for 64 bit arches (Tomas Henzl) [1088523]
+
 * Fri Sep 19 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-165.el7]
 - [virt] virtio_scsi: fix various bad behavior on aborted requests (Paolo Bonzini) [1098506]
 - [x86] perf/intel: ignore CondChgd bit to avoid false NMI handling (Don Zickus) [1110264]
-- 
1.7.1