From 9b6d7c6617286232f3f23719070c47b3a8b2929a Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Fri, 12 Sep 2014 11:48:11 -0400
Subject: [redhat] kernel-3.10.0-157.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 35ca6ba..8f5e70d 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 156
+RHEL_RELEASE = 157
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 7c61652..89de34d 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=156
+BUILD:=157
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 27251d7..6168547 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1474,6 +1474,104 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Fri Sep 12 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-157.el7]
+- [scsi] fusion: Remove use of DEF_SCSI_QCMD (Tomas Henzl) [1041675]
+- [scsi] fusion: Add free msg frames to the head, not tail of list (Tomas Henzl) [1041675]
+- [scsi] mpt2sas: Add free smids to the head, not tail of list (Tomas Henzl) [1041675]
+- [scsi] mpt2sas: Remove use of DEF_SCSI_QCMD (Tomas Henzl) [1041675]
+- [scsi] mpt2sas: Remove uses of serial_number (Tomas Henzl) [1041675]
+- [scsi] mpt3sas: Remove use of DEF_SCSI_QCMD (Tomas Henzl) [1041675]
+- [scsi] mpt3sas: Remove uses of serial_number (Tomas Henzl) [1041675]
+- [scsi] qla4xxx: 5.04.00.04.07.01-k0 (Chad Dupuis) [1089349]
+- [scsi] qla4xxx: Improve loopback failure messages (Chad Dupuis) [1089349]
+- [scsi] qla4xxx: from treewide: fix comments and printk msgs (Chad Dupuis) [1089349]
+- [scsi] qla4xxx: Use kmemdup instead of kmalloc + memcpy (Chad Dupuis) [1089349]
+- [scsi] qla4xxx: Fix smatch warning in func qla4xxx_conn_get_param (Chad Dupuis) [1089349]
+- [scsi] qla4xxx: Fix smatch warning in func qla4xxx_get_ep_param (Chad Dupuis) [1089349]
+- [scsi] qla4xxx: Fix memory leak for ha->saved_acb (Chad Dupuis) [1089349]
+- [scsi] qla4xxx: Export sysfs DDBs from DPC handler (Chad Dupuis) [1089349]
+- [scsi] qla4xxx: Disable INTx interrupt for ISP82XX (Chad Dupuis) [1089349]
+- [scsi] qla4xxx: Check for correct return status (Chad Dupuis) [1089349]
+- [scsi] qla4xxx: Move qla4_8xxx_ms_mem_write_128b to ql4_nx.c (Chad Dupuis) [1089349]
+- [scsi] qla4xxx: Added PEX DMA Support for ISP8022 Adapter (Chad Dupuis) [1089349]
+- [scsi] qla4xxx: Added new opcodes for 84XX Minidump template (Chad Dupuis) [1089349]
+- [scsi] qla4xxx: Add support of 0xFF capture mask for minidump (Chad Dupuis) [1089349]
+- [scsi] qla4xxx: Change default capture to firmware defined capture mask (Chad Dupuis) [1089349]
+- [scsi] qla4xxx: Add support for ISCSI_PARAM_LOCAL_IPADDR sysfs attr (Chad Dupuis) [1089349]
+- [scsi] scsi_transport_iscsi: Export ISCSI_PARAM_LOCAL_IPADDR attr for iscsi_connection (Chad Dupuis) [1089349]
+- [scsi] qla4xxx: Add host statistics support (Chad Dupuis) [1089349]
+- [scsi] scsi_transport_iscsi: Add host statistics support (Chad Dupuis) [1089349]
+- [scsi] qla4xxx: Added support for Diagnostics MBOX command (Chad Dupuis) [1089349]
+- [scsi] qla2xxx: Update version number to 8.07.00.08.07.1-k (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Remove wait for online from host reset handler (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Do logins from a chip reset in DPC thread instead of the error handler thread (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Remove mapped vp index iterator macro dead code (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Add MBC option for fast SFP data access (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Fix beacon blink logic for ISP26xx/83xx (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Delay driver unload if there is any pending activity going on (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Check the QLA8044_CRB_DRV_ACTIVE_INDEX register when we are not the owner of the reset (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Adjust adapter reset routine to the changes in firmware specification for ISPFx00 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: IOCB data should be copied to I/O mem using memcpy_toio (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: ISP8044 poll ipmdio bus timeout improvement (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Remove unnecessary printk_ratelimited from qla_nx2.c (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Include <delay.h> file for msleep declartion in qla_nx2.c file (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Use proper log message for flash lock failed error (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Decrease pci access for response queue processing for ISPFX00 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Change copyright year to 2014 in all the source files (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Enable fw_dump_size for ISP8044 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Introduce fw_dump_flag to track fw dump progress (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Remove unnecessary delays from fw dump code path (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Track the process when the ROM_LOCK failure happens (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Add ISP8044 serdes bsg interface (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Check for peg alive counter and clear any outstanding mailbox command (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Support of new firmware dump opcodes QLA8044_RDDFE(38), QLA8044_RDMDIO(39), QLA8044_POLLWR(40) (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Allow the next firmware dump if the previous dump capture fails for ISP8044 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Log when device state is moved to failed state (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Do not schedule reset when one is already active when receiving an invalid status handle (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Wait for reset completion without lock for ISPFX00 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Re-sync module parameter descriptions with the code (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: fix error handling of qla2x00_mem_alloc() (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Add IOCB Abort command asynchronous handling (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Fix Task Management command asynchronous handling (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Correct the port no assignment for ISP82XX (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Read capture firmware dump on mailbox timeout for ISP8044 and ISP82XX (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Simplify the ISPFX00 interrupt handler code for ISPFX00 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Avoid poisoning in the response queue for ISPFX00 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Remove ISP_ABORT_NEEDED and ISP_ABORT_RETRY checks from watchdog function for ISP8044 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Remove Marker type IOCB logic for ISPFX00 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Properly handle 32 bit mailbox register for ISPFX00 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Enable the Flash Access Control (FAC) mailbox command (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Select correct request queue for error type IOCB for ISPFX00 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Remove init control block related dead code for ISPFX00 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Use proper message for Non owner reset ACK Timeout (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Replace constant value for IOCTL IOCB abort execution status with a macro for ISPFX00 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Add handling for boot indication progress AENs for ISPFX00 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Add mutex around optrom calls to serialize accesses (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Poll during initialization for ISP25xx and ISP83xx (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Fix multiqueue MSI-X registration (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Fix warning reported by smatch (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Replace a constant with a macro definition for host->canqueue assigmnment (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Reset nic_core_reset_owner on moving from COLD to READY for ISP8044 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Only complete dcbx_comp and lb_portup_comp for virtual port index 0 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Use scnprintf() instead of snprintf() in the sysfs handlers (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Use the correct mailbox registers when acknowledging an IDC request on ISP8044 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Add changes to obtain ISPFX00 adapters product information in accordance with firmware update (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Add logic to abort BSG commands for ISPFX00 (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Clear RISC INT reg only for an event and not always while polling (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Fix undefined behavior in call to snprintf() (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Add BSG interface for read/write serdes register (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Fix issue with not displaying node name after system reboot (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Don't consider the drivers knocked out of IDC participation for future reset recovery process (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Add BPM support for ISP25xx (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Correctly set mailboxes for extended init control block (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Disable INTx interrupt for ISP82XX (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Honor execute firmware failures (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Print proper QLAFX00 product name at probe (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: print MAC via pMR (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Correction to message ids (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Correctly print out/in mailbox registers (Chad Dupuis) [1089346]
+- [scsi] qla2xxx: Add a new interface to update versions (Chad Dupuis) [1089346]
+
 * Fri Sep 12 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-156.el7]
 - [tty] hvc_console: Fix wakeup of HVC thread on hvc_kick() (Steve Best) [1123045]
 - [tty] hvc_opal: Kick the HVC thread on OPAL console events (Steve Best) [1123045]
-- 
1.7.1