From b46ba59a88f5eee42c8ef699a58ed66de54b6c34 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Thu, 22 Jan 2015 19:16:05 -0500
Subject: [redhat] kernel-3.10.0-225.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index cab016e..a2a1d7c 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 224
+RHEL_RELEASE = 225
 RHEL_DRM_VERSION = 3
 RHEL_DRM_PATCHLEVEL = 16
 RHEL_DRM_SUBLEVEL = 7
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 10cfce6..7182826 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=224
+BUILD:=225
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index a0333af..4bf49be 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1492,6 +1492,11 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Thu Jan 22 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-225.el7]
+- [net] team: avoid possible underflow of count_pending value for notify_peers and mcast_rejoin (Jiri Pirko) [1176697]
+- [fs] seq_file: don't include mm.h in genksyms calculation (Ian Kent) [1183280]
+- [scsi] Avoid crashing if device uses DIX but adapter does not support it (Ewan Milne) [1093012]
+
 * Mon Jan 19 2015 Jarod Wilson <jarod@redhat.com> [3.10.0-224.el7]
 - [fs] xfs: catch invalid negative blknos in _xfs_buf_find() (Eric Sandeen) [1164128]
 - [fs] proc: make proc_fd_permission() thread-friendly (Carlos Maiolino) [1171242]
-- 
1.7.1