From a4f5458a5dfef6d942d567b3525ffb7fcf742985 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Thu, 4 Sep 2014 10:54:29 -0400
Subject: [redhat] kernel-3.10.0-151.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 96e4d83..a259a68 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 150
+RHEL_RELEASE = 151
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 27398fe..bc52f9c 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=150
+BUILD:=151
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index dfbff70..6e2506a 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1472,6 +1472,148 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Thu Sep 04 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-151.el7]
+- [md] raid6: avoid data corruption during recovery of double-degraded RAID6 (Jes Sorensen) [1130905]
+- [block] scsi_ioctl: verify return pointer from blk_get_request (Jeff Moyer) [1104324]
+- [pnp] pnpacpi: Fix acpi_pnp_match() (Myron Stowe) [1128632]
+- [pnp] pnpacpi: Do not return errors if _DIS or _SRS are not present (Myron Stowe) [1128632]
+- [acpi] dock: Make 'docked' sysfs attribute work as documented (Myron Stowe) [1128632]
+- [acpi] dock: Use acpi_device_enumerated() to check if dock is present (Myron Stowe) [1128632]
+- [acpi] hotplug: Fix panic on eject to ejected device (Myron Stowe) [1128632]
+- [acpi] scan: Clear match_driver flag in acpi_bus_trim() (Myron Stowe) [1128632]
+- [acpi] scan: Use direct recurrence for device hierarchy walks (Myron Stowe) [1128632]
+- [acpi] Introduce acpi_set_device_status() (Myron Stowe) [1128632]
+- [acpi] hotplug: Drop unfinished global notification handling routines (Myron Stowe) [1128632]
+- [acpi] hotplug: Rework generic code to handle suprise removals (Myron Stowe) [1128632]
+- [acpi] hotplug: Move container-specific code out of the core (Myron Stowe) [1128632]
+- [acpi] hotplug: Make ACPI PCI root hotplug use common hotplug code (Myron Stowe) [1128632]
+- [acpi] hotplug: Introduce common hotplug function acpi_device_hotplug() (Myron Stowe) [1128632]
+- [acpi] hotplug: Do not fail bus and device checks for disabled hotplug (Myron Stowe) [1128632]
+- [acpi] scan: Add acpi_device objects for all device nodes in the namespace (Myron Stowe) [1128632]
+- [acpi] scan: Define non-empty device removal handler (Myron Stowe) [1128632]
+- [acpi] core: Store an ACPI device pointer in struct acpi_dev_node (Myron Stowe) [1128632]
+- [acpi] pm: Drop two functions that are not used any more (Myron Stowe) [1128632]
+- [acpi] power: Drop automaitc resume of power resource dependent devices (Myron Stowe) [1128632]
+- [acpi] Use EXPORT_SYMBOL() for acpi_bus_get_device() (Myron Stowe) [1128632]
+- [acpi] Print diagnostic messages if device links cannot be created (Myron Stowe) [1128632]
+- [acpi] Drop unnecessary label from acpi_bind_one() (Myron Stowe) [1128632]
+- [acpi] Clean up error code path in acpi_unbind_one() (Myron Stowe) [1128632]
+- [acpi] Use list_for_each_entry() in acpi_unbind_one() (Myron Stowe) [1128632]
+- [acpi] acpi_bind_one()/acpi_unbind_one() whitespace cleanups (Myron Stowe) [1128632]
+- [acpi] Create symlinks in acpi_bind_one() under physical_node_lock (Myron Stowe) [1128632]
+- [acpi] Reduce acpi_bind_one()/acpi_unbind_one() code duplication (Myron Stowe) [1128632]
+- [acpi] Do not fail acpi_bind_one() if device is already bound correctly (Myron Stowe) [1128632]
+- [acpi] Move acpi_bus_get_device() from bus.c to scan.c (Myron Stowe) [1128632]
+- [acpi] pm: Use ACPI_STATE_D3_COLD instead of ACPI_STATE_D3 everywhere (Myron Stowe) [1128632]
+- [usb] xhci: convert TRB_CYCLE to le32 before using it to set Link TRB's cycle bit (Steve Best) [1123119]
+- [usb] xhci: fix incorrect type in assignment in handle_device_notification() (Steve Best) [1123119]
+- [input] alps: Rushmore and v7 resolution support (Benjamin Tissoires) [1107819]
+- [input] alps: add support for v7 devices (Benjamin Tissoires) [1107819]
+- [input] alps: cache firmware version (Benjamin Tissoires) [1107819]
+- [input] alps: change decode function prototype to return an int (Benjamin Tissoires) [1107819]
+- [input] alps: report 2 touches when we've > 2 fingers (Benjamin Tissoires) [1107819]
+- [input] alps: add an alps_report_semi_mt_data function (Benjamin Tissoires) [1107819]
+- [input] alps: use single touch data when v3 mt data contains only one finger (Benjamin Tissoires) [1107819]
+- [input] alps: use standard contact tracking instead of DIY (Benjamin Tissoires) [1107819]
+- [input] alps: use struct input_mt_pos to track coordinates (Benjamin Tissoires) [1107819]
+- [input] alps: process_bitmap, round down when spreading adjescent fingers over 2 points (Benjamin Tissoires) [1107819]
+- [input] alps: process_bitmap, fix counting of high point bits (Benjamin Tissoires) [1107819]
+- [input] alps: process_bitmap, add alps_get_bitmap_points() helper function (Benjamin Tissoires) [1107819]
+- [input] alps: process_bitmap, don't invert the Y-axis on Rushmore (Benjamin Tissoires) [1107819]
+- [input] alps: improve 2-finger reporting on v3 models (Benjamin Tissoires) [1107819]
+- [input] alps: fix rushmore packet decoding (Benjamin Tissoires) [1107819]
+- [input] alps: add support for "Dolphin" devices (Benjamin Tissoires) [1107819]
+- [input] alps: add support for DualPoint device on Dell XT2 model (Benjamin Tissoires) [1107819]
+- [input] alps: change secondary device's name (Benjamin Tissoires) [1107819]
+- [tools] turbostat: tweak whitespace in output format (Prarit Bhargava) [1133893]
+- [tools] cpupower: bench/parse.c, Fix several minor errors (Prarit Bhargava) [1133893]
+- [tools] cpupower: Remove redundant error check (Prarit Bhargava) [1133893]
+- [tools] cpupower: mperf monitor, Correct use of ! and & (Prarit Bhargava) [1133893]
+- [tools] cpupower: drop negativity check on unsigned value (Prarit Bhargava) [1133893]
+- [tools] cpupower: Remove mc and smt power aware scheduler info/settings (Prarit Bhargava) [1133893]
+- [tools] cpupower: cpupower info -b should return 0 on success, not the perf bias value (Prarit Bhargava) [1133893]
+- [tools] cpupower: Remove dead link to homepage, and update the targets built (Prarit Bhargava) [1133893]
+- [tools] cpupower: Rename cpufrequtils -> cpupower, and libcpufreq -> libcpupower (Prarit Bhargava) [1133893]
+- [tools] cpupower: If root, try to load msr driver on x86 if /dev/cpu/0/msr is not available (Prarit Bhargava) [1133893]
+- [tools] cpupower: Install recently added cpupower-idle-{set, info} manpages (Prarit Bhargava) [1133893]
+- [tools] cpupower: Remove all manpages on make uninstall (Prarit Bhargava) [1133893]
+- [tools] cpupower: Introduce idle state disable-by-latency and enable-all (Prarit Bhargava) [1133893]
+- [tools] turbostat: Drop temperature checks (Prarit Bhargava) [1133893]
+- [tools] cpufreq: Make linux-pm@vger.kernel.org official mailing list (Prarit Bhargava) [1133893]
+- [tools] turbostat: simplify output, add Avg_MHz (Prarit Bhargava) [1133893]
+- [tools] turbostat: introduce -s to dump counters (Prarit Bhargava) [1133893]
+- [tools] turbostat: remove unused command line option (Prarit Bhargava) [1133893]
+- [tools] turbostat: Add option to report joules consumed per sample (Prarit Bhargava) [1133893]
+- [tools] turbostat: Clean up error handling; disambiguate error messages; use err and errx (Prarit Bhargava) [1133893]
+- [tools] turbostat: Add a .gitignore to ignore the compiled turbostat binary (Prarit Bhargava) [1133893]
+- [tools] turbostat: Factor out common function to open file and exit on failure (Prarit Bhargava) [1133893]
+- [tools] turbostat: Add a helper to parse a single int out of a file (Prarit Bhargava) [1133893]
+- [tools] turbostat: Check return value of fscanf (Prarit Bhargava) [1133893]
+- [tools] turbostat: Don't attempt to printf an off_t with zx (Prarit Bhargava) [1133893]
+- [tools] turbostat: Use GCC's CPUID functions to support PIC (Prarit Bhargava) [1133893]
+- [tools] turbostat: Don't put unprocessed uapi headers in the include path (Prarit Bhargava) [1133893]
+- [tools] cpupower: Fix sscanf robustness in cpufreq-set (Prarit Bhargava) [1133893]
+- [tools] cpupower: Fix segfault due to incorrect getopt_long arugments (Prarit Bhargava) [1133893]
+- [tools] cpupower: fix wrong err msg not supported vs not available (Prarit Bhargava) [1133893]
+- [tools] cpupower: Add cpupower-idle-set(1) manpage (Prarit Bhargava) [1133893]
+- [virt] virtio-scsi: Skip setting affinity on uninitialized vq (Fam Zheng) [1083860]
+- [mm] memcontrol: remove hierarchy restrictions for swappiness and oom_control (Johannes Weiner) [1124110]
+- [mm] thp: fix copy_page_rep GPF by testing is_huge_zero_pmd once only (Rik van Riel) [1123040]
+- [mm] numa: Remove BUG_ON() in __handle_mm_fault() (Rik van Riel) [1119439]
+- [iommu] Add empty stub for iommu_group_get_by_id() (Steve Best) [1123122]
+- [kernel] kexec: fix build error when hugetlbfs is disabled (Baoquan He) [1115240]
+- [kernel] kexec: export free_huge_page to VMCOREINFO (Baoquan He) [1115240]
+- [kernel] kexec: save PG_head_mask in VMCOREINFO (Baoquan He) [1115240]
+- [block] nvme: Fix START_STOP_UNIT Scsi->NVMe translation (David Milburn) [1111259]
+- [block] nvme: Use Log Page constants in SCSI emulation (David Milburn) [1111259]
+- [block] nvme: Define Log Page constants (David Milburn) [1111259]
+- [block] nvme: Fix hot cpu notification dead lock (David Milburn) [1111259]
+- [block] nvme: Rename io_timeout to nvme_io_timeout (David Milburn) [1111259]
+- [block] nvme: Use last bytes of f/w rev SCSI Inquiry (David Milburn) [1111259]
+- [block] nvme: Adhere to request queue block accounting enable/disable (David Milburn) [1111259]
+- [block] nvme: Fix nvme get/put queue semantics (David Milburn) [1111259]
+- [block] nvme: Delete NVME_GET_FEAT_TEMP_THRESH (David Milburn) [1111259]
+- [block] nvme: Make admin timeout a module parameter (David Milburn) [1111259]
+- [block] nvme: Make iod bio timeout a parameter (David Milburn) [1111259]
+- [block] nvme: Prevent possible NULL pointer dereference (David Milburn) [1111259]
+- [block] nvme: Fix the buffer size passed in GetLogPage(CDW10.NUMD) (David Milburn) [1111259]
+- [block] nvme: Update data structures for NVMe 1.2 (David Milburn) [1111259]
+- [block] nvme: Enable BUILD_BUG_ON checks (David Milburn) [1111259]
+- [block] nvme: Update namespace and controller identify structures to the 1.1a spec (David Milburn) [1111259]
+- [block] nvme: Flush with data support (David Milburn) [1111259]
+- [block] nvme: Configure support for block flush (David Milburn) [1111259]
+- [block] nvme: Add tracepoints (David Milburn) [1111259]
+- [block] nvme: Protect against badly formatted CQEs (David Milburn) [1111259]
+- [block] nvme: Improve error messages (David Milburn) [1111259]
+- [block] nvme: Update copyright headers (David Milburn) [1111259]
+- [powerpc] 64bit sendfile is capped at 2GB (Gustavo Duarte) [1107774]
+- [powerpc] sched: stop updating inside arch_update_cpu_topology() when nothing to be update (Gustavo Duarte) [1098372]
+- [firewire] ohci: fix probe failure with Agere/LSI controllers (Neil Horman) [994878]
+- [firewire] ohci: beautify some macro definitions (Neil Horman) [994878]
+- [s390] airq: silence lockdep warning (Hendrik Brueckner) [1088554]
+- [virt] virtio_scsi: don't call virtqueue_add_sgs(... GFP_NOIO) holding spinlock (Hendrik Brueckner) [1088554]
+- [virt] virtio_ccw: introduce device_lost in virtio_ccw_device (Hendrik Brueckner) [1088554]
+- [virt] virtio: virtio_break_device() to mark all virtqueues broken (Hendrik Brueckner) [1088554]
+- [virt] s390/sclp_vt220: Fix kernel panic due to early terminal input (Hendrik Brueckner) [1088554]
+- [virt] virtio: fail adding buffer on broken queues (Hendrik Brueckner) [1088554]
+- [virt] virtio_net: don't crash if virtqueue is broken (Hendrik Brueckner) [1088554]
+- [virt] s390/virtio_ccw: fix hang in set offline processing (Hendrik Brueckner) [1088554]
+- [s390] irq: improve displayed interrupt order in /proc/interrupts (Hendrik Brueckner) [1088554]
+- [virt] virtio-ccw: virtio-ccw adapter interrupt support (Hendrik Brueckner) [1088554]
+- [virt] virtio_ccw: fix vcdev pointer handling issues (Hendrik Brueckner) [1088554]
+- [virt] s390/airq: add support for irq ranges (Hendrik Brueckner) [1088554]
+- [virt] kvm/s390/virtio-ccw: Handle command rejects (Hendrik Brueckner) [1088554]
+- [virt] virtio_scsi: verify if queue is broken after virtqueue_get_buf() (Hendrik Brueckner) [1088554]
+- [virt] virtio_ring: adapt to notify() returning bool (Hendrik Brueckner) [1088554]
+- [virt] virtio_net: verify if queue is broken after virtqueue_get_buf() (Hendrik Brueckner) [1088554]
+- [virt] virtio_blk: verify if queue is broken after virtqueue_get_buf() (Hendrik Brueckner) [1088554]
+- [virt] virtio_ring: add new function virtqueue_is_broken() (Hendrik Brueckner) [1088554]
+- [virt] virtio_ring: let virtqueue_{kick()/notify()} return a bool (Hendrik Brueckner) [1088554]
+- [virt] tools/virtio: fix missing kmemleak_ignore symbol (Hendrik Brueckner) [1088554]
+- [virt] virtio_ring: plug kmemleak false positive (Hendrik Brueckner) [1088554]
+- [mm] page_alloc: do not cache reclaim distances (Larry Woodman) [1120342]
+- [mm] disable zone_reclaim_mode by default (Larry Woodman) [1120342]
+
 * Tue Sep 02 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-150.el7]
 - [net] team: set IFF_TEAM_PORT priv_flag after rx_handler is registered (Jiri Pirko) [1132943]
 - [net] ipv6: fix calculation of option len in ip6_append_data (Hannes Frederic Sowa) [1127218]
-- 
1.7.1