From b5e548f1932cf6affc8c7df9ac543239a0f407ae Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Mon, 24 Nov 2014 17:38:58 -0500
Subject: [redhat] kernel-3.10.0-210.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 51e8358..3f1541a 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 209
+RHEL_RELEASE = 210
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 8c52ed5..9261e92 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=209
+BUILD:=210
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index f2a0bd6..a37f4fd 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,9 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Mon Nov 24 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-210.el7]
+- [misc] kabi: revert two kabi additions that need updated ppc64 sums (Jarod Wilson)
+
 * Mon Nov 24 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-209.el7]
 - [fs] xfs: write failure beyond EOF truncates too much data (Brian Foster) [1032968]
 - [fs] xfs: xfs_vm_write_end truncates too much on failure (Brian Foster) [1032968]
-- 
1.7.1