From 8c8c2de5d59fdb1a92d0720dddab24821d449273 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 2 Sep 2014 12:20:57 -0400
Subject: [redhat] kernel-3.10.0-149.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index eb379f5..a813f0d 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 148
+RHEL_RELEASE = 149
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 3d08ad5..86bc4df 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=148
+BUILD:=149
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 7d5ea11..66fa0ac 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1472,6 +1472,78 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Sep 02 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-149.el7]
+- [fs] ext4: fix type declaration of ext4_validate_block_bitmap (Lukas Czerner) [1091055]
+- [fs] ext4: error out if verifying the block bitmap fails (Lukas Czerner) [1091055]
+- [fs] cifs: Fix memory leaks in SMB2_open (Sachin Prabhu) [1082049]
+- [fs] cifs: ensure that vol->username is not NULL before running strlen on it (Sachin Prabhu) [1082049]
+- [fs] cifs: Clarify SMB2/SMB3 create context and add missing ones (Sachin Prabhu) [1082049]
+- [fs] cifs: Do not send ClientGUID on SMB2.02 dialect (Sachin Prabhu) [1082049]
+- [fs] cifs: Set client guid on per connection basis (Sachin Prabhu) [1082049]
+- [fs] cifs/netmisc: convert printk to pr_foo() (Sachin Prabhu) [1082049]
+- [fs] cifs: replace seq_printf by seq_puts (Sachin Prabhu) [1082049]
+- [fs] cifs: Update cifs version number to 2.03 (Sachin Prabhu) [1082049]
+- [fs] cifs: new helper file_inode(file) (Sachin Prabhu) [1082049]
+- [fs] cifs: fix potential races in cifs_revalidate_mapping (Sachin Prabhu) [1082049]
+- [fs] cifs: new helper function: cifs_revalidate_mapping (Sachin Prabhu) [1082049]
+- [fs] cifs: convert booleans in cifsInodeInfo to a flags field (Sachin Prabhu) [1082049]
+- [fs] cifs: fix cifs_uniqueid_to_ino_t not to ever return 0 (Sachin Prabhu) [1082049]
+- [fs] cifs: fix actimeo=0 corner case when cifs_i->time == jiffies (Sachin Prabhu) [1082049]
+- [fs] cifs: fix dead code (Sachin Prabhu) [1082049]
+- [fs] cifs: fix error handling cifs_user_readv (Sachin Prabhu) [1082049]
+- [fs] cifs: remove unused variable (Sachin Prabhu) [1082049]
+- [fs] cifs: Return correct error on query of xattr on file with empty xattrs (Sachin Prabhu) [1082049]
+- [fs] cifs: fix the race in cifs_writev() (Sachin Prabhu) [1082049]
+- [fs] cifs: add __init to cifs_init_inodecache() (Sachin Prabhu) [1082049]
+- [fs] cifs: ->rename() without ->lookup() makes no sense (Sachin Prabhu) [1082049]
+- [fs] cifs: Fix too big maxBuf size for SMB3 mounts (Sachin Prabhu) [1082049]
+- [fs] cifs: Fix cifsacl mounts over smb2 to not call cifs (Sachin Prabhu) [1082049]
+- [fs] cifs: retrieving CIFS ACLs when mounted with SMB2 fails dropping session (Sachin Prabhu) [1082049]
+- [fs] cifs: Add protocol specific operation for CIFS xattrs (Sachin Prabhu) [1082049]
+- [fs] cifs: Fix SMB2 mounts so they don't try to set or get xattrs via cifs (Sachin Prabhu) [1082049]
+- [fs] cifs: Cleanup cifs open codepath (Sachin Prabhu) [1082049]
+- [fs] cifs: Remove extra indentation in cifs_sfu_type (Sachin Prabhu) [1082049]
+- [fs] cifs: Cleanup cifs_mknod (Sachin Prabhu) [1082049]
+- [fs] cifs: Cleanup CIFSSMBOpen (Sachin Prabhu) [1082049]
+- [fs] cifs: Fix memory leak in cifs_hardlink() (Sachin Prabhu) [1082049]
+- [fs] cifs: Add missing end of line termination to some cifs messages (Sachin Prabhu) [1082049]
+- [fs] cifs: Do not use btrfs refcopy ioctl for SMB2 copy offload (Sachin Prabhu) [1082049]
+- [fs] cifs: Check SMB3 dialects against downgrade attacks (Sachin Prabhu) [1082049]
+- [fs] cifs: Removed duplicated (and unneeded) goto (Sachin Prabhu) [1082049]
+- [fs] cifs: Fix SMB2/SMB3 Copy offload support (refcopy) for large files (Sachin Prabhu) [1082049]
+- [fs] cifs: Warn if SMB3 encryption required by server (Sachin Prabhu) [1082049]
+- [fs] cifs: Set copychunk defaults (Sachin Prabhu) [1082049]
+- [fs] cifs: SMB2/SMB3 Copy offload support (refcopy) phase 1 (Sachin Prabhu) [1082049]
+- [fs] cifs: Use data structures to compute NTLMv2 response offsets (Sachin Prabhu) [1082049]
+- [fs] cifs: O_DIRECT opens should work on directio mounts (Sachin Prabhu) [1082049]
+- [fs] cifs: don't spam the logs on unexpected lookup errors (Sachin Prabhu) [1082049]
+- [fs] cifs: change ERRnomem error mapping from ENOMEM to EREMOTEIO (Sachin Prabhu) [1082049]
+- [fs] cifs: Fix symbolic links usage (Sachin Prabhu) [1082049]
+- [fs] cifs: Query network adapter info at mount time for debugging (Sachin Prabhu) [1082049]
+- [fs] cifs: Fix unused variable warning when CIFS POSIX disabled (Sachin Prabhu) [1082049]
+- [fs] cifs: Allow setting per-file compression via CIFS protocol (Sachin Prabhu) [1082049]
+- [fs] cifs: Query File System Alignment (Sachin Prabhu) [1082049]
+- [fs] cifs: Query device characteristics at mount time from server on SMB2/3 not just on cifs mounts (Sachin Prabhu) [1082049]
+- [fs] cifs: Send a logoff request before removing a smb session (Sachin Prabhu) [1082049]
+- [fs] cifs: Make big endian multiplex ID sequences monotonic on the wire (Sachin Prabhu) [1082049]
+- [fs] cifs: Remove redundant multiplex identifier check from check_smb_hdr() (Sachin Prabhu) [1082049]
+- [fs] cifs: Query file system attributes from server on SMB2, not just cifs, mounts (Sachin Prabhu) [1082049]
+- [fs] cifs: Allow setting per-file compression via SMB2/3 (Sachin Prabhu) [1082049]
+- [fs] cifs: Fix corrupt SMB2 ioctl requests (Sachin Prabhu) [1082049]
+- [fs] cifs: rcu-delay unload_nls() and freeing sbi (Sachin Prabhu) [1082049]
+- [fs] cifs: new helper kfree_put_link() (Sachin Prabhu) [1082049]
+- [fs] cifs: ntstatus_to_dos_map[] is not terminated (Sachin Prabhu) [1082049]
+- [fs] cifs: Allow LANMAN auth method for servers supporting unencapsulated authentication methods (Sachin Prabhu) [1082049]
+- [fs] cifs: Fix inability to write files >2GB to SMB2/3 shares (Sachin Prabhu) [1082049]
+- [fs] cifs: Avoid umount hangs with smb2 when server is unresponsive (Sachin Prabhu) [1082049]
+- [fs] cifs: do not treat non-symlink reparse points as valid symlinks (Sachin Prabhu) [1082049]
+- [fs] cifs: update cifs.ko version (Sachin Prabhu) [1082049]
+- [fs] cifs: Provide sane values for nlink (Sachin Prabhu) [1082049]
+- [fs] cifs: FS-Cache: Uncache unread pages in cifs_readpages() before freeing them (Sachin Prabhu) [1082049]
+- [fs] cifs: Avoid calling unlock_page() twice in cifs_readpage() when using fscache (Sachin Prabhu) [1082049]
+- [fs] cifs: Do not take a reference to the page in cifs_readpage_worker() (Sachin Prabhu) [1082049]
+- [fs] cifs: Fix wrong pos argument of cifs_find_lock_conflict (Sachin Prabhu) [1082049]
+
 * Sat Aug 30 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-148.el7]
 - [ethernet] igbvf: Remove DEFINE_PCI_DEVICE_TABLE macro use (Stefan Assmann) [1091122]
 - [ethernet] igbvf: use SPEED_UNKNOWN and DUPLEX_UNKNOWN when appropriate (Stefan Assmann) [1091122]
-- 
1.7.1