From 969b309345d4117524af5688a5b39efcef3c8d10 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Sat, 22 Nov 2014 13:18:46 -0500
Subject: [redhat] kernel-3.10.0-207.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 31644cc..be90f7e 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 206
+RHEL_RELEASE = 207
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 40ad649..21593e4 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=206
+BUILD:=207
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index d5229c8..081d8ae 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,27 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Sat Nov 22 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-207.el7]
+- [powerpc] use device_online/offline() instead of cpu_up/down() (Gustavo Duarte) [1157737]
+- [ethernet] i40e: disable FCoE (Stefan Assmann) [1165175]
+- [cpufreq] intel_pstate: Add CPUID for BDW-H CPU (Steve Best) [1164379]
+- [mm] do not overwrite reserved pages counter at show_mem() (Rafael Aquini) [1125433]
+- [ethernet] enic: Do not call napi_disable when preemption is disabled (Stefan Assmann) [1145019]
+- [ethernet] enic: fix possible deadlock in enic_stop/ enic_rfs_flw_tbl_free (Stefan Assmann) [1145019]
+- [x86] uv_bau: Avoid NULL pointer reference in ptc_seq_show (Frank Ramsay) [1161183]
+- [x86] uv_bau: Increase maximum CPUs per socket/hub (Frank Ramsay) [1161183]
+- [mm] vmscan: do not throttle based on pfmemalloc reserves if node has no ZONE_NORMAL (Gustavo Duarte) [1148925]
+- [char] hwrng/pseries: port to new read API and fix stack corruption (Gustavo Duarte) [1163659]
+- [md] dm-cache: emit a warning message if there are a lot of cache blocks (Mike Snitzer) [1159001]
+- [md] dm-cache: improve discard support (Mike Snitzer) [1159001]
+- [md] dm-cache: revert "prevent corruption caused by discard_block_size > cache_block_size" (Mike Snitzer) [1159001]
+- [md] dm-cache: revert "remove remainder of distinct discard block size" (Mike Snitzer) [1159001]
+- [md] dm-bio-prison: introduce support for locking ranges of blocks (Mike Snitzer) [1159001]
+- [md] dm-btree: fix a recursion depth bug in btree walking code (Mike Snitzer) [1080894]
+- [md] dm-cache-policy-mq: simplify ability to promote sequential IO to the cache (Mike Snitzer) [1159001]
+- [md] dm-cache-policy-mq: tweak algorithm that decides when to promote a block (Mike Snitzer) [1159001]
+- [security] selinux: fix inode security list corruption (Paul Moore) [1152274]
+
 * Thu Nov 20 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-206.el7]
 - [x86] quirks: Print the Intel graphics stolen memory range (Rob Clark) [1154053]
 - [x86] quirks: Add Intel graphics stolen memory quirk for gen2 platforms (Rob Clark) [1154053]
-- 
1.7.1