From 3adced2eaf0d0128d2c150181cb377b301ffc361 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 30 Sep 2014 01:43:44 -0400
Subject: [redhat] kernel-3.10.0-175.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 3209edc..a8a62b6 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 174
+RHEL_RELEASE = 175
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 96c0d89..6b5de30 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=174
+BUILD:=175
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index d1a0508..0c1914b 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1483,6 +1483,119 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Sep 30 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-175.el7]
+- [misc] mei: ignore client writing state during cb completion (Prarit Bhargava) [1088953]
+- [misc] mei: don't use deprecated DEFINE_PCI_DEVICE_TABLE macro (Prarit Bhargava) [1088953]
+- [misc] mei: amthif: fix checkpatch error (Prarit Bhargava) [1088953]
+- [misc] mei: fix Unnecessary space after function pointer name (Prarit Bhargava) [1088953]
+- [misc] mei: use cl_dbg where appropriate (Prarit Bhargava) [1088953]
+- [misc] mei: client.h fix checkpatch errors (Prarit Bhargava) [1088953]
+- [misc] mei: report consistently copy_from/to_user failures (Prarit Bhargava) [1088953]
+- [misc] mei: drop pr_fmt macros (Prarit Bhargava) [1088953]
+- [misc] mei: make me hw headers private to me hw (Prarit Bhargava) [1088953]
+- [misc] mei: fix memory leak of pending write cb objects (Prarit Bhargava) [1088953]
+- [misc] mei: me: do not reset when less than expected data is received (Prarit Bhargava) [1088953]
+- [misc] mei: fix regressions caused by removing ext_msg (Prarit Bhargava) [1088953]
+- [misc] mei: wd: fix stop completion failure (Prarit Bhargava) [1088953]
+- [misc] mei: wd: simplify wd_send command (Prarit Bhargava) [1088953]
+- [misc] mei: make return values consistent across the driver (Prarit Bhargava) [1088953]
+- [misc] mei: revamp writing slot counting (Prarit Bhargava) [1088953]
+- [misc] mei: add mei_hbuf_acquire wrapper (Prarit Bhargava) [1088953]
+- [misc] mei: txe: include irqreturn.h for irqreturn_t etc (Prarit Bhargava) [1088953]
+- [misc] mei: set client's read_cb to NULL when flow control fails (Prarit Bhargava) [1088953]
+- [misc] mei: txe: put pm callbacks under PM_SLEEP ifdef (Prarit Bhargava) [1088953]
+- [misc] mei: me: put pm callbacks under PM_SLEEP ifdef (Prarit Bhargava) [1088953]
+- [misc] mei: don't of list_for_each_entry_safe when not deleting (Prarit Bhargava) [1088953]
+- [misc] mei: use helper function to find me client by id (Prarit Bhargava) [1088953]
+- [misc] mei: fix potential read outside of array bounds (Prarit Bhargava) [1088953]
+- [misc] mei: wd and amthif use mei_cl_ api for dis/connection (Prarit Bhargava) [1088953]
+- [misc] mei: hbm: revamp client connect and disconnection status (Prarit Bhargava) [1088953]
+- [misc] mei: Remove all bus devices from the mei_dev list when stopping the MEI (Prarit Bhargava) [1088953]
+- [misc] mei: get rid of ext_msg (Prarit Bhargava) [1088953]
+- [misc] mei: rename MEI_FOP_IOCTL to MEI_FOP_CONNECT (Prarit Bhargava) [1088953]
+- [misc] mei: fix compilation error with missing WATCHDOG_CORE (Prarit Bhargava) [1088953]
+- [misc] mei: allow multiple retries if the hw reset has failed (Prarit Bhargava) [1088953]
+- [misc] mei: export active connections to debugfs (Prarit Bhargava) [1088953]
+- [misc] mei: txe: add Kbuild for TXE device (Prarit Bhargava) [1088953]
+- [misc] mei: txe: add pci-txe.c (Prarit Bhargava) [1088953]
+- [misc] mei: txe: add hw-txe-regs.h header file (Prarit Bhargava) [1088953]
+- [misc] mei: txe: add hw-txe.c (Prarit Bhargava) [1088953]
+- [misc] mei: txe: add hw-txe.h header file (Prarit Bhargava) [1088953]
+- [misc] mei: don't unset read cb ptr on reset (Prarit Bhargava) [1088953]
+- [misc] mei: clear write cb from waiting list on reset (Prarit Bhargava) [1088953]
+- [misc] mei: limit the number of consecutive resets (Prarit Bhargava) [1088953]
+- [misc] mei: revamp mei reset state machine (Prarit Bhargava) [1088953]
+- [misc] mei: fix syntax in comments and debug output (Prarit Bhargava) [1088953]
+- [misc] mei: nfc: mei_nfc_free has to be called under lock (Prarit Bhargava) [1088953]
+- [misc] mei: use hbm idle state to prevent spurious resets (Prarit Bhargava) [1088953]
+- [misc] mei: do not run reset flow from the interrupt thread (Prarit Bhargava) [1088953]
+- [misc] mei: enable marking internal commands (Prarit Bhargava) [1088953]
+- [misc] mei: me: set dma mask using DMA mapping API (Prarit Bhargava) [1088953]
+- [misc] mei: cleanup mei_irq_read_handler (Prarit Bhargava) [1088953]
+- [misc] mei: remove flash_work_queue (Prarit Bhargava) [1088953]
+- [misc] mei: drop redundant list_del_init (Prarit Bhargava) [1088953]
+- [misc] mei: bus: propagate error code returned by mei_me_cl_by_id (Prarit Bhargava) [1088953]
+- [misc] mei: mei_cl_link remove duplicated check for open_handle_count (Prarit Bhargava) [1088953]
+- [misc] mei: replace stray pr_debug with dev_dbg (Prarit Bhargava) [1088953]
+- [misc] mei: wd: host_init propagate error codes from called functions (Prarit Bhargava) [1088953]
+- [misc] mei: print correct device state during unexpected reset (Prarit Bhargava) [1088953]
+- [misc] mei: nfc: fix memory leak in error path (Prarit Bhargava) [1088953]
+- [misc] mei: move host_clients_map cleanup to device init (Prarit Bhargava) [1088953]
+- [misc] mei: me: downgrade two errors to debug level (Prarit Bhargava) [1088953]
+- [misc] mei: amthif: mei_amthif_host_init: propagate errors from called functions (Prarit Bhargava) [1088953]
+- [misc] mei: revamp open handler counts (Prarit Bhargava) [1088953]
+- [misc] mei: simplify mei_open error handling (Prarit Bhargava) [1088953]
+- [misc] mei: remove unnecessary pci_set_drvdata() (Prarit Bhargava) [1088953]
+- [misc] mei: mei_cl_unlink: no need to loop over dev list (Prarit Bhargava) [1088953]
+- [misc] mei: push credentials inside the irq write handler (Prarit Bhargava) [1088953]
+- [misc] mei: propagate error from write routines instead of ENODEV (Prarit Bhargava) [1088953]
+- [misc] mei: fix function names in debug prints (Prarit Bhargava) [1088953]
+- [misc] mei: prefix client log messages with client me and host ids (Prarit Bhargava) [1088953]
+- [misc] mei: mei_release: drop redundant check if cb is NULL (Prarit Bhargava) [1088953]
+- [misc] mei: fix format compilation warrning on 32 bit architecture (Prarit Bhargava) [1088953]
+- [misc] mei: revamp read and write length checks (Prarit Bhargava) [1088953]
+- [misc] mei: mei_write correct checks for copy_from_user (Prarit Bhargava) [1088953]
+- [misc] mei: mei_cl_link protect open_handle_count from overflow (Prarit Bhargava) [1088953]
+- [misc] mei: make sure that me_clients_map big enough before copying (Prarit Bhargava) [1088953]
+- [misc] mei: convert bus code to use dev_groups (Prarit Bhargava) [1088953]
+- [misc] mei: client.h cleanup (Prarit Bhargava) [1088953]
+- [ethernet] e1000e: Fix Runtime PM blocks EEE link negotiation in S5 (John Greene) [1091119]
+- [ethernet] e1000e: Fix EEE in S5 w/ Runtime PM enabled (John Greene) [1091119]
+- [ethernet] e1000e: Add support for EEE in Sx states (John Greene) [1091119]
+- [ethernet] e1000e: Add code to check return values on NVM accesses (John Greene) [1091119]
+- [ethernet] e1000e: Fix CRC errors with jumbo traffic (John Greene) [1091119]
+- [ethernet] e1000e: remove unnecessary break after return (John Greene) [1091119]
+- [ethernet] e1000e: use SPEED_UNKNOWN and DUPLEX_UNKNOWN when appropriate (John Greene) [1091119]
+- [ethernet] e1000e: Out of line __ew32_prepare/__ew32 (John Greene) [1091119]
+- [ethernet] e1000e: Fix expand setting EEE link info to all affected parts (John Greene) [1091119]
+- [ethernet] e1000e: Failure to write SHRA turns on PROMISC mode (John Greene) [1091119]
+- [ethernet] e1000e: Restrict MDIO Slow Mode workaround to relevant parts (John Greene) [1091119]
+- [ethernet] e1000e: Fix issue with link flap on 82579 (John Greene) [1091119]
+- [ethernet] e1000e: Expand workaround for 10Mb HD throughput bug (John Greene) [1091119]
+- [ethernet] e1000e: Workaround for dropped packets in Gig/100 speeds on 82579 (John Greene) [1091119]
+- [ethernet] e1000e: fix message terminations (John Greene) [1091119]
+- [ethernet] e1000e: Enclose e1000e_pm_thaw() with CONFIG_PM_SLEEP (John Greene) [1091119]
+- [ethernet] e1000e: Correctly include VLAN_HLEN when changing interface MTU (John Greene) [1091119]
+- [ethernet] e1000e: Fix no connectivity when driver loaded with cable out (John Greene) [1091119]
+- [ethernet] e1000e: add timeout for TX HW time stamping work (John Greene) [1091119]
+- [ethernet] e1000e: Fix Explicitly set Transmit Control Register (John Greene) [1091119]
+- [ethernet] e1000e: Fix Hardware Unit Hang (John Greene) [1091119]
+- [ethernet] e1000e: fix the build error when PM is disabled (John Greene) [1091119]
+- [ethernet] e1000e: Fix ethtool offline tests for 82579 parts (John Greene) [1091119]
+- [ethernet] e1000e: Fix not generating an error on invalid load parameter (John Greene) [1091119]
+- [ethernet] e1000e: Feature Enable PHY Ultra Low Power Mode (ULP) (John Greene) [1091119]
+- [ethernet] e1000e: Refactor of Runtime Power Management (John Greene) [1091119]
+- [ethernet] e1000e: calls skb_set_hash (John Greene) [1091119]
+- [ethernet] e1000e: Refactor PM flows (John Greene) [1091119]
+- [ethernet] e1000e: Fix compilation warning when !CONFIG_PM_SLEEP (John Greene) [1091119]
+- [ethernet] e1000e: Fix a compile flag mis-match for suspend/resume (John Greene) [1091119]
+- [ethernet] e1000e: Remove extern from function prototypes (John Greene) [1091119]
+- [ethernet] e1000e: Cleanup - Update GPL header and Copyright (John Greene) [1091119]
+- [ethernet] e1000e: Fix 82579 sets LPI too early (John Greene) [1091119]
+- [ethernet] e1000e: 82574/82583 TimeSync errata for SYSTIM read (John Greene) [1091119]
+- [kernel] sched/fair: Stop searching for tasks in idle_balance if there are runnable tasks (Larry Woodman) [1103828]
+- [kernel] sched/numa: Initialize new idle_balance stats in sd_numa_init() (Larry Woodman) [1103828]
+
 * Mon Sep 29 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-174.el7]
 - [kernel] tty: Add C_CMSPAR(tty) (Don Zickus) [1110939]
 - [usb] pl2303: fixed handling of CS5 setting (Don Zickus) [1110939]
-- 
1.7.1