From 65cd7fe692cb2f4e626906f611f45ea8b59c7e37 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 26 Aug 2014 13:19:19 -0400
Subject: [redhat] kernel-3.10.0-146.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 3de313c..2ec827a 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 0
-RHEL_RELEASE = 145
+RHEL_RELEASE = 146
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index 7dba06b..ae7ed2d 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=145
+BUILD:=146
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index c9831c6..1434334 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1472,6 +1472,107 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Tue Aug 26 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-146.el7]
+- [edac] sb_edac: add support for Haswell based systems (Seth Jennings) [1127019]
+- [edac] add DDR4 and RDDR4 (Seth Jennings) [1127019]
+- [edac] sb_edac: update Kconfig description (Seth Jennings) [1127019]
+- [edac] sb_edac: Fix mix tab/spaces alignments (Seth Jennings) [1127019]
+- [edac] sb_edac: remove bogus assumption on mc ordering (Seth Jennings) [1127019]
+- [edac] sb_edac: make minimal use of channel_mask (Seth Jennings) [1127019]
+- [edac] sb_edac: fix socket detection on Ivy Bridge controllers (Seth Jennings) [1127019]
+- [edac] sb_edac: search devices using product id (Seth Jennings) [1127019]
+- [edac] sb_edac: Degrade log level for device registration (Seth Jennings) [1127019]
+- [edac] sb_edac: make RIR limit retrieval per model (Seth Jennings) [1127019]
+- [edac] sb_edac: make node id retrieval per model (Seth Jennings) [1127019]
+- [edac] sb_edac: make memory type detection per memory controller (Seth Jennings) [1127019]
+- [hwrng] virtio-rng: add derating factor for use by hwrng core (Amit Shah) [1129195]
+- [hwrng] Pass entropy to add_hwgenerator_randomness() in bits, not bytes (Amit Shah) [1129195]
+- [hwrng] hw_random: fix sparse warning (NULL vs 0 for pointer) (Amit Shah) [1129195]
+- [hwrng] add per-device entropy derating (Amit Shah) [1129195]
+- [hwrng] create filler thread (Amit Shah) [1129195]
+- [hwrng] random: add_hwgenerator_randomness() for feeding entropy from devices (Amit Shah) [1129195]
+- [char] hwrng: virtio - delay hwrng_register() till driver is ready (Amit Shah) [1119299]
+- [char] hwrng: virtio - re-arrange struct elements for better packing (Amit Shah) [1119299]
+- [char] hwrng: virtio - remove unused struct element (Amit Shah) [1119299]
+- [char] hwrng: virtio - ensure reads happen after successful probe (Amit Shah) [1119299]
+- [char] hwrng: fetch randomness only after device init (Amit Shah) [1119299]
+- [char] hwrng: add randomness to system from rng sources (Amit Shah) [1119299]
+- [char] hwrng: cleanup in hwrng_register() (Amit Shah) [1119299]
+- [virt] virtio/pm: use CONFIG_PM_SLEEP instead of CONFIG_PM (Amit Shah) [1119299]
+- [virt] virtio-rng: fixes for device registration/unregistration (Amos Kong) [915335]
+- [virt] virtio-rng: fix boot with virtio-rng device (Amos Kong) [915335]
+- [virt] virtio-rng: support multiple virtio-rng devices (Amos Kong) [915335]
+- [virt] virtio-rng: don't crash if virtqueue is broken (Amos Kong) [915335]
+- [pci] Fix unaligned access in AF transaction pending test (Myron Stowe) [1110896]
+- [pci] Wrong register used to check pending traffic (Myron Stowe) [1110896]
+- [pci] shpchp: Check bridge's secondary (not primary) bus speed (Myron Stowe) [1110896]
+- [pci] Update maintainer email address (Myron Stowe) [1110896]
+- [powerpc] pci: Fix NULL dereference in sys_pciconfig_iobase() list traversal (Myron Stowe) [1110896]
+- [pci] Log IDE resource quirk in dmesg (Myron Stowe) [1110896]
+- [pci] Change pci_bus_alloc_resource() type_mask to unsigned long (Myron Stowe) [1110896]
+- [pci] Check all IORESOURCE_TYPE_BITS in pci_bus_alloc_from_region() (Myron Stowe) [1110896]
+- [kernel] resources: Set type in __request_region() (Myron Stowe) [1110896]
+- [s390] pci: Use generic pci_enable_resources() (Myron Stowe) [1110896]
+- [pci] Add "weak" generic pcibios_enable_device() implementation (Myron Stowe) [1110896]
+- [pci] Don't enable decoding if BAR hasn't been assigned an address (Myron Stowe) [1110896]
+- [pci] Mark 64-bit resource as IORESOURCE_UNSET if we only support 32-bit (Myron Stowe) [1110896]
+- [pci] Don't try to claim IORESOURCE_UNSET resources (Myron Stowe) [1110896]
+- [pci] Check IORESOURCE_UNSET before updating BAR (Myron Stowe) [1110896]
+- [pci] Don't clear IORESOURCE_UNSET when updating BAR (Myron Stowe) [1110896]
+- [pci] Mark resources as IORESOURCE_UNSET if we can't assign them (Myron Stowe) [1110896]
+- [pci] Remove pci_find_parent_resource() use for allocation (Myron Stowe) [1110896]
+- [lib] vsprintf: Add support for IORESOURCE_UNSET in pR (Myron Stowe) [1110896]
+- [kernel] resource: Add resource_contains() (Myron Stowe) [1110896]
+- [message] i2o: Use pci_bus_alloc_resource(), not allocate_resource() directly (Myron Stowe) [1110896]
+- [message] i2o: Refactor i2o_iop_systab_set() PCI space allocation (Myron Stowe) [1110896]
+- [message] i2o: Fix I/O space alignment requirement (Myron Stowe) [1110896]
+- [message] i2o: Fix I/O space allocation copy/paste error (Myron Stowe) [1110896]
+- [pci] pciehp: Cleanup whitespace (Myron Stowe) [1110896]
+- [pci] pciehp: Remove a non-existent card, regardless of "surprise" capability (Myron Stowe) [1110896]
+- [pci] Remove unused SR-IOV VF Migration support (Myron Stowe) [1110896]
+- [iommu] amd: Add include of <linux/irqreturn.h> (Myron Stowe) [1110896]
+- [misc] mei: Add include of <linux/irqreturn.h> (Myron Stowe) [1110896]
+- [vfio] pci: Use pci_enable_msi_range() and pci_enable_msix_range() (Myron Stowe) [1110896]
+- [ata] ahci: Use pci_enable_msi_range() instead of pci_enable_msi_block() (Myron Stowe) [1110896]
+- [x86] pci: Set IORESOURCE_ROM_SHADOW only for the default VGA device (Myron Stowe) [1110896]
+- [pci] Remove pci_bus_b() and use list_for_each_entry() directly (Myron Stowe) [1110896]
+- [pcmcia] Use list_for_each_entry() for bus traversal (Myron Stowe) [1110896]
+- [powerpc] pci: Use list_for_each_entry() for bus traversal (Myron Stowe) [1110896]
+- [drm] Use list_for_each_entry() for bus traversal (Myron Stowe) [1110896]
+- [pci] hotplug/apci: Use list_for_each_entry() for bus traversal (Myron Stowe) [1110896]
+- [pci] pciehp: Don't turn slot off when hot-added device already exists (Myron Stowe) [1110896]
+- [pci] Cleanup per-arch list of object files (Myron Stowe) [1110896]
+- [pci] hotplug/cpqphp: Fix hex vs decimal typo in cpqhpc_probe() (Myron Stowe) [1110896]
+- [x86] pci: Fix function definition whitespace (Myron Stowe) [1110896]
+- [x86] pci: Reword comments (Myron Stowe) [1110896]
+- [x86] pci: Remove unnecessary local variable initialization (Myron Stowe) [1110896]
+- [pci] Don't scan random busses in pci_scan_bridge() (Myron Stowe) [1110896]
+- [pci] Check for child busses which use more bus numbers than allocated (Myron Stowe) [1110896]
+- [pci] Remove pci_fixup_parent_subordinate_busnr() (Myron Stowe) [1110896]
+- [pci] Make sure bus number resources stay within their parents bounds (Myron Stowe) [1110896]
+- [pci] Use request_resource_conflict() instead of insert_ for bus numbers (Myron Stowe) [1110896]
+- [pci] Assign CardBus bus number only during the second pass (Myron Stowe) [1110896]
+- [pci] Clarify the "scan anyway" comment in pci_scan_bridge() (Myron Stowe) [1110896]
+- [pci] Increment max correctly in pci_scan_bridge() (Myron Stowe) [1110896]
+- [pci] Remove unnecessary list_empty(&pci_pme_list) check (Myron Stowe) [1110896]
+- [acpi] numa: Use __weak, not the gcc-specific version (Myron Stowe) [1110896]
+- [acpi] numa: Make __acpi_map_pxm_to_node(), acpi_get_pxm() static (Myron Stowe) [1110896]
+- [acpi] numa: Simplify acpi_get_node() style (Myron Stowe) [1110896]
+- [acpi] numa: Fix acpi_get_node() prototype (Myron Stowe) [1110896]
+- [x86] pci: Remove acpi_get_pxm() usage (Myron Stowe) [1110896]
+- [x86] pci: Use NUMA_NO_NODE, not -1, for unknown node (Myron Stowe) [1110896]
+- [x86] pci: Remove unnecessary list_empty(&pci_root_infos) check (Myron Stowe) [1110896]
+- [x86] pci: Remove mp_bus_to_node[], set_mp_bus_to_node(), get_mp_bus_to_node() (Myron Stowe) [1110896]
+- [x86] pci: Use x86_pci_root_bus_node() instead of get_mp_bus_to_node() (Myron Stowe) [1110896]
+- [x86] pci: Add x86_pci_root_bus_node() to look up NUMA node from PCI bus (Myron Stowe) [1110896]
+- [x86] pci: Drop return value of pcibios_scan_root() (Myron Stowe) [1110896]
+- [x86] pci: Merge pci_scan_bus_on_node() into pcibios_scan_root() (Myron Stowe) [1110896]
+- [x86] pci: Use pcibios_scan_root() instead of pci_scan_bus_on_node() (Myron Stowe) [1110896]
+- [x86] pci: Use pcibios_scan_root() instead of pci_scan_bus_with_sysdata() (Myron Stowe) [1110896]
+- [x86] pci: Drop pcibios_scan_root() check for bus already scanned (Myron Stowe) [1110896]
+- [ata] ahci: convert ahci_init_interrupts to a static routine (Myron Stowe) [1110896]
+- [acpi] mm: use NUMA_NO_NODE (Myron Stowe) [1110896]
+
 * Tue Aug 12 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-145.el7]
 - [fs] btrfs: test for valid bdev before kobj removal in btrfs_rm_device (Eric Sandeen) [1071179]
 - [fs] btrfs: fix abnormal long waiting in fsync (Eric Sandeen) [1071179]
-- 
1.7.1