From 7b8695ee87dbb2a63782e966e2d4e7bd985bcc80 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Mon, 27 Oct 2014 21:09:58 -0400
Subject: [redhat] kernel-3.10.0-195.el7

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 51baf67..5173d8f 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
-RHEL_RELEASE = 194
+RHEL_RELEASE = 195
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/redhat/Makefile.common b/redhat/Makefile.common
index fe25cd2..6897eaa 100644
--- a/redhat/Makefile.common
+++ b/redhat/Makefile.common
@@ -9,7 +9,7 @@ RPMVERSION:=3.10.0
 # marker is git tag which we base off of for exporting patches
 MARKER:=v3.10
 PREBUILD:=
-BUILD:=194
+BUILD:=195
 DIST:=.el7
 SPECFILE:=kernel.spec
 RPM:=$(REDHAT)/rpm
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 1eab48f..7a5e597 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1484,6 +1484,41 @@ fi
 
 %changelog
 %%CHANGELOG%%
+* Mon Oct 27 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-195.el7]
+- [block] fix blk_abort_request on blk-mq (Jeff Moyer) [1146660]
+- [block] blk-timeout: fix blk_add_timer (Jeff Moyer) [1146660]
+- [block] blk-mq: limit memory consumption if a crash dump is active (Jeff Moyer) [1146660]
+- [block] blk-mq: remove unnecessary blk_clear_rq_complete() (Jeff Moyer) [1146660]
+- [block] blk-mq: pass a reserved argument to the timeout handler (Jeff Moyer) [1146660]
+- [block] blk-mq: unshared timeout handler (Jeff Moyer) [1146660]
+- [block] blk-mq: fix and simplify tag iteration for the timeout handler (Jeff Moyer) [1146660]
+- [block] blk-mq: rename blk_mq_end_io to blk_mq_end_request (Jeff Moyer) [1146660]
+- [block] blk-mq: call blk_mq_start_request from ->queue_rq (Jeff Moyer) [1146660]
+- [block] blk-mq: remove REQ_END (Jeff Moyer) [1146660]
+- [block] blk-mq: use blk_mq_start_hw_queues() when running requeue work (Jeff Moyer) [1146660]
+- [block] blk-mq: fix potential oops on out-of-memory in __blk_mq_alloc_rq_maps() (Jeff Moyer) [1146660]
+- [block] blk-mq: avoid infinite recursion with the FUA flag (Jeff Moyer) [1146660]
+- [block] blk-mq: Avoid race condition with uninitialized requests (Jeff Moyer) [1146660]
+- [block] blk-mq: request deadline must be visible before marking rq as started (Jeff Moyer) [1146660]
+- [block] blk-mq: scale depth and rq map appropriate if low on memory (Jeff Moyer) [1146660]
+- [block] blk-mq: cleanup after blk_mq_init_rq_map failures (Jeff Moyer) [1146660]
+- [block] fix unbalanced bypass-disable in blk_register_queue (Jeff Moyer) [1146660]
+- [block] bdi: an active gendisk always has a request_queue associated with it (Jeff Moyer) [1146660]
+- [block] blk-mq: pass along blk_mq_alloc_tag_set return values (Jeff Moyer) [1146660]
+- [block] blk-merge: fix blk_recount_segments (Jeff Moyer) [1146660]
+- [block] blk-mq: correct a few wrong/bad comments (Jeff Moyer) [1146660]
+- [block] blk-mq: don't allow merges if turned off for the queue (Jeff Moyer) [1146660]
+- [block] blk-mq: fix WARNING "percpu_ref_kill() called more than once!" (Jeff Moyer) [1146660]
+- [block] provide compat ioctl for BLKZEROOUT (Jeff Moyer) [1146660]
+- [block] blkcg: don't call into policy draining if root_blkg is already gone (Jeff Moyer) [1146660]
+- [block] don't assume last put of shared tags is for the host (Jeff Moyer) [1146660]
+- [block] fix SG_[GS]ET_RESERVED_SIZE ioctl when max_sectors is huge (Jeff Moyer) [1146660]
+- [block] fix BLKSECTGET ioctl when max_sectors is greater than USHRT_MAX (Jeff Moyer) [1146660]
+- [block] blk-mq: collapse __blk_mq_drain_queue() into blk_mq_freeze_queue() (Jeff Moyer) [1146660]
+- [block] blk-mq: decouble blk-mq freezing from generic bypassing (Jeff Moyer) [1146660]
+- [block] blk-mq: draining can't be skipped even if bypass_depth was non-zero (Jeff Moyer) [1146660]
+- [block] blk-mq: fix a memory ordering bug in blk_mq_queue_enter() (Jeff Moyer) [1146660]
+
 * Fri Oct 24 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-194.el7]
 - [fs] xfs: flush the range before zero range conversion (Eric Sandeen) [1145837]
 - [fs] xfs: only set extent size hint when asked (Eric Sandeen) [1145837]
-- 
1.7.1